<?xml version="1.0" encoding="UTF-8"?><html xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:h="http://www.w3.org/1999/xhtml">
   <head>
      <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style>
   </head>
   <body>
      <div class="front" title="front">
         <div class="journal-meta" tagx="journal-meta" title="journal-meta">
            <span class="nlm-ta" title="nlm-ta">PLoS One</span>
            <span class="iso-abbrev" title="iso-abbrev">PLoS ONE</span>
            <span class="publisher-id" title="publisher-id">plos</span>
            <span class="pmc" title="pmc">plosone</span>
            <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group">
               <span class="journal-title" tagx="journal-title" title="journal-title">PLoS ONE</span>
            </div>
            <span class="issn-epub" tagx="issn" title="issn-epub">1932-6203</span>
            <div class="publisher" tagx="publisher" title="publisher">
               <span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span>
               <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, USA</span>
            </div>
         </div>
         <div class="article-meta" tagx="article-meta" title="article-meta">
            <span class="pmcid" title="pmcid">
               			pmcid:
               			
               <a href="http://www.ncbi.nlm.nih.gov/pubmed/3322136">3322136</a>
            </span>
            <span class="publisher-id" title="publisher-id">PONE-D-11-19222</span>
            <span class="doi" title="doi">
               			doi:
               			
               <a href="https://dx.doi.org/10.1371/journal.pone.0032990">10.1371/journal.pone.0032990</a>
            </span>
            <div class="article-categories" title="article-categories">: 
               <span class="subject" title="subject">Research Article</span>
               <div class="sub-group" title="subj-group">: 
                  <span class="subject" title="subject">Agriculture</span>: 
                  <span class="subject" title="subject">Biofuels</span>: 
                  <span class="subject" title="subject">Forestry</span>
               </div>
               <div class="sub-group" title="subj-group">: 
                  <span class="subject" title="subject">Biology</span>
                  <div class="sub-group" title="subj-group">: 
                     <span class="subject" title="subject">Ecology</span>: 
                     <span class="subject" title="subject">Microbial Ecology</span>
                  </div>
                  <div class="sub-group" title="subj-group">: 
                     <span class="subject" title="subject">Genomics</span>: 
                     <span class="subject" title="subject">Functional Genomics</span>
                  </div>
                  <div class="sub-group" title="subj-group">: 
                     <span class="subject" title="subject">Microbiology</span>
                     <div class="sub-group" title="subj-group">: 
                        <span class="subject" title="subject">Mycology</span>: 
                        <span class="subject" title="subject">Fungal Biochemistry</span>
                        <span class="subject" title="subject">Fungi</span>
                     </div>
                  </div>
                  <div class="sub-group" title="subj-group">: 
                     <span class="subject" title="subject">Proteomics</span>: 
                     <span class="subject" title="subject">Proteomic Databases</span>
                     <span class="subject" title="subject">Sequence Analysis</span>
                     <span class="subject" title="subject">Spectrometric Identification of Proteins</span>
                  </div>
                  <div class="sub-group" title="subj-group">: 
                     <span class="subject" title="subject">Zoology</span>: 
                     <span class="subject" title="subject">Entomology</span>
                  </div>
               </div>
            </div>
            <div class="title-group">
               <div class="article-title" title="article-title">Proteomic Analysis of 
                  <i>Fusarium solani</i> Isolated from the Asian Longhorned Beetle, 
                  <i>Anoplophora glabripennis</i>
               </div>
               <div class="alt-title" title="alt-title">
                  <i>Fusarium solani</i> Proteomics
               </div>
            </div>
            <div class="contrib-group" title="contrib-group">
               <span class="contrib" title="contrib">
                  <span class="name" tagx="name" title="name">
                     <span class="surname" tagx="surname" title="surname">Scully</span>
                     <span class="given-names" tagx="given-names" title="given-names">Erin D.</span>
                  </span>
                  <a href="#aff1">
                     <sup>1</sup>
                  </a>
               </span>
               <span class="contrib" title="contrib">
                  <span class="name" tagx="name" title="name">
                     <span class="surname" tagx="surname" title="surname">Hoover</span>
                     <span class="given-names" tagx="given-names" title="given-names">Kelli</span>
                  </span>
                  <a href="#aff2">
                     <sup>2</sup>
                  </a>
               </span>
               <span class="contrib" title="contrib">
                  <span class="name" tagx="name" title="name">
                     <span class="surname" tagx="surname" title="surname">Carlson</span>
                     <span class="given-names" tagx="given-names" title="given-names">John</span>
                  </span>
                  <a href="#aff3">
                     <sup>3</sup>
                  </a>
                  <a href="#aff4">
                     <sup>4</sup>
                  </a>
               </span>
               <span class="contrib" title="contrib">
                  <span class="name" tagx="name" title="name">
                     <span class="surname" tagx="surname" title="surname">Tien</span>
                     <span class="given-names" tagx="given-names" title="given-names">Ming</span>
                  </span>
                  <a href="#aff5">
                     <sup>5</sup>
                  </a>
               </span>
               <span class="contrib" title="contrib">
                  <span class="name" tagx="name" title="name">
                     <span class="surname" tagx="surname" title="surname">Geib</span>
                     <span class="given-names" tagx="given-names" title="given-names">Scott M.</span>
                  </span>
                  <a href="#aff6">
                     <sup>6</sup>
                  </a>
                  <a href="#cor1">
                     <sup>*</sup>
                  </a>
               </span>
            </div>
            <span class="citation_author_institution" id="aff1">[1],
               	  
               <span class="addr-line" title="addr-line">Intercollege Program in Genetics, The Huck Institutes of the Life Sciences, The Pennsylvania
                  State University, University Park, Pennsylvania, United States of America
               </span>
            </span>
            <span class="citation_author_institution" id="aff2">[2],
               	  
               <span class="addr-line" title="addr-line">Department of Entomology, Center for Chemical Ecology, The Pennsylvania State University,
                  University Park, Pennsylvania, United States of America
               </span>
            </span>
            <span class="citation_author_institution" id="aff3">[3],
               	  
               <span class="addr-line" title="addr-line">School of Forest Resources, The Pennsylvania State University, University Park, Pennsylvania,
                  United States of America
               </span>
            </span>
            <span class="citation_author_institution" id="aff4">[4],
               	  
               <span class="addr-line" title="addr-line">Department of Bioenergy Science and Technology (World Class University), Chonnam National
                  University, Buk-Gu, Gwangju, Korea
               </span>
            </span>
            <span class="citation_author_institution" id="aff5">[5],
               	  
               <span class="addr-line" title="addr-line">Department of Biochemistry and Molecular Biology, The Pennsylvania State University,
                  University Park, Pennsylvania, United States of America
               </span>
            </span>
            <span class="citation_author_institution" id="aff6">[6],
               	  
               <span class="addr-line" title="addr-line">Tropical Crop and Commodity Protection Research Unit, USDA-ARS Pacific Basin Agricultural
                  Research Center, Hilo, Hawaii, United States of America
               </span>
            </span>
            <div class="contrib-group" title="contrib-group">
               <span class="contrib" title="contrib">
                  <span class="name" tagx="name" title="name">
                     <span class="surname" tagx="surname" title="surname">Freitag</span>
                     <span class="given-names" tagx="given-names" title="given-names">Michael</span>
                  </span>
                  <span class="role" tagx="role" title="role">Editor</span>
                  <a href="#edit1"></a>
               </span>
            </div>
            <span class="citation_author_institution" id="edit1">[],
               	  
            </span>
            <div class="author-notes" title="author-notes">
               <div class="corresp" title="corresp">* E-mail: 
                  <span class="email" tagx="email" title="email">scott.geib@ars.usda.gov</span>
               </div>
               <div class="fn-type-con" title="con">
                  <p>Conceived and designed the experiments: SMG KH MT JC. Performed the experiments: SMG
                     EDS. Analyzed the data: SMG EDS. Contributed reagents/materials/analysis tools: SMG
                     KH JC MT. Wrote the paper: SMG KH EDS MT JC.
                  </p>
               </div>
            </div>
            <span class="pub-date-collection" title="pub-date-collection">collection: 
               <span>2012</span>
            </span>
            <span class="pub-date-epub" title="pub-date-epub">epub: 
               <span>2012-4-4</span>
            </span>
            <span class="volume" tagx="volume" title="volume">7</span>
            <span class="issue" tagx="issue" title="issue">4</span>
            <span class="elocation-id" tagx="elocation-id" title="elocation-id">e32990</span>
            <span class="history" title="history">
               <span class="received" title="received">received: 2011-9-29</span>
               <span class="accepted" title="accepted">accepted: 2012-2-5</span>
            </span>
            <div class="permissions">
               <span class="copyright" title="copyright">(C) , 
                  <span class="copyright-year" tagx="copyright-year" title="copyright-year">2012</span>
               </span>
            </div>
            <div class="abstract" title="abstract">
               <div class="abstract-title" title="abstract-title">Abstract</div>
               <p>Wood is a highly intractable food source, yet many insects successfully colonize and
                  thrive in this challenging niche. Overcoming the lignin barrier of wood is a key challenge
                  in nutrient acquisition, but full depolymerization of intact lignin polymers has only
                  been conclusively demonstrated in fungi and is not known to occur by enzymes produced
                  by insects or bacteria. Previous research validated that lignocellulose and hemicellulose
                  degradation occur within the gut of the wood boring insect, 
                  <i>Anoplophora glabripennis</i> (Asian longhorned beetle), and that a fungal species, 
                  <i>Fusarium solani</i> (ATCC MYA 4552), is consistently associated with the larval stage. While the nature
                  of this relationship is unresolved, we sought to assess this fungal isolate's ability
                  to degrade lignocellulose and cell wall polysaccharides and to extract nutrients from
                  woody tissue. This gut-derived fungal isolate was inoculated onto a wood-based substrate
                  and shotgun proteomics using Multidimensional Protein Identification Technology (MudPIT)
                  was employed to identify 400 expressed proteins. Through this approach, we detected
                  proteins responsible for plant cell wall polysaccharide degradation, including proteins
                  belonging to 28 glycosyl hydrolase families and several cutinases, esterases, lipases,
                  pectate lyases, and polysaccharide deacetylases. Proteinases with broad substrate
                  specificities and ureases were observed, indicating that this isolate has the capability
                  to digest plant cell wall proteins and recycle nitrogenous waste under periods of
                  nutrient limitation. Additionally, several laccases, peroxidases, and enzymes involved
                  in extracellular hydrogen peroxide production previously implicated in lignin depolymerization
                  were detected. 
                  <i>In vitro</i> biochemical assays were conducted to corroborate MudPIT results and confirmed that
                  cellulases, glycosyl hydrolases, xylanases, laccases, and Mn- independent peroxidases
                  were active in culture; however, lignin- and Mn- dependent peroxidase activities were
                  not detected While little is known about the role of filamentous fungi and their associations
                  with insects, these findings suggest that this isolate has the endogenous potential
                  to degrade lignocellulose and extract nutrients from woody tissue.
               </p>
            </div>
            <div class="counts" title="counts">
               <span class="page-count" title="page-count">page-count: </span>
            </div>
         </div>
      </div>
      <div class="body" title="body">
         <div class="introduction" title="sec">
            <div class="title" tagx="title" title="title">Introduction</div>
            <p>Most beetles in the family Cerambycidae develop deep in woody tissues where access
               to sugar monomers present in plant cell wall polysaccharides is impeded by the presence
               of a recalcitrant lignin barrier and other essential nutritional resources, including
               proteins, lipids, sterols, and vitamins, are deficient or absent altogether 
               <a href="#pone.0032990-Dillon1">[1]</a>. Many cerambycids that thrive in this suboptimal environment overcome these barriers
               by preferentially targeting weakened or stressed trees, whose woody, intractable components
               have been pre-digested by wood-degrading fungi, which degrade lignocellulose into
               easily digestible mono- and di- saccharides and synthesize other essential dietary
               components 
               <a href="#pone.0032990-Kukor1">[2]</a>. Alternatively, other cerambycids harbor external wood-degrading fungi, which are
               physically inoculated into host trees; the fungi colonize the larval tunnels, digest
               carbon polymers, and serve nutrient provisioning roles for the insect or fungal enzymes
               are ingested by the insect to aid in lignocellulose digestion in the gut 
               <a href="#pone.0032990-Dowd1">[3]</a>. Unlike other cerambycids, the Asian longhorned beetle (
               <i>Anoplophora glabripennis</i>, 
               <u>Coleoptera</u>: 
               <u>Cerambycidae</u>), an exotic insect native to China first detected in the U.S. in the early 1990's,
               attacks both weakened 
               <a href="#pone.0032990-Hanks1">[4]</a> and healthy 
               <a href="#pone.0032990-Lingafelter1">[5]</a> deciduous trees in the absence of external wood-degrading fungi. This beetle also
               enjoys a broad host range, which includes over 21 deciduous tree species 
               <a href="#pone.0032990-Nowak1">[6]</a>, 
               <a href="#pone.0032990-Hu1">[7]</a>.
            </p>
            <p>
               <i>A. glabripennis</i> larvae face a number of challenges acquiring nutrients as they grow and develop deep
               in the sapwood (and heartwood in some tree species) where the lignin: nitrogen ratio
               is high. Woody tissue is primarily composed of three polymeric materials: cellulose,
               hemicellulose and lignin. Cellulose is a linear polymer of glucose linked by β-1,4glycosidic
               bonds, accounting for approximately 45% of wood by weight. Its linear structure and
               extensive hydrogen bonding increases crystallinity of the macromolecule and decreases
               accessibility of hydrolytic enzymes 
               <a href="#pone.0032990-OSullivan1">[8]</a>. Hemicellulose accounts for approximately 15 to 35% of wood by weight depending on
               tree species and is also bound by β-1,4 linkages. Unlike cellulose, hemicellulose
               has much greater structural heterogeneity and is primarily comprised of xylose chains
               forming a xylan structure in secondary cell walls of hardwood tree species. In addition,
               other lesser abundant monomers typically found in hemicellulose include galactose,
               rhamnose, arabinose, mannose and their acid derivatives 
               <a href="#pone.0032990-Pettersen1">[9]</a>. Lignin is an amorphous structural aromatic polymer, which is often esterified to
               uronic residues present in hemicellulose and cross-linked to cellulose through ether
               and glycosidic linkages, protecting these polysaccharides from hydrolytic enzymes
               
               <a href="#pone.0032990-Kirk1">[10]</a>. Furthermore, some evidence indicates that cell wall proteins present in xylem elements
               are cross-linked with lignin and other cell wall polysaccharides, protecting them
               from proteolysis, suggesting that lignocellulose and hemicellulose degradation may
               be required for protein acquisition 
               <a href="#pone.0032990-Keller1">[11]</a>. Phenylpropanoid units, including coumaryl, coniferyl, and sinapyl alcohol, are the
               precursors of lignin. Oxidation of these phenols yields free radicals that undergo
               radical coupling to form a polymer comprised of over 12 types of chemical linkages
               and dominated by C-C and ether linkages, which are invulnerable to hydrolysis and
               can only be broken through radical oxidative depolymerization 
               <a href="#pone.0032990-Ke1">[12]</a>. Its lack of stereoregularity and periodicity and its condensed, insoluble properties
               make lignin resistant to most forms of enzymatic attack 
               <a href="#pone.0032990-Martinez1">[13]</a>.
            </p>
            <p>Current methods of degradation of lignin have been described for white rot basidiomycete
               fungi. These fungal species utilize secreted heme peroxidases with high redox potentials,
               including lignin-, Mn-, and versatile- peroxidases, to depolymerize the lignin molecule
               through oxidization of non-phenolic C-C and ether linkages using hydrogen peroxide
               as an oxidant 
               <a href="#pone.0032990-Camarero1">[14]</a>. Laccases, or multicopper phenol oxidases, are also produced by some lignin-degrading
               basidiomycetes, which solely oxidize the phenolic hydroxyl groups that comprise less
               than 10% of the total linkages in an intact lignin biopolymer. Although these enzymes
               can indirectly degrade more recalcitrant linkages in the presence of synthetic redox
               mediators, such as the diammonium salt of 2,2′-azine-
               <i>bis</i> (3-ethylbenzothiazoline-6-sulfonic acid) (ABTS), natural varieties of these mediators
               have yet to be discovered; thus, the full biochemical role of laccases in natural
               lignin depolymerization processes remain obscure 
               <a href="#pone.0032990-Morozova1">[15]</a>. In brown rot basidiomycetes, hydroxy radicals generated through Fenton reactions
               initiated by iron III reductase, catalyzed by heme domains in cellobiose dehydrogeases,
               are responsible for lignin modification 
               <a href="#pone.0032990-Kirk1">[10]</a>. Although these reactive hydroxy radicals induce only partial lignin depolymerization,
               many speculate that Fenton reactions occur in tandem with radical oxidative processes
               to expedite complete lignin depolymerization since Fenton-type lignin metabolites
               have been detected in association with white-rot mediated lignin degradation 
               <a href="#pone.0032990-Regalado1">[16]</a>. Outside the basidiomycetes, the process of lignin degradation by other types of
               fungi is largely unknown; while some bacteria can degrade aromatic monolignols, dilignols,
               and other phenolic linkages found in lignin, only partial lignin depolymerization
               and modification have been documented in bacteria. Thus, bacterial lignin degradation
               is not well understood and highly speculative 
               <a href="#pone.0032990-Harazono1">[17]</a>, 
               <a href="#pone.0032990-Bugg1">[18]</a>, 
               <a href="#pone.0032990-Kajikawa1">[19]</a>, 
               <a href="#pone.0032990-Kato1">[20]</a>, 
               <a href="#pone.0032990-Shary1">[21]</a>.
            </p>
            <p>From previous studies, we demonstrated that lignin, cellulose, and hemicellulose degradation
               occur in the guts of larval 
               <i>A. glabripennis</i>
               <a href="#pone.0032990-Geib1">[22]</a>, 
               <a href="#pone.0032990-Geib2">[23]</a>, 
               <a href="#pone.0032990-Geib3">[24]</a>. Although many insects, including cerambycids, can produce endogenous cellulases
               and other cell wall degrading enzymes 
               <a href="#pone.0032990-Pauchet1">[25]</a>, 
               <a href="#pone.0032990-Pauchet2">[26]</a>, 
               <a href="#pone.0032990-Watanabe1">[27]</a>, 
               <a href="#pone.0032990-Willis1">[28]</a>, 
               <a href="#pone.0032990-Davison1">[29]</a>, 
               <a href="#pone.0032990-CalderonCortes1">[30]</a>, 
               <a href="#pone.0032990-Zhang1">[31]</a> as well as enzymes capable of oxidizing phenolic linkages present in lignin 
               <a href="#pone.0032990-Ke1">[12]</a>, 
               <a href="#pone.0032990-Pauchet1">[25]</a>, 
               <a href="#pone.0032990-Pauchet2">[26]</a>, 
               <a href="#pone.0032990-Watanabe1">[27]</a>, 
               <a href="#pone.0032990-Willis1">[28]</a>, 
               <a href="#pone.0032990-Davison1">[29]</a>, 
               <a href="#pone.0032990-CalderonCortes1">[30]</a>, 
               <a href="#pone.0032990-Sun1">[32]</a>, 
               <a href="#pone.0032990-Coy1">[33]</a>, 
               <a href="#pone.0032990-Scharf1">[34]</a>, there is no conclusive evidence that insects endogenously produce enzymes capable
               of oxidizing the predominant C-C or ether linkages required for full lignin depolymerization,
               which are definitively broken in the 
               <i>A. glabripennis</i> gut 
               <a href="#pone.0032990-Geib1">[22]</a>. However, 
               <i>A. glabripennis</i> harbors gut microbiota that likely contribute to digestive physiology and help this
               insect overcome barriers associated with extracting nutrients from woody tissue, including
               radical oxidation of non-hydrolyzable bonds in lignin. Previous 16 s rRNA-based metagenomic
               analyses of the gut bacterial communities of larvae reared in several different suitable
               host tree species revealed a community dominated by Actinobacteria and Proteobacteria
               that appears to display a tremendous degree of plasticity at lower taxonomic ranks.
               For example, striking shifts in alpha diversity and taxonomic composition were observed
               in insects reared in different host tree species with little impact on larval fitness
               or ability to degrade cellulose and xylan 
               <a href="#pone.0032990-Geib2">[23]</a>. In contrast, the fungal community was relatively static in comparison and was consistently
               dominated by a single fungal taxon regardless of collection site or host tree species,
               demonstrating a stable relationship between this fungal isolate and 
               <i>A. glabripennis</i>. Multilocus phylogenetic analysis encompassing the ITS, EF, and LSU loci convincingly
               places this isolate in the 
               <i>Fusarium solani</i> species complex 
               <a href="#pone.0032990-Geib4">[35]</a>.
            </p>
            <p>Despite their status as notorious plant and animal pathogens, 
               <i>Fusarium spp.</i> are occasionally found in non-pathogenic associations with beetles 
               <a href="#pone.0032990-Teetorbarsch1">[36]</a>. One of the most well studied examples is the relationship between Ambrosia beetles
               (Coleoptera, Curculionidae) and 
               <i>Fusarium solani</i>. Ambrosia beetles harbor fungi in special external structures called mycangia, inoculating
               the fungus into the tree to digest lignocelluose and synthesize other nutrients, including
               ergosterols required for pupation 
               <a href="#pone.0032990-MoralesRamos1">[37]</a>. However, in many cases, the nature of the relationship between beetles and their
               respective 
               <i>Fusarium</i> affiliates is unclear; although the presence of the fungus appears to directly improve
               insect growth and fecundity, their precise contributions to insect physiology and
               biochemistry are largely unknown 
               <a href="#pone.0032990-Teetorbarsch1">[36]</a>. While we are just beginning to understand the relationship between the Asian longhorned
               beetle and its gut-associated 
               <i>F. solani</i> isolate, it is known that 
               <i>A. glabripennis</i> larval galleries are typically visually free of fungal inoculum indicating that if
               this isolate is contributing to lignocellulose digestion or other physiological processes
               
               <i>in vivo</i>, these processes occur within the gut rather than in the external environment 
               <a href="#pone.0032990-Geib4">[35]</a>.
            </p>
            <p>Furthermore, it is known that 
               <i>F. solani</i> isolates comprise a metabolically versatile species complex that can colonize many
               diverse niches and persist in extreme environments. Owing to this versatility, these
               fungi can often extract glucose from exotic carbon sources, including pyrene and benzopyrene
               
               <a href="#pone.0032990-Romero1">[38]</a>, 
               <a href="#pone.0032990-Veignie1">[39]</a> and often produce an impressive arsenal of xenbiotic degrading enzymes capable of
               degrading many common aromatic hydrocarbon pollutants, including chlorobenzenes, polychlorinated
               biphenyls, and phenanthrenes 
               <a href="#pone.0032990-Colombo1">[40]</a>. One of the hallmark characteristics of lignin degrading enzymes is their lack of
               substrate specificity 
               <a href="#pone.0032990-Tien1">[41]</a>, leading many to speculate that the enzymes involved in xenobiotic oxidoreductive
               processes may also oxidize recalcitrant bonds present in lignin. Early wood block
               surveys conducted with freshwater 
               <i>Nectria</i> isolates (anamorph: 
               <i>Fusarium</i>) were initially discouraging as these isolates did not induce substantial weight
               loss characteristic of soft rot processes, despite prolific production of cell wall
               polysaccharide degrading enzymes and phenol oxidases 
               <a href="#pone.0032990-Zaremaivan1">[42]</a>, 
               <a href="#pone.0032990-Worrall1">[43]</a>. However, studies on other 
               <i>F. solani</i> isolates convincingly demonstrated their abilities to oxidize aromatic rings and
               side chains of synthetic lignin compounds, efficiently metabolize both Kraft and Klason
               lignin, and thrive on lignocellulose-based substrates as sole sources of carbon 
               <a href="#pone.0032990-Lozovaya1">[44]</a>, 
               <a href="#pone.0032990-Sutherland1">[45]</a>, 
               <a href="#pone.0032990-Rodriguez1">[46]</a>, 
               <a href="#pone.0032990-Falcon1">[47]</a>. Notably, maximal evolution of 
               <sup>14</sup>CO
               <sub>2</sub> from radiolabelled lignin rings and side chains occurred substantially earlier in
               
               <i>F. solani</i> isolates in comparison to white rot basidiomycetes and some 
               <i>Fusarium</i> isolates degraded both lignin and polysaccharides simultaneously 
               <a href="#pone.0032990-Rodriguez1">[46]</a>, clearly indicating a strong lignin-degrading propensity and suggesting that these
               fungi may harbor highly efficient lignin degrading enzymes that could be used to enhance
               industrial cellulosic ethanol production. Additionally, the recent genome sequence
               of 
               <i>Nectria haematococca</i> (anamorph: 
               <i>F. solani</i>; mating population VI (MPVI)) also includes a putative lignin peroxidase ortholog
               (protein ID 4582) 
               <a href="#pone.0032990-Coleman1">[48]</a>; to our knowledge, its activity has not been verified 
               <i>in vitro</i>, but this protein possesses all of the functional domains required for lignin peroxidase
               activity.
            </p>
            <p>Due to its strong association with 
               <i>A. glabripennis</i> and its potential metabolic versatility and lignocellulose degrading properties,
               the goal of this study was to survey and characterize the lignocellulolytic, cell
               wall polysaccharide degrading, and other nutrient extracting capacities of this 
               <i>F. solani</i> isolate using 
               <i>de novo</i> peptide sequencing and 
               <i>in vitro</i> biochemical assays of extracellular proteins produced by the fungus grown on a wood-based
               substrate.
            </p>
         </div>
         <div class="materials_methods" title="materials|methods">
            <div class="title" tagx="title" title="title">Materials and Methods</div>
            <div class="sourceoflarvala.glabripennisassociatedf.solaniculture" title="sec">
               <div class="title" tagx="title" title="title">Source of larval 
                  <i>A. glabripennis</i> associated 
                  <i>F. solani</i> culture
               </div>
               <p>Fungus cultures were obtained from 
                  <i>A. glabripennis</i> larvae maintained at Penn State University, Department of Entomology, University
                  Park, PA USA. Adult 
                  <i>A. glabripennis</i> maintained in a quarantined greenhouse were allowed to oviposit into potted sugar
                  maple (
                  <i>Acer saccharum</i>) trees, which are highly preferred hosts of these beetles 
                  <a href="#pone.0032990-Hu1">[7]</a>. Subsequently, eggs were allowed to hatch and larvae were permitted to mature in
                  the trees' woody tissues. After a period of 90 days, healthy larvae feeding on inner
                  wood were collected and fungi were cultured from larval guts as described previously
                  
                  <a href="#pone.0032990-Geib1">[22]</a> to create single spore cultures. This isolate is currently curated at the American
                  Type Culture Collection under the accession number MYA 4552.
               </p>
            </div>
            <div class="solidwoodsubstratefungalculturesandfungalenzymeextraction" title="sec">
               <div class="title" tagx="title" title="title">Solid wood substrate fungal cultures and fungal enzyme extraction</div>
               <p>Several agar plugs from a culture of 
                  <i>F. solani</i> (MYA 4552) described above were used to inoculate a solid wood substrate in polypropylene
                  growth bags (Unicorn, Commerce, TX, USA) containing 250 g red oak wood chips, 30 g
                  millet, 15 g wheat bran, and 400 mL distilled water at 30°C 
                  <a href="#pone.0032990-Varela1">[49]</a>. Previous studies in 
                  <i>Phanerochaete chrysosporium</i> revealed that white rot fungi require easily metabolizable carbohydrates to induce
                  production of peroxidases involved in lignin degradation as this process typically
                  occurs during periods of secondary metabolism and under conditions of extreme nitrogen
                  limitation 
                  <a href="#pone.0032990-Kirk1">[10]</a>. Although there is limited evidence that certain 
                  <i>Fusarium solani</i> strains can thrive on lignin as a sole source of carbon and do not require easily
                  metabolizable carbohydrates, including cellulose, glucose, or soluble starches, to
                  induce lignin degrading enzymes 
                  <a href="#pone.0032990-Falcon1">[47]</a>, other strains do require these components and will not colonize lignocelluose-based
                  substrates in their absence. Thus, general growth conditions required for induction
                  of lignocellulolytic genes in 
                  <i>Fusarium</i> are poorly understood and abilities to colonize lignocellulose substrates vary tremendously
                  within this species complex 
                  <a href="#pone.0032990-Crawford1">[50]</a>. To promote initial substrate colonization regardless of inherent metabolic potential,
                  the medium was augmented with millet and the culture was grown for an extended time
                  period to expend tractable carbohydrate and protein resources present in the growth
                  medium and to induce physiological processes characteristic of secondary metabolism.
               </p>
               <p>Approximately one month after inoculation, total fungal enzymes were extracted from
                  the entire culture as previously described 
                  <a href="#pone.0032990-Varela1">[49]</a> by mixing bag contents with one volume of 0.5 M NaCl and incubating for 2 h at 4°C
                  with constant stirring. The mixture was then squeezed through cheesecloth and centrifuged
                  at 15,000×
                  <i>g</i> for 30 min at 4°C to remove cellular debris/wood and preferentially purify proteins
                  secreted into the extracellular environment. Ammonium sulfate was added to the filtrate
                  over a 30 minute time period until the solution reached 100% saturation to precipitate
                  proteins and to preserve proteins in their native conformation. The suspension was
                  incubated overnight at 4°C with stirring and the preparation was centrifuged at 15,000×
                  <i>g</i> for 30 min at 4°C. The protein pellets were dissolved in 50 ml of water and trace
                  amounts of ammonium sulfate were removed by repeated concentration (Amicon, 10-kDa
                  cutoff) and re-suspension in 50 ml of water. The final extractions were partitioned
                  into 1 ml aliquots and stored at −80°C.
               </p>
            </div>
            <div class="mudpitanalysis" title="sec">
               <div class="title" tagx="title" title="title">MudPIT analysis</div>
               <p>One mg of total 
                  <i>F. solani</i> protein extract was digested with trypsin in solution 
                  <a href="#pone.0032990-Kinter1">[51]</a>. Briefly, the protein sample was lyophilized and resuspended in 100 µl of 6 M urea,
                  100 mM Tris buffer (pH 7.8). Denatured proteins were reduced by adding 5 µl of 200
                  mM dithiothreitol (DTT) in 100 mM Tris (pH 7.8) and incubating for 1 h at room temperature.
                  The protein sample was subsequently alkylated by adding 20 µl of 200 mM iodoacetamide
                  in 100 mM Tris (pH 7.8) and incubating for another hour at room temperature. Residual
                  iodoacetamide was then consumed by adding another 20 µl of 200 mM DTT in 100 mM Tris
                  (pH 7.8) followed by a1 hour incubation at room temperature. The sample was diluted
                  with water to a working volume of 0.9 ml; then, 20 µg of trypsin in 0.1 mL of water
                  was added to the sample, (Trypsin Gold, Promega Corporation, Madison WI) bringing
                  the final reaction volume to 1 ml. The protein sample was then completely digested
                  overnight at 37°C and the reaction was halted the following day by lowering the pH
                  to &lt;6.0 with acetic acid. Residual salts were removed from the sample through repeated
                  concentration in a SpeedVac and resuspension in water.
               </p>
               <p>Multidimensional Protein Identification Technology (MudPIT) analysis was performed
                  at the Penn State Hershey Medical Center Mass Spectrometry Core Research Facility.
                  Two-dimensional liquid chromatography was performed to highly fractionate the sample.
                  In brief, a strong cation exchange (SCX) column was used to separate the protein extract
                  into 15 fractions. These fractions were subsequently separated on a C-18 column and
                  each sub-fraction was directly spotted onto a MALDI target plate (370 spots/fraction).
                  A total of 5500 MALDI spots were prepared in this manner. Next, tandem MS was performed
                  for each spot on an ABI 4800 MALDI-TOF-TOF (Applied Biosystems, Foster City, CA, US)
                  to determine 
                  <i>de novo</i> peptide sequences. These fragments were mapped to the 
                  <i>Nectria haematococca/F. solani</i> Mating Population VI (MPVI) reference protein set (
                  <a href="http://genome.jgi-psf.org/Necha2/Necha2.home.html">http://genome.jgi-psf.org/Necha2/Necha2.home.html</a>) 
                  <a href="#pone.0032990-Coleman1">[48]</a> using ProteinPilot 3.0 Software's Paragon Algorithm (Applied Biosystems, Foster City,
                  CA, US). This genome was chosen due to its phylogenetic proximity to the 
                  <i>A. glabripennis</i>-derived 
                  <i>F. solani</i> isolate 
                  <a href="#pone.0032990-Geib4">[35]</a>, and it was the only suitable reference genome within the 
                  <i>F. solani</i> species complex that was publicly available at the time of study. In this analysis,
                  the reference proteome is digested 
                  <i>in silico</i> and 
                  <i>de novo</i> peptide sequences determined by tandem MS are mapped back to annotated peptides for
                  identification and scored based on mapping quality and coverage. Mapping quality is
                  determined by sequence similarity at the amino acid level and the number of unique
                  mapping locations in the reference proteome; thus, peptides with high amino acid similarity
                  to their predicted reference proteins and peptides that map uniquely to a single protein
                  in the reference proteome are given a higher score. Coverage is determined by the
                  number of unique peptides that map to a single reference protein and proteins covered
                  by multiple peptides are scored more favorably. These parameters are integrated into
                  the ‘unused’ score, which is used to infer the confidence of a protein match. In this
                  analysis, an unused score of 1.3 represents a significant protein match at 95% confidence;
                  significant protein matches were annotated using the 
                  <i>Nectria haematococca/F. solani</i> Mating Population VI (MPVI) 2.0 genome database (abbreviated Necha 2.0).
               </p>
               <p>KEGG, gene ontology (GO), and InterPro (IPR) annotations present in genome database
                  were applied to proteins detected by MudPIT analysis. In addition, the full 
                  <i>N. haematococca</i> MPVI amino acid sequence for each protein identified was analyzed for the presence
                  of signal peptides using both neural network and Hidden Markov model (HMM) methodologies
                  using the SignalP 3.0 web server 
                  <a href="#pone.0032990-Bendtsen1">[52]</a>. For neural network analysis, the mean S score was used to determine presence of
                  a signal peptide and HMM predictions were based on the C
                  <sub>max</sub> score. In an attempt to infer the function of proteins detected in the secretome
                  that were annotated as ‘hypothetical’ in the 
                  <i>N. haematococca</i> database, the peptide sequences were extracted from the reference genome and were
                  compared to the non-redundant protein database using blastp (blast 2.2.26+). From
                  these annotations, we summarized protein classes present, with a focus on enzymes
                  involved in plant cell wall digestion and protein acquisition. Additionally, to confirm
                  that proteins detected in the MudPIT data were enzymatically active in culture we
                  performed 
                  <i>in vitro</i> cellulase, xylanase, lignin peroxidase, manganese-dependent and –independent peroxidase
                  assays.
               </p>
            </div>
            <div class="invitrocellulaseandxylanaseactivities" title="sec">
               <div class="title" tagx="title" title="title">
                  <i>In vitro</i> cellulase and xylanase activities
               </div>
               <p>
                  <i>In vitro</i> cellulase, glycoside hydrolases and xylanase activities of fungal enzyme extracts
                  were confirmed by measuring release of reducing sugars from cellulose or xylan-based
                  substrates using the dinitrosalicylic acid (DNS) assay 
                  <a href="#pone.0032990-Bernfeld1">[53]</a>, 
                  <a href="#pone.0032990-Miller1">[54]</a>. Protein concentration was measured using Bradford chemistry 
                  <a href="#pone.0032990-Bollag1">[55]</a>, 
                  <a href="#pone.0032990-Bradford1">[56]</a> with BSA as a protein standard (0–20 µg); samples were diluted to a working concentration
                  of approximately 60 µg/ml in sodium citrate buffer (50 mM, pH 5.5). To assay for ability
                  to degrade cellulose, we quantified release of reducing sugars from cellulose-based
                  substrates, including microcrystalline cellulose (Avicel) and carboxymethyl cellulose
                  (CMC). To quantify glycoside hydrolase activity directed at β-1,4 linkages present
                  in D-glycopyranosyl containing compounds, salicin, a β-1,4 conjugated phenolic glycoside,
                  was utilized. For CMC and salicin DNS assays, 500 µl of a 2% substrate solution (in
                  50 mM sodium citrate buffer, pH 5.5) was combined with 30 µg (500 µl of 60 µg/ml)
                  of fungal extract; however, due to substrate insolubility, a 1% solution of microcrystalline
                  cellulose was substituted. For xylanase activity, 500 µl of a 1% xylan solution (in
                  50 mM sodium citrate buffer, ph 5.5, xylan from birch wood, Sigma Aldrich Corporation)
                  was combined with 30 µg (500 µl of 60 µg/ml dilution) of fungal extract. Three technical
                  replicates were performed and non-enzyme controls were run to detect background release
                  of reducing sugars from cellulose and xylan based substrates. For all assays, 100
                  µl of the reaction mixture was removed at time 0 and read at 540 nm to allow for subtraction
                  of background reducing sugars. Reactions were incubated at 37°C for 120 min and 100
                  µl aliquots were removed from each reaction after 60 and 120 minutes to record release
                  of reducing sugars over time. For each aliquot, 100 µl DNS reagent was added to halt
                  enzyme activity 
                  <a href="#pone.0032990-Miller1">[54]</a> and samples were boiled for 8 min in a water bath. 150 µL aliquots of DNSA reaction
                  were read at 540 nm on a SpectraMax™ microplate reader (Molecular Devices Corp.) and
                  were compared with a glucose standard curve (20 to 1000 µg) to quantify concentration
                  of reducing sugars at each time point.
               </p>
            </div>
            <div class="zymogramanalysis" title="sec">
               <div class="title" tagx="title" title="title">Zymogram analysis</div>
               <p>SDS-PAGE gels 
                  <a href="#pone.0032990-Laemmli1">[57]</a> were performed with modifications to independently verify cellulase or xylanase activity
                  through zymogram techniques 
                  <a href="#pone.0032990-Her1">[58]</a>, 
                  <a href="#pone.0032990-Schwarz1">[59]</a>, 
                  <a href="#pone.0032990-Chavez1">[60]</a>. Twelve percent acrylamide separation gels were prepared containing either 0.1% carboxymethyl
                  cellulose or 0.1% xylan from birch wood. Fungal enzyme extracts prepared as described
                  above were loaded onto each gel in duplicate (20 µg protein/lane) with a pre-stained
                  protein standard (SeeBlue Plus, Invitrogen, Carlsbad, CA), so that the gel could be
                  cut vertically in half after electrophoresis to produce two identical acrylamide gels.
                  The first half of the gel was stained with colloidal blue to visualize proteins as
                  a reference and imaged using a densitometer (GS-800, Bio-Rad, Hercules, CA). The second
                  half was used for zymogram analysis. Zymogram gels were rinsed in sodium citrate buffer
                  (50 mM, pH 5.5) containing 1% Triton X-100 for 1 h at room temperature to remove SDS
                  
                  <a href="#pone.0032990-Her1">[58]</a>. This was followed by 1.5 h incubation in sodium citrate buffer (50 mM, pH 5.5) to
                  allow for enzyme activity against the substrates. At this point, gels were stained
                  with 0.1% Congo red for 30 min and destained in 1 M NaCl to reveal zones of clearing
                  indicative of degradation of polysaccharide substrates. Gels were imaged under ultraviolet
                  light to highlight zones of clearing and aligned with colloidal blue stained gels
                  using the pre-stained protein standard as a reference.
               </p>
            </div>
            <div class="invitroligninperoxidase,manganeseperoxidase,andlaccaseactivities" title="sec">
               <div class="title" tagx="title" title="title">
                  <i>In vitro</i> lignin peroxidase, manganese peroxidase, and laccase activities
               </div>
               <p>Ligninolytic activity of the fungal extract was further assessed through 
                  <i>in vitro</i> approaches. Lignin peroxidase activity was assayed by the oxidation of veratryl alcohol
                  to veratraldehyde as an increase in A
                  <sub>310</sub>
                  <a href="#pone.0032990-Tien1">[41]</a>. Approximately 50 µg of protein were added to 1 mL of a solution containing 25 mM
                  sodium tartrate (pH 3.0) and 20 mM veratryl alcohol. The reaction was initiated by
                  addition of H
                  <sub>2</sub>O
                  <sub>2</sub> at a concentration of 2 mM and the absorbance at A
                  <sub>310</sub> was recorded. Mn-dependent and -independent peroxidase activities were measured by
                  the oxidation of 2,6-dimethoxyphenol as an increase in A
                  <sub>470</sub>
                  <a href="#pone.0032990-Dejong1">[61]</a>. Approximately 50 µg of protein extract were added to a solution containing 20 mM
                  2,6-dimethoxyphenol and 0.5 M sodium tartrate (pH 4.5), either with or without 20
                  mM manganese sulfate (for Mn-dependent and -independent activity, respectively). Total
                  reaction volume was 1 ml and the peroxidase reaction was initiated by addition of
                  H
                  <sub>2</sub>O
                  <sub>2</sub> to obtain a final concentration of 2 mM. Absorbance at 470 nm was recorded. Laccase
                  activity was verified using 2,6-dimethoxyphenol as a substrate 
                  <a href="#pone.0032990-Rehman1">[62]</a>. Approximately 50 µg of protein extract were added to a solution containing 20 mM
                  2,6-dimethoxyphenoland 0.5 M sodium tartrate (pH 4.5) to a total volume of 1 ml. Absorbance
                  was then read at 470 nm. All assays were performed in triplicate and mean change in
                  absorbance versus a no enzyme control over five minutes was recorded and this difference
                  was used as a measure of enzyme activity. One unit of activity was defined as the
                  amount of enzyme that oxidizes 1 µmol of substrate per minute.
               </p>
            </div>
            <div class="non-denaturingpageandhemestaining" title="sec">
               <div class="title" tagx="title" title="title">Non-denaturing PAGE and heme staining</div>
               <p>All known peroxidases with deconstructive activity directed at intact lignin polymers
                  are extracellular heme oxides containing a heme prosthetic group 
                  <a href="#pone.0032990-Kirk1">[10]</a>, 
                  <a href="#pone.0032990-Chavez1">[60]</a>, 
                  <a href="#pone.0032990-Henrissat1">[63]</a>. In an attempt to detect extracellular proteins present containing this vital prosthetic
                  group, heme stained gels were prepared. In brief, non-denaturing polyacrylamide gel
                  electrophoresis (native PAGE) was run following the methods of Laemmli 
                  <a href="#pone.0032990-Laemmli1">[57]</a> with a gel containing 12% acrylamide and lacking SDS in the gel and running buffer.
                  To further preserve the functional integrity of the proteins, the samples were not
                  boiled prior to loading on the gel. Fungal enzyme extracts were loaded onto the gel
                  in duplicate (20 µg protein/lane) with protein standards so that the gel could be
                  cut vertically in half after electrophoresis to produce two identical acrylamide gels.
                  The first half of the gel was stained with colloidal blue to visualize proteins as
                  a reference and imaged with a densitometer (GS-800, Bio-Rad, Hercules, CA). The second
                  half of the gel was stained to identify proteins containing a heme prosthetic group.
                  In brief, the second half of the gel was incubated in a Tris-MeOH solution (20 mM
                  Tris-HCl, pH 7.3, 50% MeOH) for 30 min, followed by a 45-min incubation in heme stain
                  solution (25 mM acetate buffer, pH 5.3, 0.25% benzidine HCl, 25% MeOH, and 0.75% H
                  <sub>2</sub>O
                  <sub>2</sub>). The gel was then rinsed in 25% MeOH and stored in 0.1 M Tris-HCl, pH 7.3. Heme
                  containing proteins stained dark brown and were matched with colloidal blue stained
                  proteins on the reference gel. Although other non-lignin degrading proteins that contain
                  a heme prosthetic group will stain (cytochromes) using this approach, these proteins
                  are primarily intracellular and are found in association with mitochondria.
               </p>
            </div>
         </div>
         <div class="results" title="sec">
            <div class="title" tagx="title" title="title">Results</div>
            <div class="mudpitanalysis" title="sec">
               <div class="title" tagx="title" title="title">MudPIT Analysis</div>
               <p>A total of 8400 spectra were collected from tandem MS and, of these, 4740 spectra
                  were utilized to identify 3638 unique peptides at 95% confidence (
                  <a href="#pone-0032990-t001">Table 1</a>). These 3638 peptides reliably mapped to 398 distinct proteins in the 
                  <i>N. haemotococca</i> reference genome (
                  <a href="#pone-0032990-t001">Table 1</a>). Full protein scores for all proteins detected are presented in the supplemental
                  information (
                  <a href="#pone.0032990.s001">Table S1</a>). By using the full amino acid sequence from the 
                  <i>N. haematococca</i> genome, 175 proteins were found to have a signal peptide using SignalP 3.0 neural
                  network model and 183 using HMM model (
                  <a href="#pone.0032990.s002">Table S2</a>). Over half of the proteins (224 for neural network and 203 for HMM) did not contain
                  a detectable signal peptide probably due to incidental lysis of cell membranes during
                  the protein extraction or errors in computational signal peptide prediction. Through
                  functional annotation, InterPro information was applied to 309 of these proteins,
                  GO IDs to 275 proteins, and KEGG annotations to 128 proteins. Full InterPro, GO, and
                  KEGG annotation are provided in supplemental information (
                  <a href="#pone.0032990.s003">Tables S3</a>, 
                  <a href="#pone.0032990.s004">S4</a>, and 
                  <a href="#pone.0032990.s005">S5</a> respectively). Furthermore, there were 20 InterPro IDs that had 5 or more proteins
                  classified to them (
                  <a href="#pone-0032990-t002">Table 2</a>). Many of these abundant Interpro IDs are related to cellulose/carbohydrate binding
                  and degradation, protein/peptide degradation, and general metabolic functions. Focusing
                  more closely on proteins that hydrolyze sugars, an analysis of glycoside hydrolase
                  (GH) families was performed using the InterPro IDs and blast searching the reference
                  proteins against the NCBI non-redundant database.
               </p>
               <table-wrap>
                  <a href="https://dx.doi.org/10.1371/journal.pone.0032990.t001">10.1371/journal.pone.0032990.t001</a>
                  <span class="label" tagx="label" title="label">Table 1</span>
                  <caption>
                     <div class="title" tagx="title" title="title">MudPIT summary data.</div>
                  </caption>
                  <div class="alternatives" title="alternatives">
                     <graphic></graphic>
                     <table>
                        <colgroup>
                           <col></col>
                           <col></col>
                           <col></col>
                           <col></col>
                           <col></col>
                        </colgroup>
                        <thead>
                           <tr>
                              <td>Unused (% Conf) Cutoff</td>
                              <td>Proteins Detected</td>
                              <td>Distinct Peptides</td>
                              <td>Spectra Identified</td>
                              <td>% Total Spectra Used</td>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td>&gt;2.0 (99)</td>
                              <td>264</td>
                              <td>3219</td>
                              <td>4279</td>
                              <td>51</td>
                           </tr>
                           <tr>
                              <td>&gt;1.3 (95)</td>
                              <td>398</td>
                              <td>3638</td>
                              <td>4740</td>
                              <td>56.4</td>
                           </tr>
                        </tbody>
                     </table>
                  </div>
               </table-wrap>
               <table-wrap>
                  <a href="https://dx.doi.org/10.1371/journal.pone.0032990.t002">10.1371/journal.pone.0032990.t002</a>
                  <span class="label" tagx="label" title="label">Table 2</span>
                  <caption>
                     <div class="title" tagx="title" title="title">Most Abundant InterPro IDs Identified in MudPIT analysis.</div>
                  </caption>
                  <div class="alternatives" title="alternatives">
                     <graphic></graphic>
                     <table>
                        <colgroup>
                           <col></col>
                           <col></col>
                           <col></col>
                        </colgroup>
                        <thead>
                           <tr>
                              <td>InterPro ID</td>
                              <td># Proteins with annotation</td>
                              <td>InterPro Description</td>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td>IPR000379</td>
                              <td>16</td>
                              <td>Esterase/lipase/thioesterase</td>
                           </tr>
                           <tr>
                              <td>IPR001138</td>
                              <td>14</td>
                              <td>Fungal transcriptional regulatory protein, N-terminal</td>
                           </tr>
                           <tr>
                              <td>IPR000254</td>
                              <td>8</td>
                              <td>Cellulose-binding region, fungal</td>
                           </tr>
                           <tr>
                              <td>IPR001764</td>
                              <td>8</td>
                              <td>Glycoside hydrolase, family 3, N-terminal</td>
                           </tr>
                           <tr>
                              <td>IPR000209</td>
                              <td>8</td>
                              <td>Peptidase S8 and S53, subtilisin, kexin, sedolisin</td>
                           </tr>
                           <tr>
                              <td>IPR007219</td>
                              <td>7</td>
                              <td>Fungal specific transcription factor</td>
                           </tr>
                           <tr>
                              <td>IPR002772</td>
                              <td>7</td>
                              <td>Glycoside hydrolase, family 3, C-terminal</td>
                           </tr>
                           <tr>
                              <td>IPR003439</td>
                              <td>6</td>
                              <td>ABC transporter</td>
                           </tr>
                           <tr>
                              <td>IPR001410</td>
                              <td>6</td>
                              <td>DEAD/DEAH box helicase</td>
                           </tr>
                           <tr>
                              <td>IPR001650</td>
                              <td>6</td>
                              <td>Helicase, C-terminal</td>
                           </tr>
                           <tr>
                              <td>IPR007484</td>
                              <td>6</td>
                              <td>Peptidase M28</td>
                           </tr>
                           <tr>
                              <td>IPR003137</td>
                              <td>6</td>
                              <td>Protease-associated PA</td>
                           </tr>
                           <tr>
                              <td>IPR003593</td>
                              <td>5</td>
                              <td>AAA ATPase</td>
                           </tr>
                           <tr>
                              <td>IPR001757</td>
                              <td>5</td>
                              <td>ATPase, E1–E2 type</td>
                           </tr>
                           <tr>
                              <td>IPR006045</td>
                              <td>5</td>
                              <td>Cupin</td>
                           </tr>
                           <tr>
                              <td>IPR008250</td>
                              <td>5</td>
                              <td>E1–E2 ATPase-associated region</td>
                           </tr>
                           <tr>
                              <td>IPR005834</td>
                              <td>5</td>
                              <td>Haloacid dehalogenase-like hydrolase</td>
                           </tr>
                           <tr>
                              <td>IPR000719</td>
                              <td>5</td>
                              <td>Protein kinase</td>
                           </tr>
                           <tr>
                              <td>IPR010259</td>
                              <td>5</td>
                              <td>Proteinase inhibitor I9, subtilisin propeptide</td>
                           </tr>
                           <tr>
                              <td>IPR002290</td>
                              <td>5</td>
                              <td>Serine/threonine protein kinase</td>
                           </tr>
                        </tbody>
                     </table>
                  </div>
               </table-wrap>
               <p>In total, 48 different proteins classified into 28 GH families were represented in
                  the protein extract, encompassing ∼8% of all proteins identified (
                  <a href="#pone-0032990-t003">Table 3</a>). The most abundant GH was GH 3, which are proteins represented by a broad range
                  of enzyme classes, including β-glucosidase (EC 3.2.1.21), xylan-1,4- β-xylosidase
                  (EC 3.2.1.37), β-N-actetylhexosamidase (EC 3.2.1.52), and α-L-arabinofuranosidase
                  (EC 3.2.1.55). Specifically β-glucosidase (EC 3.2.1.21) was identified in our sample
                  from this GH family through cross-annotation with the KEGG database (
                  <a href="#pone.0032990.s005">Table S5</a>). In addition, α-amylase (EC 3.2.1.1; GH 13), glucoamylase (EC 3.2.1.3; GH 15), licheninase
                  (EC 3.2.1.73; GH 16), glucan 1,3-β-glucosidase (EC 3.2.1.58; GH 17), chitinase (EC
                  3.2.1.14; GH 28), β-hexosaminidase (EC 3.2.1.52; GH 20), α-glucosidase (EC 3.2.1.20;
                  GH 37), β-galactosidase (EC 3.2.1.23; GH 35), and α,α-trehalase (EC 3.2.1.28; GH 37)
                  were identified in the proteome. A full description of potential enzymes in other
                  GH families can be found at 
                  <a href="http://www.cazy.org">www.cazy.org</a>
                  <a href="#pone.0032990-Henrissat1">[63]</a>. We were unable to annotate other GH families with additional functionality, as EC
                  numbers were not assigned to KEGG annotations for those proteins in the reference
                  genome. In addition to GH annotated proteins, other plant cell wall degrading proteins
                  that target other compounds in degrading woody tissue such as esterases, pectinases,
                  carbohydrate binding proteins, and others were identified (
                  <a href="#pone-0032990-t004">Table 4</a>).
               </p>
               <table-wrap>
                  <a href="https://dx.doi.org/10.1371/journal.pone.0032990.t003">10.1371/journal.pone.0032990.t003</a>
                  <span class="label" tagx="label" title="label">Table 3</span>
                  <caption>
                     <div class="title" tagx="title" title="title">Glycoside hydrolase (GH) families detected in MudPIT analysis.</div>
                  </caption>
                  <div class="alternatives" title="alternatives">
                     <graphic></graphic>
                     <table>
                        <colgroup>
                           <col></col>
                           <col></col>
                           <col></col>
                           <col></col>
                           <col></col>
                           <col></col>
                        </colgroup>
                        <thead>
                           <tr>
                              <td>GH Family</td>
                              <td>Number of Proteins</td>
                              <td>Protein ID</td>
                              <td>Interpro ID (or description if unknown)</td>
                              <td>KEGG EC Number (if known)</td>
                              <td>Secreted?</td>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td>Candidate GH</td>
                              <td>2</td>
                              <td>78, 80</td>
                              <td>Blast homology</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Candidate GH 39</td>
                              <td>1</td>
                              <td>156</td>
                              <td>Blast homology</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Candidate GH 39</td>
                              <td>1</td>
                              <td>170</td>
                              <td>IPR000293, IPR002860</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Candidate GH5</td>
                              <td>1</td>
                              <td>34</td>
                              <td>Blast homology</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Candidate GH7</td>
                              <td>1</td>
                              <td>44</td>
                              <td>GH 7 superfamily domain</td>
                              <td>3.2.1.58: Glucan 1,3-beta-glucosidase</td>
                              <td>N</td>
                           </tr>
                           <tr>
                              <td>Candidate GH9</td>
                              <td>1</td>
                              <td>109</td>
                              <td>Blast homology</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Candidate GH 55</td>
                              <td>1</td>
                              <td>5</td>
                              <td>Pectin lyase 3 domain</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Candidate retaining beta glucosidase</td>
                              <td>1</td>
                              <td>54</td>
                              <td>JGI User annotation</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>BNR repeat</td>
                              <td>2</td>
                              <td>35, 23</td>
                              <td>IPR002860</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Fungal cellulose binding</td>
                              <td>1</td>
                              <td></td>
                              <td>IPR000254</td>
                              <td></td>
                              <td></td>
                           </tr>
                           <tr>
                              <td>Glycoside hydrolase - starch binding</td>
                              <td>1</td>
                              <td>120</td>
                              <td>IPR002044</td>
                              <td></td>
                              <td>N</td>
                           </tr>
                           <tr>
                              <td>1</td>
                              <td>1</td>
                              <td>27</td>
                              <td>IPR001360</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>3</td>
                              <td>8</td>
                              <td>2, 56, 57, 58, 95, 149, 189, 254</td>
                              <td>IPR001764, IPR002772</td>
                              <td>3.2.1.21: Beta glucosidase</td>
                              <td>5Y 3N</td>
                           </tr>
                           <tr>
                              <td>5</td>
                              <td>3</td>
                              <td>26, 51, 209</td>
                              <td>IPR000254, IPR001547, IPR001764, IPR002772</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>6</td>
                              <td>1</td>
                              <td>20</td>
                              <td>IPR001524</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>7</td>
                              <td>4</td>
                              <td>1, 96, 115, 170</td>
                              <td>IPR001722, IPR000254</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>10</td>
                              <td>1</td>
                              <td>21</td>
                              <td>IPR001000</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>11</td>
                              <td>1</td>
                              <td>145</td>
                              <td>IPR001137</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>13</td>
                              <td>2</td>
                              <td>75, 318</td>
                              <td>IPR006046, IPR004193</td>
                              <td>3.2.1.1: Alpha amylase</td>
                              <td>1 Y 1N</td>
                           </tr>
                           <tr>
                              <td>15</td>
                              <td>1</td>
                              <td>6</td>
                              <td>IPR000165</td>
                              <td>3.2.1.3: Glucan 1,4-alpha-glucosidase</td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>16</td>
                              <td>2</td>
                              <td>16, 87</td>
                              <td>IPR000757</td>
                              <td>3.2.1.73: Lichenase</td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>17</td>
                              <td>1</td>
                              <td>237</td>
                              <td>IPR000490</td>
                              <td>3.2.1.58: Glucan 1,3-beta-glucosidase</td>
                              <td>N</td>
                           </tr>
                           <tr>
                              <td>20</td>
                              <td>1</td>
                              <td>11</td>
                              <td>IPR001540</td>
                              <td>3.2.1.52: Beta-N-acetylhexosaminidase</td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>24</td>
                              <td>1</td>
                              <td>50</td>
                              <td>IPR002196</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>28</td>
                              <td>1</td>
                              <td>112</td>
                              <td>IPR001002, IPR001223, IPR011583</td>
                              <td>3.2.1.14: Chitinase</td>
                              <td>N</td>
                           </tr>
                           <tr>
                              <td>31</td>
                              <td>1</td>
                              <td>4</td>
                              <td>IPR000322</td>
                              <td>3.2.1.20: Alpha glucosidase</td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>32</td>
                              <td>1</td>
                              <td>98</td>
                              <td>IPR001362</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>35</td>
                              <td>1</td>
                              <td>323</td>
                              <td>IPR001944</td>
                              <td>3.2.1.23: Beta galactosidase</td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>37</td>
                              <td>1</td>
                              <td>25</td>
                              <td>IPR001661</td>
                              <td>3.2.1.28: Alpha trehalase</td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>43</td>
                              <td>1</td>
                              <td>184</td>
                              <td>IPR006710</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>45</td>
                              <td>1</td>
                              <td>52</td>
                              <td>IPR000254, IPR000334</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>61</td>
                              <td>1</td>
                              <td>163</td>
                              <td>IPR005103</td>
                              <td></td>
                              <td>N</td>
                           </tr>
                        </tbody>
                     </table>
                  </div>
               </table-wrap>
               <table-wrap>
                  <a href="https://dx.doi.org/10.1371/journal.pone.0032990.t004">10.1371/journal.pone.0032990.t004</a>
                  <span class="label" tagx="label" title="label">Table 4</span>
                  <caption>
                     <div class="title" tagx="title" title="title">Other cell wall degrading proteins from MudPIT analysis.</div>
                  </caption>
                  <div class="alternatives" title="alternatives">
                     <graphic></graphic>
                     <table>
                        <colgroup>
                           <col></col>
                           <col></col>
                           <col></col>
                           <col></col>
                           <col></col>
                           <col></col>
                        </colgroup>
                        <thead>
                           <tr>
                              <td>Protein Name</td>
                              <td>Number of Proteins</td>
                              <td>Protein ID</td>
                              <td>Interpro ID (or description if unknown)</td>
                              <td>KEGG EC Number (if known)</td>
                              <td>Secreted?</td>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td>Alpha-L-arabinfuranosidase</td>
                              <td>2</td>
                              <td>9, 79</td>
                              <td>IPR010720, IPR007934</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Candidate β-N-acetylhexosaminidase</td>
                              <td>1</td>
                              <td>168</td>
                              <td>IPR002022</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Candidate carboxylesterase</td>
                              <td>1</td>
                              <td>292</td>
                              <td>IPR001087</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Candidate ester hydrolase</td>
                              <td>1</td>
                              <td>402</td>
                              <td>Blast homology</td>
                              <td></td>
                              <td>N</td>
                           </tr>
                           <tr>
                              <td>Candidate pectin lyase</td>
                              <td>1</td>
                              <td>122</td>
                              <td>Blast homology</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Carbohydrate binding protein</td>
                              <td>1</td>
                              <td>393</td>
                              <td>IPR002889</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Carboxylesterase</td>
                              <td>3</td>
                              <td>30, 77, 269</td>
                              <td>IPR000379, IPR000408, IPR002018</td>
                              <td>3.1.1.1: Carboxylesterase</td>
                              <td>2Y 1N</td>
                           </tr>
                           <tr>
                              <td>Chitin binding protein</td>
                              <td>1</td>
                              <td>287</td>
                              <td>IPR001002, IPR002889</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Chitin deacetylase</td>
                              <td>1</td>
                              <td>99</td>
                              <td>IPR009939</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Cutinase</td>
                              <td>2</td>
                              <td>73, 278</td>
                              <td>IPR000379, IPR000675, IPR011150</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Esterase</td>
                              <td>3</td>
                              <td>72, 108, 136</td>
                              <td>IPR000379, IPR007312, IPR008262</td>
                              <td></td>
                              <td>2Y 1N</td>
                           </tr>
                           <tr>
                              <td>Galactose epimerase</td>
                              <td>1</td>
                              <td>22</td>
                              <td>IPR008183</td>
                              <td>5.1.3.3: Aldose 1-epimerase</td>
                              <td>N</td>
                           </tr>
                           <tr>
                              <td>Lipolytic enzyme</td>
                              <td>2</td>
                              <td>153, 331</td>
                              <td>IPR001087</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Pectate lyase</td>
                              <td>1</td>
                              <td>116</td>
                              <td>IPR004898</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Polysaccharide deacetylase</td>
                              <td>1</td>
                              <td>134</td>
                              <td>IPR002509</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Tannase and feruloyl esterase</td>
                              <td>1</td>
                              <td>159</td>
                              <td>IPR011118</td>
                              <td></td>
                              <td>N</td>
                           </tr>
                        </tbody>
                     </table>
                  </div>
               </table-wrap>
               <p>Several proteins that may have the ability to degrade components of lignin or disrupt
                  the linkage between lignin and other components of lignocellulose were identified
                  (
                  <a href="#pone-0032990-t005">Table 5</a>). These include laccases, tyrosinases, radical copper oxidases, oxidoreductases,
                  and superoxide dismutases that are often associated with lignin degradation. Also,
                  a candidate cellobiose dehydrogenase was detected, which can generate hydroxy radicals.
                  In addition, enzymes that target aromatic compounds for degradation were found, including
                  an esterase and biphenol reductase (
                  <a href="#pone-0032990-t005">Table 5</a>).
               </p>
               <table-wrap>
                  <a href="https://dx.doi.org/10.1371/journal.pone.0032990.t005">10.1371/journal.pone.0032990.t005</a>
                  <span class="label" tagx="label" title="label">Table 5</span>
                  <caption>
                     <div class="title" tagx="title" title="title">Proteins associated with lignin metabolism from MudPIT analysis.</div>
                  </caption>
                  <div class="alternatives" title="alternatives">
                     <graphic></graphic>
                     <table>
                        <colgroup>
                           <col></col>
                           <col></col>
                           <col></col>
                           <col></col>
                           <col></col>
                           <col></col>
                        </colgroup>
                        <thead>
                           <tr>
                              <td>Protein name</td>
                              <td>Number of Proteins</td>
                              <td>Protein ID</td>
                              <td>Interpro ID (or description if unknown)</td>
                              <td>KEGG EC Number (if known)</td>
                              <td>Secreted?</td>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td>Acid phosphatase</td>
                              <td>4</td>
                              <td>18, 68, 135, 267</td>
                              <td>IPR000120, IPR000560, IPR002828, IPR003778, IPR003833, IPR004843</td>
                              <td>3.1.3.2: Acid phosphatase</td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Alkaline phosphatase</td>
                              <td>1</td>
                              <td>97</td>
                              <td>IPR001952</td>
                              <td>3.1.3.1: Alkaline phosphatase</td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Candidate 2-hydroxy-6-oxo-6-pheylhexa-2,4 dienoate hydrolase</td>
                              <td>1</td>
                              <td>201</td>
                              <td>IPR000073, IPR000379, IPR003089, IPR008262</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Candidate cellobiose dehydrogenase</td>
                              <td>1</td>
                              <td>199</td>
                              <td>CBM and DOMO domain containing protein</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Candidate copper radical oxidase</td>
                              <td>1</td>
                              <td>138</td>
                              <td>IPR002889+Blast homology</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Candidate esterase directed at aromatic compounds</td>
                              <td>1</td>
                              <td>199</td>
                              <td>IPR000379</td>
                              <td></td>
                              <td>N</td>
                           </tr>
                           <tr>
                              <td>Candidate FAD oxidoreductase</td>
                              <td>1</td>
                              <td>404</td>
                              <td>KOG2852: possible oxidoreductase+Blast homology</td>
                              <td></td>
                              <td>N</td>
                           </tr>
                           <tr>
                              <td>Carboxymuconolactone decarboxylase</td>
                              <td>1</td>
                              <td>381</td>
                              <td>IPR003779</td>
                              <td></td>
                              <td>N</td>
                           </tr>
                           <tr>
                              <td>Catalase</td>
                              <td>2</td>
                              <td>40, 114</td>
                              <td>IPR002226, IPR002818, IPR010582</td>
                              <td>1.11.1.6: Catalase</td>
                              <td>1Y 1N</td>
                           </tr>
                           <tr>
                              <td>FAD oxidoreductase</td>
                              <td>5</td>
                              <td>128, 165, 211, 219, 247</td>
                              <td>IPR006094</td>
                              <td></td>
                              <td>4Y 1N</td>
                           </tr>
                           <tr>
                              <td>Glyoxylase dioxygenase</td>
                              <td>1</td>
                              <td>398</td>
                              <td>IPR002110, IPR011588</td>
                              <td></td>
                              <td>N</td>
                           </tr>
                           <tr>
                              <td>GMC oxidoreductase</td>
                              <td>2</td>
                              <td>85, 124</td>
                              <td>IPR000172, IPR007867</td>
                              <td>1.1.99.1: Choline dehydrogenase</td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Laccase</td>
                              <td>4</td>
                              <td>33, 133, 154, 261</td>
                              <td>IPR001117, IPR002355, IPR006162</td>
                              <td>1.10.3.2: Laccase1.10.3.3: L-ascorbate oxidase</td>
                              <td>2Y 2N</td>
                           </tr>
                           <tr>
                              <td>Nickel superoxide dismutase</td>
                              <td>1</td>
                              <td>106</td>
                              <td>IPR006162</td>
                              <td>1.15.1.1: Nickel superoxide dismutase</td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Small secreted protein</td>
                              <td>3</td>
                              <td>94, 144, 290</td>
                              <td>Blast homology</td>
                              <td></td>
                              <td>1Y 2N</td>
                           </tr>
                           <tr>
                              <td>Superoxide dismutase</td>
                              <td>2</td>
                              <td>47, 248</td>
                              <td>IPR001189, IPR001424</td>
                              <td>1.15.1.1: Superoxide dismutase</td>
                              <td>N</td>
                           </tr>
                           <tr>
                              <td>Tyrosinase</td>
                              <td>2</td>
                              <td>37, 285</td>
                              <td>IPR002227</td>
                              <td></td>
                              <td>1Y 1N</td>
                           </tr>
                        </tbody>
                     </table>
                  </div>
               </table-wrap>
               <p>In addition, 47 proteins relevant to protein digestion and nitrogen scavenging were
                  detected in our protein extract, which were classified to 24 different IPR protein
                  domains (
                  <a href="#pone-0032990-t006">Table 6</a>). The most abundant domains involved in protease activity were IPR000209 (peptidase
                  S8 and S53, subtilisin, kexin, sedolisin), IPR007484 (peptidase M28), IPR000834 (peptidase
                  M14), IPR001461 (peptidase A1), and IPR 001563 (peptidase S 10 serine carboxypeptidase).
                  Through cross annotation with the KEGG database, we classified proteins containing
                  these domains to enzyme classes: proteins classified as IPR000209 were assigned to
                  ECs 3.4.21.-(serine endopeptidases), 3.4.21.48 (cerevisin), and 3.4.14.9(tripeptidyl-peptidase
                  I); proteins classified as IPR007487 were assigned to ECs 3.4.11.- (aminopeptidase),
                  3.4.11.10 (bacterial leucylaminopeptidase); proteins classified to IPR000834 were
                  assigned to ECs 3.4.17.- (metallocarboxypeptidase), 3.4.17.15 (carboxypeptidase A2),
                  3.4.17.2 (carboxypeptidase B); proteins classified to IPR001461 were assigned to ECs
                  3.4.23.1 (pepsin), 3.4.23.24 (
                  <i>Candida</i> pepsin), 3.4.23.-(aspartic endopeptidases); and proteins classified as IPR001563
                  were assigned to ECs 3.4.16.6 (carboxypeptidase D). Two proteins relevant to nitrogen
                  recycling and nitrogen scavenging were also detected, including IPR03778 (urea amidolyase
                  related) and IPR004304 (acetamidase/formamidase) respectively. Although IPR003778
                  could not be annotated with KEGG, IPR004304 was classified as EC 3.5.1.49 (formamidase).
               </p>
               <table-wrap>
                  <a href="https://dx.doi.org/10.1371/journal.pone.0032990.t006">10.1371/journal.pone.0032990.t006</a>
                  <span class="label" tagx="label" title="label">Table 6</span>
                  <caption>
                     <div class="title" tagx="title" title="title">Proteinases and nitrogen recycling proteins identified from MudPIT data.</div>
                  </caption>
                  <div class="alternatives" title="alternatives">
                     <graphic></graphic>
                     <table>
                        <colgroup>
                           <col></col>
                           <col></col>
                           <col></col>
                           <col></col>
                           <col></col>
                           <col></col>
                        </colgroup>
                        <thead>
                           <tr>
                              <td>Protein name</td>
                              <td>Number of Proteins</td>
                              <td>Protein ID</td>
                              <td>Interpro ID (or description if unknown)</td>
                              <td>KEGG EC Number (if known)</td>
                              <td>Secreted?</td>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td>Amidase</td>
                              <td>1</td>
                              <td>29</td>
                              <td>IPR000120</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Amidohydrolase</td>
                              <td>1</td>
                              <td>307</td>
                              <td>IPR006680</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Aminopeptidase</td>
                              <td>4</td>
                              <td>83, 88, 118, 256</td>
                              <td>IPR003137, IPR007484</td>
                              <td>3.4.11.-: Aminopeptidase</td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Aspartic endopeptidase</td>
                              <td>2</td>
                              <td>102, 104</td>
                              <td>IPR000250, IPR001461 IPR001969</td>
                              <td>3.4.23.-: Aspartic endopeptidase</td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Bacterial leucyl peptidase</td>
                              <td>1</td>
                              <td>63</td>
                              <td>IPR007484</td>
                              <td>3.4.11.10: Bacterial leucyl aminopeptidase</td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Candidapepsin</td>
                              <td>1</td>
                              <td>24</td>
                              <td>IPR001461</td>
                              <td>3.4.23.24: Candidapepsin</td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Candidate formylmethionine deformylase</td>
                              <td>1</td>
                              <td>322</td>
                              <td>Blast homology</td>
                              <td></td>
                              <td>N</td>
                           </tr>
                           <tr>
                              <td>Carboxypeptidase A</td>
                              <td>1</td>
                              <td>224</td>
                              <td>IPR000834</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Carboxypeptidase A2</td>
                              <td>1</td>
                              <td>64</td>
                              <td>IPR001412</td>
                              <td>3.4.17.15: Carboxypeptidase A2</td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Carboxypeptidase B</td>
                              <td>1</td>
                              <td>266</td>
                              <td>IPR000834</td>
                              <td>3.4.17.2: Carboxypeptidase B</td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Carboxypeptidase C</td>
                              <td>2</td>
                              <td>152</td>
                              <td>IPR000379, IPR001563</td>
                              <td>3.4.16.5: Carboxypeptidase C</td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Cerevisin</td>
                              <td>2</td>
                              <td>43, 131</td>
                              <td>IPR000209, IPR003137</td>
                              <td>3.4.21.48: Cerevisin</td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Cysteine peptidase</td>
                              <td>1</td>
                              <td>178</td>
                              <td>IPR000169</td>
                              <td></td>
                              <td>N</td>
                           </tr>
                           <tr>
                              <td>Di- and tri- peptidyl peptidase</td>
                              <td>1</td>
                              <td>117</td>
                              <td>IPR000379, IPR001375, IPR002088</td>
                              <td>3.4.14.-: Dipeptidyl peptidase and tripeptidyl peptidase</td>
                              <td>N</td>
                           </tr>
                           <tr>
                              <td>Formamidase</td>
                              <td>1</td>
                              <td>212</td>
                              <td>IPR002469, IPR004304</td>
                              <td>3.5.1.49: Formamidase</td>
                              <td>N</td>
                           </tr>
                           <tr>
                              <td>Fungalysin</td>
                              <td>1</td>
                              <td>55</td>
                              <td>IPR001842, IPR006025, IPR011096</td>
                              <td></td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Glutamate carboxypeptidase II</td>
                              <td>1</td>
                              <td>90</td>
                              <td>IPR003137, IPR007365, IPR007484</td>
                              <td>3.4.17.21: Glutamate carboxypeptidase II</td>
                              <td>N</td>
                           </tr>
                           <tr>
                              <td>Metallocarboxypeptidase</td>
                              <td>1</td>
                              <td>62</td>
                              <td>IPR000834</td>
                              <td>3.4.17.-: Metallocarboxypeptidase</td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Metalloendopeptidase</td>
                              <td>2</td>
                              <td>250 260</td>
                              <td>IPR001384, IPR001567, IPR006025</td>
                              <td></td>
                              <td>1Y 1N</td>
                           </tr>
                           <tr>
                              <td>Pepsin A</td>
                              <td>2</td>
                              <td>14, 380</td>
                              <td>IPR001461, IPR001969</td>
                              <td>3.4.23.1: Pepsin A</td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Peptidase</td>
                              <td>1</td>
                              <td>13</td>
                              <td>IPR000379, IPR008758</td>
                              <td>3.4.-.-: Acting on peptide bonds (peptide hydrolases).</td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Peptidase C2</td>
                              <td>1</td>
                              <td>171</td>
                              <td>IPR001300</td>
                              <td></td>
                              <td>N</td>
                           </tr>
                           <tr>
                              <td>Serine carboxypeptidase D</td>
                              <td>2</td>
                              <td>10</td>
                              <td>IPR001563</td>
                              <td>3.4.16.6: Carboxypeptidase D</td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Serine endopeptidase</td>
                              <td>4</td>
                              <td>17, 49, 277, 396</td>
                              <td>IPR000209, IPR010259</td>
                              <td>3.4.21.-: Serine endopeptidase</td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Tri-peptidyl peptidase</td>
                              <td>2</td>
                              <td>82, 221</td>
                              <td>IPR000209</td>
                              <td>3.4.14.9: Tripeptidyl-peptidase I</td>
                              <td>1Y 1N</td>
                           </tr>
                           <tr>
                              <td>Trypsin</td>
                              <td>1</td>
                              <td>46</td>
                              <td>IPR001314, IPR008256</td>
                              <td>3.4.21.4: Trypsin</td>
                              <td>Y</td>
                           </tr>
                           <tr>
                              <td>Urease</td>
                              <td>1</td>
                              <td>164</td>
                              <td>IPR000089, IPR000120, IPR003778, IPR003833</td>
                              <td></td>
                              <td>N</td>
                           </tr>
                        </tbody>
                     </table>
                  </div>
               </table-wrap>
               <p>Gene ontology (GO) annotation can be utilized to classify proteins by general function
                  and overabundance of certain categories in expression datasets, including transcriptomes
                  and proteomes, which can indicate the potential importance of these groups of genes
                  to ongoing metabolic processes. In the proteome data obtained from an 
                  <i>A. glabripennis</i> -derived 
                  <i>F. solani</i> isolate growing on solid wood substrate, the most highly abundant categories from
                  level 3 of the Molecular Function category were hydrolase activity, nucleotide binding,
                  and oxidoreductase activity (red line, 
                  <a href="#pone-0032990-g001">Figure 1</a>), accounting for 34.2%, 9.9%, and 8.7% of proteins in our secretome, respectively.
                  However, it is likely that these GO categories may be overrepresented in the reference
                  genome and the overabundance of these categories in our proteome may simply be an
                  artifact of genome enrichment. To correct for this and to identify GO categories of
                  proteins that are truly overrepresented under these growth conditions, we compared
                  the relative abundances of GO categories in our proteome against their relative abundances
                  in the genome. Through this comparison, many GO categories are enriched in our proteome
                  relative to the reference genome, including carbohydrate binding, hydrolase activity,
                  peroxidase activity, and protein binding (bar graph, 
                  <a href="#pone-0032990-g001">Figure 1</a>).
               </p>
               <div class="fig" title="fig">
                  <div class="figure" title="figure">
                     <a href="http://doi.org/10.1371/journal.pone.0032990.g001">
                        <span class="label" tagx="label" title="label">Figure 1</span>
                     </a>
                  </div>
                  <caption class="caption" title="caption">
                     <caption>
                        <div class="title" tagx="title" title="title">Enrichment of GO Molecular Function terms in proteomic analysis.</div>
                        <p>Bar graph represents the ratio of % composition of term in proteomic data vs. % composition
                           in the genome annotation. Values over 1 (dotted line) are overrepresented in the proteomic
                           data. Red line illustrates the relative abundance of the GO term in the proteomic
                           data.
                        </p>
                     </caption>
                  </caption>
               </div>
            </div>
            <div class="verificationofenzymeactivitythroughinvitrolignocellulaseactivity" title="sec">
               <div class="title" tagx="title" title="title">Verification of enzyme activity through 
                  <i>in vitro</i> lignocellulase activity
               </div>
               <p>The fungal enzyme extract created from a solid wood culture of 
                  <i>A. glabripennis</i>-derived 
                  <i>F. solani</i> was surveyed for activities characteristic of enzymes that can degrade lignin, cellulose,
                  and xylan (
                  <a href="#pone-0032990-t007">Table 7</a>). β-glucosidase activity, measured by the release of reducing sugars from salicin,
                  was 212.8 U/ml, CMCase activity was 13.1 U/ml, and cellulase activity measured by
                  release of reducing sugar from microcrystalline cellulose (Avicel), was 14.7 U/ml
                  (
                  <a href="#pone-0032990-t007">Table 7</a>). Xylanase activity, measured by release of reducing sugar from birch wood xylan,
                  was 70.5 U/ml. Although the sample did not exhibit lignin peroxidase activity as measured
                  by oxidation of veratryl alcohol, the sample exhibited low levels of Mn-dependent
                  peroxidases activity (0.021 U/ml), Mn-independent peroxidase activity (0.47 U/ml)
                  and laccase activity (0.42 U/ml) (
                  <a href="#pone-0032990-t007">Table 7</a>).
               </p>
               <table-wrap>
                  <a href="https://dx.doi.org/10.1371/journal.pone.0032990.t007">10.1371/journal.pone.0032990.t007</a>
                  <span class="label" tagx="label" title="label">Table 7</span>
                  <caption>
                     <div class="title" tagx="title" title="title">Verification of lignocellulytic activity of 
                        <i>A. glabripennis</i> derived 
                        <i>F. solani</i> solid wood culture extracts through in vitro assays.
                     </div>
                  </caption>
                  <div class="alternatives" title="alternatives">
                     <graphic></graphic>
                     <table>
                        <colgroup>
                           <col></col>
                           <col></col>
                           <col></col>
                           <col></col>
                        </colgroup>
                        <thead>
                           <tr>
                              <td></td>
                              <td>Enzyme Activity (U/ml)
                                 <a href="#nt102">*</a>
                              </td>
                              <td>Protein Conc. (mg/ml extract)</td>
                              <td>Specific Activity (U/mg protein)
                                 <a href="#nt102">*</a>
                              </td>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td>
                                 <u>Cellulose/xylan</u>
                              </td>
                              <td></td>
                              <td></td>
                              <td></td>
                           </tr>
                           <tr>
                              <td>Beta-glucosidase</td>
                              <td>12.77</td>
                              <td>0.06</td>
                              <td>212.8</td>
                           </tr>
                           <tr>
                              <td>CMCase</td>
                              <td>0.79</td>
                              <td>0.06</td>
                              <td>13.1</td>
                           </tr>
                           <tr>
                              <td>Cellulase (from Avicel)</td>
                              <td>0.88</td>
                              <td>0.06</td>
                              <td>14.7</td>
                           </tr>
                           <tr>
                              <td>Xylanase</td>
                              <td>4.23</td>
                              <td>0.06</td>
                              <td>70.5</td>
                           </tr>
                           <tr>
                              <td>
                                 <u>Lignin</u>
                              </td>
                              <td></td>
                              <td></td>
                              <td></td>
                           </tr>
                           <tr>
                              <td>Lignin Peroxidase</td>
                              <td>0</td>
                              <td>0.05</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td>Mn-dependent Peroxidase</td>
                              <td>0.021</td>
                              <td>0.05</td>
                              <td>0.42</td>
                           </tr>
                           <tr>
                              <td>Mn-independent Peroxidase</td>
                              <td>0.47</td>
                              <td>0.05</td>
                              <td>9.4</td>
                           </tr>
                           <tr>
                              <td>Laccase</td>
                              <td>0.42</td>
                              <td>0.05</td>
                              <td>8.4</td>
                           </tr>
                        </tbody>
                     </table>
                  </div>
                  <table-wrap-foot>
                     <div class="fn-type-" title="">
                        <span class="label" tagx="label" title="label"></span>
                        <p>Activities of cellulases, xylanases, and lignin peroxidases, Mn-dependent and independent
                           peroxidases, as well as laccase were measured for 
                           <i>F. solani</i> extracts with each assay containing 50 µg of protein.
                        </p>
                     </div>
                     <div class="fn-type-" title="">
                        <span class="label" tagx="label" title="label">*</span>
                        <p>1 unit of activity = amount of enzyme that releases 1 µmol of reducing sugar per minute
                           for reducing sugar assays and amount of enzyme that oxidizes 1 µmol substrate per
                           min for peroxidase assays.
                        </p>
                     </div>
                  </table-wrap-foot>
               </table-wrap>
            </div>
            <div class="verificationofenzymepresenceandactivitythroughpagegelanalysis" title="sec">
               <div class="title" tagx="title" title="title">Verification of enzyme presence and activity through PAGE gel analysis</div>
               <p>In addition to verification of activity in 
                  <i>in vitro</i> assays, PAGE analysis was performed to visualize active enzymes. A single heme containing
                  protein of approximately 70 kDa was detected on a native heme stained gel; however,
                  a corresponding protein band was not detected on the reference colloidal blue stained
                  gel (
                  <a href="#pone-0032990-g002">Figure 2</a>). A possible explanation for not observing a matching protein on the reference gel
                  is that this protein may not be highly abundant or concentrated enough to be adequately
                  visualized with the colloidal blue stain. Through zymogram analysis, carboxymethyl
                  cellulase activity was detected and six major zones of clearing ranging from 20 to
                  55 kDa were identified (
                  <a href="#pone-0032990-g002">Figure 2</a>). Comparison to the reference colloidal blue stained SDS-PAGE gel revealed that several
                  similarly sized protein bands were present, including a major band at 55 kDa and several
                  less intense bands around and below 28 kDa. Relatively high xylanase activity was
                  also detected through birch wood xylan zymogram analysis; many zones of clearing were
                  observed ranging in size from 20 kDa to 50 kDa (
                  <a href="#pone-0032990-g002">Figure 2</a>). Very large and broad zones of clearing between 25 and 45 kDa obstructed the ability
                  to define specific protein bands with activity towards xylan, but distinct bands can
                  be seen above and below this region on the zymogram gel. Again, these regions of clearing
                  can be matched to corresponding protein bands on the reference colloidal blue stained
                  gel. This analysis verifies the data from 
                  <i>in vitro</i> analysis and also demonstrates that many of these activities are from a diversity
                  of proteins, which is also represented in the proteomic dataset.
               </p>
               <div class="fig" title="fig">
                  <div class="figure" title="figure">
                     <a href="http://doi.org/10.1371/journal.pone.0032990.g002">
                        <span class="label" tagx="label" title="label">Figure 2</span>
                     </a>
                  </div>
                  <caption class="caption" title="caption">
                     <caption>
                        <div class="title" tagx="title" title="title">Heme staining and zymogram analysis of 
                           <i>A. glabripennis</i> derived 
                           <i>F. solani</i> solid wood culture extract.
                        </div>
                        <p>Twenty µg of fungal extract were loaded into each lane. Protein standard is on the
                           left, with band sizes labeled (kDa). Lane A is a colloidal blue stained lane, and
                           lane B is the corresponding heme stain/CMC/or xylan zymogram lane. For heme stain,
                           a single band is present at approximately 70 kDa. For CMC zymogram six major zones
                           of clearing are present at approximately 55, 32, 30, 27, 23 and 20 kDa and can be
                           matched to protein bands on the colloidal blue stained lane. For birch wood xylan
                           zymogram, many zones of clearing are present on the gel with major spots between 30
                           and 50 kDa. These correspond to a broad range of bands on the colloidal blue stained
                           lane.
                        </p>
                     </caption>
                  </caption>
               </div>
            </div>
         </div>
         <div class="discussion" title="sec">
            <div class="title" tagx="title" title="title">Discussion</div>
            <p>Over 30 years of research has been devoted to resolving the mechanisms and enzymology
               of lignin biodegradation, yet only three fungal enzymes have been discovered that
               conclusively depolymerize lignin, fully converting it to carbon dioxide and water.
               Reasons commonly cited for lack of progress in this field include a poorly resolved
               biochemical structure of lignin, unreliable 
               <i>in vitro</i> assays to confirm lignin degrading activities, and uncharacterized growth and physiological
               conditions required for induction of lignin degrading enzymes in non-white rot fungal
               isolates. While advances in next-generation sequencing and bioinformatics may allow
               us to detect lignin, manganese-, and versatile- peroxidase orthologs in the genomes
               of newly sequenced organisms, validating and confirming that these orthologs actually
               catalyze lignin depolymerization is not a trivial task. Despite these difficulties,
               lignin degradation has recently been documented in the guts of two evolutionarily
               distant insect species: 
               <i>Anoplophora glabripennis</i> and 
               <i>Zootermopsis angusticollis</i> (Pacific dampwood termite) 
               <a href="#pone.0032990-Ke1">[12]</a>, 
               <a href="#pone.0032990-Harazono1">[17]</a>, 
               <a href="#pone.0032990-Kato1">[20]</a>, 
               <a href="#pone.0032990-Geib1">[22]</a>, 
               <a href="#pone.0032990-Geib3">[24]</a>, 
               <a href="#pone.0032990-Zhang1">[31]</a>, 
               <a href="#pone.0032990-Sun1">[32]</a>, 
               <a href="#pone.0032990-Coy1">[33]</a>, 
               <a href="#pone.0032990-Scharf1">[34]</a>. Since this initial discovery, much effort has been devoted to dissecting lignin
               degradation mechanisms and mining for key enzymes linked to these processes in both
               termites and 
               <i>A. glabripennis</i>; however, the fundamental enzymes involved that catalyze these reactions remain elusive.
               Despite this ambiguity, there is evidence that all major reactions associated with
               large scale lignin biodegradation occur in both insects 
               <a href="#pone.0032990-Geib1">[22]</a>, but the predominant oxidative reactions are different. For example, side chain oxidation
               was the dominant reaction in the 
               <i>A. glabripennis</i> gut, while ring hydroxylation was the dominant reaction in 
               <i>Zootermopsis</i>, indicating that different enzymes may catalyze lignin degradation in these systems
               
               <a href="#pone.0032990-Geib1">[22]</a>. Despite these differences, one resounding commonality can be noted: these processes
               both occur in the absence of white rot basidiomycete fungi, leading to the possibility
               that novel lignin degrading enzymes are harbored in the insect genomes (not likely)
               or within the gut communities (more likely). While the gut bacterial communities of
               many termites and 
               <i>A. glabripennis</i> are both dominated by Actinomycete bacteria 
               <a href="#pone.0032990-Geib2">[23]</a>, 
               <a href="#pone.0032990-Pasti1">[64]</a>, including taxa known to efficiently metabolize aromatic compounds, the ability of
               these actinomycetes to completely depolymerize the lignin macromolecule remains questionable
               
               <a href="#pone.0032990-Harazono1">[17]</a>, 
               <a href="#pone.0032990-Bugg1">[18]</a>, 
               <a href="#pone.0032990-Kajikawa1">[19]</a>, 
               <a href="#pone.0032990-Kato1">[20]</a>, 
               <a href="#pone.0032990-Shary1">[21]</a>. In addition, little is known about the composition of fungal communities in the
               guts of non-fungus cultivating termites; while in contrast, 
               <i>A. glabripennis</i> larvae consistently harbor a filamentous ascomycete belonging to the 
               <i>Fusarium solani</i> species complex 
               <a href="#pone.0032990-Geib4">[35]</a>, a metabolically diverse group of fungi with prolific lignin and aromatic polymer
               degrading capabilities 
               <a href="#pone.0032990-Colombo1">[40]</a>, 
               <a href="#pone.0032990-Rodriguez1">[46]</a>. Here, we investigate the ability of this 
               <i>A. glabripennis</i> -derived 
               <i>F. solani</i> isolate to colonize and thrive on a solid wood substrate, degrade intractable woody
               polymers, including cellulose, xylan, and lignin, and extract other essential nutrients
               from this environment during periods of secondary metabolism.
            </p>
            <p>In white rot basidiomycetes, expression of lignin degrading genes occurs solely during
               periods of secondary metabolism and nutrient limitation 
               <a href="#pone.0032990-Kirk1">[10]</a>; however, the conditions required for induction of lignin degrading enzymes are not
               well characterized in 
               <i>Fusarium spp.</i> and their ability to colonize lignocellulose-based substrates varies tremendously
               
               <a href="#pone.0032990-Crawford1">[50]</a>. Our goals to ensure substrate colonization regardless of inherent metabolic potential
               and to induce growth conditions characteristic of secondary metabolism were achieved.
               Proteins detected in MudPIT data associated with secondary metabolism include cupin-,
               germin-, and patatin-containing proteins, which facilitate nutrient storage under
               nutrient limiting conditions 
               <a href="#pone.0032990-Dunwell1">[65]</a> (
               <a href="#pone.0032990.s006">Table S6</a>)). Additionally, proteins and enzymes typically induced during periods of stress
               and nutrient deprivation were detected including cell wall spore proteins, cytochrome
               p450, extracellular ribonucleases, gamma glutamyl transpeptidase, heat shock proteins,
               mucin-like glycoproteins, and woronin body proteins 
               <a href="#pone.0032990-Dunwell1">[65]</a>, 
               <a href="#pone.0032990-Ubiyvovk1">[66]</a>, 
               <a href="#pone.0032990-Soundararajan1">[67]</a> (
               <a href="#pone.0032990.s006">Table S6</a>), likely indicating that this fungus was not persisting under nutrient rich, stress
               free conditions in culture. Furthermore, proteins associated with host plant interactions
               were detected from MudPIT data, including several candidate toxin-producing proteins
               and one cerato-platanin necrosis-inducing enzyme (
               <a href="#pone.0032990.s007">Table S7</a>) 
               <a href="#pone.0032990-Pazzagli1">[68]</a>, indicating that the fungal isolate was not simply persisting on millet and bran
               and was actively colonizing wood chips present in the medium.
            </p>
            <p>Likewise, cellulase, glycoside hydrolase (directed at β-1,4 linkages), and xylanase
               activities were conclusively detected 
               <i>in vitro</i> through zymogram analysis and reducing sugar assays (
               <a href="#pone-0032990-g002">Figure 2</a> and 
               <a href="#pone-0032990-t007">Table 7</a>), demonstrating that this isolate was actively digesting carbohydrate polysaccharides
               present in the woody substrate rather than simply extracting glucose from the soluble
               starches present in millet. In tandem, GH families responsible for hydrolyzing β-1,4
               linkages present in both cellulose and xylan were identified through MudPIT analysis,
               including complete enzyme complexes for full conversion of both polysaccharides to
               glucose. Cellulose alone requires at least three distinct enzymes for efficient glucose
               liberation, including endoglucases, exoglucanases, and β-glucosidases. Endoglucanases
               cleave amorphous sites in cellulose at random, decreasing its crystallinity, increasing
               its solubility, and rapidly exposing reducing and non-reducing ends to other hydrolytic
               enzymes. Subsequently, exoglucanases act on these reducing and non-reducing termini
               to release cellobiose and other cello-oligomers, which can be efficiently converted
               to glucose by β-glucosidases 
               <a href="#pone.0032990-Holtzapple1">[69]</a>. Endoglucanases (EC 3.2.1.4) are categorized into several GH families, including
               GH 5, 6, 7, 8, 9, 12, 44, 45, 48, 51, and 61. Exoglucanases or cellobiohydrolases
               classified into two distinct KEGG ECs depending on whether they target reducing or
               nonreducing ends and whether they have inverting or retaining activities: EC 3.2.1.176
               (GH7 and 48; reducing, retaining), EC 3.2.1.91 (GH 5, 6, and 9; non-reducing, inverting).
               β-glucosidases (EC 3.2.1.21) are distributed among GH families 1, 3, 9, 30, 116 
               <a href="#pone.0032990-Cantarel1">[70]</a>. GHs with potential relevance to cellulose digestion detected through MudPIT analysis
               include GHs belonging to families 1, 3, 5, 6, 7, 45, and 61. Many of these glycoside
               hydrolases were not well-annotated in the reference genome with GO or KEGG terms 
               <a href="#pone.0032990-Coleman1">[48]</a>, so the precise reactions catalyzed by these GHs cannot be directly inferred; an
               individual GH family can harbor enzymes with very diverse catalytic and substrate
               specificities (
               <a href="#pone-0032990-t003">Table 3</a>). However, the distribution of GHs in conjunction with release of reducing ends from
               microcrystalline cellulose, carboxymethyl cellulose, and salicin suggests that this
               fungal isolate possesses the full suite of cellulolytic enzymes.
            </p>
            <p>On the other hand, hemicellulose is a much more heterogeneous polymer containing many
               monomeric subunits and greater diversity of chemical linkages, and thus, requires
               a combination of glycoside hydrolases and esterase enzymes for efficient conversion
               to sugar monomers 
               <a href="#pone.0032990-Pettersen1">[9]</a>. For example, O-acetylglucuronoxylan is the predominant polysaccharide in hardwood
               trees 
               <a href="#pone.0032990-Pinto1">[71]</a> and requires endo-, exo-, and β-xylosidases (EC 3.2.1.37: GH 3, 30, 39, 43, 52, 54,
               116, and 120) 
               <a href="#pone.0032990-Cantarel1">[70]</a> to hydrolyze β-1,4 linkages; α-uronidsases (EC 3.2.1.139) to liberate glucuronic,
               mannuronic, and galacturonic acids; and ferulic acid esterase (EC 3.1.1.73) and acetylxylan
               esterase (EC 3.2.1.72) 
               <a href="#pone.0032990-Scharf1">[34]</a>, 
               <a href="#pone.0032990-Sato1">[72]</a> to hydrolyze phenolic ester bonds that cross-link hemicellulose to lignin. Of the
               eight GHs families with documented involvement in xlyan degradation, three were detected
               in our proteome that could account for the xylanase activity we observed 
               <i>in vitro</i> in reducing sugar assays and zymogram analyses (
               <a href="#pone-0032990-t007">Table 7</a> and 
               <a href="#pone-0032990-g002">Figure 2</a>). These include GH families 3, 43, and a candidate GH 39 (
               <a href="#pone-0032990-t003">Table 3</a>). Many of these had predicted signal peptides, but no specific GO or EC annotations
               were present in the reference genome that could be utilized to determine substrate
               specificities. In addition, a number of ester hydrolyzing enzymes, including carboxylesterases,
               esterases, and a tannase/feruloyl esterase, were also detected through shotgun proteomics
               (
               <a href="#pone-0032990-t004">Table 4</a>). Surprisingly, a number of proteins with activity directed at pectin and cutin polymers,
               which are not highly abundant in woody tissues, were also detected; however, pectin
               and cutin are highly pertinent to wood decay processes because these compounds are
               often found in the central location of pit membranes, ray cell walls, and middle lamellae
               of wood cell walls 
               <a href="#pone.0032990-Ferris1">[73]</a> (
               <a href="#pone-0032990-t004">Table 4</a>). These polysaccharides are often broken down during wood rot processes and decomposition
               is often coupled to metal cation acquisition as pectin serves as a calcium reservoir
               in woody tissue 
               <a href="#pone.0032990-Sato2">[74]</a>.
            </p>
            <p>Proteins are also occasionally found impregnating xylem elements and plant cell walls
               in woody tissue, which provide vital nitrogen sources that could be assimilated by
               wood degrading bacteria and fungi. Only a few types of proteins are co-localized to
               cell walls in wood and they are often covalently cross-linked to cellulose, hemicellulose,
               and lignin in the cell wall matrix 
               <a href="#pone.0032990-Keller1">[11]</a>. Incidentally, extracellular proteinases are often highly expressed in white rot
               basidiomycetes during periods of active lignin metabolism 
               <a href="#pone.0032990-Regalado1">[16]</a>. As production of lignin degrading peroxidases is induced by nitrogen limiting conditions
               in white rot basidiomycetes, many have hypothesized that these fungi may degrade lignin
               in order to access proteins cross-linked to lignin, though this has not been directly
               tested 
               <a href="#pone.0032990-Sato1">[72]</a>, 
               <a href="#pone.0032990-Sato2">[74]</a>, 
               <a href="#pone.0032990-Abbas1">[75]</a>. In concert, we detected many extracellular proteinases in our own secretome, including
               many enzymes with broad substrate specificities that could serve to scavenge nitrogen
               from woody tissue, including aspartic peptidases, carboxypeptidases, metallopeptidases,
               and serine peptidases (
               <a href="#pone-0032990-t006">Table 6</a>). An alternative explanation for the abundance of proteinases in the secretome is
               that they may have been actively hydrolyzing proteins present in the millet, although
               these elements should have been depleted from the medium at the time of harvesting.
               In addition, fungi that thrive under these nitrogen limiting conditions for extended
               periods of time may also efficiently recycle and reuse nitrogenous waste products
               in amino acids and nucleotides. Two putative nitrogen recycling proteins were also
               detected, including formamidase and urease that actively convert nitrogenous waste
               into ammonia, which can subsequently be re-integrated into amino acids or nucleotides
               (
               <a href="#pone-0032990-t006">Table 6</a>) 
               <a href="#pone.0032990-Strope1">[76]</a>.
            </p>
            <p>Protein, cellulose, and hemicellulose in wood chips are all protected from hydrolytic
               enzymes by lignin, a structural biopolymer dominated by recalcitrant linkages that
               can only be broken through radical oxidative depolymerization catalyzed by lignin-,
               manganese-, and versatile- peroxidases 
               <a href="#pone.0032990-Kirk1">[10]</a>. In addition, highly reactive hydroxyl radicals produced from Fenton reactions may
               help to expedite complete lignin depolymerization in some white rot fungi 
               <a href="#pone.0032990-Crawford1">[50]</a>, 
               <a href="#pone.0032990-Kersten1">[77]</a>. While these enzymes all catalyze lignin depolymerization using slightly different
               mechanisms, they all require extracellular peroxide, which is generated by variety
               of enzymes including FAD oxidases, copper radical oxidases, glyoxal oxidases, and
               GMC oxidoreductases 
               <a href="#pone.0032990-Kirk1">[10]</a>, 
               <a href="#pone.0032990-Regalado1">[16]</a>, 
               <a href="#pone.0032990-Kersten1">[77]</a>, 
               <a href="#pone.0032990-Hammel1">[78]</a>. Some lignin degrading fungi also utilize laccases 
               <a href="#pone.0032990-Youn1">[79]</a> to catalyze oxidative cleavage of phenoxy linkages, which may degrade small lignin
               metabolites released by larger scale depolymerization processes or augment large scale
               oxidative depolymerization of lignin-, manganese-, or versatile- peroxidases. Although
               their catalytic potential can be expanded to oxidize more recalcitrant linkages in
               the presence of synthetic mediators 
               <a href="#pone.0032990-Morozova1">[15]</a>, no natural varieties of these redox mediators have been conclusively identified,
               though some speculate that small secreted proteins expressed during active lignin
               metabolism may fill this niche 
               <a href="#pone.0032990-Regalado1">[16]</a>.
            </p>
            <p>Despite the presence of a lignin peroxidase ortholog in our reference genome, no 
               <i>bona fide</i> lignin peroxidases or manganese peroxidases were detected through 
               <i>in vitro</i> biochemical assays or 
               <i>de novo</i> peptide sequencing; however, one unidentifiable heme protein was detected (
               <a href="#pone-0032990-g002">Figure 2</a>). Whether or not this protein functions as an extracellular lignin peroxidase is
               unknown. Additionally, many extracellular enzymes typically observed in the secretomes
               of other lignin-degrading fungi were detected, including several secreted laccases
               whose activity was verified 
               <i>in vitro</i> (
               <a href="#pone-0032990-t006">Tables 6</a> and 
               <a href="#pone-0032990-t007">7</a>) and two intracellular polyphenol oxidases (tyrosinases). Both phenol oxidases can
               oxidize similar substrates, but only laccases can oxidize syringaldazine, while trysoninases
               are more instrumental in degrading tannic and gallic acids 
               <a href="#pone.0032990-Worrall1">[43]</a>, 
               <a href="#pone.0032990-Hemingway1">[80]</a>, 
               <a href="#pone.0032990-Bending1">[81]</a>. In tandem, several extracellular peroxide generating enzymes, including FAD oxidases,
               GMC oxidoreductases, a putative copper radical oxidase, and superoxide dismutases,
               and one hydroxy radical generating enzyme (candidate cellobiose dehydrogenase) were
               also observed (
               <a href="#pone-0032990-t006">Table 6</a>), suggesting that processes requiring peroxide and hydroxyl radicals were occurring
               in the extracellular environment. In addition, several germin proteins were detected,
               which can also function as oxalate oxidases (
               <a href="#pone.0032990.s008">Table S8</a>). Oxalate is often produced during periods of active lignin metabolism and can be
               directly converted to hydrogen peroxide by oxalate oxidase and often enhance oxidative
               activities of manganese peroxidase 
               <i>in vitro</i>
               <a href="#pone.0032990-Kuan1">[82]</a>. Several enzymes that degrade small aromatic compounds were also identified, including
               a candidate reductase with activity directed at biphenyl compounds and a candidate
               esterase with activity directed at aromatic compounds 
               <a href="#pone.0032990-Ohta1">[83]</a>, 
               <a href="#pone.0032990-Magnuson1">[84]</a>, which could hydrolyze bonds in small lignin metabolites produced from larger-scale
               biodegradation processes (
               <a href="#pone-0032990-t006">Table 6</a>). Finally, several small secreted proteins and hypothetical proteins were also detected
               in the culture supernatant; whether or not these proteins could be relevant to lignin
               degradation is unknown (
               <a href="#pone-0032990-t006">Table 6</a> and 
               <a href="#pone.0032990.s008">Table S8</a>).
            </p>
            <p>Although no lignin peroxidase activity was detected 
               <i>in vitro</i>, this does not necessarily indicate that this isolate does not harbor lignin degrading
               enzymes or possess lignin degrading capabilities. In fact, veratryl alcohol oxidation
               is often not detected during active lignin degradation in white rot fungi in extracellular
               extracts prepared from isolates growing on woody substrates, even when lignin peroxidase
               isozymes were detected in 2D gels or by 
               <i>de novo</i> peptide sequencing. Furthermore, extensive glycosylation or other post translational
               modifications can interfere with trypsin digestions or alter protein masses, resulting
               in inefficient digestion or errors in predicted amino acid sequences, which can result
               in protein non-detection 
               <a href="#pone.0032990-Sato1">[72]</a>, 
               <a href="#pone.0032990-Abbas1">[75]</a>. It is also possible that the lignin peroxidase ortholog detected in the 
               <i>Nectria haematococca</i> reference genome is not present in our isolate. An alternate explanation is that
               perhaps laccases are more involved in natural lignin degradation in this system than
               originally demonstrated. Lending support to this speculation is the observation that
               the lignin degrading capacity is strongly reduced in some laccase-deficient 
               <i>Pycnoporus cinnabarinus</i> and 
               <i>Sporotrichum pulverulentum</i> mutants and the hypothesis that small secreted proteins produced during active lignin
               metabolism may serve as natural redox mediators for laccase, enhancing its oxidative
               potential 
               <a href="#pone.0032990-Regalado1">[16]</a>, 
               <a href="#pone.0032990-Ander1">[85]</a>, 
               <a href="#pone.0032990-Bermek1">[86]</a>. In addition, Scharf and colleagues 
               <a href="#pone.0032990-Scharf1">[34]</a> recently discovered an endogenous termite laccase whose phenol oxidase activity is
               enhanced in the presence of hydrogen peroxide, a characteristic more synonymous with
               lignin peroxidases. While this could indicate that this laccase has an inherently
               higher redox potential and can catalyze oxidation of more recalcitrant linkages than
               previously characterized laccases, this observation could also be an artifact of His-labeling
               and future studies are needed to validate its redox potential and determine its catalytic
               capabilities.
            </p>
            <p>In conclusion, we have demonstrated that this 
               <i>F. solani</i> isolate has definitive abilities to degrade proteins, cellulose, hemicellulose, and
               other carbohydrate polymers present in woody tissue and that it expresses many enzymes
               that are often up-regulated during periods of lignin metabolism in other lignin degrading
               fungi, including enzymes involved in extracellular peroxidase generation, laccases,
               and polyphenol oxidases. While we have not definitively documented full lignin depolymerization
               elicited by this 
               <i>A. glabripennis</i> -derived 
               <i>F. solani</i> strain, these results indicate that this isolate may have lignin degrading potential.
               In order to better assess the true metabolic potential of this fungal strain, whole
               genome sequencing is currently in progress to produce a more suitable reference genome
               for future transcriptomics and proteomics studies and more conclusively assay lignin
               degrading capabilities. Furthermore, follow up metatranscriptomic and metaproteomic
               approaches will be employed to determine if fungal transcripts and enzymes are actively
               expressed in the gut and to assess its potential contribution to lignin degrading
               activities in larval 
               <i>A. glabripennis</i>.
            </p>
         </div>
         <div class="supplementary-material" title="supplementary-material">
            <div class="title" tagx="title" title="title">Supporting Information</div>
            <div class="local-data" title="local-data">
               <span class="label" tagx="label" title="label">Table S1</span>
               <caption>
                  <p>
                     <b>Proteins identified in MudPIT analysis, with Necha 2.0 genomic database protein accession
                        number, gene name, and protein score statistics from ProteinPilot software.
                     </b>
                  </p>
                  <p>(XLS)</p>
               </caption>
               <div class="media" title="media">
                  <a href="pone.0032990.s001.xls">LINK</a>
                  <caption>
                     <p>Click here for additional data file.</p>
                  </caption>
               </div>
            </div>
            <div class="local-data" title="local-data">
               <span class="label" tagx="label" title="label">Table S2</span>
               <caption>
                  <p>
                     <b>Interpro annotation of proteins identified in MudPIT analysis.</b>
                  </p>
                  <p>(XLS)</p>
               </caption>
               <div class="media" title="media">
                  <a href="pone.0032990.s002.xls">LINK</a>
                  <caption>
                     <p>Click here for additional data file.</p>
                  </caption>
               </div>
            </div>
            <div class="local-data" title="local-data">
               <span class="label" tagx="label" title="label">Table S3</span>
               <caption>
                  <p>
                     <b>KEGG annotations of proteins identified in MudPIT analysis.</b>
                  </p>
                  <p>(XLS)</p>
               </caption>
               <div class="media" title="media">
                  <a href="pone.0032990.s003.xls">LINK</a>
                  <caption>
                     <p>Click here for additional data file.</p>
                  </caption>
               </div>
            </div>
            <div class="local-data" title="local-data">
               <span class="label" tagx="label" title="label">Table S4</span>
               <caption>
                  <p>
                     <b>Gene Ontology (GO) annotations of proteins identified in MudPIT analysis.</b>
                  </p>
                  <p>(XLS)</p>
               </caption>
               <div class="media" title="media">
                  <a href="pone.0032990.s004.xls">LINK</a>
                  <caption>
                     <p>Click here for additional data file.</p>
                  </caption>
               </div>
            </div>
            <div class="local-data" title="local-data">
               <span class="label" tagx="label" title="label">Table S5</span>
               <caption>
                  <p>
                     <b>SignalP signal peptide prediction of proteins identified in MudPIT analysis.</b>
                  </p>
                  <p>(XLS)</p>
               </caption>
               <div class="media" title="media">
                  <a href="pone.0032990.s005.xls">LINK</a>
                  <caption>
                     <p>Click here for additional data file.</p>
                  </caption>
               </div>
            </div>
            <div class="local-data" title="local-data">
               <span class="label" tagx="label" title="label">Table S6</span>
               <caption>
                  <p>
                     <b>Starvation and stress-related proteins from MudPIT analysis.</b>
                  </p>
                  <p>(XLS)</p>
               </caption>
               <div class="media" title="media">
                  <a href="pone.0032990.s006.xls">LINK</a>
                  <caption>
                     <p>Click here for additional data file.</p>
                  </caption>
               </div>
            </div>
            <div class="local-data" title="local-data">
               <span class="label" tagx="label" title="label">Table S7</span>
               <caption>
                  <p>
                     <b>Proteins related to Host plant interaction from MudPIT analysis.</b>
                  </p>
                  <p>(XLS)</p>
               </caption>
               <div class="media" title="media">
                  <a href="pone.0032990.s007.xls">LINK</a>
                  <caption>
                     <p>Click here for additional data file.</p>
                  </caption>
               </div>
            </div>
            <div class="local-data" title="local-data">
               <span class="label" tagx="label" title="label">Table S8</span>
               <caption>
                  <p>
                     <b>Hypothetical proteins detected from MudPIT data.</b>
                  </p>
                  <p>(XLS)</p>
               </caption>
               <div class="media" title="media">
                  <a href="pone.0032990.s008.xls">LINK</a>
                  <caption>
                     <p>Click here for additional data file.</p>
                  </caption>
               </div>
            </div>
         </div>
      </div>
      <div class="back" title="back">
         <div class="ack" title="ack">
            <p>We thank A. and B. Stanley at the Penn State Hershey Medical Center Mass Spectrometry
               Core Research Facility for mass spectrometry analysis. Opinions, findings, conclusions
               or recommendations expressed in this publication are those of the authors and do not
               necessarily reflect the views of the USDA. USDA is an equal opportunity provider and
               employer.
            </p>
         </div>
         <div class="fn-group" title="fn-group">
            <div class="fn-type-conflict" title="conflict">
               <p>
                  <b>Competing Interests: </b>The authors have declared that no competing interests exist.
               </p>
            </div>
            <div class="fn-type-financial-disclosure" title="financial-disclosure">
               <p>
                  <b>Funding: </b>Funding for this project was provided by USDA-NRI-CRSEES grant 2008-35504-04464, USDA-NRI-CREES
                  grant 2009-35302-05286, the Alphawood Foundation, Chicago, Illinois, a Seed Grant
                  to Dr. Hoover from the Pennsylvania State University College of Agricultural Sciences.
                  Dr. Carlson was partially supported by World Class University Project R31-2009-000-20025-0
                  grant from the Ministry of Education, Science and Technology of South Korea. The funders
                  had no role in study design, data collection and analysis, decision to publish, or
                  preparation of the manuscript.
               </p>
            </div>
         </div>
         <div class="references">References</div>
         <div tag="ref-list">
            <ul>
               <div class="title" tagx="title" title="title">References</div>
               <li tag="ref">
                  <a name="pone.0032990-Dillon1"></a>
                  <span class="label" tagx="label" title="label">1</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Dillon</span>
                           <span class="given-names" tagx="given-names" title="given-names">RJ</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Dillon</span>
                           <span class="given-names" tagx="given-names" title="given-names">VM</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2004</span>
                     <span class="mixed-article-title" title="mixed-article-title">The gut bacteria of insects: Nonpathogenic interactions.</span>
                     <span class="source" tagx="source" title="source">Annual Review of Entomology</span>
                     <span class="volume" tagx="volume" title="volume">49</span>
                     <span class="fpage" tagx="fpage" title="fpage">71</span>
                     <span class="lpage" tagx="lpage" title="lpage">92</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Kukor1"></a>
                  <span class="label" tagx="label" title="label">2</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kukor</span>
                           <span class="given-names" tagx="given-names" title="given-names">JJ</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Cowan</span>
                           <span class="given-names" tagx="given-names" title="given-names">DP</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Martin</span>
                           <span class="given-names" tagx="given-names" title="given-names">MM</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1988</span>
                     <span class="mixed-article-title" title="mixed-article-title">The role of ingested fungal enzymes in cellulose digestion in the larvae of cerambycid
                        beetles.
                     </span>
                     <span class="source" tagx="source" title="source">Physiological Zoology</span>
                     <span class="volume" tagx="volume" title="volume">61</span>
                     <span class="fpage" tagx="fpage" title="fpage">364</span>
                     <span class="lpage" tagx="lpage" title="lpage">371</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Dowd1"></a>
                  <span class="label" tagx="label" title="label">3</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Dowd</span>
                           <span class="given-names" tagx="given-names" title="given-names">PF</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1992</span>
                     <span class="mixed-article-title" title="mixed-article-title">Insect fungal symbionts - A promising source of detoxifying enzymes.</span>
                     <span class="source" tagx="source" title="source">Journal of Industrial Microbiology</span>
                     <span class="volume" tagx="volume" title="volume">9</span>
                     <span class="fpage" tagx="fpage" title="fpage">149</span>
                     <span class="lpage" tagx="lpage" title="lpage">161</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Hanks1"></a>
                  <span class="label" tagx="label" title="label">4</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hanks</span>
                           <span class="given-names" tagx="given-names" title="given-names">LM</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1999</span>
                     <span class="mixed-article-title" title="mixed-article-title">Influence of the larval host plant on reproductive strategies of cerambycid beetles.</span>
                     <span class="source" tagx="source" title="source">Annual Review of Entomology</span>
                     <span class="volume" tagx="volume" title="volume">44</span>
                     <span class="fpage" tagx="fpage" title="fpage">483</span>
                     <span class="lpage" tagx="lpage" title="lpage">505</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Lingafelter1"></a>
                  <span class="label" tagx="label" title="label">5</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Lingafelter</span>
                           <span class="given-names" tagx="given-names" title="given-names">SW</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hoebke</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">ER</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2002</span>
                     <span class="source" tagx="source" title="source">Revision of 
                        <i>Anoplophora</i> (Coleoptera: Cerambycidae)
                     </span>
                     <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Washington, DC</span>
                     <span class="publisher-name" tagx="publisher-name" title="publisher-name">Entomological Society of Washington</span>
                     <span class="size" title="size">size: 236page</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Nowak1"></a>
                  <span class="label" tagx="label" title="label">6</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Nowak</span>
                           <span class="given-names" tagx="given-names" title="given-names">DJ</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Pasek</span>
                           <span class="given-names" tagx="given-names" title="given-names">JE</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Sequeira</span>
                           <span class="given-names" tagx="given-names" title="given-names">RA</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Crane</span>
                           <span class="given-names" tagx="given-names" title="given-names">DE</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Mastro</span>
                           <span class="given-names" tagx="given-names" title="given-names">VC</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2001</span>
                     <span class="mixed-article-title" title="mixed-article-title">Potential effect of 
                        <i>Anoplophora glabripennis</i> (Coleoptera: Cerambycidae) on urban trees in the United States.
                     </span>
                     <span class="source" tagx="source" title="source">Journal of Economic Entomology</span>
                     <span class="volume" tagx="volume" title="volume">94</span>
                     <span class="fpage" tagx="fpage" title="fpage">116</span>
                     <span class="lpage" tagx="lpage" title="lpage">122</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/11233100">11233100</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Hu1"></a>
                  <span class="label" tagx="label" title="label">7</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hu</span>
                           <span class="given-names" tagx="given-names" title="given-names">JF</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Angeli</span>
                           <span class="given-names" tagx="given-names" title="given-names">S</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Schuetz</span>
                           <span class="given-names" tagx="given-names" title="given-names">S</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Luo</span>
                           <span class="given-names" tagx="given-names" title="given-names">YQ</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hajek</span>
                           <span class="given-names" tagx="given-names" title="given-names">AE</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2009</span>
                     <span class="mixed-article-title" title="mixed-article-title">Ecology and management of exotic and endemic Asian longhorned beetle 
                        <i>Anoplophora glabripennis</i>.
                     </span>
                     <span class="source" tagx="source" title="source">Agricultural and Forest Entomology</span>
                     <span class="volume" tagx="volume" title="volume">11</span>
                     <span class="fpage" tagx="fpage" title="fpage">359</span>
                     <span class="lpage" tagx="lpage" title="lpage">375</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-OSullivan1"></a>
                  <span class="label" tagx="label" title="label">8</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">OSullivan</span>
                           <span class="given-names" tagx="given-names" title="given-names">AC</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1997</span>
                     <span class="mixed-article-title" title="mixed-article-title">Cellulose: the structure slowly unravels.</span>
                     <span class="source" tagx="source" title="source">Cellulose</span>
                     <span class="volume" tagx="volume" title="volume">4</span>
                     <span class="fpage" tagx="fpage" title="fpage">173</span>
                     <span class="lpage" tagx="lpage" title="lpage">207</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Pettersen1"></a>
                  <span class="label" tagx="label" title="label">9</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Pettersen</span>
                           <span class="given-names" tagx="given-names" title="given-names">RC</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1984</span>
                     <span class="source" tagx="source" title="source">The Chemical Composition of Wood</span>
                     <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">The Chemistry of Solid Wood</span>
                     <span class="publisher-name" tagx="publisher-name" title="publisher-name">American Chemical Society</span>
                     <span class="fpage" tagx="fpage" title="fpage">57</span>
                     <span class="lpage" tagx="lpage" title="lpage">126</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Kirk1"></a>
                  <span class="label" tagx="label" title="label">10</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kirk</span>
                           <span class="given-names" tagx="given-names" title="given-names">TK</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Farrell</span>
                           <span class="given-names" tagx="given-names" title="given-names">RL</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1987</span>
                     <span class="mixed-article-title" title="mixed-article-title">Enzymatic combustion - The microbial degradation of lignin.</span>
                     <span class="source" tagx="source" title="source">Annual Review of Microbiology</span>
                     <span class="volume" tagx="volume" title="volume">41</span>
                     <span class="fpage" tagx="fpage" title="fpage">465</span>
                     <span class="lpage" tagx="lpage" title="lpage">505</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Keller1"></a>
                  <span class="label" tagx="label" title="label">11</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Keller</span>
                           <span class="given-names" tagx="given-names" title="given-names">B</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Templeton</span>
                           <span class="given-names" tagx="given-names" title="given-names">MD</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Lamb</span>
                           <span class="given-names" tagx="given-names" title="given-names">CJ</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1989</span>
                     <span class="mixed-article-title" title="mixed-article-title">Specific localization of a plant cell wall glycine-rich protein in protoxylem cells
                        of the vascular system.
                     </span>
                     <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span>
                     <span class="volume" tagx="volume" title="volume">86</span>
                     <span class="fpage" tagx="fpage" title="fpage">1529</span>
                     <span class="lpage" tagx="lpage" title="lpage">1533</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/16578841">16578841</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Ke1"></a>
                  <span class="label" tagx="label" title="label">12</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Ke</span>
                           <span class="given-names" tagx="given-names" title="given-names">J</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Singh</span>
                           <span class="given-names" tagx="given-names" title="given-names">D</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Chen</span>
                           <span class="given-names" tagx="given-names" title="given-names">SL</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2011</span>
                     <span class="mixed-article-title" title="mixed-article-title">Aromatic compound degradation by the wood-feeding termite 
                        <i>Coptotermes formosanus</i> (Shiraki).
                     </span>
                     <span class="source" tagx="source" title="source">International Biodeterioration &amp; Biodegradation</span>
                     <span class="volume" tagx="volume" title="volume">65</span>
                     <span class="fpage" tagx="fpage" title="fpage">744</span>
                     <span class="lpage" tagx="lpage" title="lpage">756</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Martinez1"></a>
                  <span class="label" tagx="label" title="label">13</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Martinez</span>
                           <span class="given-names" tagx="given-names" title="given-names">D</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Larrondo</span>
                           <span class="given-names" tagx="given-names" title="given-names">LF</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Putnam</span>
                           <span class="given-names" tagx="given-names" title="given-names">N</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Gelpke</span>
                           <span class="given-names" tagx="given-names" title="given-names">MDS</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Huang</span>
                           <span class="given-names" tagx="given-names" title="given-names">K</span>
                        </span>
                        <span class="etal" title="etal">
                           <i>et al.</i>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2004</span>
                     <span class="mixed-article-title" title="mixed-article-title">Genome sequence of the lignocellulose degrading fungus 
                        <i>Phanerochaete chrysosporium</i> strain RP78 (vol 22, pg 695, 2004).
                     </span>
                     <span class="source" tagx="source" title="source">Nature Biotechnology</span>
                     <span class="volume" tagx="volume" title="volume">22</span>
                     <span class="fpage" tagx="fpage" title="fpage">899</span>
                     <span class="lpage" tagx="lpage" title="lpage">899</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Camarero1"></a>
                  <span class="label" tagx="label" title="label">14</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Camarero</span>
                           <span class="given-names" tagx="given-names" title="given-names">S</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Sarkar</span>
                           <span class="given-names" tagx="given-names" title="given-names">S</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Ruiz-Duenas</span>
                           <span class="given-names" tagx="given-names" title="given-names">FJ</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Martinez</span>
                           <span class="given-names" tagx="given-names" title="given-names">MJ</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Martinez</span>
                           <span class="given-names" tagx="given-names" title="given-names">AT</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1999</span>
                     <span class="mixed-article-title" title="mixed-article-title">Description of a versatile peroxidase involved in the natural degradation of lignin
                        that has both manganese peroxidase and lignin peroxidase substrate interaction sites.
                     </span>
                     <span class="source" tagx="source" title="source">The Journal of Biological Chemistry</span>
                     <span class="volume" tagx="volume" title="volume">274</span>
                     <span class="fpage" tagx="fpage" title="fpage">6</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Morozova1"></a>
                  <span class="label" tagx="label" title="label">15</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Morozova</span>
                           <span class="given-names" tagx="given-names" title="given-names">OV</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Shumakovich</span>
                           <span class="given-names" tagx="given-names" title="given-names">GP</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Shleev</span>
                           <span class="given-names" tagx="given-names" title="given-names">SV</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Yaropolov</span>
                           <span class="given-names" tagx="given-names" title="given-names">YI</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2007</span>
                     <span class="mixed-article-title" title="mixed-article-title">Laccase-mediator systems and their applications: A review.</span>
                     <span class="source" tagx="source" title="source">Applied Biochemistry and Microbiology</span>
                     <span class="volume" tagx="volume" title="volume">43</span>
                     <span class="fpage" tagx="fpage" title="fpage">12</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Regalado1"></a>
                  <span class="label" tagx="label" title="label">16</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Regalado</span>
                           <span class="given-names" tagx="given-names" title="given-names">V</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Perestelo</span>
                           <span class="given-names" tagx="given-names" title="given-names">F</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Rodriguez</span>
                           <span class="given-names" tagx="given-names" title="given-names">A</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Carnicero</span>
                           <span class="given-names" tagx="given-names" title="given-names">A</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Sosa</span>
                           <span class="given-names" tagx="given-names" title="given-names">FJ</span>
                        </span>
                        <span class="etal" title="etal">
                           <i>et al.</i>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1999</span>
                     <span class="mixed-article-title" title="mixed-article-title">Activated oxygen species and two extracellular enzymes: laccase and aryl-alcohol oxidase,
                        novel for the lignin-degrading fungus 
                        <i>Fusarium proliferatum</i>.
                     </span>
                     <span class="source" tagx="source" title="source">Applied Microbiology and Biotechnology</span>
                     <span class="volume" tagx="volume" title="volume">51</span>
                     <span class="fpage" tagx="fpage" title="fpage">388</span>
                     <span class="lpage" tagx="lpage" title="lpage">390</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Harazono1"></a>
                  <span class="label" tagx="label" title="label">17</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Harazono</span>
                           <span class="given-names" tagx="given-names" title="given-names">K</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Yamashita</span>
                           <span class="given-names" tagx="given-names" title="given-names">N</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Shinzato</span>
                           <span class="given-names" tagx="given-names" title="given-names">N</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Watanabe</span>
                           <span class="given-names" tagx="given-names" title="given-names">Y</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Fukatsu</span>
                           <span class="given-names" tagx="given-names" title="given-names">T</span>
                        </span>
                        <span class="etal" title="etal">
                           <i>et al.</i>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2003</span>
                     <span class="mixed-article-title" title="mixed-article-title">Isolation and characterization of aromatics-degrading microorganisms from the gut
                        of the lower termite 
                        <i>Coptotermes formosanus</i>.
                     </span>
                     <span class="source" tagx="source" title="source">Bioscience, Biotechnology, and Biochemistry</span>
                     <span class="volume" tagx="volume" title="volume">67</span>
                     <span class="fpage" tagx="fpage" title="fpage">889</span>
                     <span class="lpage" tagx="lpage" title="lpage">892</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Bugg1"></a>
                  <span class="label" tagx="label" title="label">18</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Bugg</span>
                           <span class="given-names" tagx="given-names" title="given-names">TDH</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Ahmad</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hardiman</span>
                           <span class="given-names" tagx="given-names" title="given-names">EM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Singh</span>
                           <span class="given-names" tagx="given-names" title="given-names">R</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2011</span>
                     <span class="mixed-article-title" title="mixed-article-title">The emerging role for bacteria in lignin degradation and bio-product formation.</span>
                     <span class="source" tagx="source" title="source">Current Opinion in Biotechnology</span>
                     <span class="volume" tagx="volume" title="volume">22</span>
                     <span class="fpage" tagx="fpage" title="fpage">394</span>
                     <span class="lpage" tagx="lpage" title="lpage">400</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/21071202">21071202</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Kajikawa1"></a>
                  <span class="label" tagx="label" title="label">19</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kajikawa</span>
                           <span class="given-names" tagx="given-names" title="given-names">H</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kudo</span>
                           <span class="given-names" tagx="given-names" title="given-names">H</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kondo</span>
                           <span class="given-names" tagx="given-names" title="given-names">T</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Jodai</span>
                           <span class="given-names" tagx="given-names" title="given-names">K</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Honda</span>
                           <span class="given-names" tagx="given-names" title="given-names">Y</span>
                        </span>
                        <span class="etal" title="etal">
                           <i>et al.</i>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2000</span>
                     <span class="mixed-article-title" title="mixed-article-title">Degradation of benzyl ether bonds of lignin by ruminal microbes.</span>
                     <span class="source" tagx="source" title="source">Fems Microbiology Letters</span>
                     <span class="volume" tagx="volume" title="volume">187</span>
                     <span class="fpage" tagx="fpage" title="fpage">15</span>
                     <span class="lpage" tagx="lpage" title="lpage">20</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/10828393">10828393</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Kato1"></a>
                  <span class="label" tagx="label" title="label">20</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kato</span>
                           <span class="given-names" tagx="given-names" title="given-names">K</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kozaki</span>
                           <span class="given-names" tagx="given-names" title="given-names">S</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Sakuranaga</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1998</span>
                     <span class="mixed-article-title" title="mixed-article-title">Degradation of lignin compounds by bacteria from termite guts.</span>
                     <span class="source" tagx="source" title="source">Biotechnology Letters</span>
                     <span class="volume" tagx="volume" title="volume">20</span>
                     <span class="fpage" tagx="fpage" title="fpage">459</span>
                     <span class="lpage" tagx="lpage" title="lpage">462</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Shary1"></a>
                  <span class="label" tagx="label" title="label">21</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Shary</span>
                           <span class="given-names" tagx="given-names" title="given-names">S</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Ralph</span>
                           <span class="given-names" tagx="given-names" title="given-names">SA</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hammel</span>
                           <span class="given-names" tagx="given-names" title="given-names">KE</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2007</span>
                     <span class="mixed-article-title" title="mixed-article-title">New insights into the ligninolytic capability of a wood decay ascomycete.</span>
                     <span class="source" tagx="source" title="source">Applied and Environmental Microbiology</span>
                     <span class="volume" tagx="volume" title="volume">73</span>
                     <span class="fpage" tagx="fpage" title="fpage">6691</span>
                     <span class="lpage" tagx="lpage" title="lpage">6694</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/17766457">17766457</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Geib1"></a>
                  <span class="label" tagx="label" title="label">22</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Geib</span>
                           <span class="given-names" tagx="given-names" title="given-names">SM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Filley</span>
                           <span class="given-names" tagx="given-names" title="given-names">TR</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hatcher</span>
                           <span class="given-names" tagx="given-names" title="given-names">PG</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hoover</span>
                           <span class="given-names" tagx="given-names" title="given-names">K</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Carlson</span>
                           <span class="given-names" tagx="given-names" title="given-names">JE</span>
                        </span>
                        <span class="etal" title="etal">
                           <i>et al.</i>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2008</span>
                     <span class="mixed-article-title" title="mixed-article-title">Lignin degradation in wood-feeding insects.</span>
                     <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span>
                     <span class="volume" tagx="volume" title="volume">105</span>
                     <span class="fpage" tagx="fpage" title="fpage">12932</span>
                     <span class="lpage" tagx="lpage" title="lpage">12937</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/18725643">18725643</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Geib2"></a>
                  <span class="label" tagx="label" title="label">23</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Geib</span>
                           <span class="given-names" tagx="given-names" title="given-names">SM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Jimenez-Gasco</span>
                           <span class="given-names" tagx="given-names" title="given-names">MM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Carlson</span>
                           <span class="given-names" tagx="given-names" title="given-names">JE</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Tien</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hoover</span>
                           <span class="given-names" tagx="given-names" title="given-names">K</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2009</span>
                     <span class="mixed-article-title" title="mixed-article-title">Effect of host tree species on cellulase activity and bacterial community composition
                        in the gut of larval Asian longhorned beetle.
                     </span>
                     <span class="source" tagx="source" title="source">Environ Entomol</span>
                     <span class="volume" tagx="volume" title="volume">38</span>
                     <span class="fpage" tagx="fpage" title="fpage">686</span>
                     <span class="lpage" tagx="lpage" title="lpage">699</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/19508777">19508777</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Geib3"></a>
                  <span class="label" tagx="label" title="label">24</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Geib</span>
                           <span class="given-names" tagx="given-names" title="given-names">SM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Tien</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hoover</span>
                           <span class="given-names" tagx="given-names" title="given-names">K</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2010</span>
                     <span class="mixed-article-title" title="mixed-article-title">Identification of proteins involved in lignocellulose degradation using in gel zymogram
                        analysis combined with mass spectroscopy-based peptide analysis of gut proteins from
                        larval Asian longhorned beetles, 
                        <i>Anoplophora glabripennis</i>.
                     </span>
                     <span class="source" tagx="source" title="source">Insect Science</span>
                     <span class="volume" tagx="volume" title="volume">17</span>
                     <span class="fpage" tagx="fpage" title="fpage">253</span>
                     <span class="lpage" tagx="lpage" title="lpage">264</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Pauchet1"></a>
                  <span class="label" tagx="label" title="label">25</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Pauchet</span>
                           <span class="given-names" tagx="given-names" title="given-names">Y</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Wilkinson</span>
                           <span class="given-names" tagx="given-names" title="given-names">P</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Chauhan</span>
                           <span class="given-names" tagx="given-names" title="given-names">R</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">French-Constant</span>
                           <span class="given-names" tagx="given-names" title="given-names">RH</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2010</span>
                     <span class="mixed-article-title" title="mixed-article-title">Diversity of beetle genes encoding novel plant cell wall degrading enzymes.</span>
                     <span class="source" tagx="source" title="source">PLoS One</span>
                     <span class="volume" tagx="volume" title="volume">5</span>
                     <span class="fpage" tagx="fpage" title="fpage">-</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Pauchet2"></a>
                  <span class="label" tagx="label" title="label">26</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Pauchet</span>
                           <span class="given-names" tagx="given-names" title="given-names">Y</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Wilkinson</span>
                           <span class="given-names" tagx="given-names" title="given-names">P</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">van Munster</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Augustin</span>
                           <span class="given-names" tagx="given-names" title="given-names">S</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Pauron</span>
                           <span class="given-names" tagx="given-names" title="given-names">D</span>
                        </span>
                        <span class="etal" title="etal">
                           <i>et al.</i>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2009</span>
                     <span class="mixed-article-title" title="mixed-article-title">Pyrosequencing of the midgut transcriptome of the poplar leaf beetle 
                        <i>Chrysomela tremulae</i> reveals new gene families in Coleoptera.
                     </span>
                     <span class="source" tagx="source" title="source">Insect Biochemistry and Molecular Biology</span>
                     <span class="volume" tagx="volume" title="volume">39</span>
                     <span class="fpage" tagx="fpage" title="fpage">403</span>
                     <span class="lpage" tagx="lpage" title="lpage">413</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/19364528">19364528</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Watanabe1"></a>
                  <span class="label" tagx="label" title="label">27</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Watanabe</span>
                           <span class="given-names" tagx="given-names" title="given-names">H</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Tokuda</span>
                           <span class="given-names" tagx="given-names" title="given-names">G</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2009</span>
                     <span class="mixed-article-title" title="mixed-article-title">Cellulolytic systems in insects.</span>
                     <span class="source" tagx="source" title="source">Annu Rev Entomol</span>
                     <span class="volume" tagx="volume" title="volume">55</span>
                     <span class="fpage" tagx="fpage" title="fpage">23</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Willis1"></a>
                  <span class="label" tagx="label" title="label">28</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Willis</span>
                           <span class="given-names" tagx="given-names" title="given-names">JD</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Oppert</span>
                           <span class="given-names" tagx="given-names" title="given-names">C</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Jurat-Fuentes</span>
                           <span class="given-names" tagx="given-names" title="given-names">JL</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2010</span>
                     <span class="mixed-article-title" title="mixed-article-title">Methods for discovery and characterization of cellulolytic enzymes from insects.</span>
                     <span class="source" tagx="source" title="source">Insect Science</span>
                     <span class="volume" tagx="volume" title="volume">17</span>
                     <span class="fpage" tagx="fpage" title="fpage">184</span>
                     <span class="lpage" tagx="lpage" title="lpage">198</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Davison1"></a>
                  <span class="label" tagx="label" title="label">29</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Davison</span>
                           <span class="given-names" tagx="given-names" title="given-names">A</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Blaxter</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2005</span>
                     <span class="mixed-article-title" title="mixed-article-title">Ancient origin of glycosyl hydrolase family 9 cellulase genes.</span>
                     <span class="source" tagx="source" title="source">Molecular Biology and Evolution</span>
                     <span class="volume" tagx="volume" title="volume">22</span>
                     <span class="fpage" tagx="fpage" title="fpage">1273</span>
                     <span class="lpage" tagx="lpage" title="lpage">1284</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/15703240">15703240</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-CalderonCortes1"></a>
                  <span class="label" tagx="label" title="label">30</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Calderon-Cortes</span>
                           <span class="given-names" tagx="given-names" title="given-names">N</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Watanabe</span>
                           <span class="given-names" tagx="given-names" title="given-names">H</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Cano-Camacho</span>
                           <span class="given-names" tagx="given-names" title="given-names">H</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Zavala-Paramo</span>
                           <span class="given-names" tagx="given-names" title="given-names">G</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Quesada</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2010</span>
                     <span class="mixed-article-title" title="mixed-article-title">cDNA cloning, homology modelling and evolutionary insights into novel endogenous cellulases
                        of the borer beetle 
                        <i>Oncideres albomarginata chamela</i> (Cerambycidae).
                     </span>
                     <span class="source" tagx="source" title="source">Insect Molecular Biology</span>
                     <span class="volume" tagx="volume" title="volume">19</span>
                     <span class="fpage" tagx="fpage" title="fpage">323</span>
                     <span class="lpage" tagx="lpage" title="lpage">336</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/20201981">20201981</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Zhang1"></a>
                  <span class="label" tagx="label" title="label">31</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Zhang</span>
                           <span class="given-names" tagx="given-names" title="given-names">DH</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Lax</span>
                           <span class="given-names" tagx="given-names" title="given-names">AR</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Bland</span>
                           <span class="given-names" tagx="given-names" title="given-names">JM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Allen</span>
                           <span class="given-names" tagx="given-names" title="given-names">AB</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2011</span>
                     <span class="mixed-article-title" title="mixed-article-title">Characterization of a new endogenous endo-beta-1,4-glucanase of Formosan subterranean
                        termite (
                        <i>Coptotermes formosanus</i>).
                     </span>
                     <span class="source" tagx="source" title="source">Insect Biochemistry and Molecular Biology</span>
                     <span class="volume" tagx="volume" title="volume">41</span>
                     <span class="fpage" tagx="fpage" title="fpage">211</span>
                     <span class="lpage" tagx="lpage" title="lpage">218</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/21195179">21195179</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Sun1"></a>
                  <span class="label" tagx="label" title="label">32</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Sun</span>
                           <span class="given-names" tagx="given-names" title="given-names">JZ</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Scharf</span>
                           <span class="given-names" tagx="given-names" title="given-names">ME</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2010</span>
                     <span class="mixed-article-title" title="mixed-article-title">Exploring and integrating cellulolytic systems of insects to advance biofuel technology
                        PREFACE.
                     </span>
                     <span class="source" tagx="source" title="source">Insect Science</span>
                     <span class="volume" tagx="volume" title="volume">17</span>
                     <span class="fpage" tagx="fpage" title="fpage">163</span>
                     <span class="lpage" tagx="lpage" title="lpage">165</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Coy1"></a>
                  <span class="label" tagx="label" title="label">33</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Coy</span>
                           <span class="given-names" tagx="given-names" title="given-names">MR</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Salem</span>
                           <span class="given-names" tagx="given-names" title="given-names">TZ</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Denton</span>
                           <span class="given-names" tagx="given-names" title="given-names">JS</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kovaleva</span>
                           <span class="given-names" tagx="given-names" title="given-names">ES</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Liu</span>
                           <span class="given-names" tagx="given-names" title="given-names">Z</span>
                        </span>
                        <span class="etal" title="etal">
                           <i>et al.</i>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2010</span>
                     <span class="mixed-article-title" title="mixed-article-title">Phenol-oxidizing laccases from the termite gut.</span>
                     <span class="source" tagx="source" title="source">Insect Biochemistry and Molecular Biology</span>
                     <span class="volume" tagx="volume" title="volume">40</span>
                     <span class="fpage" tagx="fpage" title="fpage">723</span>
                     <span class="lpage" tagx="lpage" title="lpage">732</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/20691784">20691784</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Scharf1"></a>
                  <span class="label" tagx="label" title="label">34</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Scharf</span>
                           <span class="given-names" tagx="given-names" title="given-names">ME</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Tartar</span>
                           <span class="given-names" tagx="given-names" title="given-names">A</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2008</span>
                     <span class="mixed-article-title" title="mixed-article-title">Termite digestomes as sources for novel lignocellulases.</span>
                     <span class="source" tagx="source" title="source">Biofuels Bioproducts &amp; Biorefining-Biofpr</span>
                     <span class="volume" tagx="volume" title="volume">2</span>
                     <span class="fpage" tagx="fpage" title="fpage">540</span>
                     <span class="lpage" tagx="lpage" title="lpage">552</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Geib4"></a>
                  <span class="label" tagx="label" title="label">35</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Geib</span>
                           <span class="given-names" tagx="given-names" title="given-names">SM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Scully</span>
                           <span class="given-names" tagx="given-names" title="given-names">ED</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Jimenez-Gasco</span>
                           <span class="given-names" tagx="given-names" title="given-names">MM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Carlson</span>
                           <span class="given-names" tagx="given-names" title="given-names">JE</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Tien</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="etal" title="etal">
                           <i>et al.</i>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2012</span>
                     <span class="mixed-article-title" title="mixed-article-title">Phylogenetic analysis of 
                        <i>Fusarium solani</i> associated with the Asian longhorned beetle, 
                        <i>Anoplophora glabripennis</i>.
                     </span>
                     <span class="source" tagx="source" title="source">Insects</span>
                     <span class="volume" tagx="volume" title="volume">3</span>
                     <span class="fpage" tagx="fpage" title="fpage">141</span>
                     <span class="lpage" tagx="lpage" title="lpage">160</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Teetorbarsch1"></a>
                  <span class="label" tagx="label" title="label">36</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Teetorbarsch</span>
                           <span class="given-names" tagx="given-names" title="given-names">GH</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Roberts</span>
                           <span class="given-names" tagx="given-names" title="given-names">DW</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1983</span>
                     <span class="mixed-article-title" title="mixed-article-title">Entomogenous 
                        <i>Fusarium</i> species.
                     </span>
                     <span class="source" tagx="source" title="source">Mycopathologia</span>
                     <span class="volume" tagx="volume" title="volume">84</span>
                     <span class="fpage" tagx="fpage" title="fpage">3</span>
                     <span class="lpage" tagx="lpage" title="lpage">16</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/6369143">6369143</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-MoralesRamos1"></a>
                  <span class="label" tagx="label" title="label">37</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Morales-Ramos</span>
                           <span class="given-names" tagx="given-names" title="given-names">JA</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Rojas</span>
                           <span class="given-names" tagx="given-names" title="given-names">MG</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Sittertz-Bhatkar</span>
                           <span class="given-names" tagx="given-names" title="given-names">H</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Saldana</span>
                           <span class="given-names" tagx="given-names" title="given-names">G</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2000</span>
                     <span class="mixed-article-title" title="mixed-article-title">Symbiotic relationship between 
                        <i>Hypothenemus hampei</i> (Coleoptera: Scolytidae) and 
                        <i>Fusarium solani</i> (Moniliales: Tuberculariaceae).
                     </span>
                     <span class="source" tagx="source" title="source">Annals of the Entomological Society of America</span>
                     <span class="volume" tagx="volume" title="volume">93</span>
                     <span class="fpage" tagx="fpage" title="fpage">541</span>
                     <span class="lpage" tagx="lpage" title="lpage">547</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Romero1"></a>
                  <span class="label" tagx="label" title="label">38</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Romero</span>
                           <span class="given-names" tagx="given-names" title="given-names">MC</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Salvioli</span>
                           <span class="given-names" tagx="given-names" title="given-names">ML</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Cazau</span>
                           <span class="given-names" tagx="given-names" title="given-names">MC</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Arambarri</span>
                           <span class="given-names" tagx="given-names" title="given-names">AM</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2002</span>
                     <span class="mixed-article-title" title="mixed-article-title">Pyrene degradation by yeasts and filamentous fungi.</span>
                     <span class="source" tagx="source" title="source">Environmental Pollution</span>
                     <span class="volume" tagx="volume" title="volume">117</span>
                     <span class="fpage" tagx="fpage" title="fpage">159</span>
                     <span class="lpage" tagx="lpage" title="lpage">163</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/11843531">11843531</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Veignie1"></a>
                  <span class="label" tagx="label" title="label">39</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Veignie</span>
                           <span class="given-names" tagx="given-names" title="given-names">E</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Rafin</span>
                           <span class="given-names" tagx="given-names" title="given-names">C</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Woisel</span>
                           <span class="given-names" tagx="given-names" title="given-names">P</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Cazier</span>
                           <span class="given-names" tagx="given-names" title="given-names">F</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2004</span>
                     <span class="mixed-article-title" title="mixed-article-title">Preliminary evidence of the role of hydrogen peroxide in the degradation of benzo[a]pyrene
                        by a non-white rot fungus 
                        <i>Fusarium solani</i>.
                     </span>
                     <span class="source" tagx="source" title="source">Environmental Pollution</span>
                     <span class="volume" tagx="volume" title="volume">129</span>
                     <span class="fpage" tagx="fpage" title="fpage">1</span>
                     <span class="lpage" tagx="lpage" title="lpage">4</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/14749063">14749063</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Colombo1"></a>
                  <span class="label" tagx="label" title="label">40</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Colombo</span>
                           <span class="given-names" tagx="given-names" title="given-names">JC</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Cabello</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Arambarri</span>
                           <span class="given-names" tagx="given-names" title="given-names">AM</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1996</span>
                     <span class="mixed-article-title" title="mixed-article-title">Biodegradation of aliphatic and aromatic hydrocarbons by natural soil microflora and
                        pure cultures of imperfect and lignolitic fungi.
                     </span>
                     <span class="source" tagx="source" title="source">Environmental Pollution</span>
                     <span class="volume" tagx="volume" title="volume">94</span>
                     <span class="fpage" tagx="fpage" title="fpage">355</span>
                     <span class="lpage" tagx="lpage" title="lpage">362</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/15093496">15093496</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Tien1"></a>
                  <span class="label" tagx="label" title="label">41</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Tien</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kirk</span>
                           <span class="given-names" tagx="given-names" title="given-names">TK</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1984</span>
                     <span class="mixed-article-title" title="mixed-article-title">Lignin-degrading enzyme from 
                        <i>Phanerochaete chrysosporium</i>- purification, characterization, and catalytic properties of a unique H
                        <sub>2</sub>O
                        <sub>2</sub>-requiring oxygenase.
                     </span>
                     <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span>
                     <span class="volume" tagx="volume" title="volume">81</span>
                     <span class="fpage" tagx="fpage" title="fpage">2280</span>
                     <span class="lpage" tagx="lpage" title="lpage">2284</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/16593451">16593451</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Zaremaivan1"></a>
                  <span class="label" tagx="label" title="label">42</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Zaremaivan</span>
                           <span class="given-names" tagx="given-names" title="given-names">H</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Shearer</span>
                           <span class="given-names" tagx="given-names" title="given-names">CA</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1988</span>
                     <span class="mixed-article-title" title="mixed-article-title">Extracellular enzyme-production and cell wall degradation by fresh-water lignicolous
                        fungi.
                     </span>
                     <span class="source" tagx="source" title="source">Mycologia</span>
                     <span class="volume" tagx="volume" title="volume">80</span>
                     <span class="fpage" tagx="fpage" title="fpage">365</span>
                     <span class="lpage" tagx="lpage" title="lpage">375</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Worrall1"></a>
                  <span class="label" tagx="label" title="label">43</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Worrall</span>
                           <span class="given-names" tagx="given-names" title="given-names">JJ</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Anagnost</span>
                           <span class="given-names" tagx="given-names" title="given-names">SE</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Zabel</span>
                           <span class="given-names" tagx="given-names" title="given-names">RA</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1997</span>
                     <span class="mixed-article-title" title="mixed-article-title">Comparison of wood decay among diverse lignicolous fungi.</span>
                     <span class="source" tagx="source" title="source">Mycologia</span>
                     <span class="volume" tagx="volume" title="volume">89</span>
                     <span class="fpage" tagx="fpage" title="fpage">199</span>
                     <span class="lpage" tagx="lpage" title="lpage">219</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Lozovaya1"></a>
                  <span class="label" tagx="label" title="label">44</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Lozovaya</span>
                           <span class="given-names" tagx="given-names" title="given-names">V</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Lygin</span>
                           <span class="given-names" tagx="given-names" title="given-names">A</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Zernova</span>
                           <span class="given-names" tagx="given-names" title="given-names">O</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Li</span>
                           <span class="given-names" tagx="given-names" title="given-names">S</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Widholm</span>
                           <span class="given-names" tagx="given-names" title="given-names">J</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2006</span>
                     <span class="mixed-article-title" title="mixed-article-title">Lignin degradation by 
                        <i>Fusarium solani</i> f. sp. 
                        <i>glycines</i>.
                     </span>
                     <span class="source" tagx="source" title="source">Plant Disease</span>
                     <span class="volume" tagx="volume" title="volume">90</span>
                     <span class="fpage" tagx="fpage" title="fpage">77</span>
                     <span class="lpage" tagx="lpage" title="lpage">82</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Sutherland1"></a>
                  <span class="label" tagx="label" title="label">45</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Sutherland</span>
                           <span class="given-names" tagx="given-names" title="given-names">JB</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Pometto</span>
                           <span class="given-names" tagx="given-names" title="given-names">AL</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Crawford</span>
                           <span class="given-names" tagx="given-names" title="given-names">DL</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1983</span>
                     <span class="mixed-article-title" title="mixed-article-title">Lignocellulose degradation by 
                        <i>Fusarium</i> species.
                     </span>
                     <span class="source" tagx="source" title="source">Canadian Journal of Botany-Revue Canadienne De Botanique</span>
                     <span class="volume" tagx="volume" title="volume">61</span>
                     <span class="fpage" tagx="fpage" title="fpage">1194</span>
                     <span class="lpage" tagx="lpage" title="lpage">1198</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Rodriguez1"></a>
                  <span class="label" tagx="label" title="label">46</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Rodriguez</span>
                           <span class="given-names" tagx="given-names" title="given-names">A</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Perestelo</span>
                           <span class="given-names" tagx="given-names" title="given-names">F</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Carnicero</span>
                           <span class="given-names" tagx="given-names" title="given-names">A</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Regalado</span>
                           <span class="given-names" tagx="given-names" title="given-names">V</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Perez</span>
                           <span class="given-names" tagx="given-names" title="given-names">R</span>
                        </span>
                        <span class="etal" title="etal">
                           <i>et al.</i>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1996</span>
                     <span class="mixed-article-title" title="mixed-article-title">Degradation of natural lignins and lignocellulosic substrates by soil-inhabiting fungi
                        imperfecti.
                     </span>
                     <span class="source" tagx="source" title="source">Fems Microbiology Ecology</span>
                     <span class="volume" tagx="volume" title="volume">21</span>
                     <span class="fpage" tagx="fpage" title="fpage">213</span>
                     <span class="lpage" tagx="lpage" title="lpage">219</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Falcon1"></a>
                  <span class="label" tagx="label" title="label">47</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Falcon</span>
                           <span class="given-names" tagx="given-names" title="given-names">MA</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Rodriguez</span>
                           <span class="given-names" tagx="given-names" title="given-names">A</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Carnicero</span>
                           <span class="given-names" tagx="given-names" title="given-names">A</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Regalado</span>
                           <span class="given-names" tagx="given-names" title="given-names">V</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Perestelo</span>
                           <span class="given-names" tagx="given-names" title="given-names">F</span>
                        </span>
                        <span class="etal" title="etal">
                           <i>et al.</i>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1995</span>
                     <span class="mixed-article-title" title="mixed-article-title">Isolation of microorganisms with lignin transformation potential from soil of Tenerife
                        Island.
                     </span>
                     <span class="source" tagx="source" title="source">Soil Biology &amp; Biochemistry</span>
                     <span class="volume" tagx="volume" title="volume">27</span>
                     <span class="fpage" tagx="fpage" title="fpage">121</span>
                     <span class="lpage" tagx="lpage" title="lpage">126</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Coleman1"></a>
                  <span class="label" tagx="label" title="label">48</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Coleman</span>
                           <span class="given-names" tagx="given-names" title="given-names">JJ</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Rounsley</span>
                           <span class="given-names" tagx="given-names" title="given-names">SD</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Rodriguez-Carres</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kuo</span>
                           <span class="given-names" tagx="given-names" title="given-names">A</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Wasmann</span>
                           <span class="given-names" tagx="given-names" title="given-names">CC</span>
                        </span>
                        <span class="etal" title="etal">
                           <i>et al.</i>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2009</span>
                     <span class="mixed-article-title" title="mixed-article-title">The genome of 
                        <i>Nectria haematococca</i>: Contribution of supernumerary chromosomes to gene expansion.
                     </span>
                     <span class="source" tagx="source" title="source">PLoS Genet</span>
                     <span class="volume" tagx="volume" title="volume">5</span>
                     <span class="fpage" tagx="fpage" title="fpage">e1000618</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/19714214">19714214</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Varela1"></a>
                  <span class="label" tagx="label" title="label">49</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Varela</span>
                           <span class="given-names" tagx="given-names" title="given-names">E</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Mester</span>
                           <span class="given-names" tagx="given-names" title="given-names">T</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Tien</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2003</span>
                     <span class="mixed-article-title" title="mixed-article-title">Culture conditions affecting biodegradation components of the brown-rot fungus 
                        <i>Gloeophyllum trabeum</i>.
                     </span>
                     <span class="source" tagx="source" title="source">Archives of Microbiology</span>
                     <span class="volume" tagx="volume" title="volume">180</span>
                     <span class="fpage" tagx="fpage" title="fpage">251</span>
                     <span class="lpage" tagx="lpage" title="lpage">256</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/12920506">12920506</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Crawford1"></a>
                  <span class="label" tagx="label" title="label">50</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Crawford</span>
                           <span class="given-names" tagx="given-names" title="given-names">DL</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Crawford</span>
                           <span class="given-names" tagx="given-names" title="given-names">RL</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1980</span>
                     <span class="mixed-article-title" title="mixed-article-title">Microbial degradation of lignin.</span>
                     <span class="source" tagx="source" title="source">Enzyme and Microbial Technology</span>
                     <span class="volume" tagx="volume" title="volume">2</span>
                     <span class="fpage" tagx="fpage" title="fpage">11</span>
                     <span class="lpage" tagx="lpage" title="lpage">22</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Kinter1"></a>
                  <span class="label" tagx="label" title="label">51</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kinter</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Sherman</span>
                           <span class="given-names" tagx="given-names" title="given-names">NE</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2000</span>
                     <span class="source" tagx="source" title="source">Protein sequencing and identification using tandem mass spectrometry</span>
                     <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York</span>
                     <span class="publisher-name" tagx="publisher-name" title="publisher-name">John Wiley</span>
                     <span class="comment">xvi, 301 p.</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Bendtsen1"></a>
                  <span class="label" tagx="label" title="label">52</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Bendtsen</span>
                           <span class="given-names" tagx="given-names" title="given-names">JD</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Nielsen</span>
                           <span class="given-names" tagx="given-names" title="given-names">H</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">von Heijne</span>
                           <span class="given-names" tagx="given-names" title="given-names">G</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Brunak</span>
                           <span class="given-names" tagx="given-names" title="given-names">S</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2004</span>
                     <span class="mixed-article-title" title="mixed-article-title">Improved prediction of signal peptides: SignalP 3.0.</span>
                     <span class="source" tagx="source" title="source">Journal of Molecular Biology</span>
                     <span class="volume" tagx="volume" title="volume">340</span>
                     <span class="fpage" tagx="fpage" title="fpage">783</span>
                     <span class="lpage" tagx="lpage" title="lpage">795</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/15223320">15223320</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Bernfeld1"></a>
                  <span class="label" tagx="label" title="label">53</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Bernfeld</span>
                           <span class="given-names" tagx="given-names" title="given-names">P</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1955</span>
                     <span class="mixed-article-title" title="mixed-article-title">Amylases alpha and beta.</span>
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Colowick</span>
                           <span class="given-names" tagx="given-names" title="given-names">SP</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kaplan</span>
                           <span class="given-names" tagx="given-names" title="given-names">NO</span>
                        </span>
                     </span>
                     <span class="source" tagx="source" title="source">Methods in Enzymology</span>
                     <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York</span>
                     <span class="publisher-name" tagx="publisher-name" title="publisher-name">Academic Press</span>
                     <span class="fpage" tagx="fpage" title="fpage">149</span>
                     <span class="lpage" tagx="lpage" title="lpage">150</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Miller1"></a>
                  <span class="label" tagx="label" title="label">54</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Miller</span>
                           <span class="given-names" tagx="given-names" title="given-names">GL</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1959</span>
                     <span class="mixed-article-title" title="mixed-article-title">Use of dinitrosalicylic acid reagent for determination of reducing sugar.</span>
                     <span class="source" tagx="source" title="source">Analytical Chemistry</span>
                     <span class="volume" tagx="volume" title="volume">31</span>
                     <span class="fpage" tagx="fpage" title="fpage">426</span>
                     <span class="lpage" tagx="lpage" title="lpage">428</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Bollag1"></a>
                  <span class="label" tagx="label" title="label">55</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Bollag</span>
                           <span class="given-names" tagx="given-names" title="given-names">DM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Rozycki</span>
                           <span class="given-names" tagx="given-names" title="given-names">MD</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Edelstein</span>
                           <span class="given-names" tagx="given-names" title="given-names">SJ</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1996</span>
                     <span class="source" tagx="source" title="source">Protein methods</span>
                     <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York</span>
                     <span class="publisher-name" tagx="publisher-name" title="publisher-name">Wiley-Liss</span>
                     <span class="comment">xvi, 415 p.</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Bradford1"></a>
                  <span class="label" tagx="label" title="label">56</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Bradford</span>
                           <span class="given-names" tagx="given-names" title="given-names">MM</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1976</span>
                     <span class="mixed-article-title" title="mixed-article-title">A rapid and sensitive method for the quantitation of microgram quantities of protein
                        utilizing the principle of protein-dye binding.
                     </span>
                     <span class="source" tagx="source" title="source">Analytical Biochemistry</span>
                     <span class="volume" tagx="volume" title="volume">72</span>
                     <span class="fpage" tagx="fpage" title="fpage">248</span>
                     <span class="lpage" tagx="lpage" title="lpage">254</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/942051">942051</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Laemmli1"></a>
                  <span class="label" tagx="label" title="label">57</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Laemmli</span>
                           <span class="given-names" tagx="given-names" title="given-names">UK</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1970</span>
                     <span class="mixed-article-title" title="mixed-article-title">Cleavage of structural proteins during assembly of head of bacteriophage-T4.</span>
                     <span class="source" tagx="source" title="source">Nature</span>
                     <span class="volume" tagx="volume" title="volume">227</span>
                     <span class="fpage" tagx="fpage" title="fpage">680</span>
                     <span class="lpage" tagx="lpage" title="lpage">690</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/5432063">5432063</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Her1"></a>
                  <span class="label" tagx="label" title="label">58</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Her</span>
                           <span class="given-names" tagx="given-names" title="given-names">S</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Lee</span>
                           <span class="given-names" tagx="given-names" title="given-names">HS</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Choi</span>
                           <span class="given-names" tagx="given-names" title="given-names">SJ</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Choi</span>
                           <span class="given-names" tagx="given-names" title="given-names">SW</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Choi</span>
                           <span class="given-names" tagx="given-names" title="given-names">HJ</span>
                        </span>
                        <span class="etal" title="etal">
                           <i>et al.</i>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1999</span>
                     <span class="mixed-article-title" title="mixed-article-title">Cloning and sequencing of beta-1,4-endoglucanase gene (celA) from 
                        <i>Pseudomonas sp</i> YD-15.
                     </span>
                     <span class="source" tagx="source" title="source">Letters in Applied Microbiology</span>
                     <span class="volume" tagx="volume" title="volume">29</span>
                     <span class="fpage" tagx="fpage" title="fpage">389</span>
                     <span class="lpage" tagx="lpage" title="lpage">395</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/10664983">10664983</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Schwarz1"></a>
                  <span class="label" tagx="label" title="label">59</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Schwarz</span>
                           <span class="given-names" tagx="given-names" title="given-names">WH</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Bronnenmeier</span>
                           <span class="given-names" tagx="given-names" title="given-names">K</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Grabnitz</span>
                           <span class="given-names" tagx="given-names" title="given-names">F</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Staudenbauer</span>
                           <span class="given-names" tagx="given-names" title="given-names">WL</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1987</span>
                     <span class="mixed-article-title" title="mixed-article-title">Activity staining of cellulases in polyacrylamide gels containing mixed linkage beta-glucans.</span>
                     <span class="source" tagx="source" title="source">Analytical Biochemistry</span>
                     <span class="volume" tagx="volume" title="volume">164</span>
                     <span class="fpage" tagx="fpage" title="fpage">72</span>
                     <span class="lpage" tagx="lpage" title="lpage">77</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/2445222">2445222</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Chavez1"></a>
                  <span class="label" tagx="label" title="label">60</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Chavez</span>
                           <span class="given-names" tagx="given-names" title="given-names">R</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Schachter</span>
                           <span class="given-names" tagx="given-names" title="given-names">K</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Navarro</span>
                           <span class="given-names" tagx="given-names" title="given-names">C</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Peirano</span>
                           <span class="given-names" tagx="given-names" title="given-names">A</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Aguirre</span>
                           <span class="given-names" tagx="given-names" title="given-names">C</span>
                        </span>
                        <span class="etal" title="etal">
                           <i>et al.</i>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2002</span>
                     <span class="mixed-article-title" title="mixed-article-title">Differences in expression of two endoxylanase genes (xynA and xynB) from 
                        <i>Penicillium purpurogenum</i>.
                     </span>
                     <span class="source" tagx="source" title="source">Gene</span>
                     <span class="volume" tagx="volume" title="volume">293</span>
                     <span class="fpage" tagx="fpage" title="fpage">161</span>
                     <span class="lpage" tagx="lpage" title="lpage">168</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/12137954">12137954</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Dejong1"></a>
                  <span class="label" tagx="label" title="label">61</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Dejong</span>
                           <span class="given-names" tagx="given-names" title="given-names">E</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Field</span>
                           <span class="given-names" tagx="given-names" title="given-names">JA</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Debont</span>
                           <span class="given-names" tagx="given-names" title="given-names">JAM</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1992</span>
                     <span class="mixed-article-title" title="mixed-article-title">Evidence for a new extracellular peroxidase - manganese-inhibited peroxidase from
                        the white-rot fungus 
                        <i>Bjerkandera Sp Bos</i>-55.
                     </span>
                     <span class="source" tagx="source" title="source">FEBS Letters</span>
                     <span class="volume" tagx="volume" title="volume">299</span>
                     <span class="fpage" tagx="fpage" title="fpage">107</span>
                     <span class="lpage" tagx="lpage" title="lpage">110</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/1312029">1312029</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Rehman1"></a>
                  <span class="label" tagx="label" title="label">62</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Rehman</span>
                           <span class="given-names" tagx="given-names" title="given-names">AU</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Thurston</span>
                           <span class="given-names" tagx="given-names" title="given-names">CF</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1992</span>
                     <span class="mixed-article-title" title="mixed-article-title">Purification of laccase-I from 
                        <i>Armillaria mellea</i>.
                     </span>
                     <span class="source" tagx="source" title="source">Journal of General Microbiology</span>
                     <span class="volume" tagx="volume" title="volume">138</span>
                     <span class="fpage" tagx="fpage" title="fpage">1251</span>
                     <span class="lpage" tagx="lpage" title="lpage">1257</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Henrissat1"></a>
                  <span class="label" tagx="label" title="label">63</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Henrissat</span>
                           <span class="given-names" tagx="given-names" title="given-names">B</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1991</span>
                     <span class="mixed-article-title" title="mixed-article-title">A classification of glycosyl hydrolases based on amino acid sequence similarities.</span>
                     <span class="source" tagx="source" title="source">Biochem J</span>
                     <span class="volume" tagx="volume" title="volume">280</span>
                     <span class="issue" tagx="issue" title="issue">Pt 2</span>
                     <span class="fpage" tagx="fpage" title="fpage">309</span>
                     <span class="lpage" tagx="lpage" title="lpage">316</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/1747104">1747104</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Pasti1"></a>
                  <span class="label" tagx="label" title="label">64</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Pasti</span>
                           <span class="given-names" tagx="given-names" title="given-names">MB</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Pometto</span>
                           <span class="given-names" tagx="given-names" title="given-names">AL</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Nuti</span>
                           <span class="given-names" tagx="given-names" title="given-names">MP</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Crawford</span>
                           <span class="given-names" tagx="given-names" title="given-names">DL</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1990</span>
                     <span class="mixed-article-title" title="mixed-article-title">Lignin-solubilizing ability of actinomycetes isolated from termite (Termitidae) gut.</span>
                     <span class="source" tagx="source" title="source">Applied and Environmental Microbiology</span>
                     <span class="volume" tagx="volume" title="volume">56</span>
                     <span class="fpage" tagx="fpage" title="fpage">2213</span>
                     <span class="lpage" tagx="lpage" title="lpage">2218</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/2167628">2167628</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Dunwell1"></a>
                  <span class="label" tagx="label" title="label">65</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Dunwell</span>
                           <span class="given-names" tagx="given-names" title="given-names">JM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Purvis</span>
                           <span class="given-names" tagx="given-names" title="given-names">A</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Khuri</span>
                           <span class="given-names" tagx="given-names" title="given-names">S</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2004</span>
                     <span class="mixed-article-title" title="mixed-article-title">Cupins: the most functionally diverse protein superfamily?</span>
                     <span class="source" tagx="source" title="source">Phytochemistry</span>
                     <span class="volume" tagx="volume" title="volume">65</span>
                     <span class="fpage" tagx="fpage" title="fpage">7</span>
                     <span class="lpage" tagx="lpage" title="lpage">17</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/14697267">14697267</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Ubiyvovk1"></a>
                  <span class="label" tagx="label" title="label">66</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Ubiyvovk</span>
                           <span class="given-names" tagx="given-names" title="given-names">VM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Blazhenko</span>
                           <span class="given-names" tagx="given-names" title="given-names">OV</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Gigot</span>
                           <span class="given-names" tagx="given-names" title="given-names">D</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Penninckx</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Sibirny</span>
                           <span class="given-names" tagx="given-names" title="given-names">AA</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2006</span>
                     <span class="mixed-article-title" title="mixed-article-title">Role of gamma-glutamyltranspeptidase in detoxification of xenobiotics in the yeasts
                        
                        <i>Hansenula polymorpha</i> and 
                        <i>Saccharomyces cerevisiae</i>.
                     </span>
                     <span class="source" tagx="source" title="source">Cell Biology International</span>
                     <span class="volume" tagx="volume" title="volume">30</span>
                     <span class="fpage" tagx="fpage" title="fpage">665</span>
                     <span class="lpage" tagx="lpage" title="lpage">671</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/16857395">16857395</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Soundararajan1"></a>
                  <span class="label" tagx="label" title="label">67</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Soundararajan</span>
                           <span class="given-names" tagx="given-names" title="given-names">S</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Jedd</span>
                           <span class="given-names" tagx="given-names" title="given-names">G</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Li</span>
                           <span class="given-names" tagx="given-names" title="given-names">XL</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Ramos-Pamplona</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Chua</span>
                           <span class="given-names" tagx="given-names" title="given-names">NH</span>
                        </span>
                        <span class="etal" title="etal">
                           <i>et al.</i>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2004</span>
                     <span class="mixed-article-title" title="mixed-article-title">Woronin body function in 
                        <i>Magnaporthe grisea</i> is essential for efficient pathogenesis and for survival during nitrogen starvation
                        stress.
                     </span>
                     <span class="source" tagx="source" title="source">Plant Cell</span>
                     <span class="volume" tagx="volume" title="volume">16</span>
                     <span class="fpage" tagx="fpage" title="fpage">1564</span>
                     <span class="lpage" tagx="lpage" title="lpage">1574</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/15155882">15155882</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Pazzagli1"></a>
                  <span class="label" tagx="label" title="label">68</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Pazzagli</span>
                           <span class="given-names" tagx="given-names" title="given-names">L</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Cappugi</span>
                           <span class="given-names" tagx="given-names" title="given-names">G</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Manao</span>
                           <span class="given-names" tagx="given-names" title="given-names">G</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Camici</span>
                           <span class="given-names" tagx="given-names" title="given-names">G</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Santini</span>
                           <span class="given-names" tagx="given-names" title="given-names">A</span>
                        </span>
                        <span class="etal" title="etal">
                           <i>et al.</i>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1999</span>
                     <span class="mixed-article-title" title="mixed-article-title">Purification, characterization, and amino acid sequence of cerato-platanin, a new
                        phytotoxic protein from 
                        <i>Ceratocystis fimbriata f. sp platani</i>.
                     </span>
                     <span class="source" tagx="source" title="source">Journal of Biological Chemistry</span>
                     <span class="volume" tagx="volume" title="volume">274</span>
                     <span class="fpage" tagx="fpage" title="fpage">24959</span>
                     <span class="lpage" tagx="lpage" title="lpage">24964</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/10455173">10455173</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Holtzapple1"></a>
                  <span class="label" tagx="label" title="label">69</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Holtzapple</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Cognata</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Shu</span>
                           <span class="given-names" tagx="given-names" title="given-names">Y</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hendrickson</span>
                           <span class="given-names" tagx="given-names" title="given-names">C</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1990</span>
                     <span class="mixed-article-title" title="mixed-article-title">Inhibition of 
                        <i>Trichoderma reesei</i> cellulase by sugars and solvents.
                     </span>
                     <span class="source" tagx="source" title="source">Biotechnology and Bioengineering</span>
                     <span class="volume" tagx="volume" title="volume">36</span>
                     <span class="fpage" tagx="fpage" title="fpage">275</span>
                     <span class="lpage" tagx="lpage" title="lpage">287</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/18595079">18595079</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Cantarel1"></a>
                  <span class="label" tagx="label" title="label">70</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Cantarel</span>
                           <span class="given-names" tagx="given-names" title="given-names">BL</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Coutinho</span>
                           <span class="given-names" tagx="given-names" title="given-names">PM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Rancurel</span>
                           <span class="given-names" tagx="given-names" title="given-names">C</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Bernard</span>
                           <span class="given-names" tagx="given-names" title="given-names">T</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Lombard</span>
                           <span class="given-names" tagx="given-names" title="given-names">V</span>
                        </span>
                        <span class="etal" title="etal">
                           <i>et al.</i>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2009</span>
                     <span class="mixed-article-title" title="mixed-article-title">The Carbohydrate-Active EnZymes database (CAZy): An expert resource for glycogenomics.</span>
                     <span class="source" tagx="source" title="source">Nucleic Acids Research</span>
                     <span class="volume" tagx="volume" title="volume">37</span>
                     <span class="fpage" tagx="fpage" title="fpage">D233</span>
                     <span class="lpage" tagx="lpage" title="lpage">D238</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/18838391">18838391</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Pinto1"></a>
                  <span class="label" tagx="label" title="label">71</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Pinto</span>
                           <span class="given-names" tagx="given-names" title="given-names">PC</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Evtuguin</span>
                           <span class="given-names" tagx="given-names" title="given-names">DV</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Neto</span>
                           <span class="given-names" tagx="given-names" title="given-names">CP</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2005</span>
                     <span class="mixed-article-title" title="mixed-article-title">Structure of hardwood glucuronoxylans: modifications and impact on pulp retention
                        during wood kraft pulping.
                     </span>
                     <span class="source" tagx="source" title="source">Carbohydrate Polymers</span>
                     <span class="volume" tagx="volume" title="volume">60</span>
                     <span class="fpage" tagx="fpage" title="fpage">489</span>
                     <span class="lpage" tagx="lpage" title="lpage">497</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Sato1"></a>
                  <span class="label" tagx="label" title="label">72</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Sato</span>
                           <span class="given-names" tagx="given-names" title="given-names">S</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Liu</span>
                           <span class="given-names" tagx="given-names" title="given-names">F</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Koc</span>
                           <span class="given-names" tagx="given-names" title="given-names">H</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Tien</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2007</span>
                     <span class="mixed-article-title" title="mixed-article-title">Expression analysis of extracellular proteins from 
                        <i>Phanerochaete chrysosporium</i> grown on different liquid and solid substrates.
                     </span>
                     <span class="source" tagx="source" title="source">Microbiology-Sgm</span>
                     <span class="volume" tagx="volume" title="volume">153</span>
                     <span class="fpage" tagx="fpage" title="fpage">3023</span>
                     <span class="lpage" tagx="lpage" title="lpage">3033</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Ferris1"></a>
                  <span class="label" tagx="label" title="label">73</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Ferris</span>
                           <span class="given-names" tagx="given-names" title="given-names">MJ</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Muyzer</span>
                           <span class="given-names" tagx="given-names" title="given-names">G</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Ward</span>
                           <span class="given-names" tagx="given-names" title="given-names">DM</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1996</span>
                     <span class="mixed-article-title" title="mixed-article-title">Denaturing gradient gel electrophoresis profiles of 16S rRNA-defined populations inhabiting
                        a hot spring microbial mat community.
                     </span>
                     <span class="source" tagx="source" title="source">Applied and Environmental Microbiology</span>
                     <span class="volume" tagx="volume" title="volume">62</span>
                     <span class="fpage" tagx="fpage" title="fpage">340</span>
                     <span class="lpage" tagx="lpage" title="lpage">346</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/8593039">8593039</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Sato2"></a>
                  <span class="label" tagx="label" title="label">74</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Sato</span>
                           <span class="given-names" tagx="given-names" title="given-names">S</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Feltus</span>
                           <span class="given-names" tagx="given-names" title="given-names">FA</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Iyer</span>
                           <span class="given-names" tagx="given-names" title="given-names">P</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Tien</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2009</span>
                     <span class="mixed-article-title" title="mixed-article-title">The first genome-level transcriptome of the wood-degrading fungus 
                        <i>Phanerochaete chrysosporium</i> grown on red oak.
                     </span>
                     <span class="source" tagx="source" title="source">Current Genetics</span>
                     <span class="volume" tagx="volume" title="volume">55</span>
                     <span class="fpage" tagx="fpage" title="fpage">273</span>
                     <span class="lpage" tagx="lpage" title="lpage">286</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/19396602">19396602</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Abbas1"></a>
                  <span class="label" tagx="label" title="label">75</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Abbas</span>
                           <span class="given-names" tagx="given-names" title="given-names">A</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Koc</span>
                           <span class="given-names" tagx="given-names" title="given-names">H</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Liu</span>
                           <span class="given-names" tagx="given-names" title="given-names">F</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Tien</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2005</span>
                     <span class="mixed-article-title" title="mixed-article-title">Fungal degradation of wood: initial proteomic analysis of extracellular proteins of
                        
                        <i>Phanerochaete chrysosporium</i> grown on oak substrate.
                     </span>
                     <span class="source" tagx="source" title="source">Current Genetics</span>
                     <span class="volume" tagx="volume" title="volume">47</span>
                     <span class="fpage" tagx="fpage" title="fpage">49</span>
                     <span class="lpage" tagx="lpage" title="lpage">56</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/15551134">15551134</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Strope1"></a>
                  <span class="label" tagx="label" title="label">76</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Strope</span>
                           <span class="given-names" tagx="given-names" title="given-names">PK</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Nickerson</span>
                           <span class="given-names" tagx="given-names" title="given-names">KW</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Harris</span>
                           <span class="given-names" tagx="given-names" title="given-names">SD</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Moriyama</span>
                           <span class="given-names" tagx="given-names" title="given-names">EN</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2011</span>
                     <span class="mixed-article-title" title="mixed-article-title">Molecular evolution of urea amidolyase and urea carboxylase in fungi.</span>
                     <span class="source" tagx="source" title="source">Bmc Evolutionary Biology</span>
                     <span class="volume" tagx="volume" title="volume">11</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Kersten1"></a>
                  <span class="label" tagx="label" title="label">77</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kersten</span>
                           <span class="given-names" tagx="given-names" title="given-names">P</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Cullen</span>
                           <span class="given-names" tagx="given-names" title="given-names">D</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2007</span>
                     <span class="mixed-article-title" title="mixed-article-title">Extracellular oxidative systems of the lignin-degrading Basidiomycete 
                        <i>Phanerochaete chrysosporium</i>.
                     </span>
                     <span class="source" tagx="source" title="source">Fungal Genetics and Biology</span>
                     <span class="volume" tagx="volume" title="volume">44</span>
                     <span class="fpage" tagx="fpage" title="fpage">77</span>
                     <span class="lpage" tagx="lpage" title="lpage">87</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/16971147">16971147</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Hammel1"></a>
                  <span class="label" tagx="label" title="label">78</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hammel</span>
                           <span class="given-names" tagx="given-names" title="given-names">KE</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1997</span>
                     <span class="mixed-article-title" title="mixed-article-title">Fungal degradation of lignin.</span>
                     <span class="source" tagx="source" title="source">Driven by Nature: Plant Litter Quality and Decomposition</span>
                     <span class="fpage" tagx="fpage" title="fpage">33</span>
                     <span class="lpage" tagx="lpage" title="lpage">45</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Youn1"></a>
                  <span class="label" tagx="label" title="label">79</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Youn</span>
                           <span class="given-names" tagx="given-names" title="given-names">HD</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hah</span>
                           <span class="given-names" tagx="given-names" title="given-names">YC</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kang</span>
                           <span class="given-names" tagx="given-names" title="given-names">SO</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1995</span>
                     <span class="mixed-article-title" title="mixed-article-title">Role of laccase in lignin degradation by white-rot fungi.</span>
                     <span class="source" tagx="source" title="source">Fems Microbiology Letters</span>
                     <span class="volume" tagx="volume" title="volume">132</span>
                     <span class="fpage" tagx="fpage" title="fpage">183</span>
                     <span class="lpage" tagx="lpage" title="lpage">188</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Hemingway1"></a>
                  <span class="label" tagx="label" title="label">80</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hemingway</span>
                           <span class="given-names" tagx="given-names" title="given-names">RW</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Laks</span>
                           <span class="given-names" tagx="given-names" title="given-names">PE</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1992</span>
                     <span class="source" tagx="source" title="source">Plant Polyphenolics: Synthesis, Properties, Significance</span>
                     <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Houghton, MI</span>
                     <span class="publisher-name" tagx="publisher-name" title="publisher-name">Plenum Press</span>
                     <span class="size" title="size">size: 1045page</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Bending1"></a>
                  <span class="label" tagx="label" title="label">81</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Bending</span>
                           <span class="given-names" tagx="given-names" title="given-names">GD</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Read</span>
                           <span class="given-names" tagx="given-names" title="given-names">DJ</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1997</span>
                     <span class="mixed-article-title" title="mixed-article-title">Lignin and soluble phenolic degradation by ectomycorrhizal and ericoid mycorrhizal
                        fungi.
                     </span>
                     <span class="source" tagx="source" title="source">Mycological Research</span>
                     <span class="volume" tagx="volume" title="volume">101</span>
                     <span class="fpage" tagx="fpage" title="fpage">1348</span>
                     <span class="lpage" tagx="lpage" title="lpage">1354</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Kuan1"></a>
                  <span class="label" tagx="label" title="label">82</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kuan</span>
                           <span class="given-names" tagx="given-names" title="given-names">IC</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Tien</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1993</span>
                     <span class="mixed-article-title" title="mixed-article-title">Stimulation of Mn-peroxidase activity - A possible role for oxalate in lignin biodegradation.</span>
                     <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span>
                     <span class="volume" tagx="volume" title="volume">90</span>
                     <span class="fpage" tagx="fpage" title="fpage">1242</span>
                     <span class="lpage" tagx="lpage" title="lpage">1246</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/8433984">8433984</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Ohta1"></a>
                  <span class="label" tagx="label" title="label">83</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Ohta</span>
                           <span class="given-names" tagx="given-names" title="given-names">Y</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Maeda</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kudo</span>
                           <span class="given-names" tagx="given-names" title="given-names">T</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">2001</span>
                     <span class="mixed-article-title" title="mixed-article-title">
                        <i>Pseudomonas putida</i> CE2010 can degrade biphenyl by a mosaic pathway encoded by the tod operon and cmtE,
                        which are identical to those of 
                        <i>P. putida</i> F1 except for a single base difference in the operator-promoter region of the cmt
                        operon.
                     </span>
                     <span class="source" tagx="source" title="source">Microbiology-Uk</span>
                     <span class="volume" tagx="volume" title="volume">147</span>
                     <span class="fpage" tagx="fpage" title="fpage">31</span>
                     <span class="lpage" tagx="lpage" title="lpage">41</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Magnuson1"></a>
                  <span class="label" tagx="label" title="label">84</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Magnuson</span>
                           <span class="given-names" tagx="given-names" title="given-names">TS</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Crawford</span>
                           <span class="given-names" tagx="given-names" title="given-names">DL</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1992</span>
                     <span class="mixed-article-title" title="mixed-article-title">Comparison of extracellular peroxidase- and esterase-deficient mutants of 
                        <i>Streptomyces viridosporus</i> T7a.
                     </span>
                     <span class="source" tagx="source" title="source">Applied and Environmental Microbiology</span>
                     <span class="volume" tagx="volume" title="volume">58</span>
                     <span class="fpage" tagx="fpage" title="fpage">1070</span>
                     <span class="lpage" tagx="lpage" title="lpage">1072</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/1315498">1315498</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Ander1"></a>
                  <span class="label" tagx="label" title="label">85</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Ander</span>
                           <span class="given-names" tagx="given-names" title="given-names">P</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Eriksson</span>
                           <span class="given-names" tagx="given-names" title="given-names">KE</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1976</span>
                     <span class="mixed-article-title" title="mixed-article-title">Importance of phenol oxidase activity in lignin degradation by white-rot fungus 
                        <i>Sporotrichum pulverulentum</i>.
                     </span>
                     <span class="source" tagx="source" title="source">Archives of Microbiology</span>
                     <span class="volume" tagx="volume" title="volume">109</span>
                     <span class="fpage" tagx="fpage" title="fpage">1</span>
                     <span class="lpage" tagx="lpage" title="lpage">8</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0032990-Bermek1"></a>
                  <span class="label" tagx="label" title="label">86</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Bermek</span>
                           <span class="given-names" tagx="given-names" title="given-names">H</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Li</span>
                           <span class="given-names" tagx="given-names" title="given-names">KC</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Eriksson</span>
                           <span class="given-names" tagx="given-names" title="given-names">KEL</span>
                        </span>
                     </span>
                     <span class="year" tagx="year" title="year">1998</span>
                     <span class="mixed-article-title" title="mixed-article-title">Laccase-less mutants of the white-rot fungus 
                        <i>Pycnoporus cinnabarinus</i> cannot delignify kraft pulp.
                     </span>
                     <span class="source" tagx="source" title="source">Journal of Biotechnology</span>
                     <span class="volume" tagx="volume" title="volume">66</span>
                     <span class="fpage" tagx="fpage" title="fpage">117</span>
                     <span class="lpage" tagx="lpage" title="lpage">124</span>
                  </span>
               </li>
            </ul>
         </div>
      </div>
   </body>
</html>