<?xml version="1.0" encoding="UTF-8"?><html xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:h="http://www.w3.org/1999/xhtml">
   <head>
      <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style>
   </head>
   <body>
      <div class="front" title="front">
         <div class="journal-meta" tagx="journal-meta" title="journal-meta">
            <span class="nlm-ta" title="nlm-ta">Nat Commun</span>
            <span class="iso-abbrev" title="iso-abbrev">Nat Commun</span>
            <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group">
               <span class="journal-title" tagx="journal-title" title="journal-title">Nature Communications</span>
            </div>
            <span class="issn-epub" tagx="issn" title="issn-epub">2041-1723</span>
            <div class="publisher" tagx="publisher" title="publisher">
               <span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Pub. Group</span>
            </div>
         </div>
         <div class="article-meta" tagx="article-meta" title="article-meta">
            <span class="pmcid" title="pmcid">
               			pmcid:
               			
               <a href="http://www.ncbi.nlm.nih.gov/pubmed/4510693">4510693</a>
            </span>
            <span class="pii" title="pii">ncomms8618</span>
            <span class="doi" title="doi">
               			doi:
               			
               <a href="https://dx.doi.org/10.1038/ncomms8618">10.1038/ncomms8618</a>
            </span>
            <span class="pmid" title="pmid">
               			pmid:
               			
               <a href="http://www.ncbi.nlm.nih.gov/pubmed/26173063">26173063</a>
            </span>
            <div class="article-categories" title="article-categories">: 
               <span class="subject" title="subject">Article</span>
            </div>
            <div class="title-group">
               <div class="article-title" title="article-title">Gut microbiota mediate caffeine detoxification in the primary insect pest of coffee</div>
            </div>
            <div class="contrib-group" title="contrib-group">
               <span class="contrib" title="contrib">
                  <span class="name" tagx="name" title="name">
                     <span class="surname" tagx="surname" title="surname">Ceja-Navarro</span>
                     <span class="given-names" tagx="given-names" title="given-names">Javier A.</span>
                  </span>
                  <a href="#c1">a</a>
                  <a href="#a1">1</a>
               </span>
               <span class="contrib" title="contrib">
                  <span class="name" tagx="name" title="name">
                     <span class="surname" tagx="surname" title="surname">Vega</span>
                     <span class="given-names" tagx="given-names" title="given-names">Fernando E.</span>
                  </span>
                  <a href="#c2">b</a>
                  <a href="#a2">2</a>
               </span>
               <span class="contrib" title="contrib">
                  <span class="name" tagx="name" title="name">
                     <span class="surname" tagx="surname" title="surname">Karaoz</span>
                     <span class="given-names" tagx="given-names" title="given-names">Ulas</span>
                  </span>
                  <a href="#a1">1</a>
               </span>
               <span class="contrib" title="contrib">
                  <span class="name" tagx="name" title="name">
                     <span class="surname" tagx="surname" title="surname">Hao</span>
                     <span class="given-names" tagx="given-names" title="given-names">Zhao</span>
                  </span>
                  <a href="#a1">1</a>
               </span>
               <span class="contrib" title="contrib">
                  <span class="name" tagx="name" title="name">
                     <span class="surname" tagx="surname" title="surname">Jenkins</span>
                     <span class="given-names" tagx="given-names" title="given-names">Stefan</span>
                  </span>
                  <a href="#a3">3</a>
               </span>
               <span class="contrib" title="contrib">
                  <span class="name" tagx="name" title="name">
                     <span class="surname" tagx="surname" title="surname">Lim</span>
                     <span class="given-names" tagx="given-names" title="given-names">Hsiao Chien</span>
                  </span>
                  <a href="#a1">1</a>
               </span>
               <span class="contrib" title="contrib">
                  <span class="name" tagx="name" title="name">
                     <span class="surname" tagx="surname" title="surname">Kosina</span>
                     <span class="given-names" tagx="given-names" title="given-names">Petr</span>
                  </span>
                  <a href="#a4">4</a>
                  <span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-8805-0306</span>
               </span>
               <span class="contrib" title="contrib">
                  <span class="name" tagx="name" title="name">
                     <span class="surname" tagx="surname" title="surname">Infante</span>
                     <span class="given-names" tagx="given-names" title="given-names">Francisco</span>
                  </span>
                  <a href="#a5">5</a>
               </span>
               <span class="contrib" title="contrib">
                  <span class="name" tagx="name" title="name">
                     <span class="surname" tagx="surname" title="surname">Northen</span>
                     <span class="given-names" tagx="given-names" title="given-names">Trent R.</span>
                  </span>
                  <a href="#a3">3</a>
               </span>
               <span class="contrib" title="contrib">
                  <span class="name" tagx="name" title="name">
                     <span class="surname" tagx="surname" title="surname">Brodie</span>
                     <span class="given-names" tagx="given-names" title="given-names">Eoin L.</span>
                  </span>
                  <a href="#c3">c</a>
                  <a href="#a1">1</a>
                  <a href="#a6">6</a>
               </span>
               <span class="citation_author_institution" id="a1">[1],
                  	  
                  <span class="institution" title="institution">Ecology Department, Earth Sciences Division, Lawrence Berkeley National Laboratory</span>
                  <span class="country" tagx="country" title="country">USA</span>
               </span>
               <span class="citation_author_institution" id="a2">[2],
                  	  
                  <span class="institution" title="institution">Sustainable Perennial Crops Laboratory, United States Department of Agriculture, Agricultural
                     Research Service
                  </span>
                  <span class="country" tagx="country" title="country">USA</span>
               </span>
               <span class="citation_author_institution" id="a3">[3],
                  	  
                  <span class="institution" title="institution">Genome Dynamics Department, Life Sciences Division, Lawrence Berkeley National Laboratory</span>
                  <span class="country" tagx="country" title="country">USA</span>
               </span>
               <span class="citation_author_institution" id="a4">[4],
                  	  
                  <span class="institution" title="institution">International Maize and Wheat Improvement Center (CIMMYT)</span>
                  <span class="country" tagx="country" title="country">Mexico</span>
               </span>
               <span class="citation_author_institution" id="a5">[5],
                  	  
                  <span class="institution" title="institution">El Colegio de la Frontera Sur (ECOSUR)</span>
                  <span class="country" tagx="country" title="country">Mexico</span>
               </span>
               <span class="citation_author_institution" id="a6">[6],
                  	  
                  <span class="institution" title="institution">Department of Environmental Science, Policy and Management, University of California</span>
                  <span class="country" tagx="country" title="country">USA</span>
               </span>
            </div>
            <div class="author-notes" title="author-notes">
               <div class="corresp" title="corresp">
                  <span class="label" tagx="label" title="label">a</span>
                  <span class="email" tagx="email" title="email">jcnavarro@lbl.gov</span>
               </div>
               <div class="corresp" title="corresp">
                  <span class="label" tagx="label" title="label">b</span>
                  <span class="email" tagx="email" title="email">Fernando.Vega@ars.usda.gov</span>
               </div>
               <div class="corresp" title="corresp">
                  <span class="label" tagx="label" title="label">c</span>
                  <span class="email" tagx="email" title="email">elbrodie@lbl.gov</span>
               </div>
            </div>
            <span class="pub-date-epub" title="pub-date-epub">epub: 
               <span>2015-07-07</span>
            </span>
            <span class="pub-date-collection" title="pub-date-collection">collection: 
               <span>2015</span>
            </span>
            <span class="volume" tagx="volume" title="volume">6</span>
            <span class="elocation-id" tagx="elocation-id" title="elocation-id">7618</span>
            <span class="history" title="history">
               <span class="received" title="received">received: 2015-03-05</span>
               <span class="accepted" title="accepted">accepted: 2015-05-26</span>
            </span>
            <div class="permissions">
               <span class="copyright" title="copyright">(C) 
                  <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Nature Publishing Group, a division of Macmillan Publishers Limited. All Rights Reserved.</span>, 
                  <span class="copyright-year" tagx="copyright-year" title="copyright-year">2015</span>
               </span>
               <span class="license" title="license">
                  <span class="license-p" title="license-p">This work is licensed under a Creative Commons Attribution 4.0 International License.
                     The images or other third party material in this article are included in the article's
                     Creative Commons license, unless indicated otherwise in the credit line; if the material
                     is not included under the Creative Commons license, users will need to obtain permission
                     from the license holder to reproduce the material. To view a copy of this license,
                     visit 
                     <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>
                  </span>
               </span>
            </div>
            <div class="abstract" title="abstract">
               <div class="abstract-title" title="abstract-title">Abstract</div>
               <p>The coffee berry borer (
                  <i>Hypothenemus hampei</i>) is the most devastating insect pest of coffee worldwide with its infestations decreasing
                  crop yield by up to 80%. Caffeine is an alkaloid that can be toxic to insects and
                  is hypothesized to act as a defence mechanism to inhibit herbivory. Here we show that
                  caffeine is degraded in the gut of 
                  <i>H. hampei</i>, and that experimental inactivation of the gut microbiota eliminates this activity.
                  We demonstrate that gut microbiota in 
                  <i>H. hampei</i> specimens from seven major coffee-producing countries and laboratory-reared colonies
                  share a core of microorganisms. Globally ubiquitous members of the gut microbiota,
                  including prominent 
                  <i>Pseudomonas</i> species, subsist on caffeine as a sole source of carbon and nitrogen. 
                  <i>Pseudomonas</i> caffeine demethylase genes are expressed 
                  <i>in vivo</i> in the gut of 
                  <i>H. hampei</i>, and re-inoculation of antibiotic-treated insects with an isolated 
                  <i>Pseudomonas</i> strain reinstates caffeine-degradation ability confirming their key role.
               </p>
            </div>
            <div class="abstract" title="abstract">
               <p>
                  <span class="inline-graphic" title="inline-graphic"></span>The coffee berry borer, the main insect pest of coffee, feeds and lives on the caffeine-rich
                  beans despite caffeine's toxic effects. Here Ceja-Navarro 
                  <i>et al.</i> show that certain microbes, including 
                  <i>Pseudomonas</i> species, mediate caffeine detoxification in the insect's gut.
               </p>
            </div>
         </div>
      </div>
      <div class="body" title="body">
         <p>Of 124 species in the genus 
            <i>Coffea</i>
            <a href="#b1">1</a>
            <a href="#b2">2</a>, only two are commercially traded: 
            <i>Coffea arabica</i> and 
            <i>C. canephora</i>. 
            <i>C. arabica</i> is a high-altitude species endemic to southwestern Ethiopia, southeast Sudan and
            northern Kenya, while 
            <i>C. canephora</i> (commonly referred to as robusta coffee) is a lowland plant endemic to tropical Africa,
            west of the Rift Valley
            <a href="#b1">1</a>. These species differ in the content of the purine alkaloid caffeine (1,3,7-trimethylxanthine),
            representing 
            <i>ca.</i> 1% of seed dry weight in 
            <i>C. arabica</i>
            <a href="#b3">3</a> and up to 1.7–2.4% in 
            <i>C. canephora</i>. Although at low concentrations, it may have other roles
            <a href="#b4">4</a>; caffeine in plants has been hypothesized to serve as protection against herbivory
            <a href="#b5">5</a>. Caffeine imparts a bitter taste that inhibits insect feeding
            <a href="#b6">6</a>
            <a href="#b7">7</a>; it also paralyses and intoxicates insects by inhibiting phosphodiesterase activity
            and increasing the intracellular levels of cyclic AMP
            <a href="#b5">5</a>
            <a href="#b8">8</a>. Caffeine toxicity is also related to its negative effects on DNA repair and recombination
            pathways, and the delay of cell cycle progression as shown in systems such as yeast
            and zebrafish
            <a href="#b9">9</a>
            <a href="#b10">10</a>. Because of its demonstrated negative effects on insects, arachnids, slugs and snails,
            caffeine is considered a natural pest repellent
            <a href="#b5">5</a>
            <a href="#b11">11</a>
            <a href="#b12">12</a> and, while over 850 insect species can feed on other parts of the coffee plant
            <a href="#b13">13</a>, and a few of them occasionally on the coffee seed, only 
            <i>Hypothenemus hampei</i> (Ferrari) (Coleoptera; Curculionidae: Scolytinae) has developed the ability to feed
            and complete its life cycle solely on the economically important caffeine-rich coffee
            bean.
         </p>
         <p>Endemic to Africa, the coffee berry borer entered Indonesia in 1908 (ref. 
            <a href="#b14">14</a>) and Brazil in 1913 (ref. 
            <a href="#b15">15</a>) and is now present in most coffee-producing nations
            <a href="#b16">16</a>. Female coffee berry borers are minute bark beetles (
            <i>ca.</i> 2 mm long) that bore a hole in the coffee berry and build galleries within the coffee
            seed, where they deposit their eggs. Larvae feed on the seed, causing significant
            losses both in quality and yields. Yearly losses in Brazil alone have been estimated
            at $285–315 million (ref. 
            <a href="#b17">17</a>), making the insect the most devastating pest of coffee worldwide. This cryptic life
            cycle inside the berry makes insect control problematic. The devastating socioeconomic
            consequences of 
            <i>H. hampei's</i> worldwide invasion have stimulated ample research on possible pest management methods;
            however, detailed information on some basic aspects of the insect's biology, such
            as metabolism, is scarce. Acuña 
            <i>et al.</i>
            <a href="#b18">18</a> demonstrated that the natural acquisition and integration of a mannanase gene of
            bacterial origin into the 
            <i>H. hampei</i> genome has enabled the insect to exploit the primary carbohydrate reserve found in
            coffee beans. Moreover, previous work
            <a href="#b19">19</a> reported no correlation between caffeine concentrations (range: 2–17 mg g
            <sup>−1</sup>) in the beans of 12 
            <i>Coffea</i> species and resistance to 
            <i>H. hampei</i>, suggesting that the insect possesses mechanisms to degrade or efficiently excrete
            caffeine, which permits it to withstand its toxic effects, and significantly reduce
            crop yield.
         </p>
         <p>In insects as in other animals caffeine degradation can be catalysed by cytochrome
            P450 enzymes
            <a href="#b20">20</a>
            <a href="#b21">21</a>. However, no other metazoans have been reported to endure the concentrations of caffeine
            to which 
            <i>H. hampei</i> is exposed while simultaneously subsisting and completing its full life cycle exclusively
            on green coffee beans. We hypothesized that caffeine degradation in 
            <i>H. hampei</i> is primarily mediated by the activity of its gut microbiota. Here we present the
            first study of the microbiome of 
            <i>H. hampei</i> and its role in the detoxification of caffeine in the insect. Our discovery of a
            microbiota component to the transformation of caffeine and subsistence of 
            <i>H. hampei</i> on coffee beans has important implications for understanding the metabolism and ecology
            of this major pest.
         </p>
         <div class="results" title="results">
            <div class="title" tagx="title" title="title">Results</div>
            <div class="caffeinedegradationinh.hampei" title="sec">
               <div class="title" tagx="title" title="title">Caffeine degradation in 
                  <i>H. hampei</i>
               </div>
               <p>To test the hypothesis that caffeine degradation is a microbially mediated process
                  in 
                  <i>H. hampei</i>, we obtained specimens from seven geographically dispersed coffee-producing regions:
                  Kenya (Kiambu), Indonesia (East Java), India (Karnataka), Puerto Rico (Adjuntas),
                  Hawaii (Kona), Guatemala (Coatepeque) and Mexico (Chiapas). Specimens from India also
                  included insects sampled from both dominant 
                  <i>Coffea</i> species (
                  <i>C. arabica</i> and 
                  <i>C. canephora</i>). In addition, we used a laboratory-reared colony originally derived from a Colombian
                  
                  <i>H. hampei</i> population and three other bark beetle species incapable of subsisting on coffee
                  beans (
                  <i>H. crudiae</i>, 
                  <i>H. eruditus</i> and 
                  <i>Scolytodes maurus</i>).
               </p>
               <p>First, we tested for caffeine degradation following passage through the insect gut
                  and determined whether the gut microbiota were responsible for this process. This
                  hypothesis was tested with the use of a 
                  <i>H. hampei</i> laboratory-reared colony in a controlled experiment using an artificial diet (here
                  referred to as minimal diet), containing green 
                  <i>C. arabica</i> coffee beans with a final caffeine concentration of ∼1.8–2.2 mg g
                  <sup>−1</sup>. 
                  <i>Hypothenemus hampei</i> specimens were introduced to vials containing this defined diet and displayed typical
                  boring activity (
                  <a href="#S1">Supplementary Movie 1</a>) and production of frass. The quantity of diet consumed and the residence time of
                  diet in the 
                  <i>H. hampei</i> digestive tract were determined by incorporating fluorescent sphere tracers in the
                  diet. Residence time was typically 3 h (
                  <a href="#S1">Supplementary Fig. 1</a>) and corresponded to a mass of ∼14 μg of diet or 25 ng of caffeine. To quantify caffeine
                  concentrations in the low masses of produced frass, we used Fourier transform infrared
                  spectroscopy (FTIR) calibrated with pure caffeine standards
                  <a href="#b22">22</a> (
                  <a href="#S1">Supplementary Fig. 2</a>). Measurements of caffeine concentration by FTIR were further validated using gas
                  chromatography–mass spectrometry (GC–MS; Methods and 
                  <a href="#t1">Table 1</a>). After passage of diet through the 
                  <i>H. hampei</i> digestive tract, caffeine was no longer detectable (
                  <a href="#f1">Fig. 1a</a>). To determine involvement of the microbiota in caffeine detoxification, we eliminated
                  microorganisms using the defined diet supplemented with three broad-spectrum antibiotics
                  (tetracycline, rifampicin and streptomycin). Laboratory insects were reared on this
                  antibiotic-supplemented diet for 4 weeks, which eliminated culturable microorganisms,
                  before being transferred to fresh antibiotic-free diet and were allowed to feed for
                  4 h. Antibiotic-treated insects exhibited no decrease in caffeine (
                  <a href="#f1">Fig. 1b</a>), with concentrations in their frass being similar to that of the diet. This was
                  strongly suggestive of a role of the gut microbiota in caffeine removal. Elimination
                  of the gut microbiota also had a significant impact on 
                  <i>H. hampei</i> fitness with approximately a 95% decline in eggs and larvae relative to control insects,
                  and no progression to pupa or adult stage over the 44-day experiment (
                  <a href="#S1">Supplementary Fig. 3</a>).
               </p>
            </div>
            <div class="biogeographyanalysisofh.hampei'smicrobiome" title="sec">
               <div class="title" tagx="title" title="title">Biogeography analysis of 
                  <i>H. hampei</i>'s microbiome
               </div>
               <p>To identify members of the 
                  <i>H. hampei</i> gut microbiota that may contribute to caffeine degradation, we analysed the bacterial
                  composition of 
                  <i>H. hampei</i> microbial communities using 16S rRNA gene sequencing from specimens of major coffee-producing
                  regions (
                  <a href="#S1">Supplementary Data 1</a>). Phylogenetic β-diversity and hierarchical clustering showed the separation of microbiomes
                  by geographic origin (
                  <a href="#S1">Supplementary Fig. 4</a>). Nonparametric permutational multivariate analysis of variance was used to partition
                  variance in microbiome composition among the key factors. Geographic location explained
                  much of the variance (45%) in bacterial composition (
                  <a href="#S1">Supplementary Fig. 4</a>). In locations where the coffee species varied (India; 
                  <i>C. arabica</i> and 
                  <i>C. canephora</i>), coffee species accounted for 78% of the variance in microbial composition. Although
                  the dominant microbial taxa also varied geographically, network analysis showed a
                  core of organisms detected across all locations (
                  <a href="#f2">Fig. 2a</a>). These organisms were affiliated with the orders Pseudomonadales, Enterobacteriales,
                  Turicibacteriales, Rhizobiales, Alteromonadales, Actinomycetales among others; with
                  the Pseudomonadales being highly abundant in all collected specimens (
                  <a href="#f2">Fig. 2b</a>).
               </p>
            </div>
            <div class="caffeine-degradingbacteriainthegutofh.hampei" title="sec">
               <div class="title" tagx="title" title="title">Caffeine-degrading bacteria in the gut of 
                  <i>H. hampei</i>
               </div>
               <p>To determine whether the identified microorganisms in the biogeography analysis were
                  involved in caffeine detoxification in 
                  <i>H. hampei</i>, we isolated bacteria that were capable of subsisting on caffeine as a sole carbon
                  and nitrogen source using the digestive tracts of insects collected in Mexico and
                  Hawaii as inocula. More than 100 isolates comprising 12 bacterial species were obtained
                  from the orders of the Pseudomonadales, Enterobacteriales, Rhizobiales, Actinomycetales,
                  Sphingomonadales and Xanthomonadales (
                  <a href="#f3">Fig. 3a</a>). Considering the high relative abundance and frequency of the Pseudomonadales in
                  the biogeography and isolation studies (
                  <a href="#f2">Figs 2b</a> and 
                  <a href="#f3">3a</a>), all isolates were screened for the 
                  <i>ndmA</i> gene (coding for the alpha subunit of a caffeine demethylase)
                  <a href="#b23">23</a>; only 
                  <i>P. fulva</i> possessed this gene. Sequence analysis of this gene from this species demonstrated
                  a 96% amino-acid similarity to the caffeine demethylase of 
                  <i>P. putida</i>
                  <a href="#b23">23</a> (
                  <a href="#f3">Fig. 3b</a>). We confirmed expression of this gene using reverse transcriptase quantitative PCR
                  (qPCR) from RNA extracts of field-collected insects (
                  <a href="#f3">Fig. 3b</a>, 
                  <a href="#S1">Supplementary Fig. 5</a>), suggesting a contribution of 
                  <i>P. fulva</i> to the process of caffeine degradation in natural populations of 
                  <i>H. hampei</i>. This gene could not be detected in specimens from other bark beetles (
                  <i>H. crudiae</i>, 
                  <i>H. eruditus</i> and 
                  <i>S. maurus</i>) that lack the capacity to consume the coffee bean (data not shown).
               </p>
            </div>
            <div class="oxygenprofilesinthegutofh.hampei" title="sec">
               <div class="title" tagx="title" title="title">Oxygen profiles in the gut of 
                  <i>H. hampei</i>
               </div>
               <p>Oxygen gradients were measured in the gut of 
                  <i>H. hampei</i> with the use of microelectrodes. All measurements were performed in the anterior
                  midgut of the beetle (see 
                  <a href="#S1">Supplementary Fig. 1</a>). During these procedures, the tip of the electrode was driven to the surface of
                  the gut wall and maintained in position until measured concentrations of O
                  <sub>2</sub> stabilized. In all cases, once the tip penetrated the gut wall, oxygen concentrations
                  declined with steep gradients going from microaerophilic conditions to a reduced anaerobic
                  core (
                  <a href="#f4">Fig. 4</a>).
               </p>
            </div>
            <div class="reinstatingcaffeinedegradationinh.hampei" title="sec">
               <div class="title" tagx="title" title="title">Reinstating caffeine degradation in 
                  <i>H. hampei</i>
               </div>
               <p>Antibiotic-treated laboratory specimens of 
                  <i>H. hampei</i> were initially fed on the minimal diet containing 
                  <i>P. fulva</i> for 1 week. Insects were subsequently transferred to a freshly prepared sterile minimal
                  diet and were allowed to feed for 4 h before frass was collected and caffeine concentrations
                  quantified. Both GC–MS and FTIR analyses showed an almost complete depletion of caffeine
                  content in the frass of the reinfected insects (
                  <a href="#f3">Fig. 3c</a>, 
                  <a href="#t1">Table 1</a>). The reproductive fitness of the antibiotic-treated and 
                  <i>P. fulva</i>-reinfected insects was also tested. Insects did not recover their ability to reproduce
                  after 
                  <i>P. fulva</i> inoculation (
                  <a href="#S1">Supplementary Fig. 3</a>).
               </p>
            </div>
         </div>
         <div class="discussion" title="discussion">
            <div class="title" tagx="title" title="title">Discussion</div>
            <p>Although other insects have been occasionally reported to feed on the coffee seed
               <a href="#b24">24</a>, 
               <i>H. hampei</i>, is the only insect that has specialized on feeding and completing its entire life
               cycle exclusively within the coffee beans inside the berry
               <a href="#b16">16</a>
               <a href="#b25">25</a>. While surviving in this environment, 
               <i>H. hampei</i> experiences chronic exposure to caffeine which is known to be a toxic alkaloid
               <a href="#b3">3</a>
               <a href="#b5">5</a>
               <a href="#b7">7</a>
               <a href="#b9">9</a>
               <a href="#b10">10</a>, making this insect an attractive model system in which to evaluate the role of microbes
               in the adaptation of animals to unusual environments. In this study, we tested whether
               caffeine degradation in 
               <i>H. hampei</i> is mediated by the insect's gut microbiota.
            </p>
            <p>To test for the contribution of 
               <i>H. hampei</i>'s gut-associated microbes to caffeine degradation, first feeding rates and transit
               time of the fed-diet through the insect's gut were determined. Once the parameters
               for the transit time of the diet were obtained, a set of insects was fed on minimal
               diet supplemented with antibiotics to deplete its microbiota. Frass samples were collected
               from 
               <i>H. hampei</i> specimens feeding on minimal diet and diet supplemented with antibiotics and their
               caffeine content compared using FTIR and GC–MS. These results demonstrated that insects
               with an intact microbiome had a nearly complete degradation of the caffeine during
               its transit through the beetle gut. However, contrary to the control insects, antibiotic
               treated 
               <i>H. hampei</i> specimens lost the ability to degrade caffeine, demonstrating the importance of the
               insect's microbiome to this critical function. Depletion of the microbiome also had
               a pronounced impact on the insect's reproductive fitness. Across all developmental
               stages observed, antibiotic treatment resulted in significant and substantial declines
               in numbers of viable insects relative to controls. While the mechanism underlying
               this observation is likely complex and not addressed here, it has been observed previously
               that members of other insect microbiomes can be responsible for the regulation of
               behaviour, feeding habits, mating preferences, reproductive capacity and the acquisition
               or the production of nutrients required by the host
               <a href="#b26">26</a>
               <a href="#b27">27</a>
               <a href="#b28">28</a>
               <a href="#b29">29</a>.
            </p>
            <p>Members of the gut microbiota of 
               <i>H. hampei</i> were identified from insects collected from seven coffee-producing countries and
               a laboratory colony. The calculated β-diversity and variance partitioning analyses
               demonstrated that the composition of the 
               <i>H. hampei</i> microbiome was significantly associated with geographic origin of the coffee and
               also with the species of coffee where that varied (see 
               <a href="#S1">Supplementary Data 1</a>). The formation of morphocryptic units was previously demonstrated in 
               <i>H. hampei</i> samples collected from different coffee-producing countries by the analysis of microsatellite
               markers and COI gene sequences
               <a href="#b30">30</a>. These reported ‘species complexes', together with our findings of a geographic variation
               of microbial communities in 
               <i>H. hampei</i>, may be indicative of the co-evolution of insects and their microbiota. Caffeine
               content varies significantly between the coffee species assessed
               <a href="#b31">31</a> and, although not tested here, caffeine concentration may be another important factor
               having an impact on 
               <i>H. hampei</i> gut microbiome composition.
            </p>
            <p>Despite the observed variation in microbiome composition demonstrated by the biogeography
               analysis, all insect specimens shared a common core of microorganisms as demonstrated
               by network analysis. The shared microbial groups included the Pseudomonadales, Enterobacteriales,
               Turicibacteriales, Rhizobiales, Alteromonadales and Actinomycetales. Fourteen caffeine-subsisting
               bacteria from the orders of the Pseudomonadales, Rhizobiales, Enterobacteriales and
               Actinomycetales were isolated from the guts of field specimens of 
               <i>H. hampei</i> on media containing caffeine as the sole source of carbon and nitrogen. It is significant
               that many of the 
               <i>H. hampei</i> microbiome common core were also confirmed to subsist 
               <i>in vitro</i> on caffeine. To evaluate the possible mechanisms of caffeine degradation, all isolates
               were screened for the gene 
               <i>ndmA</i> that codes for a monooxygenase that participates in the initial demethylation of
               caffeine in bacteria
               <a href="#b23">23</a>. Only 
               <i>P. fulva</i> isolates yielded positive amplification. Given that the Pseudomonadales were also
               the dominant group detected in the 
               <i>H. hampei</i> gut microbiome across all sampled locations, we determined whether this pathway was
               active in field specimens of 
               <i>H. hampei</i> actively infecting coffee beans. Total RNA from these field specimens was analysed
               using quantitative PCR with reverse transcription for transcripts of the 
               <i>ndmA</i> gene and positive amplification demonstrated its expression 
               <i>in situ</i>, providing further evidence of the possible contribution of 
               <i>P. fulva</i> to caffeine degradation in the insect. Because the monooxygenase encoded by the 
               <i>ndmA</i> gene is an enzyme that requires oxygen to function, oxygen availability in the gut
               of 
               <i>H. hampei</i> was also characterized with the use of microelectrodes. As in the case of other insects
               <a href="#b32">32</a>
               <a href="#b33">33</a>, the gut of 
               <i>H. hampei</i> contains a radial oxygen gradient that changes slowly from microaerophilic to anaerobic
               conditions. The existence of this gradient suggests that the gut of this insect can
               provide the environment for the occurrence of aerobic processes such as the transformation
               of caffeine by the NdmA demethylase, and also anaerobic processes that are likely
               important such as the fermentation of the sugars from the coffee bean.
            </p>
            <p>To confirm the role of 
               <i>P. fulva</i> in the degradation of caffeine 
               <i>in vivo</i>, antibiotic-treated laboratory specimens were reinfected with 
               <i>P. fulva</i>. Insects were subsequently fed with sterile minimal diet and their frass collected
               for caffeine quantification. This demonstrated that re-inoculation with 
               <i>P. fulva</i> reinstated the ability of 
               <i>H. hampei</i> to degrade caffeine (
               <a href="#f3">Fig. 3c</a>, 
               <a href="#t1">Table 1</a>), confirming the importance of this member of the gut microbiota in the caffeine-detoxification
               process. To test whether 
               <i>P. fulva</i> alone could aid 
               <i>H. hampei</i> in the recovery of its reproductive fitness, a third set of insects treated with
               antibiotic was inoculated with 
               <i>P. fulva</i> and used for the fitness assessment. However, insects did not recover their ability
               to reproduce after inoculation. A process as complex as reproduction may require a
               proper nutritional state of the insect that relies in part on the metabolic activity
               of its associated microbes or may require the presence of other bacterial endosymbionts
               known to influence reproductive health
               <a href="#b34">34</a>. Determining the minimal set of microorganisms to reinstate reproductive fitness
               as well as comparison of the effect of coffee species with varying caffeine concentrations
               on the microbial communities associated with 
               <i>H. hampei</i> are important topics for further research.
            </p>
            <p>Here we demonstrate that 
               <i>H. hampei</i>, the most devastating coffee pest in coffee-producing countries, possesses a common
               core of gut microbiota that is shared among individuals from different geographic
               locations, and that many of these organisms can subsist on caffeine as their sole
               source of carbon and nitrogen. We also demonstrate that the 
               <i>H. hampei</i> microbiota are responsible for caffeine degradation in the insect's digestive tract.
               Microbial communities play important roles in all ecosystems, from participating in
               the cycling of nutrients in soil to shaping the immune system in humans
               <a href="#b35">35</a>
               <a href="#b36">36</a>
               <a href="#b37">37</a>. Our demonstration of the role of the gut microbiota in the detoxification of caffeine
               in 
               <i>H. hampei</i> has important implications for understanding the metabolism and ecology of this important
               coffee pest. It also provides novel avenues to develop biocontrol strategies, where
               the microbiome represents a novel target because of the role of associated microorganisms
               in the enhanced ability of the host to survive and dominate a hostile environment.
            </p>
         </div>
         <div class="methods" title="methods">
            <div class="title" tagx="title" title="title">Methods</div>
            <div class="insectspecimens" title="sec">
               <div class="title" tagx="title" title="title">Insect specimens</div>
               <p>
                  <i>H. hampei</i> specimens (
                  <i>n</i>=68; 
                  <a href="#S1">Supplementary Data 1</a>) were obtained from Kenya (Kiambu), Indonesia (East Java), India (Karnataka), Puerto
                  Rico (Adjuntas), Hawaii (Kona), Guatemala (Coatepeque) and Mexico (Chiapas; 
                  <a href="#S1">Supplementary Data 1</a>). Insects from India were collected from 
                  <i>C. arabica</i> and from a robusta hybrid (
                  <i>C. congensis</i> × 
                  <i>C. canephora</i>). 
                  <i>H. obscurus, H. crudiae</i> and 
                  <i>S. maurus</i> were obtained from 
                  <i>Cecropia</i> leaf stems in Chiapas, Mexico. For the biogeography study and comparison purposes,
                  coffee berry borers reared on artificial diet
                  <a href="#b38">38</a> (laboratory insects) were also included. At all collection sites, individual insects
                  were placed alive in tubes containing RNAlater (Ambion, Grand Island, NY, USA) and
                  shipped to our laboratory for analyses.
               </p>
            </div>
            <div class="dnaextraction,pcramplificationandhigh-throughputsequencing" title="sec">
               <div class="title" tagx="title" title="title">DNA extraction, PCR amplification and high-throughput sequencing</div>
               <p>For DNA extraction, individual specimens were removed from RNAlater, and were surface-sterilized
                  by passing the beetles through four sequential washing steps in microcentrifuge tubes
                  containing 500 μl of diethylpyrocarbonate (DEPC)-treated water, commercial bleach,
                  absolute ethanol or DEPC–water, respectively. At each step, the sample was vigorously
                  shaken for 
                  <i>ca.</i> 30 s. A Biomasher (Investigen, Hercules, CA) column was placed in a microcentrifuge
                  tube, and 180 μl of ATL buffer from the QIAmp DNA micro kit (Qiagen, Grand Island,
                  NY, USA) added to the column. Individual insects were placed on the column and ground
                  with the biomasher pestle. The biomasher column was kept in place, and 5 μl of proteinase
                  K (Ambion) was added to the column, mixed by tapping and incubated at 56 °C for 30 min.
                  The sample was centrifuged at 16,000
                  <i>g</i> for 1 min; the column and biomasher were discarded afterwards. The flow-through was
                  incubated at 56 °C for an additional 30 min. Lysozyme buffer (180 μl; 20 mM Tris-HCl
                  (pH 8.0), 2 mM EDTA, 1.2% Triton X) containing 10 mg ml
                  <sup>−1</sup> of lysozyme was then added to the flow-through and the mixture incubated at 37 °C
                  for 30 min. The DNA was then precipitated and purified following the QIAmp DNA micro
                  protocol for tissues (Qiagen). The DNA concentration was determined with the use of
                  a Qubit fluorometer (Qiagen). The DNA yields ranged from 0.5 to 5 ng μl
                  <sup>−1</sup>. The bacterial/archaeal hypervariable domains V3–V4 of the 16S rRNA gene were amplified
                  from each DNA extract in triplicate reactions using the primers 515F and 806R modified
                  with adaptors and unique barcodes for Illumina sequencing
                  <a href="#b39">39</a>. All samples were then pooled and purified with the use of SPRI beads (Agencourt
                  AMPure XP, Beckman Coulter, Brea, CA, USA). Sequencing of amplicons was carried out
                  on a GAIIX Illumina Sequencer (Illumina, San Diego, CA, USA).
               </p>
            </div>
            <div class="16srrnageneanalysis" title="sec">
               <div class="title" tagx="title" title="title">16S rRNA gene analysis</div>
               <p>Reads between 80 and 100 bp were used for downstream analysis in the Quantitative
                  Insights Into Microbial Ecology pipeline
                  <a href="#b40">40</a>. The total number of reads amounted to 15,000,000 with an average number of 169,000
                  sequences per sample (range: 19,000–200,000). All reads were deposited at the European
                  Bioinformatics Institute with the accession code PRJEB8706. Sequences were clustered
                  at 97% pairwise identity using the UCLUST
                  <a href="#b41">41</a> reference-based Operational Taxonomic Unit (OTU) picking method, where the reference
                  data set was the GreenGenes 99% (GG99_12_10; ref. 
                  <a href="#b42">42</a>). A representative sequence from each OTU was aligned to the GG99 data set using
                  PyNAST (ref. 
                  <a href="#b43">43</a>). The concatenated alignment of OTUs was filtered to remove gaps and hypervariable
                  regions using the GreenGenes Lane mask
                  <a href="#b44">44</a>. A phylogenetic tree was constructed from the filtered alignment using FastTree
                  <a href="#b45">45</a>. Taxonomic assignments were carried out with the naive Bayesian algorithm
                  <a href="#b46">46</a> developed for the Ribosomal Database Project (RDP) classifier
                  <a href="#b47">47</a> using the GG99 taxonomy data set as training
                  <a href="#b42">42</a>. OTU-generated matrices were used to create network profiles that were visualized
                  with the Cytoscape software
                  <a href="#b48">48</a>. For network visualization, only the most abundant bacterial groups were retained.
                  Node sizes were adjusted according to the degree of each node, and the edge intensity
                  adjusted to its corresponding weight (number of sequences associated with a node).
                  Phylogenetic β-diversity was quantified with a weighted Unifrac distance matrix (Weighted
                  Unifrac), which was constructed at a depth of coverage of 20,000 sequences (normalized
                  coverage to the minimum number of sequences in the data set). The resulting distance
                  matrix was used to cluster samples using the UPGMA hierarchical clustering algorithm.
                  Support values for the UPGMA dendrograms were calculated using 100 jackknife permutations,
                  and nodes having at least 50% support were deemed to have high support. The samples
                  were categorized according to their geographical origins, insect species and plant
                  species from where the samples were collected. The effect of the categories on the
                  data variations was tested with Adonis (nonparametric permutation multivariate analysis
                  of variance) performing 1,000 permutations.
               </p>
            </div>
            <div class="isolationofcaffeine-degradingbacteriaandtheiridentification" title="sec">
               <div class="title" tagx="title" title="title">Isolation of caffeine-degrading bacteria and their identification</div>
               <p>To isolate bacteria with the ability to degrade caffeine, 
                  <i>H. hampei</i> specimens collected in Hawaii and Mexico were used. Ten 
                  <i>H. hampei</i> specimens per location were dissected and macerated in 100 μl of 1 × phosphate buffer
                  and further diluted at a proportion of 1:10 in 1 × phosphate buffer. Fifty microlitres
                  of the diluted gut solution were plated on agar plates containing the following: (1)
                  mineral media (9.5 mM KH
                  <sub>2</sub>PO
                  <sub>4</sub>, 4.8 mM MgSO
                  <sub>4</sub>, 0.1 mM CaCl
                  <sub>2</sub>, 0.8 mM Na
                  <sub>2</sub>HPO
                  <sub>4</sub> and 20 g l
                  <sup>−1</sup> bacto agar), with a gradient of caffeine ranging from 5 to 12.8 mM (1 to 2.5 g l
                  <sup>−1</sup>); (2) mineral media, 5 to 12.5 mM gradient of caffeine and 2 mM of glucose; or (3)
                  mineral media, 5 to 12.5 mM caffeine gradient and 9 mM NH
                  <sub>4</sub>Cl. The inoculated plates were incubated at 30 °C for 1 week. The growth of bacteria
                  was monitored every day to follow the formation of morphologically distinct colonies
                  that were picked and transferred to fresh minimal media agar plates containing 7.7 mM
                  (1.5 g l
                  <sup>−1</sup>) caffeine. Isolated colonies were transferred to assay tubes containing 5 ml of tryptic
                  soy broth and incubated at 30 °C for 72 h to produce biomass for DNA extraction, and
                  to prepare glycerol stocks that were placed at −80 °C for storage. The bacterial DNA
                  was extracted with the DNeasy Blood and Tissue Kit (Qiagen) following the protocol
                  for Gram-positive bacteria, and used for the amplification of the complete 16S rRNA
                  genes using the primers 27F (5′-GTTTGATCCTGGCTCAG-3′) and 1492R (5′-GGTTACCTTGTTACGACTT-3′).
                  The PCR reactions were prepared as follows: 10 ng of DNA as template, 5 μl of 10 ×
                  
                  <i>ExTaq</i> Buffer, 200 μM of dNTPs, 2.5 U ExTaq DNA polymerase (Takara Mirus Bio Inc., WI, USA),
                  300 ng μl
                  <sup>−1</sup> of BSA, 10 μM of each primer and DEPC-treated water to a final volume of 25 μl. The
                  PCR conditions were 98 °C for 3 min, then 30 cycles of 98 °C for 10 s, followed by
                  57 °C for 30 s, 72 °C for 30 s and a final extension step of 72 °C for 2 min. The
                  PCR products were then purified with the MinElute PCR purification kit (Qiagen) and
                  bi-directionally sequenced with the same primers. The isolates were identified by
                  phylogenetic analysis as follows. Sequences were visualized using 4Peaks and edited
                  with the Seaview software
                  <a href="#b49">49</a>. The obtained 16S rRNA sequences were aligned using Clustal X (ref. 
                  <a href="#b50">50</a>) with corresponding reference sequences from NCBI, and phylogenetic analysis was
                  carried out with maximum parsimony criteria in PAUP
                  <a href="#b51">51</a>. Heuristic tree searches were performed using a tree bisection reconnection model
                  and a branch-swapping algorithm with 100 random stepwise swaps. One hundred trees
                  were calculated for each pseudoreplicate. A rescaled consistency index, derived from
                  trees and obtained by unweighted analysis, was used to generate an 
                  <i>a posteriori</i> weighted data set. The same heuristic search conditions as used for the unweighted
                  data were used to analyse the weighted data set. Branch support was obtained with
                  100 bootstrap replicates. 
                  <i>Desulfurobacterium thermolitotrophum</i> (NR_025270) was used as outgroup in each phylogenetic reconstruction.
               </p>
            </div>
            <div class="screeningofthendmageneinisolatesandh.hampei" title="sec">
               <div class="title" tagx="title" title="title">Screening of the 
                  <i>ndmA</i> gene in isolates and 
                  <i>H. hampei</i>
               </div>
               <p>For the screening of the 
                  <i>ndmA</i> gene, the DNA from each isolate was used as template for PCR amplification with the
                  primers A-degF1 and cdm-Rev1
                  <a href="#b23">23</a>. The PCR reactions were prepared as follows: 50 ng of DNA as template, 5 μl of 10
                  × 
                  <i>ExTaq</i> Buffer, 200 μM of dNTPs, 2.5 U ExTaq DNA polymerase (Takara Mirus Bio Inc., WI, USA),
                  300 ng μl
                  <sup>−1</sup> of BSA, 10 μM of each primer and DEPC-treated water to a final volume of 25 μl. The
                  temperature profiles were as reported in ref. 
                  <a href="#b23">23</a> for the same set of primers. Positive reactions were defined by the presence of a
                  1,000-bp band. Obtained PCR bands were purified and sequenced with the same set of
                  primers. Obtained DNA sequences were aligned, and a second set of primers designed
                  for qPCR (CBBcdmF, 5′-TGGCATCCCGTWTGTACYGT-3′; CBBcdmR, 5′-CTTGKATAACRATTCGCAACC-3′)
                  generating an ∼400-bp product. RNA was extracted from a pool of five insects collected
                  in Mexico using the ISOLATE II RNA micro kit (Bioline, MA, USA) and complementary
                  DNA prepared using 100 ng of total RNA. Total RNA, 300 ng of random hexamer primers
                  and 20 nmol of each dNTP were combined to a total volume of 11 μl and the mixture
                  incubated at 65 °C for 5 min. Then, 5 μl of 5 × first strand buffer was added to the
                  mixture, together with 20 U of SUPERase-ln (Invitrogen), and incubated at 25 °C for
                  2 min. After incubating, 200 U of SuperScript III reverse transcriptase (Invitrogen)
                  was added to the mixture and incubated at 50 °C for 50 min. The reaction was inactivated
                  at 70 °C for 10 min. cDNA was then used as a template for qPCR with the primers CBBcdmF
                  and CBBcdmR. Amplification reactions were carried out in a total volume of 25 μl,
                  which consisted of 1 μl of a diluted (1:10) cDNA template, 12.5 μl of 2 × iQ SYBR
                  green super mix (Bio-Rad), 0.4 μM of each primer and 9.5 μl of water. The PCR conditions
                  were 95 °C for 3 min, then 45 cycles of 95 °C for 10 s, followed by 57 °C for 30 s
                  with the iQ system (Bio-Rad). qPCR products from each gut region were pooled and purified
                  using the QIAquick PCR purification kit (Qiagen) and cloned into competent 
                  <i>E. coli</i> JM109 cells (Promega) using the pGEM-T Easy Vector Kit (Promega) according to the
                  manufacturer's instructions. Transformants were incubated in modified LB media (0.4%
                  glycerol and 1.7 mM KH
                  <sub>2</sub>PO
                  <sub>4</sub> and 7.2 mM K
                  <sub>2</sub>HPO
                  <sub>4</sub>) at 37 °C and 300 r.p.m. for 19 h. Plasmids were extracted using the SeqPrep 96 HP
                  Plasmid Prep Kit and sequenced with an ABI377 sequencer (Applied Biosystems, Grand
                  Island, NY, USA) using M13 universal forward and reverse sequencing primers. Sequences
                  were visualized using 4Peaks and edited with the Seaview software
                  <a href="#b49">49</a>. DNA sequences from both isolates and 
                  <i>H. hampei</i> were aligned, trimmed and translated in all frames using the CLC sequence Viewer
                  6 (CLC Bio, Cambridge, MA, USA). Translated NdmA sequences were aligned using Clustal
                  X (ref. 
                  <a href="#b50">50</a>) with related protein reference sequences from NCBI and phylogenetic analysis was
                  carried out with maximum parsimony criteria in PAUP as described before. The ammonia
                  monooxygenase protein sequence of 
                  <i>Mycobacterium rhodesiae</i> (AEV73531) was used as an outgroup for the phylogenetic reconstruction.
               </p>
            </div>
            <div class="o2measurementsusingmicroelectrodes" title="sec">
               <div class="title" tagx="title" title="title">O
                  <sub>2</sub> measurements using microelectrodes
               </div>
               <p>Oxygen gradients in the gut of 
                  <i>H. hampei</i> were measured as described before
                  <a href="#b32">32</a>. Briefly, insects were dissected keeping the head of the insects attached to the
                  gut to avoid releasing the gut content and O
                  <sub>2</sub> profiles measured at the anterior midgut. Clark-type oxygen microelectrodes (OX-25,
                  Unisense) were used for the measurement of O
                  <sub>2</sub> concentration. Before use, the electrodes were polarized overnight and calibrated
                  in water saturated with air, as well as in an anoxic solution consisting of 0.1 M
                  sodium hydroxide and 0.1 M sodium ascorbate. Calibration was carried out before each
                  experiment. The current was measured with a Unisense microsensor multimeter and recorded
                  using the SensorTracePRO software (Unisense). Before microelectrode measurements 10 ml
                  of low-melting-point agarose consisting of 2% agarose in insect Ringer's solution
                  was cast into a microchamber. A freshly dissected gut was placed on this layer of
                  agarose, fully extended and immediately covered with a second layer of molten 1% agarose
                  at 30 °C. Microelectrodes were positioned using a motorized micromanipulator (MXU2,
                  Pyro Science). Measurements were performed radially starting at the surface of the
                  gut wall (0 μm) through the beetle gut until the tip completely penetrated the whole
                  tissue. The progress of the tip was followed with a digital microscope (44032 Celestron)
                  connected to a computer. All measurements were carried out at room temperature.
               </p>
            </div>
            <div class="diet-transittimeinh.hampei" title="sec">
               <div class="title" tagx="title" title="title">Diet-transit time in 
                  <i>H. hampei</i>
               </div>
               <p>A modified diet, here referred to as minimal diet, was formulated based on the artificial
                  diet used for rearing 
                  <i>H. hampei</i> (laboratory insects)
                  <a href="#b38">38</a>, but containing only 50-g twice-ground green coffee beans, 1 g Wesson salts, 0.5 g
                  nipagin, 0.4 g benzoic acid, 1 ml 37% formaldehyde, 1.5 g of gellan gum as a solidifying
                  agent and 375 ml water. The diet was modified to reduce protein interaction with antibiotics.
                  A population of laboratory insects maintained in the regular artificial diet was transferred
                  to the minimal diet and allowed to feed for 3 weeks. A subsample of the minimal diet-fed
                  laboratory insects was used for all of the experimental procedures. Four 10-ml vials
                  containing 2 g of minimal diet and fluorescent spheres (Fluospheres, Invitrogen) at
                  a concentration of 1,800 particles per mg of diet were inoculated with 20 laboratory
                  insects. Each vial was sampled at 1, 2, 3, 4 and 24 h, and a total of four 
                  <i>H. hampei</i> specimens selected per sampling time for gut extraction
                  <a href="#b52">52</a>. After dissection, each gut was placed on a cover slide and covered with 2 μl of
                  1 × phosphate buffer. The dissected guts were then screened using a microscope at
                  × 40 magnification under ultraviolet light for the detection and counting of the fluorescent
                  spheres. Considering a homogeneous distribution of particles in the diet, the number
                  of counted fluorescent spheres per sampling point was transformed into mg of ingested
                  diet over the time. The counts were fitted to a model in the R software environment
                  <a href="#b53">53</a> using a nonlinear least squares regression to calculate the feeding rates of 
                  <i>H. hampei</i>. The time at which the fluorescent spheres were localized near the anal opening was
                  considered to be the total transit time of the diet; after this period, dropping activity
                  (defecation) and release of the fluospheres was expected and confirmed by placing
                  4-h-fed 
                  <i>H. hampei</i> into a clean PCR tube for 2 h. The insects were then removed and frass-homogenized
                  with water and screened with ultraviolet-microscopy for the detection of the fluospheres.
               </p>
            </div>
            <div class="antibiotictreatmentandreinfection" title="sec">
               <div class="title" tagx="title" title="title">Antibiotic treatment and reinfection</div>
               <p>Twenty laboratory specimens of 
                  <i>H. hampei</i> were transferred into five sets of 10-ml triplicated vials containing 2 ml of minimal
                  diet supplemented with a mixture of 50 μg ml
                  <sup>−1</sup> tetracycline, 200 μg ml
                  <sup>−1</sup> rifampicin and 100 μg ml
                  <sup>−1</sup> streptomycin; one set was prepared for direct detection of caffeine and the second
                  for the reinfection experiments. The same number of insects was transferred to control
                  vials containing 2 ml of minimal diet (no antibiotics). The insects were transferred
                  to freshly prepared sterile diet every week during a 4-week period. After the fourth
                  week, 10 insects from each vial were aseptically transferred to new 10-ml vials containing
                  sterile minimal diet with fluorescent spheres (1,800 particles mg
                  <sup>−1</sup>). Insects were allowed to feed for 4 h (time required to reach gut content equilibrium)
                  and then transferred to a clean flat-bottom 96-well plate for frass collection. After
                  4 h the insects were removed from the wells and the frass collected by washing the
                  wells with 50 μl of water. The samples were then stored at −20 °C until further analysis.
                  For the reinfection experiments, isolate iMX49 (identified as 
                  <i>P. fulva</i>) was propagated in tryptic soy broth at 30 °C and 100 r.p.m. for 24 h. The resulting
                  biomass was centrifuged and the pellet washed and stored in 5 ml of 1 × phosphate
                  buffer. The number of CFUs corresponding to this bacterial solution were determined
                  by plating a 1:100 dilution of the suspension on tryptic soy agar plates and incubated
                  for 24 h at 30 °C. New minimal diet was prepared and supplemented with the iMX49 bacterial
                  solution at rate of 10,000 CFU ml
                  <sup>−1</sup>, and left to dry for 48 h at room temperature. Antibiotic-treated insects (after
                  their fourth week) were transferred to the bacteria-inoculated diet and were allowed
                  to feed. After 1 week, the frass from the reinfected insects was collected in the
                  same way as for the control samples. Since the amount of frass was too small to be
                  weighed accurately, three 10-μl aliquots (per treatment) of the frass solution were
                  visualized under the microscope under ultraviolet illumination to count the fluospheres.
                  The number of fluospheres was then used to transform quantities to milligram of diet
                  for normalization purposes. The obtained samples from each treatment were then analysed
                  using FTIR and GC–MS to determine their corresponding levels of caffeine as described
                  below.
               </p>
            </div>
            <div class="detectionofcaffeineinfrasssamples" title="sec">
               <div class="title" tagx="title" title="title">Detection of caffeine in frass samples</div>
               <p>To determine the concentration of caffeine in the frass samples of the antibiotic-treated-,
                  antibiotic-infected- and control beetles, two strategies were followed: measurement
                  of caffeine in raw frass with FTIR and measurement of extracted caffeine from frass
                  samples with GC–MS.
               </p>
               <p>
                  <i>Measurement of caffeine in raw frass using FTIR:</i> Obtained frass solutions were first deposited and air-dried for 5 min on a 1-mm-thick
                  zinc-selenide crystal and then infrared absorption spectra of the samples collected
                  with a transmission FTIR
                  <a href="#b22">22</a> microscope (Nic-Plan IR Microscope with Nicolet Magna 760 bench, Thermo Fisher Scientific
                  Inc.) with a 100-μm aperture. A globar thermal source, a MCT detector and a KBr beamsplitter
                  were used in all the experiments. Each spectrum was averaged by 128 scans at the 4-cm
                  <sup>−1</sup> spectral resolution. The whole area of each sample was raster-scanned and the spectra
                  were averaged for each sample. Known amounts of pure caffeine (0.03, 0.12, 0.2 and
                  0.3 μg of caffeine; Sigma-Aldrich, St Louis, MO, USA) were also deposited on the same
                  crystal and their corresponding signals used as the standards for the relative quantification
                  of caffeine in the frass samples (
                  <a href="#S1">Supplementary Fig. 2</a>). The major infrared absorption peaks at ∼1,655 and ∼1,705 cm
                  <sup>−1</sup>, related to the stretching vibrations of carbon–nitrogen double bond and the carbonyl
                  groups, respectively, are used to identify the presence of caffeine and evaluate the
                  caffeine concentration in different frass samples. The two Lorentzian peaks were deconvolved
                  from each obtained absorption spectrum, and the peak area data were used to compare
                  with the areas obtained from the standards. The caffeine content of the frass samples
                  was normalized by the amount of frass deposited in the zinc-selenide crystal per sample.
               </p>
               <p>
                  <i>Caffeine extraction from frass and GC–MS:</i> Water-solubilized frass samples were nitrogen-dried and then derivatized
                  <a href="#b54">54</a>. Briefly, 10 μl of a solution containing 40 mg ml
                  <sup>−1</sup> of 98% methoxyamine hydrochloride in pyridine was added to each dry sample followed
                  by 90 min of shaking at 30 °C. A mixture of internal retention index markers was prepared
                  using fatty acid methyl esters (FAMEs) of C8, C9, C10, C12, C14, C16, C18, C20, C22,
                  C24, C26, C28 and C30 linear chain length, dissolved in chloroform at a concentration
                  of 0.8 mg ml
                  <sup>−1</sup> (C8–C16) and 0.4 mg ml
                  <sup>−1</sup> (C18–C30). The FAME mixture (20 μl) was added to 1 ml of N-Methyl-N-(trimethylsilyl)-trifluoroacetamide
                  containing 1% of Trimethylclorosilane. Ninety microlitres of N-Methyl-N-(trimethylsilyl)-trifluoroacetamide
                  spiked with the FAME mixture was then added to each extract followed by mixing for
                  30 min at 37 °C. Pure caffeine (Sigma-Aldrich) was used to generate a calibration
                  curve with levels at 0.25, 0.5, 1, 2.5, 5, 10 and 20 μg ml
                  <sup>−1</sup> in 50% methanol. One hundred microlitres of each level was dried and derivatized
                  using the same procedure described for the samples. Derivatized samples were manipulated
                  using a Gerstel automatic liner exchange MPS system (Gerstel, Muehlheim, Germany)
                  controlled with the Maestro software v.1.4.25.8 to inject 2 μl of sample into a Gerstel
                  CIS cooled injection system. The injector was operated in the splitless mode, and
                  the split vent was opened after 25 s. Samples were injected into the inlet initially
                  at 60 °C, which was then ramped to 270 °C in 12 °C s
                  <sup>−1</sup> increments and held for 3 min. Sample separation was carried out on a 30-m-long,
                  0.25-mm ID Rtx-5Sil MS column (Restek, Bellefonte, PA), 0.25 mm 5% diphenyl film with
                  a 10-m integrated guard column using an Agilent 7890 gas chromatograph (Agilent Technologies,
                  Santa Clara, CA), controlled with the Agilent GC–MS MassHunter Acquisition software
                  v.B07.00 SP1 with a constant flow rate of 1 ml min
                  <sup>−1</sup>. Initial oven temperature was 60 °C with the following gradient applied: held at
                  60 °C for 1 min; ramp at 10 °C min
                  <sup>−1</sup> to 310 °C, held for 10 min. Mass spectrometry was performed with an Agilent 5977
                  single quadrupole MS with 250 °C transfer line temperature, electron ionization at
                  70 eV, an ion source temperature of 230 °C and a quadrupole temperature of 150 °C.
                  Mass spectra were acquired in a selected ion-monitoring mode at a scan rate of 4.4
                  spectra s
                  <sup>−1</sup>. Selected ions for caffeine were 55, 82, 109 and 194. A quality-control mix was run
                  in triplicate before and after the sample set to confirm GC and MS signal reproducibility.
                  Caffeine levels were quantitated in acquired data using Agilent MassHunter Quantitative
                  Analysis v.B.07.00.
               </p>
               <p>For FTIR and GC–MS analyses, significant differences between treatments (control,
                  antibiotic-treated and antibiotic-treated/re-inoculated) were determined using one-way
                  analysis of variance and least significant differences using the R software environment
                  <a href="#b53">53</a>.
               </p>
            </div>
            <div class="antibiotictreatmenteffectonh.hampeireproduction" title="sec">
               <div class="title" tagx="title" title="title">Antibiotic treatment effect on 
                  <i>H. hampei</i> reproduction
               </div>
               <p>One hundred females recently emerged as adults were placed together with 20 males
                  in vials containing regular diet and were incubated for 14 days to maximize mating.
                  After the initial mating incubation, individual females were transferred to a vial
                  containing either regular diet (control) or diet supplemented with antibiotics (as
                  described above) and incubated for 44 days. A third set of mated females was treated
                  with antibiotics for 3 weeks, transferred to diet with 
                  <i>P. fulva</i> and also incubated for 44 days. Fifteen vials from each treatment were destructively
                  sampled on each sampling time (days 14, 24, 34 and 44) and screened for eggs, larvae,
                  pupae and adults. A logistic regression was performed on the sum of all stages in
                  each vial. Then, a model was fit with just the main effects (DIET—control, antibiotics,
                  infection; and SAMPLE—sampling time), where overdispersion was included. Both main
                  effects were significant, that is, there is a significant effect of DIET when compared
                  with control, and there is a significant effect of sample (later samples had higher
                  counts).
               </p>
            </div>
         </div>
         <div class="additionalinformation" title="sec">
            <div class="title" tagx="title" title="title">Additional information</div>
            <p>
               <b>Accession codes:</b> 16S rRNA sequences from the biogeography analysis have been deposited in the European
               Bioinformatics Institute (EBI) database with the accession code PRJEB8706. The 16S
               rRNA sequences from the isolated caffeine-subsisting bacteria and the 
               <i>ndmA</i> sequences have been deposited in the GenBank database with accession codes 
               <a href="KF913733">KF913733</a> to 
               <a href="KF913844">KF913844</a>, and 
               <a href="KJ159083">KJ159083</a> to 
               <a href="KJ159093">KJ159093</a>, respectively.
            </p>
            <p>
               <b>How to cite this article</b>: Ceja-Navarro, J. A. 
               <i>et al.</i> Gut microbiota mediate caffeine detoxification in the primary insect pest of coffee.
               
               <i>Nat. Commun.</i> 6:7618 doi: 10.1038/ncomms8618 (2015).
            </p>
         </div>
         <div class="supplementary-material" title="supplementary-material">
            <div class="title" tagx="title" title="title">Supplementary Material</div>
            <div class="local-data" title="local-data">
               <caption>
                  <div class="title" tagx="title" title="title">Supplementary Information</div>
                  <p>Supplementary Figures 1-5</p>
               </caption>
               <div class="media" title="media">
                  <a href="ncomms8618-s1.pdf">LINK</a>
               </div>
            </div>
            <div class="local-data" title="local-data">
               <caption>
                  <div class="title" tagx="title" title="title">Supplementary Movie 1</div>
                  <p>Video showing specimens of Hypothenemus hampei foraging and boring into defined media.</p>
               </caption>
               <div class="media" title="media">
                  <a href="ncomms8618-s2.mov">LINK</a>
               </div>
            </div>
            <div class="local-data" title="local-data">
               <caption>
                  <div class="title" tagx="title" title="title">Supplementary Data 1</div>
                  <p>Metadata of insects collected in seven major coffee producing countries. Data includes
                     Country, Location, Insects Species, Host plant (or diet), GPS coordinates.
                  </p>
               </caption>
               <div class="media" title="media">
                  <a href="ncomms8618-s3.xls">LINK</a>
               </div>
            </div>
         </div>
      </div>
      <div class="back" title="back">
         <div class="ack" title="ack">
            <p>This work was funded by the United States Department of Agriculture Agricultural Research
               Service, and part of this work was funded by the Laboratory Directed Research and
               Development programme at the Lawrence Berkeley National Laboratory under United States
               Department of Energy contract number DE-AC02-05CH11231. J.A.C.-N. was supported in
               part by a grant from ‘
               <i>Consejo Nacional de Ciencia y Tecnología</i>' (CONACyT, Mexico) and the Laboratory Directed Research and Development Program of
               Lawrence Berkeley National Laboratory under US Department of Energy Contract DE-AC02-05CH1121231.
               We are very grateful to S. Wiryadiputra (Indonesia), P.K. Vinod Kumar (India), A.
               Ramirez (Puerto Rico), PePe Miranda (Hawaii) and J. Jaramillo (Kenya) for providing
               specimens used in this study. We thank Susan Lynch for comments on this manuscript,
               Hoi-Ying Holman for advice on FTIR analyses, and Clark Santee for laboratory assistance.
            </p>
         </div>
         <div class="references">References</div>
         <div tag="ref-list">
            <ul>
               <li tag="ref">
                  <a name="b1"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Davis</span>
                        <span class="given-names" tagx="given-names" title="given-names">A. P.</span>
                     </span>, 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Govaerts</span>
                        <span class="given-names" tagx="given-names" title="given-names">R.</span>
                     </span>, 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Bridson</span>
                        <span class="given-names" tagx="given-names" title="given-names">D. M.</span>
                     </span> &amp; 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Stoffelen</span>
                        <span class="given-names" tagx="given-names" title="given-names">P.</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">An annotated taxonomic conspectus of the genus 
                        <i>Coffea</i> (Rubiaceae)
                     </span>. 
                     <span class="source" tagx="source" title="source">Bot. J. Linn. Soc.</span>
                     <span class="volume" tagx="volume" title="volume">152</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">465</span>–
                     <span class="lpage" tagx="lpage" title="lpage">512</span> (
                     <span class="year" tagx="year" title="year">2006</span>).
                  </span>
               </li>
               <li tag="ref">
                  <a name="b2"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Davis</span>
                        <span class="given-names" tagx="given-names" title="given-names">A. P.</span>
                     </span>, 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Tosh</span>
                        <span class="given-names" tagx="given-names" title="given-names">J.</span>
                     </span>, 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Ruch</span>
                        <span class="given-names" tagx="given-names" title="given-names">N.</span>
                     </span> &amp; 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Fay</span>
                        <span class="given-names" tagx="given-names" title="given-names">M. F.</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Growing coffee: 
                        <i>Psilanthus</i> (Rubiaceae) subsumed on the basis of molecular and morphological data; implications
                        for the size, morphology, distribution and evolutionary history of 
                        <i>Coffea</i>
                     </span>. 
                     <span class="source" tagx="source" title="source">Bot. J. Linn. Soc.</span>
                     <span class="volume" tagx="volume" title="volume">167</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">357</span>–
                     <span class="lpage" tagx="lpage" title="lpage">377</span> (
                     <span class="year" tagx="year" title="year">2011</span>).
                  </span>
               </li>
               <li tag="ref">
                  <a name="b3"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Ashihara</span>
                        <span class="given-names" tagx="given-names" title="given-names">H.</span>
                     </span> &amp; 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Crozier</span>
                        <span class="given-names" tagx="given-names" title="given-names">A.</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Caffeine: a well known but little mentioned compound in plant science</span>. 
                     <span class="source" tagx="source" title="source">Trends Plant Sci.</span>
                     <span class="volume" tagx="volume" title="volume">6</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">407</span>–
                     <span class="lpage" tagx="lpage" title="lpage">413</span> (
                     <span class="year" tagx="year" title="year">2001</span>).
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/11544129">11544129</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="b4"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Wright</span>
                        <span class="given-names" tagx="given-names" title="given-names">G. A.</span>
                     </span>
                     <i>et al.</i>
                     <span class="mixed-article-title" title="mixed-article-title">Caffeine in floral nectar enhances a pollinator's memory of reward</span>. 
                     <span class="source" tagx="source" title="source">Science</span>
                     <span class="volume" tagx="volume" title="volume">339</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">1202</span>–
                     <span class="lpage" tagx="lpage" title="lpage">1204</span> (
                     <span class="year" tagx="year" title="year">2013</span>).
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/23471406">23471406</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="b5"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Nathanson</span>
                        <span class="given-names" tagx="given-names" title="given-names">J.</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Caffeine and related methylxanthines: possible naturally occurring pesticides</span>. 
                     <span class="source" tagx="source" title="source">Science</span>
                     <span class="volume" tagx="volume" title="volume">226</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">184</span>–
                     <span class="lpage" tagx="lpage" title="lpage">187</span> (
                     <span class="year" tagx="year" title="year">1984</span>).
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/6207592">6207592</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="b6"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Uefuji</span>
                        <span class="given-names" tagx="given-names" title="given-names">H.</span>
                     </span>
                     <i>et al.</i>
                     <span class="mixed-article-title" title="mixed-article-title">Caffeine production in tobacco plants by simultaneous expression of three coffee N-methyltrasferases
                        and its potential as a pest repellant
                     </span>. 
                     <span class="source" tagx="source" title="source">Plant Mol. Biol.</span>
                     <span class="volume" tagx="volume" title="volume">59</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">221</span>–
                     <span class="lpage" tagx="lpage" title="lpage">227</span> (
                     <span class="year" tagx="year" title="year">2005</span>).
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/16247553">16247553</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="b7"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Kim</span>
                        <span class="given-names" tagx="given-names" title="given-names">Y.-S.</span>
                     </span>, 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Uefuji</span>
                        <span class="given-names" tagx="given-names" title="given-names">H.</span>
                     </span>, 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Ogita</span>
                        <span class="given-names" tagx="given-names" title="given-names">S.</span>
                     </span> &amp; 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Sano</span>
                        <span class="given-names" tagx="given-names" title="given-names">H.</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Transgenic tobacco plants producing caffeine: a potential new strategy for insect
                        pest control
                     </span>. 
                     <span class="source" tagx="source" title="source">Transgenic Res.</span>
                     <span class="volume" tagx="volume" title="volume">15</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">667</span>–
                     <span class="lpage" tagx="lpage" title="lpage">672</span> (
                     <span class="year" tagx="year" title="year">2006</span>).
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/17091387">17091387</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="b8"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Detzel</span>
                        <span class="given-names" tagx="given-names" title="given-names">A.</span>
                     </span> &amp; 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Wink</span>
                        <span class="given-names" tagx="given-names" title="given-names">M.</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Attraction, deterrence or intoxication of bees (
                        <i>Apis mellifera</i>) by plant allelochemicals
                     </span>. 
                     <span class="source" tagx="source" title="source">Chemoecology</span>
                     <span class="volume" tagx="volume" title="volume">4</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">8</span>–
                     <span class="lpage" tagx="lpage" title="lpage">18</span> (
                     <span class="year" tagx="year" title="year">1993</span>).
                  </span>
               </li>
               <li tag="ref">
                  <a name="b9"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Calvo</span>
                        <span class="given-names" tagx="given-names" title="given-names">I. A.</span>
                     </span>
                     <i>et al.</i>
                     <span class="mixed-article-title" title="mixed-article-title">Genome-wide screen of genes required for caffeine tolerance in fission yeast</span>. 
                     <span class="source" tagx="source" title="source">PLoS ONE</span>
                     <span class="volume" tagx="volume" title="volume">4</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">e6619</span> (
                     <span class="year" tagx="year" title="year">2009</span>).
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/19672306">19672306</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="b10"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Abdelkader</span>
                        <span class="given-names" tagx="given-names" title="given-names">T. S.</span>
                     </span>
                     <i>et al.</i>
                     <span class="mixed-article-title" title="mixed-article-title">Exposure time to caffeine affects heartbeat and cell damage-related gene expression
                        of zebrafish 
                        <i>Danio rerio</i> embryos at early developmental stages
                     </span>. 
                     <span class="source" tagx="source" title="source">J. Appl. Toxicol.</span>
                     <span class="volume" tagx="volume" title="volume">33</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">1277</span>–
                     <span class="lpage" tagx="lpage" title="lpage">1283</span> (
                     <span class="year" tagx="year" title="year">2012</span>).
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/22886764">22886764</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="b11"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Hollingsworth</span>
                        <span class="given-names" tagx="given-names" title="given-names">R. G.</span>
                     </span>, 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Armstrong</span>
                        <span class="given-names" tagx="given-names" title="given-names">J. W.</span>
                     </span> &amp; 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Campbell</span>
                        <span class="given-names" tagx="given-names" title="given-names">E.</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Caffeine as a repellent for slugs and snails</span>. 
                     <span class="source" tagx="source" title="source">Nature</span>
                     <span class="volume" tagx="volume" title="volume">417</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">915</span>–
                     <span class="lpage" tagx="lpage" title="lpage">916</span> (
                     <span class="year" tagx="year" title="year">2002</span>).
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/12087394">12087394</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="b12"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Ashihara</span>
                        <span class="given-names" tagx="given-names" title="given-names">H.</span>
                     </span>, 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Kato</span>
                        <span class="given-names" tagx="given-names" title="given-names">M.</span>
                     </span> &amp; 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Crozier</span>
                        <span class="given-names" tagx="given-names" title="given-names">A.</span>
                     </span> in 
                     <span class="source" tagx="source" title="source">Methylxanthines (Fredholm, B. B.)</span>
                     <span class="volume" tagx="volume" title="volume">200</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">11</span>–
                     <span class="lpage" tagx="lpage" title="lpage">31</span>Springer (
                     <span class="year" tagx="year" title="year">2011</span>).
                  </span>
               </li>
               <li tag="ref">
                  <a name="b13"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Le Pelley</span>
                        <span class="given-names" tagx="given-names" title="given-names">R. H.</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Coffee insects</span>. 
                     <span class="source" tagx="source" title="source">Annu. Rev. Entomol.</span>
                     <span class="volume" tagx="volume" title="volume">18</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">121</span>–
                     <span class="lpage" tagx="lpage" title="lpage">142</span> (
                     <span class="year" tagx="year" title="year">1973</span>).
                  </span>
               </li>
               <li tag="ref">
                  <a name="b14"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Hagedorn</span>
                        <span class="given-names" tagx="given-names" title="given-names">M.</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Wieder ein neuer kaffeeschädlinge</span>. 
                     <span class="source" tagx="source" title="source">Entomol. Blätter</span>
                     <span class="volume" tagx="volume" title="volume">6</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">1</span>–
                     <span class="lpage" tagx="lpage" title="lpage">14</span> (
                     <span class="year" tagx="year" title="year">1910</span>).
                  </span>
               </li>
               <li tag="ref">
                  <a name="b15"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Neiva</span>
                        <span class="given-names" tagx="given-names" title="given-names">A.</span>
                     </span> Os trabalhos da Commissão de Estudo e Debellação da Praga Cafeeira desde seu inicio.
                     
                     <i>Commissão de Estudo e Debellação da Praga Cafeeira</i>, Publicação No. 21, 27 (
                     <span class="year" tagx="year" title="year">1928</span>).
                  </span>
               </li>
               <li tag="ref">
                  <a name="b16"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Vega</span>
                        <span class="given-names" tagx="given-names" title="given-names">F. E.</span>
                     </span>, 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Davis</span>
                        <span class="given-names" tagx="given-names" title="given-names">A. P.</span>
                     </span> &amp; 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Jaramillo</span>
                        <span class="given-names" tagx="given-names" title="given-names">J.</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">From forest to plantation? Obscure articles reveal alternative host plants for the
                        coffee berry borer, 
                        <i>Hypothenemus hampei</i> (Coleoptera: Curculionidae)
                     </span>. 
                     <span class="source" tagx="source" title="source">Biol. J. Linn. Soc.</span>
                     <span class="volume" tagx="volume" title="volume">107</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">86</span>–
                     <span class="lpage" tagx="lpage" title="lpage">94</span> (
                     <span class="year" tagx="year" title="year">2012</span>).
                  </span>
               </li>
               <li tag="ref">
                  <a name="b17"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Oliveira</span>
                        <span class="given-names" tagx="given-names" title="given-names">C. M.</span>
                     </span>, 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Auad</span>
                        <span class="given-names" tagx="given-names" title="given-names">A. M.</span>
                     </span>, 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Mendes</span>
                        <span class="given-names" tagx="given-names" title="given-names">S. M.</span>
                     </span> &amp; 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Frizzas</span>
                        <span class="given-names" tagx="given-names" title="given-names">M. R.</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Economic impact of exotic insect pests in Brazilian agriculture</span>. 
                     <span class="source" tagx="source" title="source">J. Appl. Entomol.</span>
                     <span class="volume" tagx="volume" title="volume">137</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">1</span>–
                     <span class="lpage" tagx="lpage" title="lpage">15</span> (
                     <span class="year" tagx="year" title="year">2013</span>).
                  </span>
               </li>
               <li tag="ref">
                  <a name="b18"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Acuña</span>
                        <span class="given-names" tagx="given-names" title="given-names">R.</span>
                     </span>
                     <i>et al.</i>
                     <span class="mixed-article-title" title="mixed-article-title">Adaptive horizontal transfer of a bacterial gene to an invasive insect pest of coffee</span>. 
                     <span class="source" tagx="source" title="source">Proc. Natl Acad. Sci. USA</span>
                     <span class="volume" tagx="volume" title="volume">109</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">4197</span>–
                     <span class="lpage" tagx="lpage" title="lpage">4202</span> (
                     <span class="year" tagx="year" title="year">2012</span>).
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/22371593">22371593</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="b19"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Guerreiro Filho</span>
                        <span class="given-names" tagx="given-names" title="given-names">O.</span>
                     </span> &amp; 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Mazzafera</span>
                        <span class="given-names" tagx="given-names" title="given-names">P.</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Caffeine and resistance of coffee to the berry borer 
                        <i>Hypothenemus hampei</i> (Coleoptera: Scolytidae)
                     </span>. 
                     <span class="source" tagx="source" title="source">J. Agric. Food Chem.</span>
                     <span class="volume" tagx="volume" title="volume">51</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">6987</span>–
                     <span class="lpage" tagx="lpage" title="lpage">6991</span> (
                     <span class="year" tagx="year" title="year">2003</span>).
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/14611159">14611159</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="b20"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Gokulakrishnan</span>
                        <span class="given-names" tagx="given-names" title="given-names">S.</span>
                     </span>, 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Chandraraj</span>
                        <span class="given-names" tagx="given-names" title="given-names">K.</span>
                     </span> &amp; 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Gummadi</span>
                        <span class="given-names" tagx="given-names" title="given-names">S.</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Microbial and enzymatic methods for the removal of caffeine</span>. 
                     <span class="source" tagx="source" title="source">Enzyme Microb. Technol.</span>
                     <span class="volume" tagx="volume" title="volume">37</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">225</span>–
                     <span class="lpage" tagx="lpage" title="lpage">232</span> (
                     <span class="year" tagx="year" title="year">2005</span>).
                  </span>
               </li>
               <li tag="ref">
                  <a name="b21"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Khanna</span>
                        <span class="given-names" tagx="given-names" title="given-names">K. L.</span>
                     </span>
                     <i>et al.</i>
                     <span class="mixed-article-title" title="mixed-article-title">Metabolism of caffeine-3H in the rat*1, *2</span>. 
                     <span class="source" tagx="source" title="source">Toxicol. Appl. Pharmacol.</span>
                     <span class="volume" tagx="volume" title="volume">112</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">720</span>–
                     <span class="lpage" tagx="lpage" title="lpage">730</span> (
                     <span class="year" tagx="year" title="year">1972</span>).
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/4644703">4644703</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="b22"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Paradkar</span>
                        <span class="given-names" tagx="given-names" title="given-names">M. M.</span>
                     </span> &amp; 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Irudayaraj</span>
                        <span class="given-names" tagx="given-names" title="given-names">J.</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">A rapid FTIR spectroscopic method for estimation of caffeine in soft drinks and total
                        methylxanthines in tea and coffee
                     </span>. 
                     <span class="source" tagx="source" title="source">J. Food Sci.</span>
                     <span class="volume" tagx="volume" title="volume">67</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">2507</span>–
                     <span class="lpage" tagx="lpage" title="lpage">2511</span> (
                     <span class="year" tagx="year" title="year">2002</span>).
                  </span>
               </li>
               <li tag="ref">
                  <a name="b23"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Summers</span>
                        <span class="given-names" tagx="given-names" title="given-names">R. M.</span>
                     </span>
                     <i>et al.</i>
                     <span class="mixed-article-title" title="mixed-article-title">Novel, highly specific N-demethylases enable bacteria to live on caffeine and related
                        purine alkaloids
                     </span>. 
                     <span class="source" tagx="source" title="source">J. Bacteriol.</span>
                     <span class="volume" tagx="volume" title="volume">194</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">2041</span>–
                     <span class="lpage" tagx="lpage" title="lpage">2049</span> (
                     <span class="year" tagx="year" title="year">2012</span>).
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/22328667">22328667</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="b24"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Barrera</span>
                        <span class="given-names" tagx="given-names" title="given-names">J. F.</span>
                     </span>
                     <i>Encyclopedia of Entomology</i> (ed. Capinera, J. L.) 961–998 (Springer Dordrecht, The Netherlands (
                     <span class="year" tagx="year" title="year">2008</span>).
                  </span>
               </li>
               <li tag="ref">
                  <a name="b25"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Vega</span>
                        <span class="given-names" tagx="given-names" title="given-names">F. E.</span>
                     </span>, 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Posada</span>
                        <span class="given-names" tagx="given-names" title="given-names">F.</span>
                     </span> &amp; 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Infante</span>
                        <span class="given-names" tagx="given-names" title="given-names">F.</span>
                     </span> in 
                     <i>Encyclopedia of Pest Management</i> (ed. Pimentel, D.) 1–4 (Dekker, (
                     <span class="year" tagx="year" title="year">2006</span>).
                  </span>
               </li>
               <li tag="ref">
                  <a name="b26"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Ezenwa</span>
                        <span class="given-names" tagx="given-names" title="given-names">V. O.</span>
                     </span>, 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Gerardo</span>
                        <span class="given-names" tagx="given-names" title="given-names">N. M.</span>
                     </span>, 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Inouye</span>
                        <span class="given-names" tagx="given-names" title="given-names">D. W.</span>
                     </span>, 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Medina</span>
                        <span class="given-names" tagx="given-names" title="given-names">M.</span>
                     </span> &amp; 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Xavier</span>
                        <span class="given-names" tagx="given-names" title="given-names">J. B.</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Microbiology. Animal behavior and the microbiome</span>. 
                     <span class="source" tagx="source" title="source">Science</span>
                     <span class="volume" tagx="volume" title="volume">338</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">198</span>–
                     <span class="lpage" tagx="lpage" title="lpage">199</span> (
                     <span class="year" tagx="year" title="year">2012</span>).
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/23066064">23066064</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="b27"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Sharon</span>
                        <span class="given-names" tagx="given-names" title="given-names">G.</span>
                     </span>
                     <i>et al.</i>
                     <span class="mixed-article-title" title="mixed-article-title">Commensal bacteria play a role in mating preference of 
                        <i>Drosophila melanogaster</i>
                     </span>. 
                     <span class="source" tagx="source" title="source">Proc. Natl Acad. Sci. USA</span>
                     <span class="volume" tagx="volume" title="volume">107</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">20051</span>–
                     <span class="lpage" tagx="lpage" title="lpage">20056</span> (
                     <span class="year" tagx="year" title="year">2010</span>).
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/21041648">21041648</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="b28"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Lilburn</span>
                        <span class="given-names" tagx="given-names" title="given-names">T. G.</span>
                     </span>
                     <i>et al.</i>
                     <span class="mixed-article-title" title="mixed-article-title">Nitrogen fixation by symbiotic and free-living spirochetes</span>. 
                     <span class="source" tagx="source" title="source">Science</span>
                     <span class="volume" tagx="volume" title="volume">292</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">2495</span>–
                     <span class="lpage" tagx="lpage" title="lpage">2498</span> (
                     <span class="year" tagx="year" title="year">2001</span>).
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/11431569">11431569</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="b29"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Hoffmeister</span>
                        <span class="given-names" tagx="given-names" title="given-names">M.</span>
                     </span> &amp; 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Martin</span>
                        <span class="given-names" tagx="given-names" title="given-names">W.</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Interspecific evolution: microbial symbiosis, endosymbiosis and gene transfer</span>. 
                     <span class="source" tagx="source" title="source">Environ. Microbiol.</span>
                     <span class="volume" tagx="volume" title="volume">5</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">641</span>–
                     <span class="lpage" tagx="lpage" title="lpage">649</span> (
                     <span class="year" tagx="year" title="year">2003</span>).
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/12871231">12871231</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="b30"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Gauthier</span>
                        <span class="given-names" tagx="given-names" title="given-names">N.</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Multiple cryptic genetic units in 
                        <i>Hypothenemus hampei</i> (Coleoptera: Scolytinae): evidence from microsatellite and mitochondrial DNA sequence
                        data
                     </span>. 
                     <span class="source" tagx="source" title="source">Biol. J. Linn. Soc.</span>
                     <span class="volume" tagx="volume" title="volume">101</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">113</span>–
                     <span class="lpage" tagx="lpage" title="lpage">129</span> (
                     <span class="year" tagx="year" title="year">2010</span>).
                  </span>
               </li>
               <li tag="ref">
                  <a name="b31"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Waldhauser</span>
                        <span class="given-names" tagx="given-names" title="given-names">S. S. M.</span>
                     </span> &amp; 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Baumann</span>
                        <span class="given-names" tagx="given-names" title="given-names">T. W.</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Compartmentation of caffeine and related purine alkaloids depends exclusively on the
                        physical chemistry of their vacuolar complex formation with chlorogenic acids
                     </span>. 
                     <span class="source" tagx="source" title="source">Phytochemistry</span>
                     <span class="volume" tagx="volume" title="volume">42</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">985</span>–
                     <span class="lpage" tagx="lpage" title="lpage">996</span> (
                     <span class="year" tagx="year" title="year">1996</span>).
                  </span>
               </li>
               <li tag="ref">
                  <a name="b32"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Ceja-Navarro</span>
                        <span class="given-names" tagx="given-names" title="given-names">J. A.</span>
                     </span>
                     <i>et al.</i>
                     <span class="mixed-article-title" title="mixed-article-title">Compartmentalized microbial composition, oxygen gradients and nitrogen fixation in
                        the gut of 
                        <i>Odontotaenius disjunctus</i>
                     </span>. 
                     <span class="source" tagx="source" title="source">ISME J.</span>
                     <span class="volume" tagx="volume" title="volume">8</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">6</span>–
                     <span class="lpage" tagx="lpage" title="lpage">18</span> (
                     <span class="year" tagx="year" title="year">2014</span>).
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/23985746">23985746</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="b33"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Brune</span>
                        <span class="given-names" tagx="given-names" title="given-names">A.</span>
                     </span>, 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Emerson</span>
                        <span class="given-names" tagx="given-names" title="given-names">D.</span>
                     </span> &amp; 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Breznak</span>
                        <span class="given-names" tagx="given-names" title="given-names">J. A.</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">The termite gut microflora as an oxygen sink: microelectrode determination of oxygen
                        and pH gradients in guts of lower and higher termites
                     </span>. 
                     <span class="source" tagx="source" title="source">Appl. Environ. Microbiol.</span>
                     <span class="volume" tagx="volume" title="volume">61</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">2681</span>–
                     <span class="lpage" tagx="lpage" title="lpage">2687</span> (
                     <span class="year" tagx="year" title="year">1995</span>).
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/16535076">16535076</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="b34"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Vega</span>
                        <span class="given-names" tagx="given-names" title="given-names">F. E.</span>
                     </span>, 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Benavides</span>
                        <span class="given-names" tagx="given-names" title="given-names">P.</span>
                     </span>, 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Stuart</span>
                        <span class="given-names" tagx="given-names" title="given-names">J. A.</span>
                     </span> &amp; 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">O'Neill</span>
                        <span class="given-names" tagx="given-names" title="given-names">S. L.</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">
                        <i>Wolbachia</i> infection in the coffee berry borer (Coleoptera: Scolytidae)
                     </span>. 
                     <span class="source" tagx="source" title="source">Ann. Entomol. Soc. Am.</span>
                     <span class="volume" tagx="volume" title="volume">95</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">374</span>–
                     <span class="lpage" tagx="lpage" title="lpage">378</span> (
                     <span class="year" tagx="year" title="year">2002</span>).
                  </span>
               </li>
               <li tag="ref">
                  <a name="b35"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Ingham</span>
                        <span class="given-names" tagx="given-names" title="given-names">R. E.</span>
                     </span>, 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Trofymow</span>
                        <span class="given-names" tagx="given-names" title="given-names">J. A.</span>
                     </span>, 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Ingham</span>
                        <span class="given-names" tagx="given-names" title="given-names">E. R.</span>
                     </span> &amp; 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Coleman</span>
                        <span class="given-names" tagx="given-names" title="given-names">D. C.</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Interactions of bacteria, fungi, and their nematode grazers: effects on nutrient cycling
                        and plant growth
                     </span>. 
                     <span class="source" tagx="source" title="source">Ecol. Monogr.</span>
                     <span class="volume" tagx="volume" title="volume">55</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">119</span>–
                     <span class="lpage" tagx="lpage" title="lpage">140</span> (
                     <span class="year" tagx="year" title="year">1985</span>).
                  </span>
               </li>
               <li tag="ref">
                  <a name="b36"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Mendes</span>
                        <span class="given-names" tagx="given-names" title="given-names">R.</span>
                     </span>, 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Garbeva</span>
                        <span class="given-names" tagx="given-names" title="given-names">P.</span>
                     </span> &amp; 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Raaijmakers</span>
                        <span class="given-names" tagx="given-names" title="given-names">J. M.</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">The rhizosphere microbiome: significance of plant beneficial, plant pathogenic, and
                        human pathogenic microorganisms
                     </span>. 
                     <span class="source" tagx="source" title="source">FEMS Microbiol. Rev.</span>
                     <span class="volume" tagx="volume" title="volume">37</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">634</span>–
                     <span class="lpage" tagx="lpage" title="lpage">663</span> (
                     <span class="year" tagx="year" title="year">2013</span>).
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/23790204">23790204</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="b37"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Lee</span>
                        <span class="given-names" tagx="given-names" title="given-names">Y. K.</span>
                     </span> &amp; 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Mazmanian</span>
                        <span class="given-names" tagx="given-names" title="given-names">S. K.</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Has the microbiota played a critical role in the evolution of the adaptive immune
                        system?
                     </span>
                     <span class="source" tagx="source" title="source">Science</span>
                     <span class="volume" tagx="volume" title="volume">330</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">1768</span>–
                     <span class="lpage" tagx="lpage" title="lpage">1773</span> (
                     <span class="year" tagx="year" title="year">2010</span>).
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/21205662">21205662</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="b38"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Villacorta</span>
                        <span class="given-names" tagx="given-names" title="given-names">A.</span>
                     </span> &amp; 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Barrera</span>
                        <span class="given-names" tagx="given-names" title="given-names">J. F.</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Nova dieta merídica para criação de 
                        <i>Hypothenemus hampei</i> (Ferrari, 1867)
                     </span>. 
                     <span class="source" tagx="source" title="source">An. Soc. Entomol. Bras.</span>
                     <span class="volume" tagx="volume" title="volume">22</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">405</span>–
                     <span class="lpage" tagx="lpage" title="lpage">409</span> (
                     <span class="year" tagx="year" title="year">1867</span>).
                  </span>
               </li>
               <li tag="ref">
                  <a name="b39"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Caporaso</span>
                        <span class="given-names" tagx="given-names" title="given-names">J. G.</span>
                     </span>
                     <i>et al.</i>
                     <span class="mixed-article-title" title="mixed-article-title">Ultra-high-throughput microbial community analysis on the Illumina HiSeq and MiSeq
                        platforms
                     </span>. 
                     <span class="source" tagx="source" title="source">ISME J.</span>
                     <span class="volume" tagx="volume" title="volume">6</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">1621</span>–
                     <span class="lpage" tagx="lpage" title="lpage">1624</span> (
                     <span class="year" tagx="year" title="year">2012</span>).
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/22402401">22402401</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="b40"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Caporaso</span>
                        <span class="given-names" tagx="given-names" title="given-names">J. G.</span>
                     </span>
                     <i>et al.</i>
                     <span class="mixed-article-title" title="mixed-article-title">QIIME allows analysis of high-throughput community sequencing data</span>. 
                     <span class="source" tagx="source" title="source">Nat. Methods</span>
                     <span class="volume" tagx="volume" title="volume">7</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">335</span>–
                     <span class="lpage" tagx="lpage" title="lpage">336</span> (
                     <span class="year" tagx="year" title="year">2010</span>).
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/20383131">20383131</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="b41"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Edgar</span>
                        <span class="given-names" tagx="given-names" title="given-names">R. C.</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Search and clustering orders of magnitude faster than BLAST</span>. 
                     <span class="source" tagx="source" title="source">Bioinformatics.</span>
                     <span class="volume" tagx="volume" title="volume">26</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">2460</span>–
                     <span class="lpage" tagx="lpage" title="lpage">2461</span> (
                     <span class="year" tagx="year" title="year">2010</span>).
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/20709691">20709691</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="b42"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Werner</span>
                        <span class="given-names" tagx="given-names" title="given-names">J. J.</span>
                     </span>
                     <i>et al.</i>
                     <span class="mixed-article-title" title="mixed-article-title">Impact of training sets on classification of high-throughput bacterial 16s rRNA gene
                        surveys
                     </span>. 
                     <span class="source" tagx="source" title="source">ISME J.</span>
                     <span class="volume" tagx="volume" title="volume">6</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">94</span>–
                     <span class="lpage" tagx="lpage" title="lpage">103</span> (
                     <span class="year" tagx="year" title="year">2012</span>).
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/21716311">21716311</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="b43"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Caporaso</span>
                        <span class="given-names" tagx="given-names" title="given-names">J. G.</span>
                     </span>
                     <i>et al.</i>
                     <span class="mixed-article-title" title="mixed-article-title">PyNAST: a flexible tool for aligning sequences to a template alignment</span>. 
                     <span class="source" tagx="source" title="source">Bioinformatics</span>
                     <span class="volume" tagx="volume" title="volume">26</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">266</span>–
                     <span class="lpage" tagx="lpage" title="lpage">267</span> (
                     <span class="year" tagx="year" title="year">2010</span>).
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/19914921">19914921</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="b44"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">DeSantis</span>
                        <span class="given-names" tagx="given-names" title="given-names">T. Z.</span>
                     </span>
                     <i>et al.</i>
                     <span class="mixed-article-title" title="mixed-article-title">Greengenes, a chimera-checked 16S rRNA gene database and workbench compatible with
                        ARB
                     </span>. 
                     <span class="source" tagx="source" title="source">Appl. Environ. Microbiol.</span>
                     <span class="volume" tagx="volume" title="volume">72</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">5069</span>–
                     <span class="lpage" tagx="lpage" title="lpage">5072</span> (
                     <span class="year" tagx="year" title="year">2006</span>).
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/16820507">16820507</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="b45"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Price</span>
                        <span class="given-names" tagx="given-names" title="given-names">M. N.</span>
                     </span>, 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Dehal</span>
                        <span class="given-names" tagx="given-names" title="given-names">P. S.</span>
                     </span> &amp; 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Arkin</span>
                        <span class="given-names" tagx="given-names" title="given-names">A. P.</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">FastTree 2–approximately maximum-likelihood trees for large alignments</span>. 
                     <span class="source" tagx="source" title="source">PLoS ONE</span>
                     <span class="volume" tagx="volume" title="volume">5</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">e9490</span> (
                     <span class="year" tagx="year" title="year">2010</span>).
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/20224823">20224823</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="b46"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Wang</span>
                        <span class="given-names" tagx="given-names" title="given-names">Q.</span>
                     </span>, 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Garrity</span>
                        <span class="given-names" tagx="given-names" title="given-names">G. M.</span>
                     </span>, 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Tiedje</span>
                        <span class="given-names" tagx="given-names" title="given-names">J. M.</span>
                     </span> &amp; 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Cole</span>
                        <span class="given-names" tagx="given-names" title="given-names">J. R.</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Naive Bayesian classifier for rapid assignment of rRNA sequences into the new bacterial
                        taxonomy
                     </span>. 
                     <span class="source" tagx="source" title="source">Appl. Environ. Microbiol.</span>
                     <span class="volume" tagx="volume" title="volume">73</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">5261</span>–
                     <span class="lpage" tagx="lpage" title="lpage">5267</span> (
                     <span class="year" tagx="year" title="year">2007</span>).
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/17586664">17586664</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="b47"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Cole</span>
                        <span class="given-names" tagx="given-names" title="given-names">J. R.</span>
                     </span>
                     <i>et al.</i>
                     <span class="mixed-article-title" title="mixed-article-title">The Ribosomal Database Project: improved alignments and new tools for rRNA analysis</span>. 
                     <span class="source" tagx="source" title="source">Nucleic Acids Res.</span>
                     <span class="volume" tagx="volume" title="volume">37</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">D141</span>–
                     <span class="lpage" tagx="lpage" title="lpage">D145</span> (
                     <span class="year" tagx="year" title="year">2009</span>).
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/19004872">19004872</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="b48"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Shannon</span>
                        <span class="given-names" tagx="given-names" title="given-names">P.</span>
                     </span>
                     <i>et al.</i>
                     <span class="mixed-article-title" title="mixed-article-title">Cytoscape: a software environment for integrated models of biomolecular interaction
                        networks
                     </span>. 
                     <span class="source" tagx="source" title="source">Genome Res.</span>
                     <span class="volume" tagx="volume" title="volume">13</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">2498</span>–
                     <span class="lpage" tagx="lpage" title="lpage">2504</span> (
                     <span class="year" tagx="year" title="year">2003</span>).
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/14597658">14597658</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="b49"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Gouy</span>
                        <span class="given-names" tagx="given-names" title="given-names">M.</span>
                     </span>, 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Guindon</span>
                        <span class="given-names" tagx="given-names" title="given-names">S.</span>
                     </span> &amp; 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Gascuel</span>
                        <span class="given-names" tagx="given-names" title="given-names">O.</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">SeaView version 4: a multiplatform graphical user interface for sequence alignment
                        and phylogenetic tree building
                     </span>. 
                     <span class="source" tagx="source" title="source">Mol. Biol. Evol.</span>
                     <span class="volume" tagx="volume" title="volume">27</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">221</span>–
                     <span class="lpage" tagx="lpage" title="lpage">224</span> (
                     <span class="year" tagx="year" title="year">2010</span>).
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/19854763">19854763</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="b50"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Thompson</span>
                        <span class="given-names" tagx="given-names" title="given-names">J.</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">The CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment
                        aided by quality analysis tools
                     </span>. 
                     <span class="source" tagx="source" title="source">Nucleic Acids Res.</span>
                     <span class="volume" tagx="volume" title="volume">25</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">4876</span>–
                     <span class="lpage" tagx="lpage" title="lpage">4882</span> (
                     <span class="year" tagx="year" title="year">1997</span>).
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/9396791">9396791</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="b51"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Swofford</span>
                        <span class="given-names" tagx="given-names" title="given-names">D. L.</span>
                     </span> PAUP*: phylogenetic analysis using parsimony, ver 4.0b10. (Sinauer Associates, Sunderland,
                     Massachusetts, USA, (
                     <span class="year" tagx="year" title="year">1998</span>).
                  </span>
               </li>
               <li tag="ref">
                  <a name="b52"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Ceja-Navarro</span>
                        <span class="given-names" tagx="given-names" title="given-names">J. A.</span>
                     </span>, 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Brodie</span>
                        <span class="given-names" tagx="given-names" title="given-names">E. L.</span>
                     </span> &amp; 
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Vega</span>
                        <span class="given-names" tagx="given-names" title="given-names">F. E.</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">A technique to dissect the alimentary canal of the coffee berry borer (
                        <i>Hypothenemus hampei</i>), with isolation of internal microorganisms
                     </span>. 
                     <span class="source" tagx="source" title="source">J. Entomol. Acarol. Res.</span>
                     <span class="volume" tagx="volume" title="volume">44</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">e21</span> (
                     <span class="year" tagx="year" title="year">2012</span>).
                  </span>
               </li>
               <li tag="ref">
                  <a name="b53"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">The R Development Core Team. 
                     <span class="source" tagx="source" title="source">R: a language and environment for statistical computing</span> R Found. Stat. Comput. (
                     <span class="year" tagx="year" title="year">2010</span>) At http://www.r-project.org/ .
                  </span>
               </li>
               <li tag="ref">
                  <a name="b54"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Kind</span>
                        <span class="given-names" tagx="given-names" title="given-names">T.</span>
                     </span>
                     <i>et al.</i>
                     <span class="mixed-article-title" title="mixed-article-title">FiehnLib: mass spectral and retention index libraries for metabolomics based on quadrupole
                        and time-of-flight gas chromatography/mass spectrometry
                     </span>. 
                     <span class="source" tagx="source" title="source">Anal. Chem.</span>
                     <span class="volume" tagx="volume" title="volume">81</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">10038</span>–
                     <span class="lpage" tagx="lpage" title="lpage">10048</span> (
                     <span class="year" tagx="year" title="year">2009</span>).
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/19928838">19928838</a>
                     </span>
                  </span>
               </li>
            </ul>
         </div>
         <div class="fn-group" title="fn-group">
            <div class="fn-type-" title="">
               <p>
                  <b>Author contributions</b> E.L.B., J.A.C.-N. and F.E.V. designed experiments, analysed data and wrote the manuscript.
                  H.C.L. contributed to molecular analysis of microbiome composition. P.K. and F.I.
                  contributed to manuscript writing. J.A.C.-N., P.K. and F.I. collected insect specimens.
                  Z.H. performed FTIR analysis, U.K. performed biogeography analysis, S.J. and T.N.
                  performed mass spectrometry analysis.
               </p>
            </div>
         </div>
      </div>
   </body>
</html>