<?xml version="1.0" encoding="UTF-8"?><html xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:h="http://www.w3.org/1999/xhtml">
   <head>
      <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style>
   </head>
   <body>
      <div class="front" title="front">
         <div class="journal-meta" tagx="journal-meta" title="journal-meta">
            <span class="nlm-ta" title="nlm-ta">Front Plant Sci</span>
            <span class="iso-abbrev" title="iso-abbrev">Front Plant Sci</span>
            <span class="publisher-id" title="publisher-id">Front. Plant Sci.</span>
            <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group">
               <span class="journal-title" tagx="journal-title" title="journal-title">Frontiers in Plant Science</span>
            </div>
            <span class="issn-epub" tagx="issn" title="issn-epub">1664-462X</span>
            <div class="publisher" tagx="publisher" title="publisher">
               <span class="publisher-name" tagx="publisher-name" title="publisher-name">Frontiers Media S.A.</span>
            </div>
         </div>
         <div class="article-meta" tagx="article-meta" title="article-meta">
            <span class="pmcid" title="pmcid">
               			pmcid:
               			
               <a href="http://www.ncbi.nlm.nih.gov/pubmed/4604324">4604324</a>
            </span>
            <span class="doi" title="doi">
               			doi:
               			
               <a href="https://dx.doi.org/10.3389/fpls.2015.00817">10.3389/fpls.2015.00817</a>
            </span>
            <div class="article-categories" title="article-categories">
               <div class="sub-group" title="subj-group">: 
                  <span class="subject" title="subject">Plant Science</span>: 
                  <span class="subject" title="subject">Original Research</span>
               </div>
            </div>
            <div class="title-group">
               <div class="article-title" title="article-title">Transcriptome analysis of 
                  <i>Phoenix canariensis</i> Chabaud in response to 
                  <i>Rhynchophorus ferrugineus</i> Olivier attacks
               </div>
            </div>
            <div class="contrib-group" title="contrib-group">
               <span class="contrib" title="contrib">
                  <span class="name" tagx="name" title="name">
                     <span class="surname" tagx="surname" title="surname">Giovino</span>
                     <span class="given-names" tagx="given-names" title="given-names">Antonio</span>
                  </span>
                  <a href="#aff1">
                     <sup xmlns:ali="http://www.niso.org/schemas/ali/1.0">1</sup>
                  </a>
                  <span class="uri" title="uri">
                     <a href="http://loop.frontiersin.org/people/247214/overview"></a>
                  </span>
               </span>
               <span class="contrib" title="contrib">
                  <span class="name" tagx="name" title="name">
                     <span class="surname" tagx="surname" title="surname">Bertolini</span>
                     <span class="given-names" tagx="given-names" title="given-names">Edoardo</span>
                  </span>
                  <a href="#aff2">
                     <sup xmlns:ali="http://www.niso.org/schemas/ali/1.0">2</sup>
                  </a>
                  <span class="uri" title="uri">
                     <a href="http://loop.frontiersin.org/people/247415/overview"></a>
                  </span>
               </span>
               <span class="contrib" title="contrib">
                  <span class="name" tagx="name" title="name">
                     <span class="surname" tagx="surname" title="surname">Fileccia</span>
                     <span class="given-names" tagx="given-names" title="given-names">Veronica</span>
                  </span>
                  <a href="#aff3">
                     <sup xmlns:ali="http://www.niso.org/schemas/ali/1.0">3</sup>
                  </a>
                  <a href="#aff4">
                     <sup xmlns:ali="http://www.niso.org/schemas/ali/1.0">4</sup>
                  </a>
                  <span class="uri" title="uri">
                     <a href="http://loop.frontiersin.org/people/275688/overview"></a>
                  </span>
               </span>
               <span class="contrib" title="contrib">
                  <span class="name" tagx="name" title="name">
                     <span class="surname" tagx="surname" title="surname">Al Hassan</span>
                     <span class="given-names" tagx="given-names" title="given-names">Mohamad</span>
                  </span>
                  <a href="#aff5">
                     <sup xmlns:ali="http://www.niso.org/schemas/ali/1.0">5</sup>
                  </a>
                  <span class="uri" title="uri">
                     <a href="http://loop.frontiersin.org/people/247387/overview"></a>
                  </span>
               </span>
               <span class="contrib" title="contrib">
                  <span class="name" tagx="name" title="name">
                     <span class="surname" tagx="surname" title="surname">Labra</span>
                     <span class="given-names" tagx="given-names" title="given-names">Massimo</span>
                  </span>
                  <a href="#aff6">
                     <sup xmlns:ali="http://www.niso.org/schemas/ali/1.0">6</sup>
                  </a>
                  <span class="uri" title="uri">
                     <a href="http://loop.frontiersin.org/people/247250/overview"></a>
                  </span>
               </span>
               <span class="contrib" title="contrib">
                  <span class="name" tagx="name" title="name">
                     <span class="surname" tagx="surname" title="surname">Martinelli</span>
                     <span class="given-names" tagx="given-names" title="given-names">Federico</span>
                  </span>
                  <a href="#aff3">
                     <sup xmlns:ali="http://www.niso.org/schemas/ali/1.0">3</sup>
                  </a>
                  <a href="#aff4">
                     <sup xmlns:ali="http://www.niso.org/schemas/ali/1.0">4</sup>
                  </a>
                  <a href="#fn001">
                     <sup xmlns:ali="http://www.niso.org/schemas/ali/1.0">*</sup>
                  </a>
                  <span class="uri" title="uri">
                     <a href="http://loop.frontiersin.org/people/246864/overview"></a>
                  </span>
               </span>
            </div>
            <span class="citation_author_institution" id="aff1">[],
               	  
               <sup xmlns:ali="http://www.niso.org/schemas/ali/1.0">1</sup>
               <span class="institution" title="institution">Unità di Ricerca per il Recupero e la Valorizzazione delle Specie Floricole Mediterranee,
                  Consiglio per la Ricerca in Agricoltura e l'Analisi dell'Economia Agraria
               </span>
               <span class="country" tagx="country" title="country">Palermo, Italy</span>
            </span>
            <span class="citation_author_institution" id="aff2">[],
               	  
               <sup xmlns:ali="http://www.niso.org/schemas/ali/1.0">2</sup>
               <span class="institution" title="institution">Istituto di Scienze della Vita, Scuola Superiore Sant'Anna</span>
               <span class="country" tagx="country" title="country">Pisa, Italy</span>
            </span>
            <span class="citation_author_institution" id="aff3">[],
               	  
               <sup xmlns:ali="http://www.niso.org/schemas/ali/1.0">3</sup>
               <span class="institution" title="institution">Dipartimento di Scienze Agrarie e Forestali, Università degli Studi di Palermo</span>
               <span class="country" tagx="country" title="country">Palermo, Italy</span>
            </span>
            <span class="citation_author_institution" id="aff4">[],
               	  
               <sup xmlns:ali="http://www.niso.org/schemas/ali/1.0">4</sup>
               <span class="institution" title="institution">Istituto Euromediterraneo di Scienza e Tecnologia</span>
               <span class="country" tagx="country" title="country">Palermo, Italy</span>
            </span>
            <span class="citation_author_institution" id="aff5">[],
               	  
               <sup xmlns:ali="http://www.niso.org/schemas/ali/1.0">5</sup>
               <span class="institution" title="institution">Universitat Politècnica de València, Instituto de Biología Molecular y Celular de
                  Plantas, Consejo Superior de Investigaciones Científicas (CSIC), Ciudad Politécnica
                  de la Investigación (CPI)
               </span>
               <span class="country" tagx="country" title="country">Valencia, Spain</span>
            </span>
            <span class="citation_author_institution" id="aff6">[],
               	  
               <sup xmlns:ali="http://www.niso.org/schemas/ali/1.0">6</sup>
               <span class="institution" title="institution">Dipartimento di Biotecnologie e Bioscienze, Università degli Studi di Milano Bicocca</span>
               <span class="country" tagx="country" title="country">Milano, Italy</span>
            </span>
            <div class="author-notes" title="author-notes">
               <div class="fn-type-edited-by" title="edited-by">
                  <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">Edited by: Simone Ferrari, Sapienza Università di Roma, Italy</p>
               </div>
               <div class="fn-type-edited-by" title="edited-by">
                  <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">Reviewed by: Giulia De Lorenzo, Sapienza Università di Roma, Italy; Alberto Ferrarini,
                     University of Verona, Italy
                  </p>
               </div>
               <div class="corresp" title="corresp">*Correspondence: Federico Martinelli, Dipartimento di Scienze Agrarie e Forestali,
                  Università degli Studi di Palermo, Viale delle Scienze, Edificio 4, 90128 Palermo,
                  Italy 
                  <span class="email" tagx="email" title="email">federico.martinelli@unipa.it</span>
               </div>
               <div class="fn-type-other" title="other">
                  <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">This article was submitted to Plant Biotic Interactions, a section of the journal
                     Frontiers in Plant Science
                  </p>
               </div>
            </div>
            <span class="pub-date-epub" title="pub-date-epub">epub: 
               <span>2015-10-10</span>
            </span>
            <span class="pub-date-collection" title="pub-date-collection">collection: 
               <span>2015</span>
            </span>
            <span class="volume" tagx="volume" title="volume">6</span>
            <span class="elocation-id" tagx="elocation-id" title="elocation-id">817</span>
            <span class="history" title="history">
               <span class="received" title="received">received: 2015-6-22</span>
               <span class="accepted" title="accepted">accepted: 2015-9-18</span>
            </span>
            <div class="permissions">
               <span class="copyright" title="copyright">(C) 
                  <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Giovino, Bertolini, Fileccia, Al Hassan, Labra and Martinelli</span>, 
                  <span class="copyright-year" tagx="copyright-year" title="copyright-year">2015</span>
               </span>
               <span class="license" title="license">
                  <span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons
                     Attribution License (CC BY). The use, distribution or reproduction in other forums
                     is permitted, provided the original author(s) or licensor are credited and that the
                     original publication in this journal is cited, in accordance with accepted academic
                     practice. No use, distribution or reproduction is permitted which does not comply
                     with these terms.
                  </span>
               </span>
            </div>
            <div class="abstract" title="abstract">
               <div class="abstract-title" title="abstract-title">Abstract</div>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">Red Palm Weevil (RPW, 
                  <i>Rhynchophorus ferrugineus</i> Olivier) threatens most palm species worldwide. Until now, no studies have analyzed
                  the gene regulatory networks of 
                  <i>Phoenix canariensis</i> (Chabaud) in response to RPW attacks. The aim of this study was to fill this knowledge
                  gap. Providing this basic knowledge is very important to improve its management.
               </p>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">
                  <b>Results:</b> A deep transcriptome analysis was performed on fully expanded leaves of healthy non-infested
                  trees and attacked trees at two symptom stages (middle and late infestation). A total
                  of 54 genes were significantly regulated during middle stage. Pathway enrichment analysis
                  showed that phenylpropanoid-related pathways were induced at this stage. More than
                  3300 genes were affected during late stage of attacks. Higher transcript abundances
                  were observed for lipid fatty acid metabolism (fatty acid and glycerolipids), tryptophan
                  metabolism, phenylpropanoid metabolism. Key RPW-modulated genes involved in innate
                  response mediated by hormone crosstalk were observed belonging to auxin, jasmonate
                  and salicylic acid (SA) pathways. Among transcription factors, some WRKYs were clearly
                  induced. qRT-PCR validation confirmed the upregulation of key genes chosen as validation
                  of transcriptomic analysis.
               </p>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">
                  <b>Conclusion:</b> A subset of these genes may be further analyzed in future studies to confirm their
                  specificity to be induced by RPW infestations.
               </p>
            </div>
            <div class="kwd-group">
               <div class="kwd" title="kwd">genes</div>
               <div class="kwd" title="kwd">palm</div>
               <div class="kwd" title="kwd">
                  <i>Phoenix canariensis</i>
               </div>
               <div class="kwd" title="kwd">Red Palm Weevil</div>
               <div class="kwd" title="kwd">
                  <i>Rhynchophorus ferrugineus</i>
               </div>
               <div class="kwd" title="kwd">RNA-seq</div>
            </div>
            <div class="funding-group" title="funding-group">
               <div class="funding" title="funding">Funding</div>
               <div class="award-group" title="award-group">
                  <div class="funding-source" title="funding-source">Ministero delle politiche agricole alimentari e forestali
                     <span class="named-content" title="named-content">10.13039/501100005401</span>
                  </div>
               </div>
            </div>
            <div class="counts" title="counts">
               <span class="fig-count" title="fig-count">fig-count: </span>
               <span class="table-count" title="table-count">table-count: </span>
               <span class="equation-count" title="equation-count">equation-count: </span>
               <span class="ref-count" title="ref-count">ref-count: </span>
               <span class="page-count" title="page-count">page-count: </span>
               <span class="word-count" title="word-count">word-count: </span>
            </div>
         </div>
      </div>
      <div class="body" title="body">
         <div class="intro" title="intro">
            <div class="title" tagx="title" title="title">Introduction</div>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">
               <i>Rhynchophorus ferrugineus</i> Olivier (commonly known as the Red Palm Weevil, RPW) is considered the worst pest
               threat for palm species worldwide. More than 30 palm species are attacked by this
               insect, including 
               <i>Phoenix canariensis</i> Chabaud, 
               <i>P. dactylifera</i> L., and 
               <i>Cocos nucifera</i> L. (Ju et al., 
               <a href="#B16">2011</a>). RPW was first reported in tropical Asia and then spread worldwide, reaching several
               Middle Eastern countries, Africa and the Mediterranean basin in the 1980s (Faleiro,
               
               <a href="#B8">2006</a>). In Europe, including Spain, especially the Canary Islands, and Southern Italy,
               RPW inflicted great economic damage in areas where palm distinguishes the scenic beauty
               of the public and private gardens of the cities and countryside. In the United States
               a species of RPW has been identified (Rugman-Jones et al., 
               <a href="#B29">2013</a>). RPW belongs to the Curculionidae family of Coleoptera. The damage caused to palm
               is due to the large larvae forming large tunnels within soft and terminal plant tissues.
               The larvae can achieve a size of 5 cm, and different generations can be present in
               the same infected tree. Adults may deposit approximately 200 eggs at the base of leaves
               or in wounded tissues, and larvae migrate within all parts of trees, including the
               roots, destroying all the plant organs, and structures (Gutierrez et al., 
               <a href="#B13">2010</a>). A major issue in the management of pest attacks is the late detection of infestations.
               Usually, symptoms are only visible when the larvae have reached the latest developmental
               stage and when plant organs are already compromised (Ju et al., 
               <a href="#B16">2011</a>). Indeed, at this point, it is too late to save the infected trees. The symptoms
               of infestation include a gnawing sound caused by larvae feeding inside infested trees,
               chewed plant material at the external entrances of tunnels generating particular smells,
               the presence of empty pupal and dead adult bodies close to infested palms, the breaking
               of the palm crown and trunk (Faleiro, 
               <a href="#B8">2006</a>) and the asymmetry of the palm crown. These symptoms are not visible prior to 5–6
               months of infestation, when any prophylaxis is useless (Faleiro, 
               <a href="#B8">2006</a>). The evaluation of the signs of larval mines and chewed material at the leaf bases
               is the typical way to detect signs of early infestation. However, early infestations
               can be difficult to detect in adult palms because actively growing portions are usually
               at the top of the plants. Therefore, a wide range of physical, chemical and biological
               methodologies are under development to detect the insect inside the palms as soon
               as infestations occur. Computer-assisted tomography shows interesting results for
               the inspection of the granary weevil in wheat (Haff and Slaughter, 
               <a href="#B14">2004</a>). However, these methods are expensive and difficult to be applied in infected palms
               where the whole tree needs to be inspected. Indeed, these methods may not be used
               for the large-scale scouting of infestations. The analysis of volatiles that are emitted
               by the fermentation processes of infested trees was also addressed by means of trained
               dogs that recognized the characteristic odors of plants that were attacked by RPW
               (Nakash et al., 
               <a href="#B27">2000</a>). However, the low selectivity and reliability due to the presence of several volatiles
               that are unrelated to the RPW infection limit the use of such methods (Mielle and
               Marquis, 
               <a href="#B25">1999</a>). Bioacoustic sensors could represent an alternative system for early detection (Gutierrez
               et al., 
               <a href="#B13">2010</a>). However, this detection is still expensive and requires trained technical staff
               to discriminate the RPW noise from others. Moreover, these methods are not very effective
               at early feeding phases. Until now, no large-scale analyses of palm responses to RPW
               attacks have been conducted. Transcriptomics represent a powerful tool not only to
               elucidate the physiological effects of RPW attacks on infected palms but also to the
               identify biomarkers that are usable to improve detection when symptoms are still unclear.
               In addition, transcriptomics is essential to develop therapeutics based on gene biotechnology.
               Next-Generation Sequencing (NGS) has revolutionized transcriptomic studies because
               this technology can detect rare and unknown transcripts and splice variants that are
               not present in microarrays, offering a more detailed and profound analysis of the
               transcriptome. The power of this method is fully exploitable when extensive genomic
               information is available for the organism under investigation. In plants, RNA-seq
               has proven effective in studying the transcriptomic profile after pathogenic infections
               (Martinelli et al., 
               <a href="#B24">2012</a>, 
               <a href="#B22">2013</a>; Chen et al., 
               <a href="#B4">2014</a>; Zhang et al., 
               <a href="#B37">2014</a>) and in identifying possible targets for their early detection (Tremblay et al.,
               
               <a href="#B34">2013</a>).
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">Recent studies have been published on the genomic analysis of 
               <i>P. dactylifera</i> L. A genome assembly was performed for Khalas, an elite cultivar (Al-Dous et al.,
               
               <a href="#B1">2011</a>). The assembled sequence was approximately 380 Mb spanning mostly gene-rich regions
               (90% of genes were covered) and including &gt;25,000 gene models. Another genome sequence
               of date palm is available (Al-Mssallem et al., 
               <a href="#B2">2013</a>).
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">The aim of this study was to gain insight into gene regulatory networks of responses
               to RPW attack in 
               <i>Phoenix canariensis</i>. This research will allow us to (1) clarify the gene regulation mechanisms of leaf
               metabolism in response to RPW attacks at different stage of infestations and (2) identify
               possible host biomarkers that may confirm RPW typical symptomatology.
            </p>
         </div>
         <div class="materials_methods" title="materials|methods">
            <div class="title" tagx="title" title="title">Materials and methods</div>
            <div class="plantmaterialandexperimentaldesign" title="sec">
               <div class="title" tagx="title" title="title">Plant material and experimental design</div>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">This study was performed on 
                  <i>Phoenix canariensis</i> Chabaud. We selected 15–20-year-old trees with a diameter of 70–90 cm. Analyzed leaves
                  were 4–5 m long with 80–100 segments on each side of the spine. The plants were field-grown
                  in a former cultivation of approximately 2 ha, located in Trabia (Palermo, Italy).
                  These plants were genetically different although they have a similar genetic background
                  since they were derived from the same mother plant. No specific permissions were required
                  for these locations and activities. The field studies did not involve endangered or
                  protected species. These are the GPS coordinates of the area where plants were analyzed:
                  38°00′00″N 13°39′00″E; 38°00′00″N 13°39′00″E.
               </p>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">The plants were arranged on the sides of a private driveway. The trees were divided
                  into three groups: Healthy unattacked trees (He), stage 1 (the middle stage of infection—S1),
                  and stage 2 (plants with late symptoms—S2). The pathogen presence was visually confirmed.
                  No plants were available at very early stage of infection. Although trees at S1 showed
                  symptoms, analyzed leaves were green and did not show any symptoms such as the healthy
                  ones.
               </p>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">The He category was composed of palm trees with no symptoms of Red Palm Weevil attacks
                  and no signs of other common diseases or pathogen attacks (Figure 
                  <a href="#F1">1</a>). The sampled individuals showed fully expanded leaves without any sign of gnawing.
                  To exclude sampling errors (i.e., sampling asymptomatic infested plants instead of
                  healthy uninfected plants), we monitored the sampled He trees for the following 90
                  days from sampling. This period was defined based on the timing needed to show early
                  symptoms of infestations (Ju et al., 
                  <a href="#B16">2011</a>). The S1 category represents trees that showed anomalous behavior of the canopy with
                  the beginning of characteristic retracted asymmetry of the crown. Individuals showed
                  fully expanded leaves without any sign of gnawing at the sampling time; however, after
                  30–40 days, some of the leaves showed a flattened and nibbled vegetative apex. Moreover,
                  after 50 days, some larvae were detected in the S1 individuals. The S2 category included
                  trees with loss of leaves for subsidence of foliar rachis. In these trees, most of
                  the leaves showed the typical recline induced by RPW. Foliar desiccation was much
                  more apparent than in S1. While analyzed leaves of S1 looked green as the healthy
                  ones, S2 leaves showed evident yellowing.
               </p>
               <div class="fig" title="fig">
                  <div class="figure" title="figure">
                     <a href="http://doi.org/">
                        <span class="label" tagx="label" title="label">Figure 1</span>
                     </a>
                  </div>
                  <caption class="caption" title="caption">
                     <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0">
                        <p>
                           <b>Palm trees of the three analyzed categories: (A) healthy (He); (B) stage S1 (middle
                              stage of infestation); (C) stage S2 (intense level of infestation)
                           </b>.
                        </p>
                     </caption>
                  </caption>
               </div>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">Each of the three conditions was analyzed in duplicate; thus, a total of six samples
                  (biological replicates) were analyzed by deep sequencing. Each biological replicate
                  was composed of 10–15 leaf portions of three distinct trees. The leaf portions were
                  collected from the middle of the palm leaves at the same time during the day. The
                  samples were immediately frozen in liquid nitrogen and stored at −80°C.
               </p>
            </div>
            <div class="analysisoftheplanthealthstatus" title="sec">
               <div class="title" tagx="title" title="title">Analysis of the plant health status</div>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">The rhizosphere was analyzed to identify any other pathogenic organism in addition
                  to RPW. The soil samples were collected from all four plant sides at a depth of 30
                  cm (500 g per plant). The pathogens were isolated through different traditional techniques.
                  Substrates of the soil, roots and bark were prepared for both generic and selective
                  pathogen isolation. The soil samples were repeatedly mixed, and 10 g was taken, dissolved
                  in 100 ml of sterile distilled water and agitated for 15 min. A 1 ml aliquot of the
                  suspension was inoculated in Petri dishes with cultured generic potato dextrose agar
                  (PDA). The root and bark tissues were previously superficially sterilized and immersed
                  in an aqueous solution of sodium hypochlorite at 10% for 3–5 min, rinsed in sterile
                  distilled water and dried. The tissue fragments were placed on the culture substrates
                  PDA, corn meal agar (CMA) plus streptomycin (20–30 mg/L) and PARPNH substrate-selective
                  culture. All the plates were dark-incubated at 24 ± 1°C. The developed fungal colonies
                  were grown in purity and characterized based on both macroscopic (morphology, color,
                  and speed of growth of the colony) and microscopic features (fruiting bodies, spores,
                  and conidia). The genera and species were identified based on dichotomous recognized
                  keys. All the analyzed palms were tested for the presence of common bacterial and
                  fungal pathogens (i.e., Penicillium sp., Alternaria sp., Mortierella sp., Aspergillus
                  sp.). None of these common pathogens were present.
               </p>
            </div>
            <div class="rnaextraction" title="sec">
               <div class="title" tagx="title" title="title">RNA extraction</div>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">The total RNA from each biological replicate was isolated using a rapid RNA extraction
                  method that was developed by Gambino et al. (
                  <a href="#B9">2008</a>). The RNA concentrations were determined using a NanoDrop ND-1000 spectrophotometer
                  (NanoDrop Technologies, Wilmington, DE). The RNA quality and purity were assessed
                  by an Agilent Bioanalyzer (Folsom, CA).
               </p>
            </div>
            <div class="rna-seqanalysis" title="sec">
               <div class="title" tagx="title" title="title">RNA-seq analysis</div>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">The RNA samples were processed using the TruSeq RNA-seq sample prep kit from Illumina
                  (Illumina, Inc., CA, USA). Briefly, the poly-A containing mRNA molecules were purified
                  using poly-T oligo-attached magnetic beads and fragmented into small pieces using
                  divalent cations at an elevated temperature. The cDNA was synthesized by reverse transcription,
                  and standard blunt-ending plus add “A” was performed. Then, Illumina TruSeq adapters
                  with indexes were ligated to the ends of the cDNA fragments. After the ligation reaction
                  and separation of the unligated adapters, the samples were amplified by PCR to selectively
                  enrich those cDNA fragments in the library with adapter molecules at both ends. The
                  six samples (RNA pools of 10–15 leaves of three individual plants) were loaded into
                  one lane of an Illumina flow cell, and clusters were created by Illumina cBot. The
                  clusters were sequenced at ultra-high throughput on the Illumina HiSeq 2000 (Illumina
                  Inc.). One lane in 6-plex was run, obtaining between approximately 21.1 and 29.8 million
                  single reads per sample, each 50 bp long. The data were produced on an IGA Technology
                  Services Srl (Udine, Italy) Illumina platform. All the sequenced reads were compared
                  to the Date Palm Genome Draft Sequence Version 3.0 (
                  <a href="http://qatar-weill.cornell.edu/">http://qatar-weill.cornell.edu/</a>). The gene prediction from the Date Palm Genome Draft Sequence Version 3.0 (file
                  PDK30-mrna.fsa.gz downloaded from 
                  <a href="http://qatar-weill.cornell.edu/">http://qatar-weill.cornell.edu/</a>) was used. The Software CLC Genomic Workbench 5.5.1 (CLC Bio, Denmark) was used for
                  reads trimming and alignment on the reference. The reads were quality trimmed using
                  the modified-Mott trimming algorithm (parameter: trim using quality score = 0.05).
                  The alignment on a reference parameters were set-up in CLC Genomics Workbench as following:
                  Mismatch cost = 2; Insertion cost = 3; Deletion cost = 3; Length fraction = 0.9; Similarity
                  fraction = 0.95. BLASTx was used to determine the date palm gene predictions of the
                  putative orthologous 
                  <i>Arabidopsis thaliana</i> genes (
                  <i>e</i> &lt; 10
                  <sup>−4</sup>). DESeq 2 package from Bioconductor in the R statistical software suite (Love et
                  al., 
                  <a href="#B21">2014</a>) was used to estimate the euclidean sample distance and the expression level of transcripts
                  among different conditions. DESeq 2 program performs normalization, variance estimation
                  and differential expression of the raw read counts and works best with experiments
                  with replicates. The log
                  <sub>2</sub>-fold ratio and adjusted 
                  <i>p</i>-values (FDR) based on the t-distribution for each gene for the two pairwise comparisons
                  (He-S1 and He-S2) were calculated. Genes with a log
                  <sub>2</sub>-fold ratio &gt;1 or &lt; −1 and with an adjusted 
                  <i>p</i>-value (FDR) below 0.1 were considered differentially expressed. RNA-seq and details
                  of the samples were submitted on Squence Read Archive (NCBI) (SAMN04031658).
               </p>
            </div>
            <div class="functionalcategorizationofthepredictedrpw-regulatedtranscripts" title="sec">
               <div class="title" tagx="title" title="title">Functional categorization of the predicted RPW-regulated transcripts</div>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">The sequenced transcripts were mapped to the Date Palm Genome Draft Sequence Version
                  3.0 and used as a reference genome. For each of the palm genes, the gene length, unique
                  and total gene reads, annotation and RPKM expression values were obtained. The closest
                  
                  <i>Arabidopsis</i> putative orthologs were determined for each gene to allow the use of functional genomics
                  tools. The integrated data-mining approach used different web tools, such as MapMan,
                  PageMan, PathExpress, and Cytoscape to dissect the transcriptome responses and decipher
                  the gene regulatory networks. A list of predicted transcripts that were differentially
                  expressed at a significant level (
                  <i>p</i> &lt; 0.05 and an absolute value of log
                  <sub>2</sub>-fold change &gt;1 or &lt; -1) was obtained for each of the two pairwise comparisons (S1
                  vs. He and S2 vs. He). These input data files were used for all the data mining tools.
               </p>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">The functions of the differentially expressed genes (as 
                  <i>Arabidopsis</i> putative orthologs) were visualized using the MapMan web-tool (Thimm et al., 
                  <a href="#B33">2004</a>; 
                  <a href="http://mapman.gabipd.org/web/guest/mapman">http://mapman.gabipd.org/web/guest/mapman</a>) through the Ath_AGI_isoform_model_Tair10_Aug2012.m02 mapping file that was downloaded
                  from the MapMan server. The PageMan visualization tool was used for GSEA analysis
                  using the Wilcoxon test (no correction and ORA cutoff = 1.0). A pathway enrichment
                  analysis was performed using PathExpress (Goffard and Weiller, 
                  <a href="#B12">2007</a>) using the 
                  <i>Arabidopsis</i> putative orthologs of the differentially expressed palm transcripts.
               </p>
            </div>
            <div class="rt-pcrvalidation" title="sec">
               <div class="title" tagx="title" title="title">RT-PCR validation</div>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">Nine genes were chosen for qRT-PCR validation of RNA-seq data. Four biological replicates
                  were considered for each of three conditions (Healthy, S1, and S2). Each replicate
                  was a pool of 10–15 mature leaves from the same plant. The four chosen plants belonged
                  to those used for RNA-seq analysis: two belonged to the first replicate and the other
                  two belonged to the second replicate of deep sequencing analysis. Primers were designed
                  basing on each target sequence using Primer Express software (Applied Biosystems,
                  Foster City, CA, Table 
                  <a href="#SM1">S1</a>). RNA was extracted as previously described. Retrotranscription was performed following
                  the Quantitect Reverse Transcription Kit (Qiagen) instructions. A standard curve was
                  generated for each gene. Amplifications used 25 ng cDNA in a 15 μL final volume were
                  performed on a Biorad iQ5 PCR system (Biorad) using standard amplification conditions:
                  10 min at 95°C; 40 cycles of 15 s at 95°C; and 1 min at 60°C. All PCR reactions were
                  performed in duplicate (technical replicates). Fluorescent signals were collected
                  during the annealing step and C
                  <sub>T</sub> values extracted with an auto calculated threshold followed by baseline subtraction.
                  
                  <i>18S</i> (AF206991.1) was used as an endogenous reference and ΔΔ
                  <i>C</i>
                  <sub>T</sub> was calculated by subtracting the average of 
                  <i>18S</i> from the average 
                  <i>C</i>
                  <sub>T</sub> of the gene of interest. The reference gene was tested on the 12 analyzed samples
                  and any significant changes of expression were observed between the three samples
                  categories.
               </p>
            </div>
         </div>
         <div class="results" title="results">
            <div class="title" tagx="title" title="title">Results</div>
            <div class="illuminarnasequencing" title="sec">
               <div class="title" tagx="title" title="title">Illumina RNA sequencing</div>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">RNA-seq was used to evaluate palm responses to RPW attack by comparing the expression
                  profiles of healthy trees (He) with infected trees during the middle and late stages
                  of infection (S1 and S2). The genome sequence of 
                  <i>Phoenix dactylifera</i> was used as reference genome for sequence annotation (Al-Dous et al., 
                  <a href="#B1">2011</a>). This allowed obtain a list of palm genes correspondent to the assembled sequences.
                  Then we identified the correspondent 
                  <i>Arabidopsis</i> ortholog to each of these genes. We obtained a total of 21.3–29.8 million raw reads
                  for the six palm samples (Table 
                  <a href="#T1">1</a>). Of them 21.1–29.5 were trimmed. A total of approximately 8.6–12.3 million 50-nt
                  single-end reads from each cDNA library were uniquely mapped to the genome published
                  by Al-Dous et al. (
                  <a href="#B1">2011</a>). Unique mapped reads were approximately 40.6–41.6 % of the corresponding total trimmed
                  reads per sample. Multiple mapped reads were approximately 1.1 % and they were discarded
                  from the analysis.
               </p>
               <table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0">
                  <span class="label" tagx="label" title="label">Table 1</span>
                  <caption>
                     <p>
                        <b>Number of raw reads, trimmed raw reads, mapped reads (unique), mapped reads (multiple)</b>.
                     </p>
                  </caption>
                  <table>
                     <thead>
                        <tr>
                           <th>
                              <b>Sample</b>
                           </th>
                           <th>
                              <b>Healthy 1</b>
                           </th>
                           <th>
                              <b>Healthy 2</b>
                           </th>
                           <th>
                              <b>Stage 1-1</b>
                           </th>
                           <th>
                              <b>Stage 1-2</b>
                           </th>
                           <th>
                              <b>Stage 2-1</b>
                           </th>
                           <th>
                              <b>Stage 2-2</b>
                           </th>
                        </tr>
                     </thead>
                     <tbody>
                        <tr>
                           <td>Raw reads</td>
                           <td>21,333,300</td>
                           <td>25,657,081</td>
                           <td>22,350,317</td>
                           <td>29,707,097</td>
                           <td>29,781,177</td>
                           <td>26,491,474</td>
                        </tr>
                        <tr>
                           <td>Trimmed raw reads</td>
                           <td>21,139,613</td>
                           <td>25,438,175</td>
                           <td>22,142,923</td>
                           <td>29,456,600</td>
                           <td>29,527,217</td>
                           <td>26,123,111</td>
                        </tr>
                        <tr>
                           <td>Mapped reads (unique)</td>
                           <td>8,591,083</td>
                           <td>10,460,021</td>
                           <td>9,023,479</td>
                           <td>12,174,309</td>
                           <td>12,291,541</td>
                           <td>10,860,082</td>
                        </tr>
                        <tr>
                           <td>% of mapped reads (unique)</td>
                           <td>40.64</td>
                           <td>41.12</td>
                           <td>40.75</td>
                           <td>41.33</td>
                           <td>41.63</td>
                           <td>41.57</td>
                        </tr>
                        <tr>
                           <td>Mapped reads (multiple)</td>
                           <td>227,933</td>
                           <td>268,690</td>
                           <td>233,928</td>
                           <td>337,254</td>
                           <td>337,657</td>
                           <td>297,427</td>
                        </tr>
                        <tr>
                           <td>% of mapped reads (multiple)</td>
                           <td>1.08</td>
                           <td>1.06</td>
                           <td>1.06</td>
                           <td>1.14</td>
                           <td>1.14</td>
                           <td>1.14</td>
                        </tr>
                     </tbody>
                  </table>
                  <table-wrap-foot>
                     <p>
                        <i>Percentages of mapped reads (unique or multiple) were calculated from trimmed raw
                           reads
                        </i>.
                     </p>
                  </table-wrap-foot>
               </table-wrap>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">The gene expression data of the two pairwise comparisons (S1 vs. He and S2 vs. He)
                  were provided in Tables 
                  <a href="#SM2">S2</a>, 
                  <a href="#SM3">S3</a>. Figure 
                  <a href="#F2">2</a> showed a Venn diagram of up- and down-regulated genes by RPW in both pairwise comparisons.
                  A total of 44 genes were upregulated, and 10 genes were downregulated in S1 vs. He
                  comparison. Most of them were reported in Table 
                  <a href="#T2">2</a>. A total of 3373 genes were differentially expressed in the S2 vs. He comparison
                  (approximately 13.5 % of the predicted genes in the genome reference); 1938 were upregulated,
                  and 1435 were downregulated. A dendrogram was constructed based on the overall transcriptome
                  analysis of the six analyzed samples (Figure 
                  <a href="#F3">3</a>). As expected He and S1 were grouped together while S2 showed to be clearly distinguished
                  by the other two sample categories. Volcano plots were performed for both S1/He and
                  S2/He comparisons (Figure 
                  <a href="#F4">4</a>). In the S1/He comparison, although many genes have a log fold ratio higher than
                  1, they had a non-significant FDR. In the S2/He comparison a considerable portions
                  of the analyzed genes were significantly regulated. These genes were mainly those
                  that presented a higher fold ratio.
               </p>
               <div class="fig" title="fig">
                  <div class="figure" title="figure">
                     <a href="http://doi.org/">
                        <span class="label" tagx="label" title="label">Figure 2</span>
                     </a>
                  </div>
                  <caption class="caption" title="caption">
                     <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0">
                        <p>
                           <b>Venn diagrams of RPW-regulated genes at He vs. S1 and He vs. S2 comparisons</b>. Numbers of up- or down-regulated genes were shown.
                        </p>
                     </caption>
                  </caption>
               </div>
               <table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0">
                  <span class="label" tagx="label" title="label">Table 2</span>
                  <caption>
                     <p>
                        <b>List of the main differentially regulated genes during stage 1 (FDR &lt; 0.1)</b>.
                     </p>
                  </caption>
                  <table>
                     <thead>
                        <tr>
                           <th>
                              <b>Annotation</b>
                           </th>
                           <th>
                              <b>AGI</b>
                           </th>
                           <th>
                              <b>Log
                                 <sub>2</sub> fold ratio
                              </b>
                           </th>
                        </tr>
                     </thead>
                     <tbody>
                        <tr>
                           <td>
                              <b>REDOX PATHWAYS</b>
                           </td>
                        </tr>
                        <tr>
                           <td>Peroxidase superfamily protein</td>
                           <td>AT1G05260.1</td>
                           <td>−4.0</td>
                        </tr>
                        <tr>
                           <td>Peroxidase superfamily protein</td>
                           <td>AT1G68850.1</td>
                           <td>2.4</td>
                        </tr>
                        <tr>
                           <td>(2OG) and Fe(II)-dependent oxygenase protein</td>
                           <td>AT5G05600.1</td>
                           <td>3.0</td>
                        </tr>
                        <tr>
                           <td>(2OG) and Fe(II)-dependent oxygenase protein</td>
                           <td>AT4G10490.1</td>
                           <td>1.7</td>
                        </tr>
                        <tr>
                           <td>
                              <b>LARGE ENZYME FAMILIES</b>
                           </td>
                        </tr>
                        <tr>
                           <td>GDSL-like Lipase/Acylhydrolase protein</td>
                           <td>AT1G74460.1</td>
                           <td>2.6</td>
                        </tr>
                        <tr>
                           <td>Glutathione S-transferase TAU 18</td>
                           <td>AT1G10360.1</td>
                           <td>2.0</td>
                        </tr>
                        <tr>
                           <td>Cytochrome P450 superfamily protein</td>
                           <td>AT5G07990.1</td>
                           <td>−3.9</td>
                        </tr>
                        <tr>
                           <td>Cytochrome P450 family 94 C polypeptide 1</td>
                           <td>AT2G27690.1</td>
                           <td>3.4</td>
                        </tr>
                        <tr>
                           <td>
                              <b>SIGNAL PERCEPTION AND SIGNALING</b>
                           </td>
                        </tr>
                        <tr>
                           <td>Protein kinase superfamily protein</td>
                           <td>AT1G18670.1</td>
                           <td>1.6</td>
                        </tr>
                        <tr>
                           <td>Calcium-dependent protein kinase 16</td>
                           <td>AT2G17890.1</td>
                           <td>1.5</td>
                        </tr>
                        <tr>
                           <td>Protein of unknown function (DUF604)</td>
                           <td>AT2G37730.1</td>
                           <td>−3.3</td>
                        </tr>
                        <tr>
                           <td>Protein of unknown function (DUF616)</td>
                           <td>AT1G53040.1</td>
                           <td>1.6</td>
                        </tr>
                        <tr>
                           <td>Calcium-dependent protein kinase 28</td>
                           <td>AT5G66210.2</td>
                           <td>2.0</td>
                        </tr>
                        <tr>
                           <td>
                              <b>TRANSCRIPTION FACTORS</b>
                           </td>
                        </tr>
                        <tr>
                           <td>WRKY40</td>
                           <td>AT1G80840.1</td>
                           <td>2.8</td>
                        </tr>
                        <tr>
                           <td>WRKY51</td>
                           <td>AT5G64810.1</td>
                           <td>2.7</td>
                        </tr>
                        <tr>
                           <td>NAC domain containing protein 32</td>
                           <td>AT1G77450.1</td>
                           <td>1.8</td>
                        </tr>
                        <tr>
                           <td>
                              <b>PROTEIN MODIFICATIONS</b>
                           </td>
                        </tr>
                        <tr>
                           <td>RING/U-box superfamily protein</td>
                           <td>AT1G78420.1</td>
                           <td>−7.6</td>
                        </tr>
                        <tr>
                           <td>RING/U-box superfamily protein</td>
                           <td>AT1G53820.1</td>
                           <td>3.1</td>
                        </tr>
                        <tr>
                           <td>RING/U-box superfamily protein</td>
                           <td>AT2G18650.1</td>
                           <td>−4.1</td>
                        </tr>
                        <tr>
                           <td>U-box domain-containing protein kinase protein</td>
                           <td>AT2G45910.1</td>
                           <td>1.6</td>
                        </tr>
                        <tr>
                           <td>
                              <b>SECONDARY METABOLISM</b>
                           </td>
                        </tr>
                        <tr>
                           <td>Cinnamate-4-hydroxylase</td>
                           <td>AT2G30490.1</td>
                           <td>3.0</td>
                        </tr>
                        <tr>
                           <td>Laccase 7</td>
                           <td>AT3G09220.1</td>
                           <td>2.9</td>
                        </tr>
                        <tr>
                           <td>Laccase 12</td>
                           <td>AT5G05390.1</td>
                           <td>2.3</td>
                        </tr>
                        <tr>
                           <td>Chalcone and stilbene synthase protein</td>
                           <td>AT5G13930.1</td>
                           <td>1.8</td>
                        </tr>
                        <tr>
                           <td>Leucoanthocyanidin dioxygenase</td>
                           <td>AT4G22880.1</td>
                           <td>−3.6</td>
                        </tr>
                        <tr>
                           <td>
                              <b>DEFENSE RESPONSES</b>
                           </td>
                        </tr>
                        <tr>
                           <td>Disease resistance family protein / LRR protein</td>
                           <td>AT2G34930.1</td>
                           <td>1.7</td>
                        </tr>
                        <tr>
                           <td>Pathogenesis-related thaumatin protein</td>
                           <td>AT2G17860.1</td>
                           <td>1.8</td>
                        </tr>
                        <tr>
                           <td>Lipid-transfer protein/seed storage 2S albumin</td>
                           <td>AT3G22600.1</td>
                           <td>2.5</td>
                        </tr>
                        <tr>
                           <td>
                              <b>CELL WALL MODIFICATIONS</b>
                           </td>
                        </tr>
                        <tr>
                           <td>Xyloglucan endotransglycosylase 6</td>
                           <td>AT4G25810.1</td>
                           <td>2.1</td>
                        </tr>
                        <tr>
                           <td>Beta-1 3-glucanase 5</td>
                           <td>AT5G20340.1</td>
                           <td>2.1</td>
                        </tr>
                        <tr>
                           <td>D-arabinono-1 4-lactone oxidase family protein</td>
                           <td>AT2G46740.1</td>
                           <td>3.4</td>
                        </tr>
                        <tr>
                           <td>Pectin lyase-like superfamily protein</td>
                           <td>AT3G62110.1</td>
                           <td>−3.7</td>
                        </tr>
                        <tr>
                           <td>
                              <b>HORMONE-RELATED</b>
                           </td>
                        </tr>
                        <tr>
                           <td>Ethylene response factor 110</td>
                           <td>AT5G50080.1</td>
                           <td>1.2</td>
                        </tr>
                        <tr>
                           <td>Auxin amidohydrolase</td>
                           <td>AT1G51760.1</td>
                           <td>3.4</td>
                        </tr>
                        <tr>
                           <td>
                              <b>OTHERS</b>
                           </td>
                        </tr>
                        <tr>
                           <td>Nitrate transporter 1.5</td>
                           <td>AT1G32450.1</td>
                           <td>1.3</td>
                        </tr>
                        <tr>
                           <td>LOB domain-containing protein 1</td>
                           <td>AT1G07900.1</td>
                           <td>2.2</td>
                        </tr>
                        <tr>
                           <td>Amino acid permease 3</td>
                           <td>AT1G77380.1</td>
                           <td>1.8</td>
                        </tr>
                        <tr>
                           <td>Formin homology5</td>
                           <td>AT5G54650.1</td>
                           <td>1.5</td>
                        </tr>
                        <tr>
                           <td>HXXXD-type acyl-transferase protein</td>
                           <td>AT5G41040.1</td>
                           <td>2.4</td>
                        </tr>
                        <tr>
                           <td>Alpha/beta-Hydrolases superfamily protein</td>
                           <td>AT2G39420.1</td>
                           <td>3.4</td>
                        </tr>
                        <tr>
                           <td>P-loop nucleoside triphosphate hydrolases protein</td>
                           <td>AT3G45080.1</td>
                           <td>−2.7</td>
                        </tr>
                        <tr>
                           <td>Glutamate receptor 2.8</td>
                           <td>AT2G29110.1</td>
                           <td>1.5</td>
                        </tr>
                        <tr>
                           <td>Glutamate receptor 2.7</td>
                           <td>AT2G29120.1</td>
                           <td>1.8</td>
                        </tr>
                        <tr>
                           <td>Di-glucose binding protein Kinesin motor domain</td>
                           <td>AT2G22610.1</td>
                           <td>−3.1</td>
                        </tr>
                        <tr>
                           <td>Integrase-type DNA-binding protein</td>
                           <td>AT4G36920.1</td>
                           <td>−1.9</td>
                        </tr>
                        <tr>
                           <td>Glycosyl hydrolase superfamily protein</td>
                           <td>AT4G16260.1</td>
                           <td>2.1</td>
                        </tr>
                     </tbody>
                  </table>
                  <table-wrap-foot>
                     <p>
                        <i>The Log FD (Fold Ratio) was indicated. The complete list with the palm gene ID and
                           Arabidopsis putative orthologsis available in the supplemental material (Table 
                           <a href="#SM2">S2</a>)
                        </i>.
                     </p>
                  </table-wrap-foot>
               </table-wrap>
               <div class="fig" title="fig">
                  <div class="figure" title="figure">
                     <a href="http://doi.org/">
                        <span class="label" tagx="label" title="label">Figure 3</span>
                     </a>
                  </div>
                  <caption class="caption" title="caption">
                     <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0">
                        <p>
                           <b>A dendrogram based on mapped read counts was constructed for the six analyzed samples
                              belonging to Healthy (He), Stage 1 (S1)—middle stage of infestation, Stage 2 (S2)—intense
                              level of infestation
                           </b>.
                        </p>
                     </caption>
                  </caption>
               </div>
               <div class="fig" title="fig">
                  <div class="figure" title="figure">
                     <a href="http://doi.org/">
                        <span class="label" tagx="label" title="label">Figure 4</span>
                     </a>
                  </div>
                  <caption class="caption" title="caption">
                     <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0">
                        <p>
                           <b>(A)</b> Expression changes for each analyzed palm gene in the two pairwise comparisons (He
                           vs. S1 and He vs. S2). In abscissa axis mean expression and in ordinate axis log fold
                           changes were shown. 
                           <b>(B)</b> Volcano plots for both He/S1 and He/S2 comparisons were shown. In abscissa axis log
                           <sub>2</sub> fold change while in ordinate axis log
                           <sub>10</sub>
                           <i>p</i>-value were shown for each identified gene.
                        </p>
                     </caption>
                  </caption>
               </div>
            </div>
            <div class="metabolicpathwayandgenesetenrichmentanalysis" title="sec">
               <div class="title" tagx="title" title="title">Metabolic pathway and gene set enrichment analysis</div>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">The metabolic pathway enrichment analysis indicated that at stage S1 of infestation,
                  phenylalanine metabolism, phenylpropanoid biosynthesis, flavonoid biosynthesis, and
                  xenobiotic metabolism mediated by cytochrome P450 were significantly upregulated by
                  RPW attacks (Table 
                  <a href="#T3">3</a>). In contrast, flavone and flavonol biosynthesis were significantly repressed at
                  S1.
               </p>
               <table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0">
                  <span class="label" tagx="label" title="label">Table 3</span>
                  <caption>
                     <p>
                        <b>PathExpress analysis of the up and downregulated genes in each of the two pairwise
                           comparisons
                        </b>.
                     </p>
                  </caption>
                  <table>
                     <thead>
                        <tr>
                           <th>
                              <b>S1 vs. He</b>
                           </th>
                           <th>
                              <b>
                                 <i>P</i>-values
                              </b>
                           </th>
                        </tr>
                     </thead>
                     <tbody>
                        <tr>
                           <td>
                              <b>UPREGULATED</b>
                           </td>
                        </tr>
                        <tr>
                           <td>Phenylalaninemetabolism</td>
                           <td>4.1 × 10
                              <sup>−3</sup>
                           </td>
                        </tr>
                        <tr>
                           <td>Phenylpropanoidbiosynthesis</td>
                           <td>5.1 × 10
                              <sup>−3</sup>
                           </td>
                        </tr>
                        <tr>
                           <td>Flavonoidbiosynthesis</td>
                           <td>7.9 × 10
                              <sup>−3</sup>
                           </td>
                        </tr>
                        <tr>
                           <td>Metabolism of xenobiotics by cytochrome P450</td>
                           <td>2.6 × 10
                              <sup>−2</sup>
                           </td>
                        </tr>
                        <tr>
                           <td>
                              <b>DOWNREGULATED</b>
                           </td>
                        </tr>
                        <tr>
                           <td>Flavone and flavonolbiosynthesis</td>
                           <td>1.3 × 10
                              <sup>−2</sup>
                           </td>
                        </tr>
                        <tr>
                           <td>
                              <b>S2 vs. He</b>
                           </td>
                           <td>
                              <i>
                                 <b>P</b>
                              </i>
                              <b>-values</b>
                           </td>
                        </tr>
                        <tr>
                           <td>
                              <b>UPREGULATED</b>
                           </td>
                        </tr>
                        <tr>
                           <td>Fatty acid metabolism</td>
                           <td>3.3 × 10
                              <sup>−4</sup>
                           </td>
                        </tr>
                        <tr>
                           <td>Sphingolipid metabolism</td>
                           <td>3.6 × 10
                              <sup>−2</sup>
                           </td>
                        </tr>
                        <tr>
                           <td>Glycerolipid metabolism</td>
                           <td>3.7 × 10
                              <sup>−2</sup>
                           </td>
                        </tr>
                        <tr>
                           <td>Tryptophan metabolism</td>
                           <td>0.05</td>
                        </tr>
                        <tr>
                           <td>Alkaloid biosynthesis</td>
                           <td>0.05</td>
                        </tr>
                        <tr>
                           <td>
                              <b>DOWNREGULATED</b>
                           </td>
                        </tr>
                        <tr>
                           <td>Starch and sucrosemetabolism</td>
                           <td>2.1 × 10
                              <sup>−3</sup>
                           </td>
                        </tr>
                        <tr>
                           <td>Phosphatidylinositol signaling system</td>
                           <td>3.6 × 10
                              <sup>−3</sup>
                           </td>
                        </tr>
                        <tr>
                           <td>Inositol phosphate metabolism</td>
                           <td>9.3 × 10
                              <sup>−3</sup>
                           </td>
                        </tr>
                        <tr>
                           <td>Glycosaminoglycan degradation</td>
                           <td>0.04</td>
                        </tr>
                     </tbody>
                  </table>
                  <table-wrap-foot>
                     <p>
                        <i>Pathways with a P &lt; 0.05 (no corrections) were affected by the RPW attacks. He, Healthy;
                           S1, stage 1; S2, stage 2
                        </i>.
                     </p>
                  </table-wrap-foot>
               </table-wrap>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">At S2, RPW significantly induced lipid-related pathways such as fatty acid, sphingolipid,
                  glycerolipid metabolism. In contrast RPW repressed starch and sucrose metabolism and
                  inositol-related pathways.
               </p>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">Key primary metabolism pathways were influenced by RPW at S2, such as the sucrose
                  (upregulated) and callose metabolism (downregulated) (Figure 
                  <a href="#F5">5</a>). Interestingly key players in transcription regulation of biotic stress responses
                  were upregulated such as WRKYs. Genes encoding receptor kinases (including leucine
                  rich repeat III) and cell organization-related proteins were repressed.
               </p>
               <div class="fig" title="fig">
                  <div class="figure" title="figure">
                     <a href="http://doi.org/">
                        <span class="label" tagx="label" title="label">Figure 5</span>
                     </a>
                  </div>
                  <caption class="caption" title="caption">
                     <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0">
                        <p>
                           <b>Gene set enrichment analysis of the transcriptomic changes during stage 2 (S2)</b>. Pageman web-tool was used. Wilcoxon test with ORA cut off = 1 was used. A scale
                           bar between -4 and 4 was chosen. Increased intensity of red and green respectively
                           represented higher level of upregulation and downregulation.
                        </p>
                     </caption>
                  </caption>
               </div>
            </div>
            <div class="secondarymetabolism,transcriptionfactors,signaling,redox" title="sec">
               <div class="title" tagx="title" title="title">Secondary metabolism, transcription factors, signaling, redox</div>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">At S1, some genes involved in phenylpropanoids were upregulated (Table 
                  <a href="#T2">2</a>). At S2, a high number of genes belonging to secondary metabolism were induced. Some
                  phenylpropanoid genes were strongly activated during the late stage S2 such as O-
                  <i>methyltransferasefamily protein 1</i> and 
                  <i>2, 3-coumarate-CoA ligase, ferulic acid 5-hydroxylase 1, cinnamoyl-CoA reductase</i>.
               </p>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">A list of differentially regulated genes at S2 encoding transcription factors was
                  provided (Table 
                  <a href="#SM4">S4</a>). Regarding signal perception and transduction signaling, different classes of receptor
                  kinases showed different trends of expression (Table 
                  <a href="#SM4">S4</a>). At S1, genes encoding a specific category of signal receptors DUF were differentially
                  regulated: 
                  <i>DUF616</i> was slightly enhanced and 
                  <i>DUF604</i> was repressed (−3.3). At S2, most of the genes of DUF26 protein kinases and LRK10-like
                  (serine/threonine protein kinases) were more abundant while leucine-rich repeat III,
                  VI, and XII were mostly repressed. The transcript abundance of genes of large enzyme
                  families (thioredoxin-related and cytochrome-related) that are involved in redox was
                  severely affected by RPW attack.
               </p>
            </div>
            <div class="hormone-relatedpathways" title="sec">
               <div class="title" tagx="title" title="title">Hormone-related pathways</div>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">At S1, 
                  <i>Auxin amidohydrolase</i> and 
                  <i>ethylene response factor 110</i> were slightly higher in abundance. At S2, significant transcriptional changes were
                  observed for a group of genes that are involved in auxin signal transduction (
                  <i>PIN2</i> and 
                  <i>PIN4</i>) (Table 
                  <a href="#SM4">S4</a>). ABA-related pathways were affected as demonstrated by the induction of 
                  <i>HVA22J</i> and two 
                  <i>GRAM-domain-containing proteins</i>. In contrast, the expressions of 
                  <i>ABA1</i> and 
                  <i>AREB3</i> were lower at S2 compared to He. In general hormone-related pathways were drastically
                  affected by RPW at S2. Two 
                  <i>ACC oxidases</i> that are involved in ethylene biosynthesis were upregulated. Two cytokinin receptors
                  were significantly repressed, 
                  <i>HK2</i> and 
                  <i>HK3</i>. Some significant increases in transcript abundance were observed for genes that
                  were involved in jasmonate synthesis (
                  <i>allene oxide synthase</i> and 
                  <i>OPR2</i>) and salicylic acid-mediated response (glucosyltransferases).
               </p>
            </div>
            <div class="bioticstressresponses" title="sec">
               <div class="title" tagx="title" title="title">Biotic stress responses</div>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">Defense-related pathways were activated at S1 as shown by the upregulation of genes
                  encoding a 
                  <i>pathogenesis-related thaumatin protein</i> and a 
                  <i>disease resistance/LRR protein</i>. An overview of the transcriptomic changes involved in biotic stress pathways at
                  S2 was provided (Figure 
                  <a href="#F6">6</a>). Extensive induction of genes encoding enzymes involved in redox state and peroxidases
                  was observed in response to RPW at both disease stages. Two peroxidase genes were
                  differentially regulated at S1: one was induced (Log FD = 2.4) and the other was repressed
                  (Log FD = −4.0). Key genes encoding antioxidant enzymes were slightly upregulated
                  at S1 (Table 
                  <a href="#T2">2</a>).
               </p>
               <div class="fig" title="fig">
                  <div class="figure" title="figure">
                     <a href="http://doi.org/">
                        <span class="label" tagx="label" title="label">Figure 6</span>
                     </a>
                  </div>
                  <caption class="caption" title="caption">
                     <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0">
                        <p>
                           <b>Transcriptomic changes involved in biotic stress responses at stage S2</b>. The same scale bar used for the other figures was shown. Negative values represented
                           repressed genes (green) at S2 in comparison to He and positive values (red) represented
                           upregulated genes.
                        </p>
                     </caption>
                  </caption>
               </div>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">As far as it concerns, cell wall modification genes were upregulated at S1. Two laccase
                  genes were induced at both S1 and S2. At S2, five genes involved in cell wall restructuring
                  and encoding five 
                  <i>beta-1,3,- glucan hydrolases</i> were enhanced.
               </p>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">Pathogenesis-related (PR) proteins were mostly repressed although three disease-resistance
                  family proteins were upregulated at late symptomatic stage. At S1, two WRKYs were
                  upregulated. At S2, several 
                  <i>WRKY</i> genes were clearly enhanced, such as 
                  <i>WRKY9, WRKY14, WRKY40, WRKY47, WRKY28, WRKY72, WRKY75</i>, and 
                  <i>WRKY51</i>. In contrast, 
                  <i>WRKY2</i> was downregulated.
               </p>
            </div>
            <div class="qrt-pcranalysis" title="sec">
               <div class="title" tagx="title" title="title">qRT-PCR analysis</div>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">qRT-PCR were performed to validate expression trend of nine RPW-regulated genes (Table
                  
                  <a href="#T4">4</a>). Statistical differences of 15 of the 18 pairwise comparisons of gene expressions
                  observed by RNA-seq were confirmed by qRT-PCR analysis. 
                  <i>Alpha-amylase</i> and 
                  <i>invertase</i> belonged to sugar and starch metabolism. RNA-seq analysis showed that both these
                  two genes were not regulated at stage S1 while they were induced at stage S2. qRT-PCR
                  confirmed data on S2 but showed that they were also significantly induced at S1. 
                  <i>UDP-glycosyltransferase</i> was confirmed to be induced at S2 while 
                  <i>UDP-glucose-SA glycosyltransferase</i> was enhanced at both S1 and S2. 
                  <i>Laccase7</i>, involved in secondary metabolism, was upregulated at S1. 
                  <i>Auxin amidohydrolase</i> was upregulated at both stages of infestation. 
                  <i>WRKY72</i> and 
                  <i>WRKY75</i> were confirmed to be higher in abundance at S2.
               </p>
               <table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0">
                  <span class="label" tagx="label" title="label">Table 4</span>
                  <caption>
                     <p>
                        <b>qRT-PCR analysis of 9 chosen genes</b>.
                     </p>
                  </caption>
                  <table>
                     <thead>
                        <tr>
                           <th>
                              <b>Date palm ID</b>
                           </th>
                           <th>
                              <b>Description</b>
                           </th>
                           <th>
                              <b>ANOVA (RNA-seq)</b>
                           </th>
                           <th>
                              <b>ANOVA (qRT-PCR)</b>
                           </th>
                        </tr>
                        <tr>
                           <th></th>
                           <th></th>
                           <th>
                              <b>He vs. S1</b>
                           </th>
                           <th>
                              <b>He vs. S2</b>
                           </th>
                           <th>
                              <b>He vs. S1</b>
                           </th>
                           <th>
                              <b>He vs. S2</b>
                           </th>
                        </tr>
                     </thead>
                     <tbody>
                        <tr>
                           <td>PDK_30s654931g003</td>
                           <td>Alpha-Amylase</td>
                           <td>n.s.</td>
                           <td>6.4 (
                              <a href="#TN1">
                                 <sup>*</sup>
                              </a>)
                           </td>
                           <td>1.5 (
                              <a href="#TN1">
                                 <sup>*</sup>
                              </a>)
                           </td>
                           <td>4.9 (
                              <a href="#TN1">
                                 <sup>*</sup>
                              </a>)
                           </td>
                        </tr>
                        <tr>
                           <td>PDK_30s705371g005</td>
                           <td>UDP-Glycosyltransf.</td>
                           <td>n.s.</td>
                           <td>5.6 (
                              <a href="#TN1">
                                 <sup>*</sup>
                              </a>)
                           </td>
                           <td>n.s.</td>
                           <td>80.9 (
                              <a href="#TN1">
                                 <sup>*</sup>
                              </a>)
                           </td>
                        </tr>
                        <tr>
                           <td>PDK_30s888011g002</td>
                           <td>Laccase 7</td>
                           <td>2.9 (
                              <a href="#TN1">
                                 <sup>*</sup>
                              </a>)
                           </td>
                           <td>n.s.</td>
                           <td>0.37 (
                              <a href="#TN1">
                                 <sup>*</sup>
                              </a>)
                           </td>
                           <td>n.s.</td>
                        </tr>
                        <tr>
                           <td>PDK_30s835131g001</td>
                           <td>WRKY 75</td>
                           <td>n.s.</td>
                           <td>7.5 (
                              <a href="#TN1">
                                 <sup>*</sup>
                              </a>)
                           </td>
                           <td>n.s.</td>
                           <td>4.3 (
                              <a href="#TN1">
                                 <sup>*</sup>
                              </a>)
                           </td>
                        </tr>
                        <tr>
                           <td>PDK_30s883821g030</td>
                           <td>UDP-glucose-SA glycosyltransf.</td>
                           <td>n.s.</td>
                           <td>1.5 (
                              <a href="#TN1">
                                 <sup>*</sup>
                              </a>)
                           </td>
                           <td>2.4 (
                              <a href="#TN1">
                                 <sup>*</sup>
                              </a>)
                           </td>
                           <td>3.2 (
                              <a href="#TN1">
                                 <sup>*</sup>
                              </a>)
                           </td>
                        </tr>
                        <tr>
                           <td>PDK_30s936891g001</td>
                           <td>WRKY 40</td>
                           <td>2.8 (
                              <a href="#TN1">
                                 <sup>*</sup>
                              </a>)
                           </td>
                           <td>2.7 (
                              <a href="#TN1">
                                 <sup>*</sup>
                              </a>)
                           </td>
                           <td>9.2 (
                              <a href="#TN1">
                                 <sup>*</sup>
                              </a>)
                           </td>
                           <td>1.4 (
                              <a href="#TN1">
                                 <sup>*</sup>
                              </a>)
                           </td>
                        </tr>
                        <tr>
                           <td>PDK_30s1151561g006</td>
                           <td>Invertase</td>
                           <td>n.s.</td>
                           <td>3.0 (
                              <a href="#TN1">
                                 <sup>*</sup>
                              </a>)
                           </td>
                           <td>0.8 (
                              <a href="#TN1">
                                 <sup>*</sup>
                              </a>)
                           </td>
                           <td>1.4 (
                              <a href="#TN1">
                                 <sup>*</sup>
                              </a>)
                           </td>
                        </tr>
                        <tr>
                           <td>PDK_30s1173851g004</td>
                           <td>Auxin amidohydrolase</td>
                           <td>3.4 (
                              <a href="#TN1">
                                 <sup>*</sup>
                              </a>)
                           </td>
                           <td>1.7 (
                              <a href="#TN1">
                                 <sup>*</sup>
                              </a>)
                           </td>
                           <td>10.1 (
                              <a href="#TN1">
                                 <sup>*</sup>
                              </a>)
                           </td>
                           <td>1.2 (
                              <a href="#TN1">
                                 <sup>*</sup>
                              </a>)
                           </td>
                        </tr>
                        <tr>
                           <td>PDK_30s1138471g004</td>
                           <td>WRKY 72</td>
                           <td>n.s.</td>
                           <td>3.3</td>
                           <td>n.s.</td>
                           <td>0.7 (
                              <a href="#TN1">
                                 <sup>*</sup>
                              </a>)
                           </td>
                        </tr>
                     </tbody>
                  </table>
                  <table-wrap-foot>
                     <p>
                        <i>Statistical analysis using ANOVA (P &lt; 0.05) was shown</i>:
                     </p>
                     <div class="fn-type-" title="">
                        <span class="label" tagx="label" title="label">*</span>
                        <p>
                           <i>means significant, n.s. means not significant. Log
                              <sub>2</sub> fold ratio were indicated
                           </i>.
                        </p>
                     </div>
                  </table-wrap-foot>
               </table-wrap>
            </div>
         </div>
         <div class="discussion" title="discussion">
            <div class="title" tagx="title" title="title">Discussion</div>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">In this work, we addressed the lack of knowledge on transcriptomic responses to RPW
               attacks. Plant secondary metabolites are well-known compounds involved in plant-insect
               interactions (Becerra, 
               <a href="#B3">2007</a>). Phenylpropanoid genes were strongly affected by RPW attacks. This evidence highlighted
               that palms are activating counteracting responses to RPW from the middle stage of
               infection. We observed that BACT3 and IIL1 genes and other genes (
               <i>allinase</i> and 
               <i>TAR2</i>) involved in the biosynthesis of sulfur-containing compounds were enhanced. These
               compounds have well-known defensive properties. Plants have been engineered to produce
               a cyanogenic glycoside, obtaining enhanced resistance to Phyllotretanemorum in 
               <i>Arabidopsis</i> (Tattersall et al., 
               <a href="#B32">2001</a>). These genes were not induced at S1 implying that palm responses to RPW attacks
               are activated too late and in an inefficient manner. Genes encoding strictosidine
               synthases were clearly enhanced only in response to RPW during the late stage. These
               genes are involved in alkaloid metabolism, a pathway that is generally stimulated
               in defense against insect herbivory. Flavonoids are known to be important compounds
               playing a key role in plant-insect interactions (Treutter, 
               <a href="#B35">2006</a>). We observed an unclear pattern of expression of some members of this pathway. 
               <i>Flavanone isomerase</i> was repressed while 
               <i>chalcone synthase</i> was induced. Taken together all these data related to secondary metabolism let us
               to conclude that 
               <i>Phoenix canariensis</i> did not trigger appropriate defenses to promptly counterbalance RPW attacks.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">This is might be due to two factors: (1) the RPW-induced genes involved in phenylpropanoids
               are induced only at late stage when plants are already compromised, (2) it is possible
               that the genes playing a major roles in the transcriptional regulation of secondary
               metabolism were not sufficiently enhanced at middle stage of infection (S1).
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">Other key genes involved in volatile biosynthesis were differentially regulated. The
               emission of plant volatiles may represent a direct defensive benefit by precluding
               oviposition (De Moraes et al., 
               <a href="#B7">2001</a>) or attracting predators (Kessler and Baldwin, 
               <a href="#B18">2001</a>). The important transcript changes that were observed in volatile pathways confirmed
               that infestations may lead to drastic changes in the volatile profiles detectable
               by biological methods (Nakash et al., 
               <a href="#B27">2000</a>). Novel methods based on detection of induced volatile organic compounds have been
               recently proposed and should be applied to detect RPW attacks (Dandekar et al., 
               <a href="#B6">2010</a>; Martinelli et al., 
               <a href="#B23">2015</a>).
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">Figure 
               <a href="#F7">7</a> summarized a global view of the main changes that were detected in leaf metabolism
               considering both analyzed symptom stages. We identified some RPW-induced genes encoding
               receptor kinases belonging to receptor kinase VIII, DUF26 protein kinases, RLK10-like,
               and S-locus glycoprotein-like categories. LRR receptor like serine/threonine-protein
               kinase, S-locus receptor kinase, and TIR-NBS-LRR resistance proteins have been previously
               found to be upregulated by aphid attacks (Coppola et al., 
               <a href="#B5">2013</a>). Interestingly 
               <i>LRR III</i> and 
               <i>LRR XII</i> were mostly repressed. It is possible that the repression of some of these classes
               might be involved in a delay or complete lack of signal perception of RPW attacks
               and consequently in inadequate immune responses to RPW infestations. However, the
               size and complexity of these gene families make it extremely difficult to understand
               which family could be involved in susceptible reactions to RPW.
            </p>
            <div class="fig" title="fig">
               <div class="figure" title="figure">
                  <a href="http://doi.org/">
                     <span class="label" tagx="label" title="label">Figure 7</span>
                  </a>
               </div>
               <caption class="caption" title="caption">
                  <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0">
                     <p>
                        <b>Global view of the transcriptomic changes in palm leaves in response to RPW attacks</b>. The genes, pathways, and cell functions that were differentially expressed are indicated
                        with a square (red for upregulated and green for downregulated).
                     </p>
                  </caption>
               </caption>
            </div>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">Plant-induced defense responses are modulated by a network of interconnecting hormone
               crosstalk in which salicylic acid (SA), jasmonic acid (JA), and ethylene (ET) play
               pivotal roles (Glazebrook, 
               <a href="#B11">2001</a>). Jasmonic-related genes were induced by RPW attacks at S2. The synthesis of JA and
               its precursors and derivatives (collectively termed jasmonates) occurs when plants
               are wounded. Jasmonates play a central role in regulating defense responses to herbivores
               (Heil and Ton, 
               <a href="#B15">2008</a>). It is expected that a trauma caused by chewing insects or mechanical damage results
               in the rapid accumulation of JA at the site of wounding. Previous data have demonstrated
               that the repression of JA responses induces susceptibility to herbivorous insects
               (Kessler et al., 
               <a href="#B19">2004</a>). Here, we observed the upregulation of two key genes (
               <i>AOS</i> and 
               <i>OPR2</i>) that are involved in JA biosynthesis during S2. Interestingly, jasmonic-related
               genes were not upregulated during the middle stage (S1), although their roles in herbivore
               attack have been demonstrated. This evidence is another clue that demonstrates the
               inefficient response of 
               <i>Phoenix canariensis</i> to RPW infestations. Although crosstalk between defense signaling pathways presents
               a powerful regulatory instrument, it may also represent a weakness. Here, we observed
               that a 
               <i>UDP-glucosyltransferase</i> that is involved in the conversion of SA to methyl-salicylate was induced during
               S2. QRT-PCR showed that this gene was significantly upregulated at both stages. It
               is well known that systemic acquired resistance-against biotroph attacks is antagonist
               to JA-mediated responses to necrotrophs such as insects. It is possible that the induction
               of 
               <i>salicylic acid methyl transferase</i> would benefit the insect by regulating negatively jasmonic acid-mediated responses.
               However, previous transcriptomic studies highlighted the upregulation of SA-related
               genes in response to aphid attacks while the effect on JA-related genes showed a more
               complex regulation (Coppola et al., 
               <a href="#B5">2013</a>). These differences in hormonal plant responses may be due to their diverse type
               of insect feeding. ABA is connected to the SA-JA-ET network, stimulating JA biosynthesis
               and antagonizing the onset of SA-dependent defenses. Here, we reported that RPW induced
               genes that are involved in ABA metabolism and response (
               <i>HVA22J, C3HC4-type ring finger</i>, two 
               <i>GRAM-domain-containing proteins</i>). The antagonistic effect of auxin on SA signaling has been documented (Pieterse
               et al., 
               <a href="#B28">2009</a>).
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">PR proteins were associated with the induction of both locally and systemically induced
               resistance to pathogens (Van Loon, 
               <a href="#B36">1997</a>). Here, we showed that PR genes were differentially regulated by RPW infestations.
               The majority of PR genes were downregulated implying again a probably weak and inefficacious
               response to RPW invasion. WRKY proteins are a well-known family of transcription factors
               involved in biotic stress responses and comprise more than 70 members in 
               <i>Arabidopsis</i> (Rushton et al., 
               <a href="#B30">2010</a>). Induction of WRKY members have been observed in longitudinal studies of Macrosyphum
               euphorbiae aphid attacks (Coppola et al., 
               <a href="#B5">2013</a>). Upregulation of some members of this family was observed mainly at S2. However,
               
               <i>WRKY40</i> and 
               <i>WRKY51</i> were enhanced also at S1 and might be considered candidate host biomarkers for RPW
               infestations. 
               <i>WRKY40</i> gene has been linked to response to the aphid B. brassicae attacks (Kusnierczyk et
               al., 
               <a href="#B20">2008</a>). A total of 8 WRKY members were significantly enhanced by RPW at stage S2. This
               important evidence indicates that palms activated a transcription induction of defense
               responses to RPW larvae feeding although these responses are too late. 
               <i>WRKY51</i> transcript was the most abundant (log FD = 7.7). Gao et al. (
               <a href="#B10">2011</a>) showed that 
               <i>WRKY51</i> plays a key role in stimulating SA-mediated responses and inhibiting JA-inducible
               defenses, resulting in enhanced resistance to biotrophs but increased susceptibility
               to necrotrophs. Basing on these data, we may speculate that the induction of this
               gene as observed in RPW-attacked trees may have negative effects on host plant.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">Laccases are enzymes leading to the polymerization of monolignol precursors of lignin.
               The activation of these genes at middle stage of infestation confirmed that attacked
               palms responded to RPW invasion. A large number of 
               <i>peroxidase</i> genes were induced at stage S2. Two of them were differentially regulated also at
               stage S1. These transcript changes were expected since peroxidases are well known
               to be involved in ROS-detoxifying reactions, in the modulation of the redox and Ca
               <sup>2+</sup> homeostasis as well as the regulation of defense-related genes (Kawano, 
               <a href="#B17">2003</a>). Peroxidases act against biotic attacks in a passive way building up stronger walls
               or actively producing ROS against pathogens (Moura et al., 
               <a href="#B26">2010</a>). When the attacking organism overcomes lignin barriers, peroxidases may be important
               to isolate the intruder. The evidence that these genes were induced at S1 is a clue
               that palm are activating defense responses against pathogen colonization.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">The ability to rapidly identify RPW infestations is of particular interest because
               it would allow to rapidly activate the management practices against RPW. In addition,
               the qRT-PCR analysis of RPW-regulated genes might help in speeding up detection of
               infestations. Genes upregulated at S1, involved in biotic stress responses, may be
               further analyzed to check their specificity for RPW attacks such as WRKY40 and WRKY51.
               A panel of genes may be analyzed in response to abiotic stresses and other typical
               biotic attacks to check their level of reliability in improving RPW management.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">Some possible short-term therapeutic approaches might be proposed. A first approach
               could focus on boosting JA-mediated defense responses by applying molecules, such
               as methyl jasmonate, JA, and linolenic acid, which would most likely shift the hormone
               crosstalk to JA, penalizing SA and allowing trees to localize efforts to pathways
               that might impair the larvae feeding aptitude. Another possible strategy could be
               the over-expression of genes playing a key role in the production of compounds that
               are repellant or not attractive to larvae or adults. Because auxin-responsive genes
               are antagonists of SA, a worth while experiment would be to test the effects of auxin
               inhibitor compounds on an attacked tree. It is possibly worthwhile to test the combination
               of sucrose with agents that stimulate xenobiotic responses through ROS signaling induction
               as previously suggested (Sulmon et al., 
               <a href="#B31">2006</a>). Although an effective RPW management is still absent, the data presented in this
               work may lead to novel methods of detection of infestations and offer an essential
               contribution to save palms from RPW attacks, especially in designated World Heritage
               areas.
            </p>
         </div>
         <div class="authorcontributions" title="sec">
            <div class="title" tagx="title" title="title">Author contributions</div>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">FM and AG conceived and designed the experiments. Experimental work was performed
               by FM, VF, MA, AG. Data were analyzed by FM, EB, MA, AG, VF, and ML. Manuscript was
               mainly written by FM in collaboration with AG.
            </p>
         </div>
         <div class="funding" title="sec">
            <div class="title" tagx="title" title="title">Funding</div>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">The present research work was financially supported by Propalma project funded by
               the “Ministero delle Politiche Agricole, Alimentari e Forestali.”
            </p>
            <div class="conflictofintereststatement" title="sec">
               <div class="title" tagx="title" title="title">Conflict of interest statement</div>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">The authors declare that the research was conducted in the absence of any commercial
                  or financial relationships that could be construed as a potential conflict of interest.
               </p>
            </div>
         </div>
      </div>
      <div class="back" title="back">
         <div class="supplementary-material" title="supplementary-material">
            <div class="title" tagx="title" title="title">Supplementary material</div>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0">The Supplementary Material for this article can be found online at: 
               <a href="http://journal.frontiersin.org/article/10.3389/fpls.2015.00817">http://journal.frontiersin.org/article/10.3389/fpls.2015.00817</a>
            </p>
            <div class="local-data" title="local-data">
               <div class="media" title="media">
                  <a href="Table1.XLSX">LINK</a>
                  <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0">
                     <p>Click here for additional data file.</p>
                  </caption>
               </div>
            </div>
            <div class="local-data" title="local-data">
               <div class="media" title="media">
                  <a href="Table2.XLSX">LINK</a>
                  <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0">
                     <p>Click here for additional data file.</p>
                  </caption>
               </div>
            </div>
            <div class="local-data" title="local-data">
               <div class="media" title="media">
                  <a href="Table3.XLSX">LINK</a>
                  <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0">
                     <p>Click here for additional data file.</p>
                  </caption>
               </div>
            </div>
            <div class="local-data" title="local-data">
               <div class="media" title="media">
                  <a href="Table4.XLSX">LINK</a>
                  <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0">
                     <p>Click here for additional data file.</p>
                  </caption>
               </div>
            </div>
         </div>
         <div class="references">References</div>
         <div tag="ref-list">
            <ul>
               <div class="title" tagx="title" title="title">References</div>
               <li tag="ref">
                  <a name="B1"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Al-Dous</span>
                           <span class="given-names" tagx="given-names" title="given-names">E. K.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">George</span>
                           <span class="given-names" tagx="given-names" title="given-names">B.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Al-Mahmoud</span>
                           <span class="given-names" tagx="given-names" title="given-names">M. E.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Al-Jaber</span>
                           <span class="given-names" tagx="given-names" title="given-names">M. Y.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Wang</span>
                           <span class="given-names" tagx="given-names" title="given-names">H.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Salameh</span>
                           <span class="given-names" tagx="given-names" title="given-names">Y. M.</span>
                        </span>
                        <span class="etal" title="etal">
                           <i>et al.</i>
                        </span>
                     </span>. (
                     <span class="year" tagx="year" title="year">2011</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">De novo genome sequencing and comparative genomics of date palm (
                        <i>Phoenix dactylifera</i>)
                     </span>. 
                     <span class="source" tagx="source" title="source">Nat. Biotechnol.</span>
                     <span class="volume" tagx="volume" title="volume">29</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">521</span>–
                     <span class="lpage" tagx="lpage" title="lpage">527</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1038/nbt.1860">10.1038/nbt.1860</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/21623354">21623354</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B2"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Al-Mssallem</span>
                           <span class="given-names" tagx="given-names" title="given-names">I. S.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hu</span>
                           <span class="given-names" tagx="given-names" title="given-names">S.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Zhang</span>
                           <span class="given-names" tagx="given-names" title="given-names">X.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Lin</span>
                           <span class="given-names" tagx="given-names" title="given-names">Q.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Liu</span>
                           <span class="given-names" tagx="given-names" title="given-names">W.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Tan</span>
                           <span class="given-names" tagx="given-names" title="given-names">J.</span>
                        </span>
                        <span class="etal" title="etal">
                           <i>et al.</i>
                        </span>
                     </span>. (
                     <span class="year" tagx="year" title="year">2013</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">Genome sequence of the date palm 
                        <i>Phoenix dactylifera</i> L
                     </span>. 
                     <span class="source" tagx="source" title="source">Nat. Commun.</span>
                     <span class="volume" tagx="volume" title="volume">4</span>:
                     <span class="fpage" tagx="fpage" title="fpage">2274</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1038/ncomms3274">10.1038/ncomms3274</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/23917264">23917264</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B3"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Becerra</span>
                           <span class="given-names" tagx="given-names" title="given-names">J. X.</span>
                        </span>
                     </span> (
                     <span class="year" tagx="year" title="year">2007</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">The impact of herbivore-plant coevolution on plant community structure</span>. 
                     <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U.S.A.</span>
                     <span class="volume" tagx="volume" title="volume">104</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">7483</span>–
                     <span class="lpage" tagx="lpage" title="lpage">7488</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1073/pnas.0608253104">10.1073/pnas.0608253104</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/17456606">17456606</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B4"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Chen</span>
                           <span class="given-names" tagx="given-names" title="given-names">X. R.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Zhang</span>
                           <span class="given-names" tagx="given-names" title="given-names">B. Y.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Xing</span>
                           <span class="given-names" tagx="given-names" title="given-names">Y. P.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Li</span>
                           <span class="given-names" tagx="given-names" title="given-names">Q. Y.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Li</span>
                           <span class="given-names" tagx="given-names" title="given-names">Y. P.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Tong</span>
                           <span class="given-names" tagx="given-names" title="given-names">Y. H.</span>
                        </span>
                        <span class="etal" title="etal">
                           <i>et al.</i>
                        </span>
                     </span>. (
                     <span class="year" tagx="year" title="year">2014</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">Transcriptomic analysis of the phytopathogenic oomycete Phytophthoracactorum provides
                        insights into infection-related effectors
                     </span>. 
                     <span class="source" tagx="source" title="source">BMC Genomics</span>
                     <span class="volume" tagx="volume" title="volume">15</span>:
                     <span class="fpage" tagx="fpage" title="fpage">980</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1186/1471-2164-15-980">10.1186/1471-2164-15-980</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/25406848">25406848</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B5"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Coppola</span>
                           <span class="given-names" tagx="given-names" title="given-names">V.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Coppola</span>
                           <span class="given-names" tagx="given-names" title="given-names">M.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Rocco</span>
                           <span class="given-names" tagx="given-names" title="given-names">M.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Digilio</span>
                           <span class="given-names" tagx="given-names" title="given-names">M. C.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">D'Ambrosio</span>
                           <span class="given-names" tagx="given-names" title="given-names">C.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Renzone</span>
                           <span class="given-names" tagx="given-names" title="given-names">G.</span>
                        </span>
                        <span class="etal" title="etal">
                           <i>et al.</i>
                        </span>
                     </span>. (
                     <span class="year" tagx="year" title="year">2013</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">Transcriptomic and proteomic analysis of a compatible tomato-aphid interaction reveals
                        a predominant salicylic acid-dependent plant response
                     </span>. 
                     <span class="source" tagx="source" title="source">BMC Genomics</span>
                     <span class="volume" tagx="volume" title="volume">14</span>:
                     <span class="fpage" tagx="fpage" title="fpage">515</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1186/1471-2164-14-515">10.1186/1471-2164-14-515</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/23895395">23895395</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B6"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Dandekar</span>
                           <span class="given-names" tagx="given-names" title="given-names">A. M.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Martinelli</span>
                           <span class="given-names" tagx="given-names" title="given-names">F.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Davis</span>
                           <span class="given-names" tagx="given-names" title="given-names">C. E.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Bhushan</span>
                           <span class="given-names" tagx="given-names" title="given-names">A.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Zhao</span>
                           <span class="given-names" tagx="given-names" title="given-names">W.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Fiehn</span>
                           <span class="given-names" tagx="given-names" title="given-names">O.</span>
                        </span>
                        <span class="etal" title="etal">
                           <i>et al.</i>
                        </span>
                     </span>. (
                     <span class="year" tagx="year" title="year">2010</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">Analysis of early host responses for asymptomatic disease detection and management
                        of specialty crops
                     </span>. 
                     <span class="source" tagx="source" title="source">Crit. Rev. Immunol.</span>
                     <span class="volume" tagx="volume" title="volume">30</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">277</span>–
                     <span class="lpage" tagx="lpage" title="lpage">289</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1615/CritRevImmunol.v30.i3.50">10.1615/CritRevImmunol.v30.i3.50</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/20370635">20370635</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B7"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">De Moraes</span>
                           <span class="given-names" tagx="given-names" title="given-names">C. M.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Mescher</span>
                           <span class="given-names" tagx="given-names" title="given-names">M. C.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Tumlinson</span>
                           <span class="given-names" tagx="given-names" title="given-names">J. H.</span>
                        </span>
                     </span> (
                     <span class="year" tagx="year" title="year">2001</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">Caterpillar-induced nocturnal plant volatiles repel conspecific females</span>. 
                     <span class="source" tagx="source" title="source">Nature</span>
                     <span class="volume" tagx="volume" title="volume">410</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">577</span>–
                     <span class="lpage" tagx="lpage" title="lpage">580</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1038/35069058">10.1038/35069058</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/11279494">11279494</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B8"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Faleiro</span>
                           <span class="given-names" tagx="given-names" title="given-names">J.</span>
                        </span>
                     </span> (
                     <span class="year" tagx="year" title="year">2006</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">A review of the issues and management of the Red Palm Weevil 
                        <i>Rhynchophorus ferrugineus</i> (Coleoptera: Rhynchophoridae) in coconut and date palm during the last one hundred
                        years
                     </span>. 
                     <span class="source" tagx="source" title="source">Int. J. Trop. Insect Sci.</span>
                     <span class="volume" tagx="volume" title="volume">26</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">135</span>–
                     <span class="lpage" tagx="lpage" title="lpage">154</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1079/IJT2006113">10.1079/IJT2006113</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B9"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Gambino</span>
                           <span class="given-names" tagx="given-names" title="given-names">G.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Perrone</span>
                           <span class="given-names" tagx="given-names" title="given-names">I.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Gribaudo</span>
                           <span class="given-names" tagx="given-names" title="given-names">I.</span>
                        </span>
                     </span> (
                     <span class="year" tagx="year" title="year">2008</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">A rapid and effective method for RNA extraction from different tissues of grapevine
                        and other woody plants
                     </span>. 
                     <span class="source" tagx="source" title="source">Phytochem. Anal.</span>
                     <span class="volume" tagx="volume" title="volume">19</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">520</span>–
                     <span class="lpage" tagx="lpage" title="lpage">525</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1002/pca.1078">10.1002/pca.1078</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/18618437">18618437</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B10"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Gao</span>
                           <span class="given-names" tagx="given-names" title="given-names">Q. M.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Venugopal</span>
                           <span class="given-names" tagx="given-names" title="given-names">S.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Navarre</span>
                           <span class="given-names" tagx="given-names" title="given-names">D.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kachroo</span>
                           <span class="given-names" tagx="given-names" title="given-names">A.</span>
                        </span>
                     </span> (
                     <span class="year" tagx="year" title="year">2011</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">Low oleic acid-derived repression of jasmonic acid-inducible defense responses requires
                        the WRKY50 and WRKY51 proteins
                     </span>. 
                     <span class="source" tagx="source" title="source">Plant Physiol.</span>
                     <span class="volume" tagx="volume" title="volume">155</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">464</span>–
                     <span class="lpage" tagx="lpage" title="lpage">476</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1104/pp.110.166876">10.1104/pp.110.166876</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/21030507">21030507</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B11"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Glazebrook</span>
                           <span class="given-names" tagx="given-names" title="given-names">J.</span>
                        </span>
                     </span> (
                     <span class="year" tagx="year" title="year">2001</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">Genes controlling expression of defense responses in 
                        <i>Arabidopsis</i>—2001 status
                     </span>. 
                     <span class="source" tagx="source" title="source">Curr. Opin. Plant Biol.</span>
                     <span class="volume" tagx="volume" title="volume">4</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">301</span>–
                     <span class="lpage" tagx="lpage" title="lpage">308</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1016/S1369-5266(00)00177-1">10.1016/S1369-5266(00)00177-1</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/11418339">11418339</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B12"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Goffard</span>
                           <span class="given-names" tagx="given-names" title="given-names">N.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Weiller</span>
                           <span class="given-names" tagx="given-names" title="given-names">G.</span>
                        </span>
                     </span> (
                     <span class="year" tagx="year" title="year">2007</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">PathExpress: a web-based tool to identify relevant pathways in gene expression data</span>. 
                     <span class="source" tagx="source" title="source">Nucleic Acids Res.</span>
                     <span class="volume" tagx="volume" title="volume">35</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">176</span>–
                     <span class="lpage" tagx="lpage" title="lpage">181</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1093/nar/gkm261">10.1093/nar/gkm261</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/17586825">17586825</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B13"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Gutierrez</span>
                           <span class="given-names" tagx="given-names" title="given-names">A.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Ruiz</span>
                           <span class="given-names" tagx="given-names" title="given-names">V.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Moltó</span>
                           <span class="given-names" tagx="given-names" title="given-names">E.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Tapia</span>
                           <span class="given-names" tagx="given-names" title="given-names">G.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Téllez</span>
                           <span class="given-names" tagx="given-names" title="given-names">M. M.</span>
                        </span>
                     </span> (
                     <span class="year" tagx="year" title="year">2010</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">Development of a bioacoustic sensor for the early detection of Red Palm Weevil (
                        <i>Rhynchophorus ferrugineus</i> Olivier)
                     </span>. 
                     <span class="source" tagx="source" title="source">Crop Prot.</span>
                     <span class="volume" tagx="volume" title="volume">29</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">671</span>–
                     <span class="lpage" tagx="lpage" title="lpage">676</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1016/j.cropro.2010.02.001">10.1016/j.cropro.2010.02.001</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B14"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Haff</span>
                           <span class="given-names" tagx="given-names" title="given-names">R. P.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Slaughter</span>
                           <span class="given-names" tagx="given-names" title="given-names">D. C.</span>
                        </span>
                     </span> (
                     <span class="year" tagx="year" title="year">2004</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">Real-time X-ray inspection of wheat for infestation by the granary weevil, 
                        <i>Sitophilus granarius</i> (L)
                     </span>. 
                     <span class="source" tagx="source" title="source">Trans. Am. Soc. Agric. Eng.</span>
                     <span class="volume" tagx="volume" title="volume">47</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">531</span>–
                     <span class="lpage" tagx="lpage" title="lpage">537</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.13031/2013.16022">10.13031/2013.16022</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B15"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Heil</span>
                           <span class="given-names" tagx="given-names" title="given-names">M.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Ton</span>
                           <span class="given-names" tagx="given-names" title="given-names">J.</span>
                        </span>
                     </span> (
                     <span class="year" tagx="year" title="year">2008</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">Long-distance signalling in plant defence</span>. 
                     <span class="source" tagx="source" title="source">Trends Plant Sci.</span>
                     <span class="volume" tagx="volume" title="volume">13</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">264</span>–
                     <span class="lpage" tagx="lpage" title="lpage">272</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1016/j.tplants.2008.03.005">10.1016/j.tplants.2008.03.005</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/18487073">18487073</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B16"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Ju</span>
                           <span class="given-names" tagx="given-names" title="given-names">R. T.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Wang</span>
                           <span class="given-names" tagx="given-names" title="given-names">F.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Wan</span>
                           <span class="given-names" tagx="given-names" title="given-names">F. H.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Li</span>
                           <span class="given-names" tagx="given-names" title="given-names">B.</span>
                        </span>
                     </span> (
                     <span class="year" tagx="year" title="year">2011</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">Effect of host plants on development and reproduction of 
                        <i>Rhynchophorus ferrugineus</i> (Olivier) (
                        <i>Coleoptera: Curculionidae</i>)
                     </span>. 
                     <span class="source" tagx="source" title="source">J. Pest. Sci.</span>
                     <span class="volume" tagx="volume" title="volume">84</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">33</span>–
                     <span class="lpage" tagx="lpage" title="lpage">39</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1007/s10340-010-0323-4">10.1007/s10340-010-0323-4</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B17"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kawano</span>
                           <span class="given-names" tagx="given-names" title="given-names">T.</span>
                        </span>
                     </span> (
                     <span class="year" tagx="year" title="year">2003</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">Roles of the reactive oxygen species-generating peroxidase reactions in plant defense
                        and growth induction
                     </span>. 
                     <span class="source" tagx="source" title="source">Plant Cell Rep.</span>
                     <span class="volume" tagx="volume" title="volume">21</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">829</span>–
                     <span class="lpage" tagx="lpage" title="lpage">837</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1007/s00299-003-0591-z">10.1007/s00299-003-0591-z</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/12789499">12789499</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B18"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kessler</span>
                           <span class="given-names" tagx="given-names" title="given-names">A.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Baldwin</span>
                           <span class="given-names" tagx="given-names" title="given-names">I. T.</span>
                        </span>
                     </span> (
                     <span class="year" tagx="year" title="year">2001</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">Defensive function of herbivore-induced plant volatile emissions in nature</span>. 
                     <span class="source" tagx="source" title="source">Science</span>
                     <span class="volume" tagx="volume" title="volume">291</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">2141</span>–
                     <span class="lpage" tagx="lpage" title="lpage">2144</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1126/science.291.5511.2141">10.1126/science.291.5511.2141</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/11251117">11251117</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B19"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kessler</span>
                           <span class="given-names" tagx="given-names" title="given-names">A.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Halitschke</span>
                           <span class="given-names" tagx="given-names" title="given-names">R.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Baldwin</span>
                           <span class="given-names" tagx="given-names" title="given-names">I. T.</span>
                        </span>
                     </span> (
                     <span class="year" tagx="year" title="year">2004</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">Silencing the jasmonate cascade: induced plant defenses and insect populations</span>. 
                     <span class="source" tagx="source" title="source">Science</span>
                     <span class="volume" tagx="volume" title="volume">305</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">665</span>–
                     <span class="lpage" tagx="lpage" title="lpage">668</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1126/science.1096931">10.1126/science.1096931</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/15232071">15232071</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B20"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kusnierczyk</span>
                           <span class="given-names" tagx="given-names" title="given-names">A.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Winge</span>
                           <span class="given-names" tagx="given-names" title="given-names">P.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Jørstad</span>
                           <span class="given-names" tagx="given-names" title="given-names">T. S.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Troczynska</span>
                           <span class="given-names" tagx="given-names" title="given-names">J.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Rossiter</span>
                           <span class="given-names" tagx="given-names" title="given-names">J. T.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Bones</span>
                           <span class="given-names" tagx="given-names" title="given-names">A. M.</span>
                        </span>
                     </span> (
                     <span class="year" tagx="year" title="year">2008</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">Towards global understanding of plant defence against aphids—timing and dynamics of
                        early 
                        <i>Arabidopsis</i> defence responses to cabbage aphid (Brevicorynebrassicae) attack
                     </span>. 
                     <span class="source" tagx="source" title="source">Plant Cell Environ.</span>
                     <span class="volume" tagx="volume" title="volume">31</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">1097</span>–
                     <span class="lpage" tagx="lpage" title="lpage">1115</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1111/j.1365-3040.2008.01823.x">10.1111/j.1365-3040.2008.01823.x</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/18433442">18433442</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B21"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Love</span>
                           <span class="given-names" tagx="given-names" title="given-names">M. I.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Huber</span>
                           <span class="given-names" tagx="given-names" title="given-names">W.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Anders</span>
                           <span class="given-names" tagx="given-names" title="given-names">S.</span>
                        </span>
                     </span> (
                     <span class="year" tagx="year" title="year">2014</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq 2</span>. 
                     <span class="source" tagx="source" title="source">Genome Biol.</span>
                     <span class="volume" tagx="volume" title="volume">15</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">550</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1101/002832">10.1101/002832</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/25516281">25516281</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B22"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Martinelli</span>
                           <span class="given-names" tagx="given-names" title="given-names">F.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Reagan</span>
                           <span class="given-names" tagx="given-names" title="given-names">R. L.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Uratsu</span>
                           <span class="given-names" tagx="given-names" title="given-names">S. L.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Phu</span>
                           <span class="given-names" tagx="given-names" title="given-names">M. L.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Albrecht</span>
                           <span class="given-names" tagx="given-names" title="given-names">U.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Zhao</span>
                           <span class="given-names" tagx="given-names" title="given-names">W.</span>
                        </span>
                        <span class="etal" title="etal">
                           <i>et al.</i>
                        </span>
                     </span>. (
                     <span class="year" tagx="year" title="year">2013</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">Gene regulatory networks elucidating huanglongbing disease mechanisms</span>. 
                     <span class="source" tagx="source" title="source">PLoS ONE</span>
                     <span class="volume" tagx="volume" title="volume">8</span>:
                     <span class="fpage" tagx="fpage" title="fpage">e74256</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1371/journal.pone.0074256">10.1371/journal.pone.0074256</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/24086326">24086326</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B23"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Martinelli</span>
                           <span class="given-names" tagx="given-names" title="given-names">F.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Scalenghe</span>
                           <span class="given-names" tagx="given-names" title="given-names">R.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Davino</span>
                           <span class="given-names" tagx="given-names" title="given-names">S. W.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Panno</span>
                           <span class="given-names" tagx="given-names" title="given-names">S.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Suderi</span>
                           <span class="given-names" tagx="given-names" title="given-names">G.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Ruisi</span>
                           <span class="given-names" tagx="given-names" title="given-names">P.</span>
                        </span>
                        <span class="etal" title="etal">
                           <i>et al.</i>
                        </span>
                     </span> (
                     <span class="year" tagx="year" title="year">2015</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">Advanced methods of plant disease detection, A Review</span>. 
                     <span class="source" tagx="source" title="source">Agron. Sustain. Dev.</span>
                     <span class="volume" tagx="volume" title="volume">35</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">1</span>–
                     <span class="lpage" tagx="lpage" title="lpage">25</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1007/s13593-014-0246-1">10.1007/s13593-014-0246-1</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B24"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Martinelli</span>
                           <span class="given-names" tagx="given-names" title="given-names">F.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Uratsu</span>
                           <span class="given-names" tagx="given-names" title="given-names">S. L.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Albrecht</span>
                           <span class="given-names" tagx="given-names" title="given-names">U.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Reagan</span>
                           <span class="given-names" tagx="given-names" title="given-names">R. L.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Phu</span>
                           <span class="given-names" tagx="given-names" title="given-names">M. L.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Britton</span>
                           <span class="given-names" tagx="given-names" title="given-names">M.</span>
                        </span>
                        <span class="etal" title="etal">
                           <i>et al.</i>
                        </span>
                     </span>. (
                     <span class="year" tagx="year" title="year">2012</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">Transcriptome profiling of citrus fruit response to huanglongbing disease</span>. 
                     <span class="source" tagx="source" title="source">PLoS ONE</span>
                     <span class="volume" tagx="volume" title="volume">7</span>:
                     <span class="fpage" tagx="fpage" title="fpage">e38039</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1371/journal.pone.0038039">10.1371/journal.pone.0038039</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/22675433">22675433</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B25"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Mielle</span>
                           <span class="given-names" tagx="given-names" title="given-names">P.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Marquis</span>
                           <span class="given-names" tagx="given-names" title="given-names">F.</span>
                        </span>
                     </span> (
                     <span class="year" tagx="year" title="year">1999</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">An alternative way to improve the sensitivity of electronic olfactometers</span>. 
                     <span class="source" tagx="source" title="source">Sens. Actuators B Chem.</span>
                     <span class="volume" tagx="volume" title="volume">58</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">526</span>–
                     <span class="lpage" tagx="lpage" title="lpage">535</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1016/S0925-4005(99)00158-6">10.1016/S0925-4005(99)00158-6</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B26"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Moura</span>
                           <span class="given-names" tagx="given-names" title="given-names">J. C.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Bonine</span>
                           <span class="given-names" tagx="given-names" title="given-names">C. A.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">de Oliverira Fernandes Viana</span>
                           <span class="given-names" tagx="given-names" title="given-names">J.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Dornelas</span>
                           <span class="given-names" tagx="given-names" title="given-names">M. C.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Mazzafera</span>
                           <span class="given-names" tagx="given-names" title="given-names">P.</span>
                        </span>
                     </span> (
                     <span class="year" tagx="year" title="year">2010</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">Abiotic and biotic stresses and changes in the lignin content and composition in plants</span>. 
                     <span class="source" tagx="source" title="source">J. Integr. Plant Biol</span>. 
                     <span class="volume" tagx="volume" title="volume">52</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">360</span>–
                     <span class="lpage" tagx="lpage" title="lpage">376</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1111/j.1744-7909.2010.00892.x">10.1111/j.1744-7909.2010.00892.x</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/20377698">20377698</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B27"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Nakash</span>
                           <span class="given-names" tagx="given-names" title="given-names">J.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Osem</span>
                           <span class="given-names" tagx="given-names" title="given-names">Y.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kehat</span>
                           <span class="given-names" tagx="given-names" title="given-names">M.</span>
                        </span>
                     </span> (
                     <span class="year" tagx="year" title="year">2000</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">A suggestion to use dogs for detecting red palm weevil (
                        <i>Rhynchophorus ferrugineus</i>) infestation in date palms in Israel
                     </span>. 
                     <span class="source" tagx="source" title="source">Phytoparasitica</span>
                     <span class="volume" tagx="volume" title="volume">28</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">153</span>–
                     <span class="lpage" tagx="lpage" title="lpage">155</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1007/BF02981745">10.1007/BF02981745</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B28"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Pieterse</span>
                           <span class="given-names" tagx="given-names" title="given-names">C. M. J.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Leon-Reyes</span>
                           <span class="given-names" tagx="given-names" title="given-names">A.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Van der Ent</span>
                           <span class="given-names" tagx="given-names" title="given-names">S.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Van Wees</span>
                           <span class="given-names" tagx="given-names" title="given-names">S. C. M.</span>
                        </span>
                     </span> (
                     <span class="year" tagx="year" title="year">2009</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">Networking by small-molecule hormones in plant immunity</span>. 
                     <span class="source" tagx="source" title="source">Nat. Chem. Biol.</span>
                     <span class="volume" tagx="volume" title="volume">5</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">308</span>–
                     <span class="lpage" tagx="lpage" title="lpage">316</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1038/nchembio.164">10.1038/nchembio.164</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/19377457">19377457</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B29"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Rugman-Jones</span>
                           <span class="given-names" tagx="given-names" title="given-names">P. F.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hoddle</span>
                           <span class="given-names" tagx="given-names" title="given-names">C. D.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hoddle</span>
                           <span class="given-names" tagx="given-names" title="given-names">M. S.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Stouthamer</span>
                           <span class="given-names" tagx="given-names" title="given-names">R. S.</span>
                        </span>
                     </span> (
                     <span class="year" tagx="year" title="year">2013</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">The lesser of two weevils: molecular-genetics of pest palm weevil populations confirm
                        Rhynchophorus vulneratus (Panzer 1798) as a valid species distinct from R. ferrugineus
                        (Olivier 1790), and reveal the global extent of both
                     </span>. 
                     <span class="source" tagx="source" title="source">PLoS ONE</span>
                     <span class="volume" tagx="volume" title="volume">8</span>:
                     <span class="fpage" tagx="fpage" title="fpage">e78379</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1371/journal.pone.0078379">10.1371/journal.pone.0078379</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/24143263">24143263</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B30"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Rushton</span>
                           <span class="given-names" tagx="given-names" title="given-names">P. J.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Somssich</span>
                           <span class="given-names" tagx="given-names" title="given-names">I. E.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Ringler</span>
                           <span class="given-names" tagx="given-names" title="given-names">P.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Shen</span>
                           <span class="given-names" tagx="given-names" title="given-names">Q. J.</span>
                        </span>
                     </span> (
                     <span class="year" tagx="year" title="year">2010</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">WRKY transcription factors</span>. 
                     <span class="source" tagx="source" title="source">Trends Plant Sci.</span>
                     <span class="volume" tagx="volume" title="volume">15</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">247</span>–
                     <span class="lpage" tagx="lpage" title="lpage">258</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1016/j.tplants.2010.02.006">10.1016/j.tplants.2010.02.006</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/20304701">20304701</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B31"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Sulmon</span>
                           <span class="given-names" tagx="given-names" title="given-names">C.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Gouesbet</span>
                           <span class="given-names" tagx="given-names" title="given-names">G.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">El Amrani</span>
                           <span class="given-names" tagx="given-names" title="given-names">A.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Couée</span>
                           <span class="given-names" tagx="given-names" title="given-names">I.</span>
                        </span>
                     </span> (
                     <span class="year" tagx="year" title="year">2006</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">Sugar-induced tolerance to the herbicide atrazine in 
                        <i>Arabidopsis</i> seedlings involves activation of oxidative and xenobiotic stress responses
                     </span>. 
                     <span class="source" tagx="source" title="source">Plant Cell Rep.</span>
                     <span class="volume" tagx="volume" title="volume">25</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">489</span>–
                     <span class="lpage" tagx="lpage" title="lpage">498</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1007/s00299-005-0062-9">10.1007/s00299-005-0062-9</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/16397785">16397785</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B32"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Tattersall</span>
                           <span class="given-names" tagx="given-names" title="given-names">D. B.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Bak</span>
                           <span class="given-names" tagx="given-names" title="given-names">S.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Jones</span>
                           <span class="given-names" tagx="given-names" title="given-names">P. R.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Olsen</span>
                           <span class="given-names" tagx="given-names" title="given-names">C. E.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Nielsen</span>
                           <span class="given-names" tagx="given-names" title="given-names">J. K.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hansen</span>
                           <span class="given-names" tagx="given-names" title="given-names">M. L.</span>
                        </span>
                        <span class="etal" title="etal">
                           <i>et al.</i>
                        </span>
                     </span>. (
                     <span class="year" tagx="year" title="year">2001</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">Resistance to an herbivore through engineered cyanogenic glucoside synthesis</span>. 
                     <span class="source" tagx="source" title="source">Science</span>
                     <span class="volume" tagx="volume" title="volume">293</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">1826</span>–
                     <span class="lpage" tagx="lpage" title="lpage">1828</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1126/science.1062249">10.1126/science.1062249</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/11474068">11474068</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B33"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Thimm</span>
                           <span class="given-names" tagx="given-names" title="given-names">O.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Bläsing</span>
                           <span class="given-names" tagx="given-names" title="given-names">O.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Gibon</span>
                           <span class="given-names" tagx="given-names" title="given-names">Y.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Nagel</span>
                           <span class="given-names" tagx="given-names" title="given-names">A.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Meyer</span>
                           <span class="given-names" tagx="given-names" title="given-names">S.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Krüger</span>
                           <span class="given-names" tagx="given-names" title="given-names">P.</span>
                        </span>
                        <span class="etal" title="etal">
                           <i>et al.</i>
                        </span>
                     </span>. (
                     <span class="year" tagx="year" title="year">2004</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">MAPMAN: a user-driven tool to display genomics data sets onto diagrams of metabolic
                        pathways and other biological processes
                     </span>. 
                     <span class="source" tagx="source" title="source">Plant J.</span>
                     <span class="volume" tagx="volume" title="volume">37</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">914</span>–
                     <span class="lpage" tagx="lpage" title="lpage">939</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1111/j.1365-313X.2004.02016.x">10.1111/j.1365-313X.2004.02016.x</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/14996223">14996223</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B34"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Tremblay</span>
                           <span class="given-names" tagx="given-names" title="given-names">A.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hosseini</span>
                           <span class="given-names" tagx="given-names" title="given-names">P.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Li</span>
                           <span class="given-names" tagx="given-names" title="given-names">S.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Alkharouf</span>
                           <span class="given-names" tagx="given-names" title="given-names">N. W.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Matthews</span>
                           <span class="given-names" tagx="given-names" title="given-names">B. F.</span>
                        </span>
                     </span> (
                     <span class="year" tagx="year" title="year">2013</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">Analysis of Phakopsorapachyrhizi transcript abundance in critical pathways at four
                        time-points during infection of a susceptible soybean cultivar using deep sequencing
                     </span>. 
                     <span class="source" tagx="source" title="source">BMC Genomics</span>
                     <span class="volume" tagx="volume" title="volume">14</span>:
                     <span class="fpage" tagx="fpage" title="fpage">614</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1186/1471-2164-14-614">10.1186/1471-2164-14-614</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/24025037">24025037</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B35"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Treutter</span>
                           <span class="given-names" tagx="given-names" title="given-names">D.</span>
                        </span>
                     </span> (
                     <span class="year" tagx="year" title="year">2006</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">Significance of flavonoids in plant resistance: a review</span>. 
                     <span class="source" tagx="source" title="source">Environ. Chem. Lett.</span>
                     <span class="volume" tagx="volume" title="volume">4</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">147</span>–
                     <span class="lpage" tagx="lpage" title="lpage">157</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1007/s10311-006-0068-8">10.1007/s10311-006-0068-8</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B36"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Van Loon</span>
                           <span class="given-names" tagx="given-names" title="given-names">L. C.</span>
                        </span>
                     </span> (
                     <span class="year" tagx="year" title="year">1997</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">Induced resistance in plants and the role of pathogenesis-related proteins</span>. 
                     <span class="source" tagx="source" title="source">Eur. J. Plant Pathol.</span>
                     <span class="volume" tagx="volume" title="volume">103</span>, 
                     <span class="fpage" tagx="fpage" title="fpage">753</span>–
                     <span class="lpage" tagx="lpage" title="lpage">765</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1023/A:1008638109140">10.1023/A:1008638109140</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B37"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Zhang</span>
                           <span class="given-names" tagx="given-names" title="given-names">H.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Yang</span>
                           <span class="given-names" tagx="given-names" title="given-names">Y.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Wang</span>
                           <span class="given-names" tagx="given-names" title="given-names">C.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Liu</span>
                           <span class="given-names" tagx="given-names" title="given-names">M.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Li</span>
                           <span class="given-names" tagx="given-names" title="given-names">H.</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Fu</span>
                           <span class="given-names" tagx="given-names" title="given-names">Y.</span>
                        </span>
                        <span class="etal" title="etal">
                           <i>et al.</i>
                        </span>
                     </span>. (
                     <span class="year" tagx="year" title="year">2014</span>). 
                     <span class="mixed-article-title" title="mixed-article-title">Large-scale transcriptome comparison reveals distinct gene activations in wheat responding
                        to stripe rust and powdery mildew
                     </span>. 
                     <span class="source" tagx="source" title="source">BMC Genomics</span>
                     <span class="volume" tagx="volume" title="volume">15</span>:
                     <span class="fpage" tagx="fpage" title="fpage">898</span>. 
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1186/1471-2164-15-898">10.1186/1471-2164-15-898</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/25318379">25318379</a>
                     </span>
                  </span>
               </li>
            </ul>
         </div>
      </div>
   </body>
</html>