<?xml version="1.0" encoding="UTF-8"?><html xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:h="http://www.w3.org/1999/xhtml">
   <head>
      <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style>
   </head>
   <body>
      <div class="front" title="front">
         <div class="journal-meta" tagx="journal-meta" title="journal-meta">
            <span class="nlm-ta" title="nlm-ta">BMC Evol Biol</span>
            <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group">
               <span class="journal-title" tagx="journal-title" title="journal-title">BMC Evolutionary Biology</span>
            </div>
            <span class="issn-epub" tagx="issn" title="issn-epub">1471-2148</span>
            <div class="publisher" tagx="publisher" title="publisher">
               <span class="publisher-name" tagx="publisher-name" title="publisher-name">BioMed Central</span>
            </div>
         </div>
         <div class="article-meta" tagx="article-meta" title="article-meta">
            <span class="pmcid" title="pmcid">
               			pmcid:
               			
               <a href="http://www.ncbi.nlm.nih.gov/pubmed/2804672">2804672</a>
            </span>
            <span class="publisher-id" title="publisher-id">1471-2148-9-288</span>
            <span class="pmid" title="pmid">
               			pmid:
               			
               <a href="http://www.ncbi.nlm.nih.gov/pubmed/20003282">20003282</a>
            </span>
            <span class="doi" title="doi">
               			doi:
               			
               <a href="https://dx.doi.org/10.1186/1471-2148-9-288">10.1186/1471-2148-9-288</a>
            </span>
            <div class="article-categories" title="article-categories">: 
               <span class="subject" title="subject">Research article</span>
            </div>
            <div class="title-group">
               <div class="article-title" title="article-title">Genome scan to assess the respective role of host-plant and environmental constraints
                  on the adaptation of a widespread insect
               </div>
            </div>
            <div class="contrib-group" title="contrib-group">
               <span class="contrib" title="contrib">
                  <span class="name" tagx="name" title="name">
                     <span class="surname" tagx="surname" title="surname">Manel</span>
                     <span class="given-names" tagx="given-names" title="given-names">Stéphanie</span>
                  </span>
                  <a href="#I1">1</a>
                  <a href="#I2">2</a>
                  <span class="email" tagx="email" title="email">stephanie.manel@ujf-grenoble.fr</span>
               </span>
               <span class="contrib" title="contrib">
                  <span class="name" tagx="name" title="name">
                     <span class="surname" tagx="surname" title="surname">Conord</span>
                     <span class="given-names" tagx="given-names" title="given-names">Cyrille</span>
                  </span>
                  <a href="#I1">1</a>
                  <span class="email" tagx="email" title="email">Cyrille.Conord@ujf-grenoble.fr</span>
               </span>
               <span class="contrib" title="contrib">
                  <span class="name" tagx="name" title="name">
                     <span class="surname" tagx="surname" title="surname">Després</span>
                     <span class="given-names" tagx="given-names" title="given-names">Laurence</span>
                  </span>
                  <a href="#I1">1</a>
                  <span class="email" tagx="email" title="email">laurence.despres@ujf-grenoble.fr</span>
               </span>
            </div>
            <span class="citation_author_institution" id="I1">[1],
               	  
            </span>
            <span class="citation_author_institution" id="I2">[2],
               	  
            </span>
            <span class="pub-date-collection" title="pub-date-collection">collection: 
               <span>2009</span>
            </span>
            <span class="pub-date-epub" title="pub-date-epub">epub: 
               <span>2009-12-12</span>
            </span>
            <span class="volume" tagx="volume" title="volume">9</span>
            <span class="fpage" tagx="fpage" title="fpage">288</span>
            <span class="lpage" tagx="lpage" title="lpage">288</span>
            <span class="history" title="history">
               <span class="received" title="received">received: 2009-7-21</span>
               <span class="accepted" title="accepted">accepted: 2009-12-10</span>
            </span>
            <div class="permissions">
               <span class="copyright" title="copyright">(C) 
                  <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Manel et al; licensee BioMed Central Ltd.</span>, 
                  <span class="copyright-year" tagx="copyright-year" title="copyright-year">2009</span>
               </span>
               <span class="license" title="license">
                  <span class="license-p" title="license-p">This is an Open Access article distributed under the terms of the Creative Commons
                     Attribution License (
                     <a href="http://creativecommons.org/licenses/by/2.0">http://creativecommons.org/licenses/by/2.0</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided
                     the original work is properly cited.
                  </span>
               </span>
            </div>
            <a class="self-uri" href="http://www.biomedcentral.com/1471-2148/9/288" title="self-uri">http://www.biomedcentral.com/1471-2148/9/288</a>
            <div class="abstract" title="abstract">
               <div class="abstract-title" title="abstract-title">Abstract</div>
               <div class="background" title="sec">
                  <div class="title" tagx="title" title="title">Background</div>
                  <p>The evolutionary success of phytophagous insects could result from their adaptation
                     to different host-plants. Alternatively, the diversification of widespread species
                     might be driven by adaptation along environmental gradients. To disentangle the respective
                     roles of host-plant versus abiotic environmental variables acting on the genome of
                     an oligophagous insect, we performed a genome scan using 83 unlinked AFLP markers
                     on larvae of the large pine weevil collected on two host-plants (pine and spruce)
                     in four forestry regions across Europe.
                  </p>
               </div>
               <div class="results" title="sec">
                  <div class="title" tagx="title" title="title">Results</div>
                  <p>At this large geographic scale, the global genetic differentiation was low and there
                     was no isolation by distance pattern, suggesting that migration is overwhelming genetic
                     drift in this species. In this context, the widely used frequentist methods to detect
                     outliers (e.g. Dfdist), which assume migration - drift equilibrium are not the most
                     appropriate approach. The implementation of a recently developed Bayesian approach,
                     conceived to detect outliers even in non-equilibrium situations, consistently detected
                     9 out of 83 loci as outliers. Eight of these were validated as outliers by multiple
                     logistic regressions: six correlated with environmental variables, one with host-plant
                     and one with the interaction between environmental variables and host-plant.
                  </p>
               </div>
               <div class="conclusion" title="sec">
                  <div class="title" tagx="title" title="title">Conclusion</div>
                  <p>These results suggest a relatively greater importance of abiotic environmental variables,
                     as opposed to factors linked with the host-plant, in shaping genetic differentiation
                     across the genome in this species. Logistic regression allows the nature of factors
                     involved in locus-specific selection to be precisely identified and represents another
                     step forward in the process of identifying adaptive loci.
                  </p>
               </div>
            </div>
         </div>
      </div>
      <div class="body" title="body">
         <div class="background" title="sec">
            <div class="title" tagx="title" title="title">Background</div>
            <p>Natural selection is expected to increase the frequency of locally advantageous alleles,
               resulting in higher among-population differentiation at these adaptive loci (measured
               by locus-specific 
               <i>F</i>
               <sub>ST</sub>), compared with differentiation in the rest of the genome (neutral loci) [
               <a href="#B1">1</a>,
               <a href="#B2">2</a>]. Identifying adaptive divergence among populations at specific loci from genome
               scans is an active and challenging research area (see [
               <a href="#B3">3</a>] for a review). This task is especially demanding for dominant biallelic markers
               such as Amplified Fragment Length Polymorphism (AFLP) markers, although they represent
               an easy way to scan a large number of markers scattered throughout the genome in non-model
               species [
               <a href="#B4">4</a>-
               <a href="#B6">6</a>]. So far, the most widely used method to detect outliers from AFLP genome scans is
               implemented in Dfdist, an extension of Fdist software that allows the use of dominant
               markers [
               <a href="#B7">7</a>]. Dfidst is a frequentist method based on summary statistics of a symmetrical island
               model (i.e. drift-migration equilibrium, [
               <a href="#B8">8</a>]). Each locus is compared to the neutral distribution built from the mean 
               <i>F</i>
               <sub>ST </sub>averaged across all populations. Dfdist is likely to generate false positives (i.e.
               loci with higher than expected 
               <i>F</i>
               <sub>ST</sub>) when gene flows are asymmetric across populations, and/or when some populations
               experiment bottlenecks. To overcome that limitation, Foll and Gaggiotti [
               <a href="#B9">9</a>] have recently developed a new hierarchical Bayesian method (BayeScan) that also
               allows the accommodation of AFLPs data. Their method is derived from the method of
               Beaumont and Baldwin [
               <a href="#B10">10</a>]. It produces a posterior probability for each locus being under selection. The main
               advantage of BayeScan is that it estimates population-specific 
               <i>F</i>
               <sub>ST </sub>coefficients, therefore allowing for different demographic histories and different
               amounts of genetic drift between populations. In structured populations, the Bayesian
               approach is less likely to detect false positives. The proportion of false positive
               in detected outliers cannot be easily estimated in the Bayesian approaches as it requires
               simulating datasets under different scenarios [
               <a href="#B9">9</a>]. By contrast, it can be estimated using false discovery rate [
               <a href="#B11">11</a>] in frequentist methods.
            </p>
            <p>After detecting outlier loci, it is then a challenging prospect to verify whether
               they are involved in local adaptation and to isolate the different ecological factors
               responsible for the behaviour of each outlier from a complex natural environment [
               <a href="#B4">4</a>,
               <a href="#B12">12</a>]. Indeed, despite recent advances in tracking adaptive genes out from the neutral
               background genetic variability across populations of a species, the relative role
               of biotic versus abiotic constraints acting on genomes in their natural environment
               remains a largely under-explored area. For non-model organisms, outlier loci cannot
               be mapped and hence their functional role will remain unknown. An alternative approach
               is to correlate their variation in frequency throughout the sampling area with that
               of continuous environmental variables, such as altitude or climate [
               <a href="#B13">13</a>-
               <a href="#B15">15</a>], or qualitative variables such as different host-plants for insects [
               <a href="#B6">6</a>,
               <a href="#B16">16</a>,
               <a href="#B17">17</a>] or different life habits (limnic or benthic) for fish ecotypes [
               <a href="#B18">18</a>].
            </p>
            <p>The evolutionary success of phytophagous insects is thought to result mainly from
               their adaptation to various host-plants, with insect adaptation driving plant diversification
               in a co-evolutionary process [
               <a href="#B19">19</a>-
               <a href="#B21">21</a>]. Alternatively, the diversification of widespread species could be driven by adaptation
               along environmental abiotic gradients. The large pine weevil 
               <i>Hylobius abietis L. </i>(Curculionidae) is a good model for addressing this question because it is widespread
               in Europe (large environmental variation) and because, during larval development,
               it depends exclusively on only two plant genera: spruce (
               <i>Picea</i>) and pine (
               <i>Pinus</i>). The large pine weevil is one of the most important economic pests of European conifer
               forests. The larvae feed under the bark of stumps and roots of recently felled trees,
               and take from three months to two years to develop into adults, depending on location
               [
               <a href="#B22">22</a>], presumably because of climatic conditions and/or host plant quality [
               <a href="#B23">23</a>,
               <a href="#B24">24</a>]. The adults are active only under cool climatic conditions, usually in spring and
               autumn, and burrow into the soil during hot summers and cold winters; adults can fly
               large distances and can live up to four years [
               <a href="#B22">22</a>]. Because of this complex life cycle, several climatic factors including temperature,
               precipitation, soil, frost and wind speed may have either a direct impact on larval/adult
               survival, or a more indirect impact on fitness through host plant quality, and represent
               therefore potential selective forces acting on the pine weevil genome at a large geographical
               scale. Adults are attracted to recently felled trees (spruce or pine) where the females
               lay eggs under the bark. Managed pine and spruce forests planted in Western Europe
               200-300 years ago offer many oviposition opportunities for this weevil, allowing large
               stable populations to be sustained in contrasting climatic environments.
            </p>
            <p>The first study on the population genetic structure of the pine weevils, at the European
               scale [
               <a href="#B25">25</a>] revealed that genetic variation of this insect is better explained by geography
               than host-plant (5% versus 1% of total variation). Furthermore, a locus by locus AMOVA
               identified some loci with significant 
               <i>F</i>
               <sub>ST </sub>across different host-plant groups, suggesting that host-plant linked selection might
               occur in this species. A second analysis consistently identified 2 out of 83 unlinked
               AFLP markers as outliers by using univariate logistic regressions to search for correlations
               between molecular markers scattered throughout the genome and several environmental
               variables suggesting an effect of climate on weevil adaptation [
               <a href="#B14">14</a>]. However, the effect of the host-plant type was not tested.
            </p>
            <p>In the present analysis, we focused on disentangling the role of abiotic environmental
               variables on one hand from the effect of the host-plant (pine or spruce) on the other.
               We therefore excluded adult weevils from the original AFLP dataset because they cannot
               be assigned to a host-plant, reducing the analysed dataset to 296 individual larvae.
               We firstly detected outlier loci across geographic and host-specific groups of individuals
               in 16 managed forest sites distributed across 4 large forestry regions (Table 
               <a href="#T1">1</a>, Figure 
               <a href="#F1">1</a>) using population genetic approaches (Dfdist and BayeScan). We then used a correlative
               approach to disentangle the effects of host-plants from those of various environmental
               variables. To further confirm the involvement of selection in genetic patterns of
               differentiation, we tested for the drift-migration equilibrium (i.e. isolation by
               distance pattern) on neutral loci. If adaptation to host-plant is promoting divergence,
               then we would expect to find outliers loci when comparing different host populations
               whereas loci candidate to diverge independently from host-plant would be rather correlated
               with other ecological pressures, such as climate.
            </p>
            <div class="fig" title="fig">
               <div class="figure" title="figure">
                  <a href="http://doi.org/">
                     <span class="label" tagx="label" title="label">Figure 1</span>
                  </a>
               </div>
               <caption class="caption" title="caption">
                  <caption>
                     <p>
                        <b>Group-based structure of the large pine weevil larvae</b>. Description of the 5 group-based structures defined for the two population genetic
                        approaches. Rectangles: regions. Circles: host-plant groups; P = Pine, S = Spruce;
                        sample size in parentheses. Structure 1: individuals were grouped according to their
                        geographic origin: Finland, Limousin, Ardèche, and Landes (the four rectangles 1,
                        2, 3, 4). Structure 2: individuals were grouped according to their geographic origin
                        and host-plant (7 circles) Structures 3, 4, 5: within each of the three regions (rectangles
                        1, 2, 3) where both alternative host-plants are found (Finland, Limousin, and Ardèche),
                        pairwise comparisons of host-plant samples were performed. Two circles in rectangle
                        1, 2, 3 correspond to structures 3, 4, 5 respectively.
                     </p>
                  </caption>
               </caption>
            </div>
            <table-wrap>
               <span class="label" tagx="label" title="label">Table 1</span>
               <caption>
                  <p>Geographic location, sample sizes and host-association characteristics of 
                     <i>Hylobius abietis </i>sites collected.
                  </p>
               </caption>
               <table>
                  <thead>
                     <tr>
                        <th>Site</th>
                        <th>Country-region</th>
                        <th>Longitude/latitude</th>
                        <th>Sample Size</th>
                        <th>Host-plant</th>
                     </tr>
                  </thead>
                  <tbody>
                     <tr>
                        <td>1 Jalkala</td>
                        <td>Finland</td>
                        <td>27°15'E/62°33'N</td>
                        <td>10</td>
                        <td>spruce</td>
                     </tr>
                     <tr>
                        <td>2 Kalakukkokangas</td>
                        <td>Finland</td>
                        <td>27°15'E/62°43'N</td>
                        <td>73</td>
                        <td>mixed
                           <sup>1</sup>
                        </td>
                     </tr>
                     <tr>
                        <td>3 Les Quatre Vios</td>
                        <td>France - Ardèche</td>
                        <td>4°13'E/44°28'N</td>
                        <td>56</td>
                        <td>mixed
                           <sup>1</sup>
                        </td>
                     </tr>
                     <tr>
                        <td>4 Lachamp Raphaël</td>
                        <td>France - Ardèche</td>
                        <td>4°18'E/44°49'N</td>
                        <td>8</td>
                        <td>spruce</td>
                     </tr>
                     <tr>
                        <td>5 Mézilhac</td>
                        <td>France - Ardèche</td>
                        <td>4°21'E/44°48'N</td>
                        <td>10</td>
                        <td>pine</td>
                     </tr>
                     <tr>
                        <td>6 Etienne de Lugdares</td>
                        <td>France - Ardèche</td>
                        <td>3°57'E/44°39'N</td>
                        <td>4</td>
                        <td>spruce</td>
                     </tr>
                     <tr>
                        <td>7 Annouillards</td>
                        <td>France - Limousin</td>
                        <td>2°12'E/45°40'N</td>
                        <td>28</td>
                        <td>mixed
                           <sup>1</sup>
                        </td>
                     </tr>
                     <tr>
                        <td>8 Basville</td>
                        <td>France - Limousin</td>
                        <td>2°24'E/45°52'N</td>
                        <td>10</td>
                        <td>pine</td>
                     </tr>
                     <tr>
                        <td>9 Bellechassagne</td>
                        <td>France - Limousin</td>
                        <td>2°13'E/45°39'N</td>
                        <td>7</td>
                        <td>pine</td>
                     </tr>
                     <tr>
                        <td>10 Ebraly</td>
                        <td>France - Limousin</td>
                        <td>2°22'E/45°34'N</td>
                        <td>29</td>
                        <td>pine</td>
                     </tr>
                     <tr>
                        <td>11 Maussac</td>
                        <td>France - Limousin</td>
                        <td>2°09'E/45°28'N</td>
                        <td>19</td>
                        <td>pine</td>
                     </tr>
                     <tr>
                        <td>12 Pontgibaud</td>
                        <td>France - Limousin</td>
                        <td>2°52'E/45°49'N</td>
                        <td>10</td>
                        <td>pine</td>
                     </tr>
                     <tr>
                        <td>13 Puits de la Blanche</td>
                        <td>France - Limousin</td>
                        <td>2°01'E/45°17'N</td>
                        <td>7</td>
                        <td>spruce</td>
                     </tr>
                     <tr>
                        <td>14 Royère</td>
                        <td>France - Limousin</td>
                        <td>1°54'E/45°49'N</td>
                        <td>5</td>
                        <td>pine</td>
                     </tr>
                     <tr>
                        <td>15 Le Sen</td>
                        <td>France - Landes</td>
                        <td>1°30'W/44°07'N</td>
                        <td>6</td>
                        <td>pine</td>
                     </tr>
                     <tr>
                        <td>16 Pontenx</td>
                        <td>France - Landes</td>
                        <td>2°52'W/44°15'N</td>
                        <td>14</td>
                        <td>pine</td>
                     </tr>
                  </tbody>
               </table>
               <table-wrap-foot>
                  <p>
                     <sup>1 </sup>Both pine and spruce locally co-exist
                  </p>
               </table-wrap-foot>
            </table-wrap>
         </div>
         <div class="results" title="sec">
            <div class="title" tagx="title" title="title">Results</div>
            <div class="populationgeneticapproaches" title="sec">
               <div class="title" tagx="title" title="title">Population genetic approaches</div>
               <p>The global 
                  <i>F</i>
                  <sub>ST</sub>, calculated with AFLP-SURV 1.0 [
                  <a href="#B26">26</a>] was low but significant (
                  <i>F</i>
                  <sub>ST </sub>= 0.02, p &lt; 0.0001). Most pairwise 
                  <i>F</i>
                  <sub>ST </sub>between sites within region were non-significantly different from zero, while all
                  pairwise 
                  <i>F</i>
                  <sub>ST </sub>between the four forestry regions were significant (range 0.057-0.058, p &lt; 0.0001),
                  indicating that forestry region is a coherent population genetic entity. When larvae
                  were grouped according to their forestry region (structure 1, Figure 
                  <a href="#F1">1</a>), four markers were consistently detected by both population genetic approaches as
                  being under directional selection: loci 52, 38, 68 and 63 (Table 
                  <a href="#T2">2</a>). Dfdist detected one outlier (locus 10) that was not detected by BayeScan, and BayeScan
                  detected two outliers (loci 13 and 47) that were not detected by Dfdist.
               </p>
               <table-wrap>
                  <span class="label" tagx="label" title="label">Table 2</span>
                  <caption>
                     <p>Results of outlier detection among the 83 AFLP markers in larvae of the large pine
                        weevil using the frequentist method Dfdist and the Bayesian inference method BayeScan.
                     </p>
                  </caption>
                  <table>
                     <thead>
                        <tr>
                           <th>Method of detection</th>
                           <th></th>
                           <th>Frequentist</th>
                           <th>Bayesian inference</th>
                        </tr>
                     </thead>
                     <tbody>
                        <tr>
                           <td>
                              <b>Dataset</b>
                           </td>
                           <td>
                              <b>Locus</b>
                           </td>
                           <td>
                              <b>
                                 <i>p</i>-value
                              </b>
                           </td>
                           <td>
                              <b>
                                 <i>F</i>
                                 <sub>ST</sub>
                              </b>
                           </td>
                           <td>
                              <b>Posterior probability</b>
                           </td>
                           <td>
                              <b>A</b>
                           </td>
                           <td>
                              <b>
                                 <i>F</i>
                                 <sub>ST</sub>
                              </b>
                           </td>
                        </tr>
                        <tr>
                           <td>
                              <hr></hr>
                           </td>
                        </tr>
                        <tr>
                           <td>Geography (Structure 1
                              <sup>1</sup>)
                           </td>
                           <td>
                              <b>52
                                 <sup>3</sup>
                              </b>
                           </td>
                           <td>0.000</td>
                           <td>0.231</td>
                           <td>1</td>
                           <td>2.010</td>
                           <td>0.221</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>
                              <b>68</b>
                           </td>
                           <td>0.000</td>
                           <td>0.338</td>
                           <td>1</td>
                           <td>1.790</td>
                           <td>0.191</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>
                              <b>38</b>
                           </td>
                           <td>0.000</td>
                           <td>0.276</td>
                           <td>1</td>
                           <td>1.810</td>
                           <td>0.194</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>10</td>
                           <td>0.000</td>
                           <td>0.248</td>
                           <td>0.758</td>
                           <td>1.000</td>
                           <td>0.105</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>
                              <b>63</b>
                           </td>
                           <td>0.004</td>
                           <td>0.180</td>
                           <td>0.999</td>
                           <td>1.520</td>
                           <td>0.156</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>13</td>
                           <td>0.099</td>
                           <td>0.061</td>
                           <td>0.974</td>
                           <td>1.320</td>
                           <td>0.136</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>47</td>
                           <td>0.045</td>
                           <td>0.080</td>
                           <td>0.931</td>
                           <td>1.190</td>
                           <td>0.122</td>
                        </tr>
                        <tr>
                           <td>
                              <hr></hr>
                           </td>
                        </tr>
                        <tr>
                           <td>Geography + host-plant</td>
                           <td>
                              <b>38</b>
                           </td>
                           <td>0.000</td>
                           <td>0.259</td>
                           <td>1</td>
                           <td>2.090</td>
                           <td>0.208</td>
                        </tr>
                        <tr>
                           <td>(Structure 2
                              <sup>1</sup>)
                           </td>
                           <td>
                              <b>52</b>
                           </td>
                           <td>0.000</td>
                           <td>0.256</td>
                           <td>1</td>
                           <td>2.180</td>
                           <td>0.220</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>
                              <b>63</b>
                           </td>
                           <td>0.000</td>
                           <td>0.225</td>
                           <td>1</td>
                           <td>1.950</td>
                           <td>0.186</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>
                              <b>68</b>
                           </td>
                           <td>0.000</td>
                           <td>0.231</td>
                           <td>0.999</td>
                           <td>1.650</td>
                           <td>0.151</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>10</td>
                           <td>0.000</td>
                           <td>0.181</td>
                           <td>0.743</td>
                           <td>0.893</td>
                           <td>0.082</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>30</td>
                           <td>0.016</td>
                           <td>0.088</td>
                           <td>0.908</td>
                           <td>1.110</td>
                           <td>0.098</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>33</td>
                           <td>0.018</td>
                           <td>0.095</td>
                           <td>0.876</td>
                           <td>1.030</td>
                           <td>0.091</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>27</td>
                           <td>0.102</td>
                           <td>0.066</td>
                           <td>0.882</td>
                           <td>1.040</td>
                           <td>0.092</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>13</td>
                           <td>0.069</td>
                           <td>0.054</td>
                           <td>0.981</td>
                           <td>1.300</td>
                           <td>0.116</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>47</td>
                           <td>0.043</td>
                           <td>0.065</td>
                           <td>0.865</td>
                           <td>1.030</td>
                           <td>0.092</td>
                        </tr>
                        <tr>
                           <td>
                              <hr></hr>
                           </td>
                        </tr>
                        <tr>
                           <td>Local host-plant differentiation</td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                        </tr>
                        <tr>
                           <td>Regions</td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                        </tr>
                        <tr>
                           <td>Finland
                              <sup>2 </sup>(Structure 3
                              <sup>1</sup>)
                           </td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                        </tr>
                        <tr>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                        </tr>
                        <tr>
                           <td>Limousin (Structure 4
                              <sup>1</sup>)
                           </td>
                           <td>
                              <b>27</b>
                           </td>
                           <td>0.000</td>
                           <td>0.217</td>
                           <td>0.915</td>
                           <td>1.460</td>
                           <td>0.222</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                        </tr>
                        <tr>
                           <td>Ardeche
                              <sup>2 </sup>(Structure 5
                              <sup>1</sup>)
                           </td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                        </tr>
                     </tbody>
                  </table>
                  <table-wrap-foot>
                     <p>
                        <sup>1</sup>As in Figure 1.
                     </p>
                     <p>
                        <sup>2</sup>No outliers were found with the significance level used.
                     </p>
                     <p>
                        <sup>3</sup>Bold type indicates markers that are detected by both methods with a type-I error
                        (α) = 0.0006 for Dfdist, and with a posterior probability &gt; 0.79 for BayeScan (see
                        text for explanation about these values).
                     </p>
                  </table-wrap-foot>
               </table-wrap>
               <p>Subdividing the geographic data set into host-plant groups (pine and spruce) (structure
                  2, Figure 
                  <a href="#F1">1</a>) allowed three additional outliers, loci 27, 30 and 33, to be detected only with
                  BayeScan (Table 
                  <a href="#T2">2</a>).
               </p>
               <p>When local pairwise comparisons between host-plants were performed within each region
                  (structures 3, 4, 5, Figure 
                  <a href="#F1">1</a>), only one locus was detected as host-specific by both genetic methods in Limousin
                  (locus 27). This locus was also detected in host-plant groups (structure 2, Figure
                  
                  <a href="#F1">1</a>) by the Bayesian approach. No host-plant outliers were detected in Finland and in
                  Ardèche (Table 
                  <a href="#T2">2</a>).
               </p>
               <p>These last years, it has been suggested to use false discovery rate or 
                  <i>q</i>-values to assess significance in multiple tests [
                  <a href="#B11">11</a>]. The maximum estimated 
                  <i>q</i>-value among all 
                  <i>p</i>-values less than or equal to 0.0006 was 0.014: this means that among 100 loci predicted
                  as outliers, less than 2 are false positives (i.e. neutral), so that it is very unlikely
                  that there is any false positive among the 6 loci identified as outliers.
               </p>
            </div>
            <div class="resultsfromthecorrelativeapproachandcomparisonwithpopulationgeneticmethods" title="sec">
               <div class="title" tagx="title" title="title">Results from the correlative approach and comparison with population genetic methods</div>
               <p>The first two axes of the principal component analysis (PCA) used to investigate correlation
                  between environmental factors explained 96% of the variance (76% and 20% respectively),
                  suggesting that all these variables were highly correlated (correlations of climatic
                  variables with axis 1: 80%; correlation of altitude with axis 2: 53%). The first two
                  axes of the PCA were used as orthogonal predictors in the logistic regressions.
               </p>
               <p>Based on both a minimal Akaike criterion and the likelihood ratio test significant
                  at level α = 6 × 10
                  <sup>-4</sup>, the multiple logistic regressions identified 17 markers correlated with at least
                  one of the 6 explanatory variable tested (Table 
                  <a href="#T3">3</a>). The regression coefficients of the climatic variables were significant for 14 markers,
                  those of host-plant were significant for one marker (27), and those of interactions
                  between host-plant and one of the PCA axis representing abiotic environmental variables
                  for two markers (62 and 68). The maximum estimated 
                  <i>q</i>-value among all 
                  <i>p</i>-values less than or equal to 0.0006 was 0.0033. Again it is very unlikely that there
                  is any false positive among the 17 loci identified as outliers.
               </p>
               <table-wrap>
                  <span class="label" tagx="label" title="label">Table 3</span>
                  <caption>
                     <p>Results from stepwise logistic regressions relating the presence and absence of each
                        marker to the two first axes of the principal component analysis (PCA)
                        <sup>1</sup>, the host-plant (HP), and the three interactions
                        <sup>2</sup>.
                     </p>
                  </caption>
                  <table>
                     <thead>
                        <tr>
                           <th>Markers</th>
                           <th>Axis1</th>
                           <th>Axis2</th>
                           <th>HP</th>
                           <th>Axis1 × HP</th>
                           <th>Axis2 × HP</th>
                           <th>Axis1 × Axis 2</th>
                        </tr>
                     </thead>
                     <tbody>
                        <tr>
                           <td>11</td>
                           <td>
                              <b>0.19</b>
                              <br></br>
                              <b>(5.7 × 10
                                 <sup>-4</sup>)
                              </b>
                           </td>
                           <td>NS</td>
                           <td>NS</td>
                           <td>NS</td>
                           <td>NS</td>
                           <td>NS</td>
                        </tr>
                        <tr>
                           <td>13</td>
                           <td>
                              <b>0.39</b>
                              <br></br>
                              <b>(8.22 × 10
                                 <sup>-6</sup>)
                              </b>
                           </td>
                           <td>
                              <b>-0.34</b>
                              <br></br>
                              <b>(5.3 × 10
                                 <sup>-4</sup>)
                              </b>
                           </td>
                           <td>0.53
                              <br></br>(0.15)
                           </td>
                           <td>NS</td>
                           <td>NS</td>
                           <td>NS</td>
                        </tr>
                        <tr>
                           <td>19</td>
                           <td>NS</td>
                           <td>
                              <b>-0.38</b>
                              <br></br>
                              <b>(4.2 × 10
                                 <sup>-4</sup>)
                              </b>
                           </td>
                           <td>0.56
                              <br></br>(0.13)
                           </td>
                           <td>NS</td>
                           <td>NS</td>
                           <td>NS</td>
                        </tr>
                        <tr>
                           <td>26</td>
                           <td>0.10
                              <br></br>(0.03)
                           </td>
                           <td>
                              <b>-0.25</b>
                              <br></br>
                              <b>(5.4 × 10
                                 <sup>-4</sup>)
                              </b>
                           </td>
                           <td>0.98
                              <br></br>(2.7 × 10
                              <sup>-3</sup>)
                           </td>
                           <td>NS</td>
                           <td>NS</td>
                           <td>NS</td>
                        </tr>
                        <tr>
                           <td>27</td>
                           <td>0.09
                              <br></br>(7 × 10
                              <sup>-2</sup>)
                           </td>
                           <td>NS</td>
                           <td>
                              <b>-0.96</b>
                              <br></br>
                              <b>(1 × 10
                                 <sup>-4</sup>)
                              </b>
                           </td>
                           <td>
                              <b>NS</b>
                           </td>
                           <td>
                              <b>NS</b>
                           </td>
                           <td>
                              <b>NS</b>
                           </td>
                        </tr>
                        <tr>
                           <td>29</td>
                           <td>
                              <b>-0.57</b>
                              <br></br>
                              <b>(4 × 10
                                 <sup>-4</sup>)
                              </b>
                           </td>
                           <td>2.40
                              <br></br>(0.24)
                           </td>
                           <td>2.48
                              <br></br>(0.43)
                           </td>
                           <td>0.37
                              <br></br>(0.45)
                           </td>
                           <td>-2.74
                              <br></br>(3 × 10
                              <sup>-3</sup>)
                           </td>
                           <td>NS</td>
                        </tr>
                        <tr>
                           <td>30</td>
                           <td>-0.19
                              <br></br>(5.4 × 10
                              <sup>-3</sup>)
                           </td>
                           <td>
                              <b>1.37</b>
                              <br></br>
                              <b>(2.42 × 10
                                 <sup>-5</sup>)
                              </b>
                           </td>
                           <td>0.74
                              <br></br>(0.59)
                           </td>
                           <td>NS</td>
                           <td>-1.13
                              <br></br>(1 × 10
                              <sup>-3</sup>)
                           </td>
                           <td>
                              <i>NS</i>
                           </td>
                        </tr>
                        <tr>
                           <td>33</td>
                           <td>
                              <b>-0.26</b>
                              <br></br>
                              <b>(3.64 × 10
                                 <sup>-9</sup>)
                              </b>
                           </td>
                           <td>NS</td>
                           <td>0.37
                              <br></br>(0.13)
                           </td>
                           <td>NS</td>
                           <td>NS</td>
                           <td>NS</td>
                        </tr>
                        <tr>
                           <td>38</td>
                           <td>
                              <b>-0.45</b>
                              <br></br>
                              <b>(4.31 × 10
                                 <sup>-21</sup>)
                              </b>
                           </td>
                           <td>
                              <b>0.90</b>
                              <br></br>
                              <b>(2.37 × 10
                                 <sup>-7</sup>)
                              </b>
                           </td>
                           <td>NS</td>
                           <td>NS</td>
                           <td>NS</td>
                           <td>NS</td>
                        </tr>
                        <tr>
                           <td>39</td>
                           <td>
                              <b>0.16</b>
                              <br></br>
                              <b>(3.8 × 10
                                 <sup>-5</sup>)
                              </b>
                           </td>
                           <td>NS</td>
                           <td>-0.090
                              <br></br>(0.45)
                           </td>
                           <td>0.27
                              <br></br>(.09)
                           </td>
                           <td>NS</td>
                           <td>NS</td>
                        </tr>
                        <tr>
                           <td>40</td>
                           <td>
                              <b>-0.18</b>
                              <br></br>
                              <b>(8 × 10
                                 <sup>-6</sup>)
                              </b>
                           </td>
                           <td>-0.26
                              <br></br>(0.93)
                           </td>
                           <td>0.29
                              <br></br>(0.19)
                           </td>
                           <td>NS</td>
                           <td>0.43
                              <br></br>(0.10)
                           </td>
                           <td>
                              <i>NS</i>
                           </td>
                        </tr>
                        <tr>
                           <td>42</td>
                           <td>-0.18
                              <br></br>(0.37)
                           </td>
                           <td>
                              <b>-0.66</b>
                              <br></br>
                              <b>(6 × 10
                                 <sup>-5</sup>)
                              </b>
                           </td>
                           <td>-0.74
                              <br></br>(0.02)
                           </td>
                           <td>0.34
                              <br></br>(0.02)
                           </td>
                           <td>NS</td>
                           <td>NS</td>
                        </tr>
                        <tr>
                           <td>52</td>
                           <td>
                              <b>0.33</b>
                              <br></br>
                              <b>(1 × 10
                                 <sup>-7</sup>)
                              </b>
                           </td>
                           <td>
                              <b>-1.6</b>
                              <br></br>
                              <b>(6 × 16
                                 <sup>-16</sup>)
                              </b>
                           </td>
                           <td>-0.22
                              <br></br>(0.86)
                           </td>
                           <td>NS</td>
                           <td>0.97
                              <br></br>(1 × 10
                              <sup>-3</sup>)
                           </td>
                           <td>NS</td>
                        </tr>
                        <tr>
                           <td>62</td>
                           <td>-0.18
                              <br></br>(0.37)
                           </td>
                           <td>0.46
                              <br></br>(4 × 10
                              <sup>-3</sup>)
                           </td>
                           <td>0.34
                              <br></br>(0.39)
                           </td>
                           <td>
                              <b>0.31</b>
                              <br></br>
                              <b>(3 × 10
                                 <sup>-4</sup>)
                              </b>
                           </td>
                           <td>NS</td>
                           <td>NS</td>
                        </tr>
                        <tr>
                           <td>63</td>
                           <td>
                              <b>0.59</b>
                              <br></br>
                              <b>(1 × 10
                                 <sup>-22</sup>)
                              </b>
                           </td>
                           <td>-0.76
                              <br></br>(0.89)
                           </td>
                           <td>-0.86
                              <br></br>(0.16)
                           </td>
                           <td>-0.18
                              <br></br>(0.20)
                           </td>
                           <td>0.80
                              <br></br>(0.04)
                           </td>
                           <td>NS</td>
                        </tr>
                        <tr>
                           <td>
                              <hr></hr>
                           </td>
                        </tr>
                        <tr>
                           <td>65</td>
                           <td>-0.19
                              <br></br>(0.37)
                           </td>
                           <td>
                              <b>-0.44</b>
                              <br></br>
                              <b>(8 × 10
                                 <sup>-5</sup>)
                              </b>
                           </td>
                           <td>0.45
                              <br></br>(0.47)
                           </td>
                           <td>0.20
                              <br></br>(0.09)
                           </td>
                           <td>NS</td>
                           <td>NS</td>
                        </tr>
                        <tr>
                           <td>
                              <hr></hr>
                           </td>
                        </tr>
                        <tr>
                           <td>68</td>
                           <td>NS</td>
                           <td>0.93
                              <br></br>(0.05)
                           </td>
                           <td>0.04
                              <br></br>(0.52)
                           </td>
                           <td>NS</td>
                           <td>
                              <b>-1.05</b>
                              <br></br>
                              <b>(4.3 × 10
                                 <sup>-5</sup>)
                              </b>
                           </td>
                           <td>NS</td>
                        </tr>
                     </tbody>
                  </table>
                  <table-wrap-foot>
                     <p>
                        <sup>1 </sup>Axis 1 and axis 2 are the two first PCA components (representing respectively 76 and
                        20% of total inertia) of the projection of altitude and nine climatic variables calculated
                        from the latitude/longitude as yearly mean of monthly values for the period 1961 to
                        2001 (see Methods for more details).
                     </p>
                     <p>
                        <sup>2</sup>Statistically significant logistic regression coefficients are indicated in bold (
                        <i>p</i>-values for each coefficient are given in parenthesis). NS: the correlation was not
                        significant.
                     </p>
                  </table-wrap-foot>
               </table-wrap>
               <p>Finally, seven loci were identified as under positive selection by the regression
                  method and at least one genetic based method (Table 
                  <a href="#T2">2</a> and 
                  <a href="#T3">3</a>: loci 13, 27, 52, 38, 63, 30, 33 and 68). Six of these were correlated with one or
                  two axes of the PCA representing the climatic variables, one (locus 27) was correlated
                  with host-plant, and one (68) with the interaction between the PCA axis 2 and host-plant.
               </p>
               <p>The correlative method did not detect the locus 10 detected as outlier by Dfdist but
                  not detected by the Bayesian approach, suggesting that this is site-specific locus
                  related to a particular demographic history of a site rather than to the ecological
                  pressures analysed.
               </p>
               <p>BayeScan suggested that loci 27, 33 and 30 were involved in host-plant adaptation,
                  which is confirmed by logistic regression for locus 27 (Table 
                  <a href="#T3">3</a>). However for marker 30 and 33, the logistic regression only found an effect of environmental
                  variables (Table 
                  <a href="#T3">3</a>). Finally, one locus (locus 68) detected by BayeScan both in structure 1 and 2 (Figure
                  
                  <a href="#F1">1</a>) was found significantly associated with the interaction between host-plant and PCA
                  axis 2.
               </p>
               <p>When the outliers were omitted from the dataset, there was no correlation between
                  neutral genetic distance (
                  <i>F</i>
                  <sub>ST</sub>/(1-
                  <i>F</i>
                  <sub>ST</sub>)) and geographical distance (r = 0.35, 
                  <i>p </i>= 0.067) or the logarithm of the geographical distance (r = 0.33, 
                  <i>p </i>= 0.077).
               </p>
            </div>
         </div>
         <div class="discussion" title="sec">
            <div class="title" tagx="title" title="title">Discussion</div>
            <div class="detectionofoutliers-complimentaryapproaches" title="sec">
               <div class="title" tagx="title" title="title">Detection of outliers - complimentary approaches</div>
               <p>The lack of correlation between genetic and geographical distances, together with
                  low 
                  <i>F</i>
                  <sub>ST </sub>observed throughout the range of comparisons, suggest that large pine weevils populations
                  are not at the drift-migration equilibrium; the low 
                  <i>F</i>
                  <sub>ST </sub>consistently observed throughout the sampling range suggests that migration is overwhelming
                  drift in this species. This is a further justification to use the Bayesian approach
                  rather than Dfdist to detect outliers. Therefore we consider outliers detected by
                  BayeScan as more reliable. However, both population genetic methods rely on the identification
                  of loci that differ more than the rest of the genome between two or more groups of
                  individuals. One caveat of such approaches to detect outliers is the possible detection
                  of loci with different inherent features such as markers on sexual chromosomes, or
                  markers that exhibit different intrinsic mutation rates. For instance, a sex-linked
                  locus would have a different effective population size (Ne) than autosomal loci and
                  thus more extreme 
                  <i>F</i>
                  <sub>ST </sub>values; and, as a consequence, potentially mis-classified as outlier. Finally, Excoffier
                  
                  <i>et al</i>. [
                  <a href="#B27">27</a>] highlight the need to have a good understanding of the population genetic structure
                  of the studied organism, to accurately identify loci with unusual levels of differentiation
                  using population genetic approaches. In that context, the validation of outliers detected
                  using population genetic approaches by correlative approaches appear to be particularly
                  important. Eight loci were identified both by logistic regression and by BayeScan
                  as being under positive selection. Out of these loci, one was correlated with host-plant,
                  and one was correlated with the interaction between host-plant and abiotic factors.
                  The congruence of the results obtained with two different methods using totally different
                  algorithms (population genetic based vs correlative approach), makes us confident
                  about these findings. Locus 27 was detected as an outlier linked to host-plant only
                  in Limousin (Table 
                  <a href="#T2">2</a>), which raises the question of local adaptation at the regional scale. In the Limousin,
                  locus 27 is much more frequent on spruce than on pine (0.8 versus 0.36). Although
                  it was not detected as an outlier in the two other regions (Ardèche and Finland),
                  it was also more frequent on spruce, suggesting that it is a global trend as also
                  suggested by the logistic regression.
               </p>
               <p>In contrast, the loci identified as outliers by only one of the methods should be
                  interpreted with caution. First, the lack of congruence observed between the two genetic
                  methods could be linked to the presence of false positives, which can be generated
                  by both methods [
                  <a href="#B9">9</a>,
                  <a href="#B10">10</a>].
               </p>
               <p>Second, the identification of an outlier by Dfdist but not by BayeScan could result
                  from the fact that Dfdist does not take into account population-specific demographic
                  effects. This was the case for one locus (locus 10) over-represented in one region
                  (Landes) compared with the other regions (frequency 0.9 vs 0.30 to 0.39). Because
                  the Bayesian model takes into account site-specific effects in modelling population
                  divergence (by calculating population-specific 
                  <i>F</i>
                  <sub>ST</sub>), it did not detect this locus as differing more than expected given the overall
                  divergence of this isolated south-western region (Landes) compared to the rest of
                  the European weevil distribution. Accordingly, the 
                  <i>F</i>
                  <sub>ST </sub>estimates for this locus was strikingly different depending on the method used, with
                  much higher estimates obtained using Dfdist than BayeScan (Table 
                  <a href="#T2">2</a>). Another hypothesis that can support the fact that locus 10 was only detected as
                  an outlier by Dfdist and not BayeScan in Landes is the "allele surfing effect" [
                  <a href="#B28">28</a>]. In a 2D spatial demographic expansion model, it is possible that one front of the
                  wave drift caused by a spatial bottleneck could push low frequency alleles to increase
                  their proportion and possibly become fixed in those populations at the edge of the
                  geographical range of species distribution. The non-identification of such locus by
                  the logistic regression could be a way to further confirm a false positive.
               </p>
            </div>
            <div class="ecologicaldivergence:possibleselectivepressures" title="sec">
               <div class="title" tagx="title" title="title">Ecological divergence: possible selective pressures</div>
               <p>The logistic regression identified 9 loci not consistently found by the genetic methods
                  (11, 19, 26, 29, 39, 40, 42, 62, 65). Such results could be interpreted as spurious
                  correlations, not linked to adaptation. A second explanation could be that these loci
                  are indeed under directional selection, but that both genetic methods failed to detect
                  them as outliers. Indeed, both genetic methods have a high rate of false negatives,
                  i.e. loci that are true outliers but that are not detected. For example, Beaumont
                  &amp; Balding [
                  <a href="#B10">10</a>], using simulated datasets, found that as many as 50% of true directionally selected
                  biallelic codominant loci (selection coefficient 0.05) were mis-classified as being
                  neutral by Fdist, a proportion that might be even higher with dominant markers. With
                  BayeScan, Foll and Gaggiotti [
                  <a href="#B9">9</a>] showed that for simulated AFLP datasets with an average 
                  <i>F</i>
                  <sub>ST </sub>of 0.05, 6 populations, 30 individuals sampled per population, and a selection coefficient
                  of 0.05, the false negative rate was as high as 33%. The parameters used in these
                  simulations were similar to those of our dataset (average 
                  <i>F</i>
                  <sub>ST </sub>= 0.05, 7 populations with an average of 42 individuals sampled per population).
               </p>
               <p>Two strategies can be used to search for loci under environmental selection. The first
                  consists of applying population genetic methods to identify outliers, then using logistic
                  regression only on the identified outliers to characterise the selective pressure
                  that best explains their variability [
                  <a href="#B17">17</a>]. The second strategy is the one we used in the present study, which was also used
                  by Joost 
                  <i>et al. </i>[
                  <a href="#B14">14</a>]. The approach consists of systematically applying the logistic regression to all
                  loci to identify all marker correlations and thus perhaps identify additional markers,
                  correlated to the environmental variable, that were not detected as outliers by the
                  genetic approaches. As a second step in this strategy, one returns to the genetic
                  method to investigate these newly detected relations in more depth, for example by
                  using different groupings that better reflect the selective pressure identified by
                  the logistic regression.
               </p>
               <p>In fact, only logistic regression allows assumptions to be made on the selective pressure
                  shaping observed patterns of adaptive genetic diversity. Although it is only a correlative
                  approach, it can help us to formulate biological hypothesis. However, we need to keep
                  in mind i) the possibility of spurious relations, and ii) the problem of the choice
                  of the explanatory variables, which is not an easy task. Indeed, climatic variables
                  are often highly inter-correlated and in most cases also correlated with elevation,
                  longitude and latitude. In our case, to avoid the correlations between the climatic
                  and elevation variables, we applied a PCA to produce orthogonal predictors, reducing
                  a total of 10 highly correlated explanatory variables to two 'environmental' predictors.
                  However, to have more precise information about the involvement of each climatic/elevation
                  variable on the behaviour of each locus, it would be necessary to use the climatic
                  variables directly rather than the PCA axes as explanatory factors, as done in Joost
                  
                  <i>et al</i>. [
                  <a href="#B14">14</a>]. In this later paper based on a larger dataset but not testing for the impact of
                  the host-plant on locus-specific differentiation, two outliers (loci 38 and 52, also
                  detected in the present analysis) were shown to be correlated with climatic variables,
                  locus 38 being positively correlated with the number of days of ground frost and negatively
                  with precipitation, sunshine and diurnal temperature range, while locus 52 was positively
                  correlated with diurnal temperature range. 
                  <i>A priori </i>biological assumptions and knowledge about the selective pressure at play in nature
                  can help with the choice of predictors to be used in the model. However, when no 
                  <i>a priori </i>assumption is available, all variables (after checking for correlations) should be
                  included in the model. We do nevertheless need to keep in mind that it is always better
                  to favour direct predictors (e.g. gradient of humidity) over indirect predictors (e.g.
                  elevation) [
                  <a href="#B29">29</a>].
               </p>
               <p>Although migration is likely to be important in this flying weevil, which may also
                  be enhanced due to the high connectivity of pine and spruce planted forests in Europe
                  and perhaps by human-mediated translocations, we detected 8 loci out of 83 (9.6%)
                  under directional selection across Europe - corresponding to a selection coefficient
                  5 times stronger than migration [
                  <a href="#B10">10</a>]. Out of these adaptive loci, only one was correlated with the host-plant, and one
                  was correlated with the interaction between host-plant and abiotic variables. 
                  <i>Hylobius </i>larvae feed on decaying plant material, so that the food quality might depend both
                  on the host-plant type and on climatic variables. Furthermore, host-plant genotypes
                  might vary from a region to another, explaining why the interaction between both biotic
                  and abiotic factors might be more important than each factor taken separately in explaining
                  genetic variability at some particular locus. This shows that in this widespread oligophagous
                  weevil, climatic variables have a relatively larger effect than the host-plant in
                  shaping genetic differentiation across the genome. Managed conifer forests in Western
                  Europe are relatively recent from an evolutionary perspective and have been planted
                  with both spruce and pine throughout large areas outside the natural range of the
                  two tree genera, thereby possibly limiting the opportunities for selection based on
                  host-plant. In contrast, the wide geographic range of these managed forests means
                  that 
                  <i>Hylobius abietis </i>encounters a large variability in climatic conditions. For instance, this weevil lives
                  underground during its diapause stage, and the length of the soil frost period is
                  likely to be a selective factor [
                  <a href="#B14">14</a>]. We found only limited evidence for host-plant-specific outliers, which represented
                  only 1.2% of all loci. This is comparable to the 1-2% of AFLP loci detected as host-specific
                  in a genome scan performed on different ecotypes of 
                  <i>Timema </i>walking sticks, using two host-plant species [
                  <a href="#B17">17</a>], and slightly less than that observed across maple and willow beetles (5% of host-specific
                  loci, [
                  <a href="#B16">16</a>]). Most of the host groups in this latter study were however taken from different
                  geographic localities, making it difficult to distinguish environmental effects from
                  host-specific adaptations. Finally, the limited evidence for host-plant adaptive loci
                  in 
                  <i>Hylobius abietis </i>could be related to the presence of obligate endosymbiotic bacteria in the midgut
                  of larvae [
                  <a href="#B30">30</a>]. Although the precise role of these bacteria in 
                  <i>Hylobius abietis </i>fitness has not yet been elucidated, the presence of this third genome (not included
                  in our genome scan because insect gut was removed before DNA extraction) might be
                  an alternative way to specialize on various host-plants, thereby limiting the selection
                  for adaptive loci within the genome of the insect itself. Obligate endosymbionts are
                  widespread in insects and might play a key role in the adaptation of phytophagous
                  insects to alternative host-plants [
                  <a href="#B31">31</a>].
               </p>
            </div>
         </div>
         <div class="conclusion" title="sec">
            <div class="title" tagx="title" title="title">Conclusion</div>
            <p>Our study shows that correlative methods are very promising to investigate the respective
               role of many environmental variables in shaping the geographical distribution at a
               locus level (allele frequency). They are particularly useful to validate outliers
               detected using population genetic methods, especially when populations are not at
               their demographic equilibrium. Finally, correlative methods represent the only way
               to investigate for adaptive loci when sampling is conducted at the individual rather
               that population level, and no 
               <i>a priori </i>grouping of individuals into panmictic populations can be performed, rendering population
               genetic methods inappropriate.
            </p>
         </div>
         <div class="methods" title="methods">
            <div class="title" tagx="title" title="title">Methods</div>
            <div class="studysitesanddatacollection" title="sec">
               <div class="title" tagx="title" title="title">Study sites and data collection</div>
               <p>This study is based on a sub-sample (296 larvae) of a large AFLP dataset of great
                  pine weevils comprising 367 individuals, including adults and larvae, collected in
                  20 localities throughout Europe [
                  <a href="#B14">14</a>,
                  <a href="#B25">25</a>]. Larvae were collected on either spruce (
                  <i>Picea abies </i>Karst) or pine (
                  <i>Pinus sylvestris </i>L. and 
                  <i>Pinus pinaster </i>Ait.) in 16 managed forest sites distributed across 4 large forestry regions: three
                  in France (Limousin, Ardeche, and Landes) and one in Finland (Table 
                  <a href="#T1">1</a>). Some sites were planted only with spruce, other only with pine, and several both
                  with spruce and pine. Three out of the four regions analysed comprised the two host-plants
                  (Finland, Limousin, Ardeche), while the region Landes was planted only with pine (
                  <i>Pinus pinaster</i>) (Figure 
                  <a href="#F1">1</a>). For each sampled site, previously geo-referenced [
                  <a href="#B23">23</a>], we extracted nine environmental variables (same variables used by [
                  <a href="#B12">12</a>]) as yearly mean of monthly values for the period 1961-2001 (available at: 
                  <a href="http://www.cru.uea.ac.uk">http://www.cru.uea.ac.uk</a>[
                  <a href="#B28">28</a>]): mean diurnal temperature range (°C), number of days with ground frost, precipitation
                  (mm/month), coefficient of variation of monthly precipitation (%), relative humidity
                  (%), maximum possible sunshine (% of day length), mean temperature (°C), wet days
                  (number of days with &gt; 0.1 mm rain per month), wind speed (m/s at 10 m above the ground).
                  The altitude of the site was also noted.
               </p>
               <p>Larvae were genotyped at 83 unlinked AFLP loci as described in Conord 
                  <i>et al</i>. (2006) [
                  <a href="#B25">25</a>]. Fragment length ranged from 50 to 375 bp, and most fragments (86%) were longer
                  than 100 bp limiting the probability of homoplasy (co-migrating non-homologous fragments)
                  which is known to affect mostly small fragments [
                  <a href="#B25">25</a>]. AFLP patterns were then visualised with GENOGRAPHER V1.6.0 software, in which a
                  fluorescent peak corresponds to the presence of an amplified restriction fragment.
                  Polymorphic peaks were checked individually and a presence/absence (i.e. 1/0) matrix
                  was constructed. A previous genetic structure analysis showed that regional forests
                  are a pertinent geographic scale for defining populations in the large pine [
                  <a href="#B25">25</a>].
               </p>
            </div>
            <div class="outlierdetectionusingapopulationgeneticapproach" title="sec">
               <div class="title" tagx="title" title="title">Outlier detection using a population genetic approach</div>
               <p>To detect outlier loci presenting a signature of positive selection in our AFLP dataset
                  we used two population genetic approaches. Both methods aimed at detecting loci with
                  a higher than expected differentiation between populations in contrasting situations
                  (e.g. contrasting climatic conditions, and/or alternative host-plant species): (1)
                  the popular Beaumont and Nichols [
                  <a href="#B7">7</a>] Dfdist program developed from Fdist for the analysis of dominant data (available
                  at 
                  <a href="http://www.rubic.rdg.ac.uk/~mab/stuff/">http://www.rubic.rdg.ac.uk/~mab/stuff/</a>) and (2) a new hierarchical Bayesian method (BayeScan, 
                  <a href="http://www-leca.ujf-grenoble.fr/logiciels.htm">http://www-leca.ujf-grenoble.fr/logiciels.htm</a>[
                  <a href="#B9">9</a>] that also allows the accommodation of AFLP data. Dfidst is a frequentist method
                  based on summary statistics of a symmetrical island model (i.e. drift-migration equilibrium,
                  [
                  <a href="#B8">8</a>]. Dfdist implements the Bayesian method of Zhivotovsky [
                  <a href="#B32">32</a>] for dominant markers [
                  <a href="#B32">32</a>] to estimate allelic frequencies. Outliers were identified by plotting 
                  <i>F</i>
                  <sub>ST </sub>against heterozygosity under the assumption of Hardy-Weinberg equilibrium. Significance
                  values were obtained by generating a null distribution of 
                  <i>F</i>
                  <sub>ST </sub>values based on 50 000 simulated loci with a mean 
                  <i>F</i>
                  <sub>ST </sub>equal to the trimmed mean 
                  <i>F</i>
                  <sub>ST </sub>calculated by removing the 30% highest and lowest 
                  <i>F</i>
                  <sub>ST </sub>values observed in the empirical dataset [
                  <a href="#B4">4</a>,
                  <a href="#B10">10</a>]. Finally Dfdist plots the observed 
                  <i>F</i>
                  <sub>ST </sub>value for each locus to detect those falling outside the neutral expectation given
                  by the null distribution.
               </p>
               <p>A first shortcoming of Dfdist is the possibility of detecting false positives (i.e.
                  committing type-I errors) [
                  <a href="#B33">33</a>,
                  <a href="#B34">34</a>]. This risk, largely due to multiple tests (one at each locus), can be reduced by
                  using a conservative significance level [
                  <a href="#B10">10</a>,
                  <a href="#B35">35</a>], as done here by adopting the smallest type I error of our analysis which is the
                  one calculated for logistic regressions α = 0.0006 (see later the justification).
                  It is also possible to estimate the number of false positives among detected outlier
                  loci (= false discovery rate) for a given level of significance by applying Storey
                  and Tibshirani's [
                  <a href="#B11">11</a>]
                  <i>q</i>-value method. False discovery rate is a statistical method used in multiple hypotheses
                  testing to correct for multiple comparisons. In our case, the method estimates the
                  false discovery rate (or 
                  <i>q</i>-value) from the set of each locus 
                  <i>p</i>-values provided by Dfdist. For a significance level of 0.0006, the corresponding
                  
                  <i>q</i>-value is the expected proportion of false positives among detected outliers. We used
                  the 
                  <i>q</i>-value package in R [
                  <a href="#B36">36</a>] for computations [
                  <a href="#B11">11</a>].
               </p>
               <p>A second shortcoming of Dfdist is that it assumes that populations are at drift-migration
                  equilibrium, which is unrealistic in most natural situations. In that context, the
                  main advantage of BayeScan is that it estimates population-specific 
                  <i>F</i>
                  <sub>ST </sub>coefficients, therefore allowing for different demographic histories and different
                  amounts of genetic drift between populations. The method is based on a logistic regression
                  model in which each logit value of genetic differentiation 
                  <i>F</i>
                  <sub>ST </sub>(
                  <i>i, j</i>) for locus 
                  <i>i </i>in population 
                  <i>j </i>is decomposed as a linear combination of the coefficients of the logistic regression,
                  α
                  <i>i </i>and β
                  <i>j</i>, corresponding, respectively, to a locus effect and to a population effect. To identify
                  adaptive loci, we focused on the posterior distribution of α
                  <i>i</i>: a positive value suggests that locus 
                  <i>i </i>is subject to adaptive selection, whereas a negative value suggests that balancing
                  selection is tending to homogenize allele frequencies over the populations. The posterior
                  probability of locus 
                  <i>i </i>being under selection is estimated by defining two alternative models, one that includes
                  α
                  <i>i </i>and another that excludes it. The respective posterior probabilities of these two
                  models are estimated using a Reversible Jump Markov Chain Monte Carlo (MCMC) approach
                  [
                  <a href="#B9">9</a>]. The posterior probability that a locus is subject to selection, corresponding to
                  
                  <i>P </i>(α
                  <i>i </i>≠ 0), is then estimated from the output of the MCMC by counting the number of times
                  α
                  <i>i </i>is included in the model. Another advantage of the Bayesian method compared with the
                  frequentist method (Dfdist), is that it explicitly takes all loci into account in
                  the analyses. Finally, the Bayesian approach also deals with the problem of multiple
                  testing of a large number of genomic locations, as the number of tested loci is taken
                  into account through the prior distribution. In this study, for the population and
                  locus effects Gaussian priors with means of -2 and zero were used with standard deviations
                  of 1.8 and 1, respectively, according to Beaumont and Balding [
                  <a href="#B10">10</a>]. The estimation of model parameters was automatically tuned on the basis of short
                  pilot runs (10 pilot runs, length 2000). Preliminary tests indicated that a burn-in
                  of 10 000 iterations was enough to have the MCMC converging. The sample size was set
                  to 10 000 and the thinning interval to 50 as suggested by Foll and Gaggiotti [
                  <a href="#B9">9</a>], resulting in a total chain length of 500 000 iterations. Four independent runs
                  were performed for each of the two datasets to account for the consistency of the
                  detected outliers. The loci were ranked according to their estimated posterior probability
                  and all loci with a value over 0.79 were retained as outliers. This corresponds to
                  a Bayes Factor &gt;3 as defined by Jeffreys [
                  <a href="#B37">37</a>], which provides substantial support for acceptation of the model.
               </p>
               <p>In practice, applying population genetic approaches requires to define populations.
                  We defined 5 different population-based structures in order to disentangle abiotic
                  and biotic factors (Figure 
                  <a href="#F1">1</a>). Separate analyses were done for these 5 structures using both Dfdist and BayeScan.
                  We firstly grouped individuals according to their geographic origin only (4 populations:
                  Finland, Limousin, Ardeche, and Landes; population size 83, 115, 78 and 20 respectively)
                  (structure 1, Figure 
                  <a href="#F1">1</a>), and secondly, according to their geographic origin and host-plant (7 populations:
                  Finland Pine, Finland Spruce, Limousin Pine, Limousin Spruce, Ardeche Pine, Ardeche
                  Spruce and Landes Pine; population size 38, 45, 30, 85, 28, 50 and 20 respectively)
                  (structure 2, Figure 
                  <a href="#F1">1</a>). Finally, within each of the three regions where both alternative host-plants are
                  found (Finland, Limousin, and Ardeche), pairwise comparisons of host-plant samples
                  were performed (structure 3, 4, 5, Figure 
                  <a href="#F1">1</a>). Multiple independent pairwise comparisons of populations submitted to the same
                  ecological constraints in different geographic areas allow confirmation of the involvement
                  of a putative outlier in adaptation to contrasting situations, in our case alternative
                  host-plants.
               </p>
            </div>
            <div class="outlierdetectionusingacorrelativeapproach" title="sec">
               <div class="title" tagx="title" title="title">Outlier detection using a correlative approach</div>
               <p>Our objective was to assess the effect of the host-plant variable versus all the other
                  measurable variables potentially involved in genetic differentiation. These include
                  elevation, 9 climatic variables and geographic coordinates of the sites. Preliminary
                  analyses have shown strong correlations between climatic variables and site geographic
                  coordinates. We therefore decided to remove geographic coordinates since it was not
                  possible, given our sampling design, to disentangle the effect of climate from the
                  effect of geography. This would not affect our conclusion about the potential effect
                  of host-plant type. To avoid correlation between the explanatory variables, a principal
                  component analysis (PCA) was applied on the correlation matrix of climatic and elevation
                  variables. The two first axes of the PCA represent the best way to represent variation
                  in environmental abiotic factors at the studied scale. Multiple logistic regressions
                  with a logit link and binomial error distribution [
                  <a href="#B38">38</a>] were then used to explain the presence and absence of each marker by the coordinates
                  of the two first axes of PCA (orthogonal predictors) representing environmental abiotic
                  variables, the host-plant variable and all the three possible interactions between
                  these three explanatory variables. Models were fitted using a maximum likelihood method
                  [
                  <a href="#B38">38</a>]. We used backwards elimination to select the variables in the final models using
                  Akaike's information criterion (AIC). Then, for each selected model, we also tested
                  each regression coefficient with a likelihood ratio test [
                  <a href="#B38">38</a>]. The significance of the likelihood ratio tests were tested using a Bonferroni correction
                  to correct for multiple testing (significance level = 0.05/83 where 83 is the number
                  of markers = 6 × 10
                  <sup>-4</sup>). Although initially all explanatory variables were potential predictors, only those
                  variables selected by the above criteria were used in the final models. False discovery
                  rates were also calculated using the type I error of 0.0006. All analyses were conducted
                  using R software [
                  <a href="#B36">36</a>].
               </p>
            </div>
            <div class="isolationbydistancepatternonneutralloci" title="sec">
               <div class="title" tagx="title" title="title">Isolation by distance pattern on neutral loci</div>
               <p>To test for isolation by distance on neutral loci, we applied a Mantel test between
                  genetic and Euclidian geographic distances calculated between the 16 sites. Genetic
                  distances were computed with AFLP-SURV 1.0 [
                  <a href="#B26">26</a>] as pairwise 
                  <i>F</i>
                  <sub>ST </sub>using the Bayesian method with non-uniform prior distribution to estimate allele frequencies
                  with dominant markers [
                  <a href="#B32">32</a>]. Geographic distances (in km) and Mantel tests were calculated using R software
                  [
                  <a href="#B36">36</a>]. We not only tested the relation between 
                  <i>F</i>
                  <sub>ST</sub>/(1-
                  <i>F</i>
                  <sub>ST</sub>) and the geographic distance (km), but also used the logarithm of the geographic
                  distance as recommended by Rousset [
                  <a href="#B39">39</a>].
               </p>
            </div>
         </div>
         <div class="authors'contributions" title="sec">
            <div class="title" tagx="title" title="title">Authors' contributions</div>
            <p>SM performed the statistical analysis and drafted the manuscript. CC participated
               to the design and carried out the molecular analysis. LD conceived the study, performed
               the population genetic analyses, coordinated and drafted the manuscript. All the authors
               read and approved the final manuscript.
            </p>
         </div>
      </div>
      <div class="back" title="back">
         <div class="acknowledgements" title="sec">
            <div class="title" tagx="title" title="title">Acknowledgements</div>
            <p>The authors thank G. Lempérière and H. Viiri (Forest Research Institute, Finland)
               for collecting weevils, M. Foll and O. Gaggiotti for their helpful comments and H.
               McCombie-Boudry for correcting our English.
            </p>
         </div>
         <div class="references">References</div>
         <div tag="ref-list">
            <ul>
               <li tag="ref">
                  <a name="B1"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Black</span>
                        <span class="given-names" tagx="given-names" title="given-names">WC</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Baer</span>
                        <span class="given-names" tagx="given-names" title="given-names">CF</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Antolin</span>
                        <span class="given-names" tagx="given-names" title="given-names">MF</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">DuTeau</span>
                        <span class="given-names" tagx="given-names" title="given-names">NM</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Population genomics: genome-wide sampling of insect populations</span>
                     <span class="source" tagx="source" title="source">Annual Review of Entomology</span>
                     <span class="year" tagx="year" title="year">2001</span>
                     <span class="volume" tagx="volume" title="volume">46</span>
                     <span class="fpage" tagx="fpage" title="fpage">441</span>
                     <span class="lpage" tagx="lpage" title="lpage">469</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1146/annurev.ento.46.1.441">10.1146/annurev.ento.46.1.441</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/11112176">11112176</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B2"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Schlötterer</span>
                        <span class="given-names" tagx="given-names" title="given-names">C</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Hitchhiking mapping-functional genomics from the population genetics perspective</span>
                     <span class="source" tagx="source" title="source">Trends in Genetics</span>
                     <span class="year" tagx="year" title="year">2003</span>
                     <span class="volume" tagx="volume" title="volume">19</span>
                     <span class="fpage" tagx="fpage" title="fpage">32</span>
                     <span class="lpage" tagx="lpage" title="lpage">38</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1016/S0168-9525(02)00012-4">10.1016/S0168-9525(02)00012-4</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/12493246">12493246</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B3"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Stinchcombe</span>
                        <span class="given-names" tagx="given-names" title="given-names">JR</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Hoekstra</span>
                        <span class="given-names" tagx="given-names" title="given-names">HE</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Combining population genomics and quantitative genetics: finding the genes underlying
                        ecologically important traits
                     </span>
                     <span class="source" tagx="source" title="source">Heredity</span>
                     <span class="year" tagx="year" title="year">2008</span>
                     <span class="volume" tagx="volume" title="volume">100</span>
                     <span class="fpage" tagx="fpage" title="fpage">158</span>
                     <span class="lpage" tagx="lpage" title="lpage">170</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1038/sj.hdy.6800937">10.1038/sj.hdy.6800937</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/17314923">17314923</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B4"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Bonin</span>
                        <span class="given-names" tagx="given-names" title="given-names">A</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Miaud</span>
                        <span class="given-names" tagx="given-names" title="given-names">C</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Taberlet</span>
                        <span class="given-names" tagx="given-names" title="given-names">P</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Pompanon</span>
                        <span class="given-names" tagx="given-names" title="given-names">F</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Explorative genome scan to detect candidate loci for adaptation along a gradient of
                        altitude in the common frog (
                        <i>Rana temporaria</i>)
                     </span>
                     <span class="source" tagx="source" title="source">Molecular Biology and Evolution</span>
                     <span class="year" tagx="year" title="year">2006</span>
                     <span class="volume" tagx="volume" title="volume">23</span>
                     <span class="fpage" tagx="fpage" title="fpage">773</span>
                     <span class="lpage" tagx="lpage" title="lpage">783</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1093/molbev/msj087">10.1093/molbev/msj087</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/16396915">16396915</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B5"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Despres</span>
                        <span class="given-names" tagx="given-names" title="given-names">L</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Loriot</span>
                        <span class="given-names" tagx="given-names" title="given-names">S</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Gaudeul</span>
                        <span class="given-names" tagx="given-names" title="given-names">M</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Geographic pattern of genetic variation in the European globeflower Trollius europaeus
                        L. (Ranunculaceae) inferred from amplified fragment length polymorphism markers
                     </span>
                     <span class="source" tagx="source" title="source">Molecular Ecology</span>
                     <span class="year" tagx="year" title="year">2002</span>
                     <span class="volume" tagx="volume" title="volume">11</span>
                     <span class="fpage" tagx="fpage" title="fpage">2337</span>
                     <span class="lpage" tagx="lpage" title="lpage">2347</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1046/j.1365-294X.2002.01618.x">10.1046/j.1365-294X.2002.01618.x</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/12406244">12406244</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B6"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Emelianov</span>
                        <span class="given-names" tagx="given-names" title="given-names">I</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Marec</span>
                        <span class="given-names" tagx="given-names" title="given-names">F</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Mallet</span>
                        <span class="given-names" tagx="given-names" title="given-names">J</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Genomic evidence for divergence with gene flow in host races of the larch budmoth</span>
                     <span class="source" tagx="source" title="source">Proceedings of the Royal Society of London, Series B</span>
                     <span class="year" tagx="year" title="year">2004</span>
                     <span class="volume" tagx="volume" title="volume">271</span>
                     <span class="fpage" tagx="fpage" title="fpage">97</span>
                     <span class="lpage" tagx="lpage" title="lpage">105</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1098/rspb.2003.2574">10.1098/rspb.2003.2574</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B7"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Beaumont</span>
                        <span class="given-names" tagx="given-names" title="given-names">MA</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Nichols</span>
                        <span class="given-names" tagx="given-names" title="given-names">RA</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Evaluating loci for use in the genetic analysis of population structure</span>
                     <span class="source" tagx="source" title="source">Proceedings of the Royal Society of London, Series B</span>
                     <span class="year" tagx="year" title="year">1996</span>
                     <span class="volume" tagx="volume" title="volume">263</span>
                     <span class="fpage" tagx="fpage" title="fpage">1619</span>
                     <span class="lpage" tagx="lpage" title="lpage">1626</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1098/rspb.1996.0237">10.1098/rspb.1996.0237</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B8"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Wood</span>
                        <span class="given-names" tagx="given-names" title="given-names">HM</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Grahame</span>
                        <span class="given-names" tagx="given-names" title="given-names">JW</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Humphray</span>
                        <span class="given-names" tagx="given-names" title="given-names">S</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Rogers</span>
                        <span class="given-names" tagx="given-names" title="given-names">J</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Butlin</span>
                        <span class="given-names" tagx="given-names" title="given-names">RK</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Sequence differentiation in regions identified by a genome scan for local adaptation</span>
                     <span class="source" tagx="source" title="source">Molecular Ecology</span>
                     <span class="year" tagx="year" title="year">2008</span>
                     <span class="volume" tagx="volume" title="volume">17</span>
                     <span class="fpage" tagx="fpage" title="fpage">3123</span>
                     <span class="lpage" tagx="lpage" title="lpage">3135</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1111/j.1365-294X.2008.03755.x">10.1111/j.1365-294X.2008.03755.x</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/18510587">18510587</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B9"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Foll</span>
                        <span class="given-names" tagx="given-names" title="given-names">M</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Gaggiotti</span>
                        <span class="given-names" tagx="given-names" title="given-names">O</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">A genome scan method to identify selected loci appropriate for both dominant and codominant
                        markers: a bayesian perspective
                     </span>
                     <span class="source" tagx="source" title="source">Genetics</span>
                     <span class="year" tagx="year" title="year">2008</span>
                     <span class="volume" tagx="volume" title="volume">180</span>
                     <span class="fpage" tagx="fpage" title="fpage">977</span>
                     <span class="lpage" tagx="lpage" title="lpage">993</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1534/genetics.108.092221">10.1534/genetics.108.092221</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/18780740">18780740</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B10"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Beaumont</span>
                        <span class="given-names" tagx="given-names" title="given-names">MA</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Balding</span>
                        <span class="given-names" tagx="given-names" title="given-names">DJ</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Identifying adaptive genetic divergence among populations from genome scans</span>
                     <span class="source" tagx="source" title="source">Molecular Ecology</span>
                     <span class="year" tagx="year" title="year">2004</span>
                     <span class="volume" tagx="volume" title="volume">13</span>
                     <span class="fpage" tagx="fpage" title="fpage">969</span>
                     <span class="lpage" tagx="lpage" title="lpage">980</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1111/j.1365-294X.2004.02125.x">10.1111/j.1365-294X.2004.02125.x</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/15012769">15012769</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B11"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Storey</span>
                        <span class="given-names" tagx="given-names" title="given-names">JD</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Tibshirani</span>
                        <span class="given-names" tagx="given-names" title="given-names">R</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Statistical significance for genomewide studies</span>
                     <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences of the United States of America</span>
                     <span class="year" tagx="year" title="year">2003</span>
                     <span class="volume" tagx="volume" title="volume">100</span>
                     <span class="fpage" tagx="fpage" title="fpage">9440</span>
                     <span class="lpage" tagx="lpage" title="lpage">9445</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1073/pnas.1530509100">10.1073/pnas.1530509100</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/12883005">12883005</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B12"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Holderegger</span>
                        <span class="given-names" tagx="given-names" title="given-names">R</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Herrmann</span>
                        <span class="given-names" tagx="given-names" title="given-names">D</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Poncet</span>
                        <span class="given-names" tagx="given-names" title="given-names">B</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Gugerli</span>
                        <span class="given-names" tagx="given-names" title="given-names">F</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Thuiller</span>
                        <span class="given-names" tagx="given-names" title="given-names">W</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Taberlet</span>
                        <span class="given-names" tagx="given-names" title="given-names">P</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Gielly</span>
                        <span class="given-names" tagx="given-names" title="given-names">L</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Rioux</span>
                        <span class="given-names" tagx="given-names" title="given-names">D</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Brodbeck</span>
                        <span class="given-names" tagx="given-names" title="given-names">S</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Aubert</span>
                        <span class="given-names" tagx="given-names" title="given-names">S</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Land ahead: Using genome scans to identify molecular markers of adaptive relevance</span>
                     <span class="source" tagx="source" title="source">Plant Ecology and Diversity</span>
                     <span class="year" tagx="year" title="year">2008</span>
                     <span class="volume" tagx="volume" title="volume">1</span>
                     <span class="fpage" tagx="fpage" title="fpage">273</span>
                     <span class="lpage" tagx="lpage" title="lpage">283</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1080/17550870802338420">10.1080/17550870802338420</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B13"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Bonin</span>
                        <span class="given-names" tagx="given-names" title="given-names">A</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Pompanon</span>
                        <span class="given-names" tagx="given-names" title="given-names">F</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Taberlet</span>
                        <span class="given-names" tagx="given-names" title="given-names">P</span>
                     </span>
                     <span class="person-group'">Zimmer EA, Roalson E</span>
                     <span class="mixed-article-title" title="mixed-article-title">Use of Amplified Fragment Length Polymorphism (AFLP) markers in surveys of vertebrate
                        diversity
                     </span>
                     <span class="source" tagx="source" title="source">Molecular Evolution: Producing the Biochemical Data, Part B</span>
                     <span class="year" tagx="year" title="year">2005</span>
                     <span class="volume" tagx="volume" title="volume">395</span>
                     <span class="publisher-name" tagx="publisher-name" title="publisher-name">New York City: Academic Press</span>
                     <span class="fpage" tagx="fpage" title="fpage">145</span>
                     <span class="lpage" tagx="lpage" title="lpage">161</span>
                     <span class="comment">full_text</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B14"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Joost</span>
                        <span class="given-names" tagx="given-names" title="given-names">S</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Bonin</span>
                        <span class="given-names" tagx="given-names" title="given-names">A</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Bruford</span>
                        <span class="given-names" tagx="given-names" title="given-names">MW</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Despres</span>
                        <span class="given-names" tagx="given-names" title="given-names">L</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Conord</span>
                        <span class="given-names" tagx="given-names" title="given-names">C</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Erhardt</span>
                        <span class="given-names" tagx="given-names" title="given-names">G</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Taberlet</span>
                        <span class="given-names" tagx="given-names" title="given-names">P</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">A spatial analysis method (SAM) to detect candidate loci for selection: towards a
                        landscape genomics approach to adaptation
                     </span>
                     <span class="source" tagx="source" title="source">Molecular Ecology</span>
                     <span class="year" tagx="year" title="year">2007</span>
                     <span class="volume" tagx="volume" title="volume">16</span>
                     <span class="fpage" tagx="fpage" title="fpage">3955</span>
                     <span class="lpage" tagx="lpage" title="lpage">3969</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1111/j.1365-294X.2007.03442.x">10.1111/j.1365-294X.2007.03442.x</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/17850556">17850556</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B15"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Jump</span>
                        <span class="given-names" tagx="given-names" title="given-names">AS</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Hunt</span>
                        <span class="given-names" tagx="given-names" title="given-names">JM</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Martinez-Izquierdo</span>
                        <span class="given-names" tagx="given-names" title="given-names">JA</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Penuelas</span>
                        <span class="given-names" tagx="given-names" title="given-names">J</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Natural selection and climate change: temperature-linked spatial and temporal trends
                        in gene frequency in 
                        <i>Fagus sylvatica</i>
                     </span>
                     <span class="source" tagx="source" title="source">Molecular Ecology</span>
                     <span class="year" tagx="year" title="year">2006</span>
                     <span class="volume" tagx="volume" title="volume">15</span>
                     <span class="fpage" tagx="fpage" title="fpage">3469</span>
                     <span class="lpage" tagx="lpage" title="lpage">3480</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1111/j.1365-294X.2006.03027.x">10.1111/j.1365-294X.2006.03027.x</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/16968284">16968284</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B16"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Egan</span>
                        <span class="given-names" tagx="given-names" title="given-names">SP</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Nosil</span>
                        <span class="given-names" tagx="given-names" title="given-names">P</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Funk</span>
                        <span class="given-names" tagx="given-names" title="given-names">DJ</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Selection and genomic differentiation during ecological speciation: Isolating the
                        contributions of host association via a comparative genome scan of Neochlamisus bebbianae
                        leaf beetles
                     </span>
                     <span class="source" tagx="source" title="source">Evolution</span>
                     <span class="year" tagx="year" title="year">2008</span>
                     <span class="volume" tagx="volume" title="volume">62</span>
                     <span class="fpage" tagx="fpage" title="fpage">1162</span>
                     <span class="lpage" tagx="lpage" title="lpage">1181</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1111/j.1558-5646.2008.00352.x">10.1111/j.1558-5646.2008.00352.x</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/18298645">18298645</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B17"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Nosil</span>
                        <span class="given-names" tagx="given-names" title="given-names">P</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Egan</span>
                        <span class="given-names" tagx="given-names" title="given-names">SP</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Funk</span>
                        <span class="given-names" tagx="given-names" title="given-names">DJ</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Heterogeneous genomic differentiation between walking-stick ecotypes: "Isolation by
                        adaptation" and multiple roles for divergent selection
                     </span>
                     <span class="source" tagx="source" title="source">Evolution</span>
                     <span class="year" tagx="year" title="year">2008</span>
                     <span class="volume" tagx="volume" title="volume">62</span>
                     <span class="fpage" tagx="fpage" title="fpage">316</span>
                     <span class="lpage" tagx="lpage" title="lpage">336</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1111/j.1558-5646.2007.00299.x">10.1111/j.1558-5646.2007.00299.x</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/17999721">17999721</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B18"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Campbell</span>
                        <span class="given-names" tagx="given-names" title="given-names">D</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Bernatchez</span>
                        <span class="given-names" tagx="given-names" title="given-names">L</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Generic scan using AFLP markers as a means to assess the role of directional selection
                        in the divergence of sympatric whitefish ecotypes
                     </span>
                     <span class="source" tagx="source" title="source">Molecular Biology and Evolution</span>
                     <span class="year" tagx="year" title="year">2004</span>
                     <span class="volume" tagx="volume" title="volume">21</span>
                     <span class="fpage" tagx="fpage" title="fpage">945</span>
                     <span class="lpage" tagx="lpage" title="lpage">956</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1093/molbev/msh101">10.1093/molbev/msh101</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/15014172">15014172</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B19"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Berlocher</span>
                        <span class="given-names" tagx="given-names" title="given-names">SH</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Feder</span>
                        <span class="given-names" tagx="given-names" title="given-names">JL</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Sympatric speciation in phytophagous insects: moving beyond controversy?</span>
                     <span class="source" tagx="source" title="source">Annual Review of Entomology</span>
                     <span class="year" tagx="year" title="year">2002</span>
                     <span class="volume" tagx="volume" title="volume">47</span>
                     <span class="fpage" tagx="fpage" title="fpage">773</span>
                     <span class="lpage" tagx="lpage" title="lpage">815</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1146/annurev.ento.47.091201.145312">10.1146/annurev.ento.47.091201.145312</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/11729091">11729091</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B20"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Dres</span>
                        <span class="given-names" tagx="given-names" title="given-names">M</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Mallet</span>
                        <span class="given-names" tagx="given-names" title="given-names">J</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Host races in plant-feeding insects and their importance in sympatric speciation</span>
                     <span class="source" tagx="source" title="source">Philosophical Transactions of the Royal Society of London Series B-Biological Sciences</span>
                     <span class="year" tagx="year" title="year">2002</span>
                     <span class="volume" tagx="volume" title="volume">357</span>
                     <span class="fpage" tagx="fpage" title="fpage">471</span>
                     <span class="lpage" tagx="lpage" title="lpage">492</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1098/rstb.2002.1059">10.1098/rstb.2002.1059</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B21"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Via</span>
                        <span class="given-names" tagx="given-names" title="given-names">S</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Reproductive isolation between sympatric races of pea aphids. I. Gene flow restriction
                        and habitat choice
                     </span>
                     <span class="source" tagx="source" title="source">Evolution</span>
                     <span class="year" tagx="year" title="year">1999</span>
                     <span class="volume" tagx="volume" title="volume">53</span>
                     <span class="fpage" tagx="fpage" title="fpage">1446</span>
                     <span class="lpage" tagx="lpage" title="lpage">1457</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.2307/2640891">10.2307/2640891</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B22"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Day</span>
                        <span class="given-names" tagx="given-names" title="given-names">K</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Nordlander</span>
                        <span class="given-names" tagx="given-names" title="given-names">G</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Kenis</span>
                        <span class="given-names" tagx="given-names" title="given-names">M</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Halldorson</span>
                        <span class="given-names" tagx="given-names" title="given-names">G</span>
                     </span>
                     <span class="person-group'">Lieutier FDK, Battisti AGJ, Evans HF</span>
                     <span class="mixed-article-title" title="mixed-article-title">General biology and life cycles of bark weevils</span>
                     <span class="source" tagx="source" title="source">Insects in Living Trees in Europe, a Synthesis</span>
                     <span class="year" tagx="year" title="year">2004</span>
                     <span class="publisher-name" tagx="publisher-name" title="publisher-name">Dordrecht: Kluwer Academi</span>
                     <span class="fpage" tagx="fpage" title="fpage">331</span>
                     <span class="lpage" tagx="lpage" title="lpage">349</span>
                     <span class="comment">full_text</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B23"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Thorpe</span>
                        <span class="given-names" tagx="given-names" title="given-names">K</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Day</span>
                        <span class="given-names" tagx="given-names" title="given-names">KR</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">The impact of host plant species on the larval development of the large pine weevil
                        Hylobius abietis L
                     </span>
                     <span class="source" tagx="source" title="source">Agricultural and Forest Entomology</span>
                     <span class="year" tagx="year" title="year">2002</span>
                     <span class="volume" tagx="volume" title="volume">4</span>
                     <span class="fpage" tagx="fpage" title="fpage">187</span>
                     <span class="lpage" tagx="lpage" title="lpage">194</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1046/j.1461-9563.2002.00144.x">10.1046/j.1461-9563.2002.00144.x</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B24"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">von Sydow</span>
                        <span class="given-names" tagx="given-names" title="given-names">F</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Bigersson</span>
                        <span class="given-names" tagx="given-names" title="given-names">G</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Conifer stump condition and pine weevil (Hylobius abietis) reproduction</span>
                     <span class="source" tagx="source" title="source">Canadian Journal of Forest Research</span>
                     <span class="year" tagx="year" title="year">1997</span>
                     <span class="volume" tagx="volume" title="volume">22</span>
                     <span class="fpage" tagx="fpage" title="fpage">1254</span>
                     <span class="lpage" tagx="lpage" title="lpage">1262</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B25"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Conord</span>
                        <span class="given-names" tagx="given-names" title="given-names">C</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Lemperiere</span>
                        <span class="given-names" tagx="given-names" title="given-names">G</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Taberlet</span>
                        <span class="given-names" tagx="given-names" title="given-names">P</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Despres</span>
                        <span class="given-names" tagx="given-names" title="given-names">L</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Genetic structure of the forest pest Hylobius abietis on conifer plantations at different
                        spatial scales in Europe
                     </span>
                     <span class="source" tagx="source" title="source">Heredity</span>
                     <span class="year" tagx="year" title="year">2006</span>
                     <span class="volume" tagx="volume" title="volume">97</span>
                     <span class="fpage" tagx="fpage" title="fpage">46</span>
                     <span class="lpage" tagx="lpage" title="lpage">55</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1038/sj.hdy.6800837">10.1038/sj.hdy.6800837</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/16705324">16705324</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B26"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Vekemans</span>
                        <span class="given-names" tagx="given-names" title="given-names">X</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Beauwens</span>
                        <span class="given-names" tagx="given-names" title="given-names">T</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Lemaire</span>
                        <span class="given-names" tagx="given-names" title="given-names">M</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Roldan-Ruiz</span>
                        <span class="given-names" tagx="given-names" title="given-names">I</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Data from amplified fragment length polymorphism (AFLP) markers show indication of
                        size homoplasy and of a relationship between degree of homoplasy and fragment size
                     </span>
                     <span class="source" tagx="source" title="source">Molecular Ecology</span>
                     <span class="year" tagx="year" title="year">2002</span>
                     <span class="volume" tagx="volume" title="volume">11</span>
                     <span class="fpage" tagx="fpage" title="fpage">139</span>
                     <span class="lpage" tagx="lpage" title="lpage">151</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1046/j.0962-1083.2001.01415.x">10.1046/j.0962-1083.2001.01415.x</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/11903911">11903911</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B27"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Excoffier</span>
                        <span class="given-names" tagx="given-names" title="given-names">L</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Hofer</span>
                        <span class="given-names" tagx="given-names" title="given-names">T</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Foll</span>
                        <span class="given-names" tagx="given-names" title="given-names">M</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Detecting loci under selection in a hierarchically structured population</span>
                     <span class="source" tagx="source" title="source">Heredity</span>
                     <span class="year" tagx="year" title="year">2009</span>
                     <span class="volume" tagx="volume" title="volume">103</span>
                     <span class="fpage" tagx="fpage" title="fpage">285</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1038/hdy.2009.74">10.1038/hdy.2009.74</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/19623208">19623208</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B28"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Excoffier</span>
                        <span class="given-names" tagx="given-names" title="given-names">L</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Ray</span>
                        <span class="given-names" tagx="given-names" title="given-names">N</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Surfing during population expansions promotes genetic revolutions and structuration</span>
                     <span class="source" tagx="source" title="source">Trends in Ecology &amp; Evolution</span>
                     <span class="year" tagx="year" title="year">2008</span>
                     <span class="volume" tagx="volume" title="volume">23</span>
                     <span class="fpage" tagx="fpage" title="fpage">347</span>
                     <span class="lpage" tagx="lpage" title="lpage">351</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1016/j.tree.2008.04.004">10.1016/j.tree.2008.04.004</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B29"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Vaughan</span>
                        <span class="given-names" tagx="given-names" title="given-names">I</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Ormerod</span>
                        <span class="given-names" tagx="given-names" title="given-names">S</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Improving the quality of distribution models for conservation biology by addressing
                        shortcomings in the field collection of training data
                     </span>
                     <span class="source" tagx="source" title="source">conservation biology</span>
                     <span class="year" tagx="year" title="year">2003</span>
                     <span class="volume" tagx="volume" title="volume">17</span>
                     <span class="fpage" tagx="fpage" title="fpage">1601</span>
                     <span class="lpage" tagx="lpage" title="lpage">1611</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1111/j.1523-1739.2003.00359.x">10.1111/j.1523-1739.2003.00359.x</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B30"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Conord</span>
                        <span class="given-names" tagx="given-names" title="given-names">C</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Despres</span>
                        <span class="given-names" tagx="given-names" title="given-names">L</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Vallier</span>
                        <span class="given-names" tagx="given-names" title="given-names">A</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Balmand</span>
                        <span class="given-names" tagx="given-names" title="given-names">S</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Miquel</span>
                        <span class="given-names" tagx="given-names" title="given-names">C</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Zundel</span>
                        <span class="given-names" tagx="given-names" title="given-names">S</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Lemperiere</span>
                        <span class="given-names" tagx="given-names" title="given-names">G</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Heddi</span>
                        <span class="given-names" tagx="given-names" title="given-names">A</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Long-term evolutionary stability of bacterial endosymbiosis in Curculionoidea: additional
                        evidence of symbiont replacement in the Dryophthoridae family
                     </span>
                     <span class="source" tagx="source" title="source">Molecular Biology and Evolution</span>
                     <span class="year" tagx="year" title="year">2008</span>
                     <span class="volume" tagx="volume" title="volume">25</span>
                     <span class="fpage" tagx="fpage" title="fpage">859</span>
                     <span class="lpage" tagx="lpage" title="lpage">868</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1093/molbev/msn027">10.1093/molbev/msn027</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/18310662">18310662</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B31"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Tsuchida</span>
                        <span class="given-names" tagx="given-names" title="given-names">T</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Koga</span>
                        <span class="given-names" tagx="given-names" title="given-names">R</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Fukatsu</span>
                        <span class="given-names" tagx="given-names" title="given-names">T</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Host plant specialization governed by facultative symbiont</span>
                     <span class="source" tagx="source" title="source">Science</span>
                     <span class="year" tagx="year" title="year">2004</span>
                     <span class="volume" tagx="volume" title="volume">303</span>
                     <span class="fpage" tagx="fpage" title="fpage">1989</span>
                     <span class="lpage" tagx="lpage" title="lpage">1989</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1126/science.1094611">10.1126/science.1094611</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/15044797">15044797</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B32"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Zhivotovsky</span>
                        <span class="given-names" tagx="given-names" title="given-names">LA</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Estimating population structure in diploids with multilocus dominant DNA markers</span>
                     <span class="source" tagx="source" title="source">Molecular Ecology</span>
                     <span class="year" tagx="year" title="year">1999</span>
                     <span class="volume" tagx="volume" title="volume">8</span>
                     <span class="fpage" tagx="fpage" title="fpage">907</span>
                     <span class="lpage" tagx="lpage" title="lpage">913</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1046/j.1365-294x.1999.00620.x">10.1046/j.1365-294x.1999.00620.x</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/10434412">10434412</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B33"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Bonin</span>
                        <span class="given-names" tagx="given-names" title="given-names">A</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Ehrich</span>
                        <span class="given-names" tagx="given-names" title="given-names">D</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Manel</span>
                        <span class="given-names" tagx="given-names" title="given-names">S</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Statistical analysis of amplified fragment length polymorphism data: a toolbox for
                        molecular ecologists and evolutionists
                     </span>
                     <span class="source" tagx="source" title="source">Molecular Ecology</span>
                     <span class="year" tagx="year" title="year">2007</span>
                     <span class="volume" tagx="volume" title="volume">16</span>
                     <span class="fpage" tagx="fpage" title="fpage">3737</span>
                     <span class="lpage" tagx="lpage" title="lpage">3758</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1111/j.1365-294X.2007.03435.x">10.1111/j.1365-294X.2007.03435.x</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/17850542">17850542</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B34"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Luikart</span>
                        <span class="given-names" tagx="given-names" title="given-names">G</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">England</span>
                        <span class="given-names" tagx="given-names" title="given-names">PR</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Tallmon</span>
                        <span class="given-names" tagx="given-names" title="given-names">D</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Jordan</span>
                        <span class="given-names" tagx="given-names" title="given-names">S</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Taberlet</span>
                        <span class="given-names" tagx="given-names" title="given-names">P</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">The power and promise of population genomics: from genotyping to genome typing</span>
                     <span class="source" tagx="source" title="source">Nature Reviews Genetics</span>
                     <span class="year" tagx="year" title="year">2003</span>
                     <span class="volume" tagx="volume" title="volume">4</span>
                     <span class="fpage" tagx="fpage" title="fpage">981</span>
                     <span class="lpage" tagx="lpage" title="lpage">994</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1038/nrg1226">10.1038/nrg1226</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/14631358">14631358</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B35"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Caballero</span>
                        <span class="given-names" tagx="given-names" title="given-names">A</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Quesada</span>
                        <span class="given-names" tagx="given-names" title="given-names">H</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Rolan-Alvarez</span>
                        <span class="given-names" tagx="given-names" title="given-names">E</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Impact of amplified fragment length polymorphism size homoplasy on the estimation
                        of population genetic diversity and the detection of selective loci
                     </span>
                     <span class="source" tagx="source" title="source">Genetics</span>
                     <span class="year" tagx="year" title="year">2008</span>
                     <span class="volume" tagx="volume" title="volume">179</span>
                     <span class="fpage" tagx="fpage" title="fpage">539</span>
                     <span class="lpage" tagx="lpage" title="lpage">554</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1534/genetics.107.083246">10.1534/genetics.107.083246</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/18493070">18493070</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B36"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="collab'">collab: Team RDC</span>
                     <span class="mixed-article-title" title="mixed-article-title">R: A language and environment for statistical computing</span>
                     <span class="source" tagx="source" title="source">R Foundation for Statistical Computing</span>
                     <span class="year" tagx="year" title="year">2008</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B37"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Jeffreys</span>
                        <span class="given-names" tagx="given-names" title="given-names">H</span>
                     </span>
                     <span class="source" tagx="source" title="source">Theory of probability</span>
                     <span class="year" tagx="year" title="year">1961</span>
                     <span class="edition" tagx="edition" title="edition">Third</span>
                     <span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford Universityt press</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B38"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">McCullagh</span>
                        <span class="given-names" tagx="given-names" title="given-names">P</span>
                     </span>
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Nelder</span>
                        <span class="given-names" tagx="given-names" title="given-names">J</span>
                     </span>
                     <span class="source" tagx="source" title="source">Generalized linear models</span>
                     <span class="year" tagx="year" title="year">1989</span>
                     <span class="publisher-name" tagx="publisher-name" title="publisher-name">London, UK: Chapman &amp; Hall</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="B39"></a>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">
                     <span class="name" tagx="name" title="name">
                        <span class="surname" tagx="surname" title="surname">Rousset</span>
                        <span class="given-names" tagx="given-names" title="given-names">F</span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Genetic differentiation and estimation of gene flow from F-statistics under isolation
                        by distance
                     </span>
                     <span class="source" tagx="source" title="source">Genetics</span>
                     <span class="year" tagx="year" title="year">1997</span>
                     <span class="volume" tagx="volume" title="volume">145</span>
                     <span class="fpage" tagx="fpage" title="fpage">1219</span>
                     <span class="lpage" tagx="lpage" title="lpage">1228</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/9093870">9093870</a>
                     </span>
                  </span>
               </li>
            </ul>
         </div>
      </div>
   </body>
</html>