<?xml version="1.0" encoding="UTF-8"?><html xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:h="http://www.w3.org/1999/xhtml">
   <head>
      <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style>
   </head>
   <body>
      <div class="front" title="front">
         <div class="journal-meta" tagx="journal-meta" title="journal-meta">
            <span class="nlm-ta" title="nlm-ta">BMC Genomics</span>
            <span class="iso-abbrev" title="iso-abbrev">BMC Genomics</span>
            <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group">
               <span class="journal-title" tagx="journal-title" title="journal-title">BMC Genomics</span>
            </div>
            <span class="issn-epub" tagx="issn" title="issn-epub">1471-2164</span>
            <div class="publisher" tagx="publisher" title="publisher">
               <span class="publisher-name" tagx="publisher-name" title="publisher-name">BioMed Central</span>
               <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span>
            </div>
         </div>
         <div class="article-meta" tagx="article-meta" title="article-meta">
            <span class="pmcid" title="pmcid">
               			pmcid:
               			
               <a href="http://www.ncbi.nlm.nih.gov/pubmed/4299006">4299006</a>
            </span>
            <span class="pmid" title="pmid">
               			pmid:
               			
               <a href="http://www.ncbi.nlm.nih.gov/pubmed/25495900">25495900</a>
            </span>
            <span class="publisher-id" title="publisher-id">6803</span>
            <span class="doi" title="doi">
               			doi:
               			
               <a href="https://dx.doi.org/10.1186/1471-2164-15-1096">10.1186/1471-2164-15-1096</a>
            </span>
            <div class="article-categories" title="article-categories">: 
               <span class="subject" title="subject">Research Article</span>
            </div>
            <div class="title-group">
               <div class="article-title" title="article-title">Functional genomics and microbiome profiling of the Asian longhorned beetle (
                  <i>Anoplophora glabripennis</i>) reveal insights into the digestive physiology and nutritional ecology of wood feeding
                  beetles
               </div>
            </div>
            <div class="contrib-group" title="contrib-group">
               <span class="contrib" title="contrib">
                  <span class="name" tagx="name" title="name">
                     <span class="surname" tagx="surname" title="surname">Scully</span>
                     <span class="given-names" tagx="given-names" title="given-names">Erin D</span>
                  </span>
                  <span class="address" title="address">
                     <span class="email" tagx="email" title="email">erin.scully@ars.usda.gov</span>
                  </span>
                  <a href="#Aff42"></a>
                  <a href="#Aff43"></a>
               </span>
               <span class="contrib" title="contrib">
                  <span class="name" tagx="name" title="name">
                     <span class="surname" tagx="surname" title="surname">Geib</span>
                     <span class="given-names" tagx="given-names" title="given-names">Scott M</span>
                  </span>
                  <span class="address" title="address">
                     <span class="email" tagx="email" title="email">scott.geib@ars.usda.gov</span>
                  </span>
                  <a href="#Aff44"></a>
               </span>
               <span class="contrib" title="contrib">
                  <span class="name" tagx="name" title="name">
                     <span class="surname" tagx="surname" title="surname">Carlson</span>
                     <span class="given-names" tagx="given-names" title="given-names">John E</span>
                  </span>
                  <span class="address" title="address">
                     <span class="email" tagx="email" title="email">jec16@psu.edu</span>
                  </span>
                  <a href="#Aff45"></a>
               </span>
               <span class="contrib" title="contrib">
                  <span class="name" tagx="name" title="name">
                     <span class="surname" tagx="surname" title="surname">Tien</span>
                     <span class="given-names" tagx="given-names" title="given-names">Ming</span>
                  </span>
                  <span class="address" title="address">
                     <span class="email" tagx="email" title="email">mxt3@psu.edu</span>
                  </span>
                  <a href="#Aff46"></a>
               </span>
               <span class="contrib" title="contrib">
                  <span class="name" tagx="name" title="name">
                     <span class="surname" tagx="surname" title="surname">McKenna</span>
                     <span class="given-names" tagx="given-names" title="given-names">Duane</span>
                  </span>
                  <span class="address" title="address">
                     <span class="email" tagx="email" title="email">dmckenna@memphis.edu</span>
                  </span>
                  <a href="#Aff47"></a>
               </span>
               <span class="contrib" title="contrib">
                  <span class="name" tagx="name" title="name">
                     <span class="surname" tagx="surname" title="surname">Hoover</span>
                     <span class="given-names" tagx="given-names" title="given-names">Kelli</span>
                  </span>
                  <span class="address" title="address">
                     <span class="email" tagx="email" title="email">kxh25@psu.edu</span>
                  </span>
                  <a href="#Aff48"></a>
               </span>
               <span class="citation_author_institution" id="Aff42">[],
                  	  
               </span>
               <span class="citation_author_institution" id="Aff43">[],
                  	  
               </span>
               <span class="citation_author_institution" id="Aff44">[],
                  	  
               </span>
               <span class="citation_author_institution" id="Aff45">[],
                  	  
               </span>
               <span class="citation_author_institution" id="Aff46">[],
                  	  
               </span>
               <span class="citation_author_institution" id="Aff47">[],
                  	  
               </span>
               <span class="citation_author_institution" id="Aff48">[],
                  	  
               </span>
            </div>
            <span class="pub-date-epub" title="pub-date-epub">epub: 
               <span>2014-12-12</span>
            </span>
            <span class="pub-date-collection" title="pub-date-collection">collection: 
               <span>2014</span>
            </span>
            <span class="volume" tagx="volume" title="volume">15</span>
            <span class="issue" tagx="issue" title="issue">1</span>
            <span class="elocation-id" tagx="elocation-id" title="elocation-id">1096</span>
            <span class="history" title="history">
               <span class="received" title="received">received: 2014-8-5</span>
               <span class="accepted" title="accepted">accepted: 2014-11-24</span>
            </span>
            <div class="permissions">
               <span class="copyright" title="copyright">(C) , </span>
               <span class="license" title="license">
                  <span class="license-p" title="license-p">This article is published under license to BioMed Central Ltd. This is an Open Access
                     article distributed under the terms of the Creative Commons Attribution License (
                     <a href="http://creativecommons.org/licenses/by/4.0">http://creativecommons.org/licenses/by/4.0</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided
                     the original work is properly credited. The Creative Commons Public Domain Dedication
                     waiver (
                     <a href="http://creativecommons.org/publicdomain/zero/1.0/">http://creativecommons.org/publicdomain/zero/1.0/</a>) applies to the data made available in this article, unless otherwise stated.
                  </span>
               </span>
            </div>
            <div class="abstract" title="abstract">
               <div class="abstract-title" title="abstract-title">Abstract</div>
               <div class="background" title="sec">
                  <div class="title" tagx="title" title="title">Background</div>
                  <p>Wood-feeding beetles harbor an ecologically rich and taxonomically diverse assemblage
                     of gut microbes that appear to promote survival in woody tissue, which is devoid of
                     nitrogen and essential nutrients. Nevertheless, the contributions of these apparent
                     symbionts to digestive physiology and nutritional ecology remain uncharacterized in
                     most beetle lineages.
                  </p>
               </div>
               <div class="results" title="sec">
                  <div class="title" tagx="title" title="title">Results</div>
                  <p>Through parallel transcriptome profiling of beetle- and microbial- derived mRNAs,
                     we demonstrate that the midgut microbiome of the Asian longhorned beetle (
                     <i>Anoplophora glabripennis</i>), a member of the beetle family Cerambycidae, is enriched in biosynthetic pathways
                     for the synthesis of essential amino acids, vitamins, and sterols. Consequently, the
                     midgut microbiome of 
                     <i>A. glabripennis</i> can provide essential nutrients that the beetle cannot obtain from its woody diet
                     or synthesize itself. The beetle gut microbiota also produce their own suite of transcripts
                     that can enhance lignin degradation, degrade hemicellulose, and ferment xylose and
                     wood sugars. An abundance of cellulases from several glycoside hydrolase families
                     are expressed endogenously by 
                     <i>A. glabripennis</i>, as well as transcripts that allow the beetle to convert microbe-synthesized essential
                     amino acids into non-essential amino acids
                     <i>. A. glabripennis</i> and its gut microbes likely collaborate to digest carbohydrates and convert released
                     sugars and amino acid intermediates into essential nutrients otherwise lacking from
                     their woody host plants.
                  </p>
               </div>
               <div class="conclusions" title="sec">
                  <div class="title" tagx="title" title="title">Conclusions</div>
                  <p>The nutritional provisioning capabilities of the 
                     <i>A. glabripennis</i> gut microbiome may contribute to the beetles’ unusually broad host range. The presence
                     of some of the same microbes in the guts of other Cerambycidae and other wood-feeding
                     beetles suggests that partnerships with microbes may be a facilitator of evolutionary
                     radiations in beetles, as in certain other groups of insects, allowing access to novel
                     food sources through enhanced nutritional provisioning.
                  </p>
               </div>
               <div class="electronicsupplementarymaterial" title="sec">
                  <div class="title" tagx="title" title="title">Electronic supplementary material</div>
                  <p>The online version of this article (doi:10.1186/1471-2164-15-1096) contains supplementary
                     material, which is available to authorized users.
                  </p>
               </div>
            </div>
            <div class="kwd-group">
               <div class="title" tagx="title" title="title">Keywords</div>
               <div class="kwd" title="kwd">Metatranscriptome</div>
               <div class="kwd" title="kwd">Cerambycidae</div>
               <div class="kwd" title="kwd">
                  <i>Fusarium solani</i>
               </div>
               <div class="kwd" title="kwd">Nutrient-provisioning</div>
               <div class="kwd" title="kwd">Amino acids</div>
               <div class="kwd" title="kwd">Nitrogen recycling</div>
               <div class="kwd" title="kwd">Xylose fermentation</div>
            </div>
            <div class="custom-meta-group" title="custom-meta-group">
               <meta name="issue-copyright-statement" value="© The Author(s) 2014"></meta>
            </div>
         </div>
      </div>
      <div class="body" title="body">
         <div class="background" title="sec">
            <div class="title" tagx="title" title="title">Background</div>
            <p>Insects are among the most ecologically versatile and taxonomically diverse groups
               of organisms on the planet, and their symbiotic interactions with microbes are pervasive
               phenomena [
               <a href="#CR1">1</a>]. Insect–microbe interactions are known to drive colonization of novel niches, promote
               survival under harsh environmental conditions [
               <a href="#CR2">2</a>], provide sources of exogenous genetic material [
               <a href="#CR3">3</a>], and can facilitate rapid niche expansion and adaptation [
               <a href="#CR4">4</a>]. The microbial partners of insects include archaea, fungi, protists, and more than
               10 phyla of bacteria [
               <a href="#CR5">5</a>–
               <a href="#CR7">7</a>]. The taxonomic composition and diversity of insect-associated microbial communities
               can vary tremendously, even between individuals of the same insect species [
               <a href="#CR8">8</a>]. Even so, the importance of symbionts to insect nutrition and digestive physiology
               has been demonstrated repeatedly. For example, symbionts have been shown to manipulate
               host plant physiology to modulate expression of defense-related genes [
               <a href="#CR9">9</a>], detoxify ingested plant defensive compounds [
               <a href="#CR10">10</a>], synthesize essential nutrients [
               <a href="#CR11">11</a>], fix atmospheric nitrogen, recycle nitrogenous waste products [
               <a href="#CR12">12</a>], promote host tolerance to extreme environmental conditions [
               <a href="#CR2">2</a>], digest plant cell wall components [
               <a href="#CR6">6</a>], and provide protection from pathogenic microbes [
               <a href="#CR13">13</a>]. Microbial communities associated with insects vary from relatively simple and static
               intracellular bacterial assemblages [
               <a href="#CR14">14</a>, 
               <a href="#CR15">15</a>] to highly complex and dynamic communities comprised of different taxonomic groups
               of microbes (e.g., fungi and bacteria) such as those associated with wood-feeding
               (xylophagous) beetles in the family Cerambycidae [
               <a href="#CR7">7</a>]. In these diverse and complex microbial systems, which are often dominated by facultative
               symbionts with high degrees of plasticity in terms of community richness and composition
               [
               <a href="#CR16">16</a>], it is difficult to determine if and how each of the various microbes directly enhance
               host fitness. Yet, evidence is mounting that the taxonomic diversity of endosymbionts
               may be directly correlated with host plant range [
               <a href="#CR17">17</a>, 
               <a href="#CR18">18</a>].
            </p>
            <p>Metagenomic approaches using next generation sequencing technologies have successfully
               elucidated the taxonomic composition and metabolic potentials of a variety of microbial
               communities associated with phytophagous insects [
               <a href="#CR6">6</a>, 
               <a href="#CR19">19</a>–
               <a href="#CR21">21</a>]. However, one common pitfall of such approaches is that DNA from dead, transient,
               or metabolically inactive microbes can be sequenced and included in gene annotations,
               making it difficult to discern between these and microbial taxa that may provide direct
               fitness benefits to the insect host [
               <a href="#CR22">22</a>]. Metatranscriptome and/or metaproteome analyses can alleviate these challenges in
               part, by focusing on microbial taxa and genes that are actively expressed in a particular
               community and thus potentially contribute to nutritional ecology.
            </p>
            <p>The Asian longhorned beetle (
               <i>Anoplophora glabripennis</i>) belongs to the subfamily Lamiinae in the beetle family Cerambycidae (e.g., cerambycids
               or longhorned beetles; &gt;30,000 described extant species) [
               <a href="#CR23">23</a>]. It has an unusually broad host range, attacking and reproducing in at least 47
               angiosperm tree species from at least 14 plant families in its native (Asia) and introduced
               ranges (Europe and North America) [
               <a href="#CR24">24</a>, 
               <a href="#CR25">25</a>]. Larvae feed in the sapwood and heartwood of host trees, requiring 1–2 years to
               complete development, while adult beetles feed on leaves and twigs. 
               <i>A. glabripennis</i> is considered to be among the 100 most threatening invasive plant or animal species
               worldwide [
               <a href="#CR26">26</a>]. Since the 1980s, 
               <i>A. glabripennis</i> has killed millions of trees in China and is responsible for significant losses of
               high-value shade and timber tree species where it has been introduced in the U.S.,
               Canada, and Europe, with a preference for maples (
               <i>Acer spp.</i>) in its introduced range [
               <a href="#CR24">24</a>]. Larvae of 
               <i>A. glabripennis</i> harbor a complex, diverse, and plastic gut microbiota [
               <a href="#CR17">17</a>, 
               <a href="#CR20">20</a>, 
               <a href="#CR27">27</a>]. Understanding the interactions between 
               <i>A. glabripennis</i> and its gut microbial community could lead to new and more effective control practices
               for this insect and for other xylophagous beetles, in addition to providing new insights
               into this beetle’s ecology (e.g., symbioses with microbes), evolution, and digestive
               physiology. For example, control of xylem-feeding insects by insecticides is ineffective
               [
               <a href="#CR28">28</a>]; as an alternative the gut microbes that enhance fitness by making key contributions
               to digestion and nutrient acquisition may be disrupted as a means of biological control
               as has been previously observed in other wood-feeding insects (e.g., Zootermopsis
               
               <i>angusticollis</i>) [
               <a href="#CR29">29</a>].
            </p>
            <p>The larval midgut transcriptome of 
               <i>A. glabripennis</i> is rich in insect-derived transcripts predicted to encode cellulases, xylanases,
               polygalacturonases, detoxification enzymes, and enzymes that could enhance degradation
               of lignin biopolymers [
               <a href="#CR20">20</a>, 
               <a href="#CR30">30</a>]. Even so, functional gaps exist in the beetle digestome, suggesting a role for collaboration
               with microbes in lignin degradation and the synthesis of essential nutrients [
               <a href="#CR31">31</a>]. The midgut lumen of 
               <i>A. glabripennis</i> is host to a diverse microbial community containing over 300 bacterial and at least
               18 fungal operational taxonomic units (OTUs) [
               <a href="#CR20">20</a>]. Many of the genes detected in the larval midgut metagenome [
               <a href="#CR31">31</a>] were derived from microbial taxa previously detected in 
               <i>A. glabripennis</i>[
               <a href="#CR17">17</a>, 
               <a href="#CR27">27</a>, 
               <a href="#CR32">32</a>, 
               <a href="#CR33">33</a>], suggesting that a subset of the midgut microbiome is consistently associated with
               larvae and that enzymes produced by these microbes may play integral roles in digestive
               physiology. For example, the 
               <i>A. glabripennis</i> midgut metagenome [
               <a href="#CR31">31</a>] contains genes capable of producing a full suite of plant cell wall-degrading enzymes,
               fixing and recycling nitrogen, synthesizing essential nutrients, detoxifying plant
               secondary metabolites, and degrading the dominant linkages in hardwood lignin, further
               demonstrating that the gut microbiome has the metabolic potential to fill functional
               gaps that exist in the beetle digestome, thereby facilitating survival in woody tissue
               [
               <a href="#CR20">20</a>]. Despite these potential contributions to the beetle’s digestive physiology and
               nutritional ecology, it is not known which gut microbial taxa are metabolically active.
               Furthermore, although previous studies have characterized the composition of bacterial
               and fungal communities of the 
               <i>A. glabripennis</i> midgut through culture-dependent and culture-independent approaches [
               <a href="#CR17">17</a>, 
               <a href="#CR27">27</a>, 
               <a href="#CR31">31</a>–
               <a href="#CR33">33</a>], biological replicates were not sampled and it is not known whether any of these
               microbes are found consistently in the midgut.
            </p>
            <p>The goals of this study were to overcome the limitations of the previous studies by
               documenting bacterial and fungal taxa that are consistently associated with the midgut
               of multiple 
               <i>A. glabripennis</i> larvae through 16S and ITS amplicon sequencing (“microbiome profiling”). These persistant
               microbes may have deep evolutionary relationships with 
               <i>A. glabripennis</i> and we hypothesize that these microbes are primed to serve key roles in digestion
               and nutrient provisioning. Additionally, metatranscriptome profiling of the midgut
               microbiome was performed to identify transcriptionally active microbes in the midgut
               and to characterize their metabolic potential. Using these methods, we are able to
               provide an overview of the microbial gene expression profile in actively-feeding larvae
               and detail the relative contributions of these microbes to digestive and physiological
               processes. We compared the functions of the genes expressed by the gut microbiota
               to those endogenously expressed by 
               <i>A. glabripennis</i> midgut in order to identify gaps in insect digestive physiology that can be complemented
               by genes expressed by gut symbionts. Taken together, this study reveals important
               new insights into 
               <i>A. glabripennis’</i> digestive physiology and nutritional biology, which will ultimately facilitate further
               studies of the ecology, physiological relevance and evolutionary history of symbioses
               between xylophagous beetles and microbes.
            </p>
         </div>
         <div class="resultsanddiscussion" title="sec">
            <div class="title" tagx="title" title="title">Results and discussion</div>
            <div class="midgutbacterialcommunitystructureinferredfrom16srdnaampliconsequencing" title="sec">
               <div class="title" tagx="title" title="title">Midgut bacterial community structure inferred from 16S rDNA amplicon sequencing</div>
               <p>Species richness varied slightly among individual larval midgut communities, ranging
                  from 82 to 198 operational taxonomic units (OTUs) at a 97% sequence similarity threshold.
                  Combined richness for all bacterial midgut communities sampled was 317 OTUs. Measures
                  of community complexity and diversity varied slightly among individuals (Table 
                  <a href="#Tab1">1</a>). Rarefaction curves computed for each individual midgut community failed to reach
                  saturation (Figure 
                  <a href="#Fig1">1</a>), suggesting that the full richness of the individual midgut communities was not
                  entirely sampled by sequencing. The midgut bacterial communities were predominately
                  composed of members of the following classes: Actinobacteria, Alphaproteobacteria,
                  Bacilli, Bacteroidetes, Clostridia, Gammaproteobacteria, and Sphingobacteria (Figure 
                  <a href="#Fig2">2</a>). The communities were generally dominated by OTUs assigned to class Gammaproteobacteria
                  and contained approximately equivalent relative abundances of OTUs classified to Bacilli
                  and Betaproteobacteria, while the relative abundances of Actinobacteria, Alphaproteobacteria,
                  Gammaproteobacteria, and Sphingobacteria differed slightly among individual beetles.
                  Twenty-two OTUs were detected in association with all midgut communities sampled (Additional
                  file 
                  <a href="#MOESM1">1</a>: Table S1), while two OTUs assigned to the family Enterobacteriaceae were highly
                  abundant in all midguts sampled for sequencing. These could not be definitively classified
                  to the genus level, but had highest scoring BLASTN alignments to bacteria in the genus
                  
                  <i>Enterobacter</i> in the RDP classifier database. Interestingly, unclassified members of the family
                  Enterobacteriaceae were consistently associated with the egg, oviposition site, and
                  
                  <i>A. glabripennis</i> larvae in a previous study and were flagged as candidates for vertical transmission,
                  suggesting an intimate association with the larval stage of this insect [
                  <a href="#CR34">34</a>].
                  <table-wrap>
                     <span class="label" tagx="label" title="label">Table 1</span>
                     <caption>
                        <p>
                           <b>Ecological indices for 16S bacterial communities sampled from the midguts of each
                              of 4 individual
                           </b>
                           <b>
                              <i>A. glabripennis</i>
                           </b>
                           <b>larvae feeding in sugar maple</b>
                        </p>
                     </caption>
                     <table>
                        <thead>
                           <tr>
                              <th>Sample</th>
                              <th>Number OTUs</th>
                              <th>Chao</th>
                              <th>Chao 95% CI</th>
                              <th>Ace</th>
                              <th>Ace 95% CI</th>
                              <th>Shannon</th>
                              <th>Shannon 95% CI</th>
                              <th>Simpson</th>
                              <th>Simpson 95% CI</th>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td>1</td>
                              <td>198</td>
                              <td>451</td>
                              <td>342–641</td>
                              <td>659</td>
                              <td>565–776</td>
                              <td>3.69</td>
                              <td>3.62–3.74</td>
                              <td>0.043</td>
                              <td>0.041–0.046</td>
                           </tr>
                           <tr>
                              <td>2</td>
                              <td>91</td>
                              <td>156</td>
                              <td>121–230</td>
                              <td>361</td>
                              <td>199–315</td>
                              <td>2.40</td>
                              <td>2.29–2.50</td>
                              <td>0.200</td>
                              <td>0.185–0.217</td>
                           </tr>
                           <tr>
                              <td>3</td>
                              <td>111</td>
                              <td>246</td>
                              <td>178–385</td>
                              <td>200</td>
                              <td>295–451</td>
                              <td>2.90</td>
                              <td>2.81–2.98</td>
                              <td>0.100</td>
                              <td>0.093–0.107</td>
                           </tr>
                           <tr>
                              <td>4</td>
                              <td>82</td>
                              <td>141</td>
                              <td>107–218</td>
                              <td>142</td>
                              <td>84–287</td>
                              <td>2.80</td>
                              <td>2.70–2.99</td>
                              <td>0.115</td>
                              <td>0.106–0.125</td>
                           </tr>
                        </tbody>
                     </table>
                  </table-wrap>
                  <div class="fig" title="fig">
                     <div class="figure" title="figure">
                        <a href="http://doi.org/">
                           <span class="label" tagx="label" title="label">Figure 1</span>
                        </a>
                     </div>
                     <caption class="caption" title="caption">
                        <caption>
                           <p>
                              <b>Rarefaction analysis of 16S amplicons sequenced from four individual third instar
                                 larvae of
                              </b>
                              <b>
                                 <i>A. glabripennis</i>
                              </b>
                              <b>feeding on sugar maple.</b> Curves failed to reach saturation, indicating that the community may harbor additional
                              OTUs not sampled for sequencing. This hypothesis is supported by 20 additional OTUs
                              detected by further sequencing of Sample 1 (dotted line).
                           </p>
                        </caption>
                     </caption>
                  </div>
                  <div class="fig" title="fig">
                     <div class="figure" title="figure">
                        <a href="http://doi.org/">
                           <span class="label" tagx="label" title="label">Figure 2</span>
                        </a>
                     </div>
                     <caption class="caption" title="caption">
                        <caption>
                           <p>
                              <b>Relative abundances of bacterial classes detected through 16S amplicon analysis of
                                 four individual third instar larvae of
                              </b>
                              <b>
                                 <i>A. glabripennis</i>
                              </b>
                              <b>.</b>
                           </p>
                        </caption>
                     </caption>
                  </div>
               </p>
               <p>The midgut microbial communities of larval 
                  <i>A. glabripennis</i> sampled herein were relatively distinct from those sampled in previous studies from
                  larval 
                  <i>A. glabripennis</i> collected in Worcester, MA (introduced range) [
                  <a href="#CR20">20</a>] or from China (native range) [
                  <a href="#CR17">17</a>]; instead the larvae sampled for this study more strongly resembled microbial communities
                  from 
                  <i>A. glabripennis</i> larvae collected in Brooklyn, NY [
                  <a href="#CR27">27</a>]. The bacterial communities of larval 
                  <i>A. glabripennis</i> from Worcester, MA and China were dominated by Bacilli [
                  <a href="#CR17">17</a>, 
                  <a href="#CR20">20</a>] while those sampled from New York were predominately assigned to class Gammaproteobacteria
                  [
                  <a href="#CR27">27</a>]. The differences in the taxonomic compositions of these communities are not surprising
                  since the Worcester, MA and New York populations were introduced separately and originated
                  from different source populations in Asia [
                  <a href="#CR35">35</a>]. Additionally, our 
                  <i>A. glabripennis</i> colony was derived primarily from the New York population and, therefore, the resemblance
                  of the midgut bacterial communities was expected.
               </p>
               <p>Because a different region of the 16S rRNA locus was targeted for sequencing in the
                  prior studies referenced in the above paragraph, direct OTU comparisons cannot be
                  performed. However, several genera were detected in both communities, including 
                  <i>Aeromicrobium</i>, 
                  <i>Aurantimonas, Agrococcus, Brachybacterium, Brevibacterium</i>, 
                  <i>Chrysobacterium, Curtobacterium</i>, 
                  <i>Enterococcus, Flavobacterium</i>, 
                  <i>Leucobacter</i>, 
                  <i>Microbacterium</i>, 
                  <i>Olivibacter</i>, 
                  <i>Pseudomonas</i>, 
                  <i>Pseudonocardia</i>, 
                  <i>Sphingobacterium</i>, 
                  <i>Staphylococcus, Stenotrophomonas</i>, and 
                  <i>Streptococcus</i>. Further, OTUs assigned to the genus 
                  <i>Pseudomonas</i> were highly abundant in all three populations (i.e., New York, Massachusetts, and
                  the Penn State University (PSU) colony).
               </p>
            </div>
            <div class="midgutfungalcommunitystructureinferredfromitsampliconsequencing" title="sec">
               <div class="title" tagx="title" title="title">Midgut fungal community structure inferred from ITS amplicon sequencing</div>
               <p>Observed richness values ranged from 15 to 28 OTUs, which is in agreement with the
                  fungal community richness detected previously through 18S amplicon sequencing [
                  <a href="#CR20">20</a>]. Total richness among all midgut fungal communities sampled was 44 OTUs (Table 
                  <a href="#Tab2">2</a>). Rarefaction curves approached saturation and computed values for various community
                  richness estimators were similar to the number of observed OTUs in communities from
                  each individual beetle midgut sampled, indicating adequate sampling (Figure 
                  <a href="#Fig3">3</a>). The Boneh estimator, which predicts the detection of additional OTUs with subsequent
                  sampling, projected detection of 1–10 additional OTUs, supporting the hypothesis that
                  the sequencing depth achieved for the fungal ITS communities was adequate to detect
                  the majority of the fungal OTUs associated with the 
                  <i>A. glabripennis</i> larval midgut.
                  <table-wrap>
                     <span class="label" tagx="label" title="label">Table 2</span>
                     <caption>
                        <p>
                           <b>Ecological indices for ITS fungal communities sampled from the midguts of each of
                              four individual third instar larvae of
                           </b>
                           <b>
                              <i>A. glabripennis</i>
                           </b>
                           <b>feeding in the heartwood of sugar maple (</b>
                           <b>
                              <i>Acer saccharum</i>
                           </b>
                           <b>)</b>
                        </p>
                     </caption>
                     <table>
                        <thead>
                           <tr>
                              <th>Sample</th>
                              <th>Number OTUs</th>
                              <th>Chao</th>
                              <th>Chao 95% CI</th>
                              <th>Ace</th>
                              <th>Ace 95% CI</th>
                              <th>Shannon</th>
                              <th>Shannon 95% CI</th>
                              <th>Simpson</th>
                              <th>Simpson 95% CI</th>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td>1</td>
                              <td>20</td>
                              <td>23</td>
                              <td>20–37</td>
                              <td>25</td>
                              <td>21–45</td>
                              <td>1.74</td>
                              <td>1.70–1.79</td>
                              <td>0.22</td>
                              <td>0.21–0.23</td>
                           </tr>
                           <tr>
                              <td>2</td>
                              <td>20</td>
                              <td>20</td>
                              <td>20–26</td>
                              <td>22</td>
                              <td>20–33</td>
                              <td>2.52</td>
                              <td>2.44–2.60</td>
                              <td>0.10</td>
                              <td>0.09–0.11</td>
                           </tr>
                           <tr>
                              <td>3</td>
                              <td>28</td>
                              <td>43</td>
                              <td>32–92</td>
                              <td>83</td>
                              <td>57–134</td>
                              <td>1.89</td>
                              <td>1.85–1.94</td>
                              <td>0.22</td>
                              <td>0.21–0.23</td>
                           </tr>
                           <tr>
                              <td>4</td>
                              <td>15</td>
                              <td>21</td>
                              <td>16–53</td>
                              <td>22</td>
                              <td>17–39</td>
                              <td>1.67</td>
                              <td>1.52–1.82</td>
                              <td>0.33</td>
                              <td>0.28–0.39</td>
                           </tr>
                        </tbody>
                     </table>
                  </table-wrap>
                  <div class="fig" title="fig">
                     <div class="figure" title="figure">
                        <a href="http://doi.org/">
                           <span class="label" tagx="label" title="label">Figure 3</span>
                        </a>
                     </div>
                     <caption class="caption" title="caption">
                        <caption>
                           <p>
                              <b>Rarefaction analysis of ITS fungal amplicons sequenced from the midguts of four individual
                                 third instar larvae of
                              </b>
                              <b>
                                 <i>A. glabripennis</i>
                              </b>
                              <b>.</b> Rarefaction curves appeared to reach saturation, indicating sufficient sampling to
                              detect the majority of the fungal community diversity, although deeper sampling of
                              samples 1 and 3 revealed the detection of 6 and 12 additional OTUs, respectively (dotted
                              line).
                           </p>
                        </caption>
                     </caption>
                  </div>
               </p>
               <p>The midgut fungal community was more stable than the bacterial community and contained
                  a greater percentage of OTUs that were detected in all beetle gut microbiota sampled
                  in this study. For example, gut communities were consistently dominated by ITS amplicons
                  assigned to the order Hypocreales (Figure 
                  <a href="#Fig4">4</a>). Seven OTUs, including six assigned to the genus 
                  <i>Fusarium</i> and a single OTU assigned to the genus 
                  <i>Pichia</i>, were associated with all 
                  <i>A. glabripennis</i> larval midguts sampled in this study. Fungi belonging to the 
                  <i>Fusarium solani</i> species complex 6 (FSSC6) have been consistently detected in the midgut of several
                  populations of 
                  <i>A. glabripennis</i> sampled for sequencing [
                  <a href="#CR33">33</a>]. Further, one of the OTUs detected in this analysis was 98% similar at the nucleotide
                  sequence level to a 
                  <i>Fusarium solani</i> OTU detected previously from beetles sampled from the PSU 
                  <i>A. glabripennis</i> colony [
                  <a href="#CR33">33</a>], indicating persistent maintenance of these OTUs over multiple generations, likely
                  through vertical transmission. Further study is necessary to elucidate whether the
                  consistent association of 
                  <i>F. solani</i> with 
                  <i>A. glabripennis</i> signifies the importance of this fungus to the beetle’s fitness and digestive physiology.
                  Phylogenetic analysis under maximum likelihood inference illustrates that this OTU
                  is more closely related to an isolate detected in the PSU colony than 
                  <i>Fusarium</i>-derived OTUs isolated from 
                  <i>A. glabripennis</i> midguts from the Worcester, MA population (Figure 
                  <a href="#Fig5">5</a>) [
                  <a href="#CR33">33</a>]. Six OTUs with highest scoring BLASTN alignments to 
                  <i>Fusarium oxysporum</i> were also detected through this analysis, which was detected previously in the 
                  <i>A. glabripennis</i> midgut (unpublished data)
                  <i>.</i>
                  <div class="fig" title="fig">
                     <div class="figure" title="figure">
                        <a href="http://doi.org/">
                           <span class="label" tagx="label" title="label">Figure 4</span>
                        </a>
                     </div>
                     <caption class="caption" title="caption">
                        <caption>
                           <p>
                              <b>Abundance of fungal orders detected in ITS amplicon data sampled from the midguts
                                 of four individual third instar larvae of
                              </b>
                              <b>
                                 <i>A. glabripennis</i>
                              </b>
                              <b>.</b> Fungal reads were exclusively classified to phylum Ascomycota. At the ordinal level,
                              the communities were dominated by Hypocreales and Saccharomycetales.
                           </p>
                        </caption>
                     </caption>
                  </div>
                  <div class="fig" title="fig">
                     <div class="figure" title="figure">
                        <a href="http://doi.org/">
                           <span class="label" tagx="label" title="label">Figure 5</span>
                        </a>
                     </div>
                     <caption class="caption" title="caption">
                        <caption>
                           <p>
                              <b>Maximum likelihood analysis of fungal ITS amplicon sequences taxonomically assigned
                                 to
                              </b>
                              <b>
                                 <i>Fusarium solani</i>
                              </b>
                              <b>that were detected in</b>
                              <b>
                                 <i>A. glabripennis</i>
                              </b>
                              <b>larval midguts.</b> The star designates the 
                              <i>F. solani</i> derived OTU that was detected in the current study and scale bars represent the number
                              of substitutions per site. PSU: 
                              <i>F. solani</i> isolates cultivated from 
                              <i>A. glabripennis</i> larvae reared at Penn State University; MA: represents 
                              <i>F. solani</i> cultivated from 
                              <i>A. glabripennis</i> larvae collected from a field site in Worcester, MA; NYC represents 
                              <i>F. solani</i> isolated from 
                              <i>A. glabripennis</i> larvae collected from a field site in Brooklyn, NY. MP designates the 
                              <i>F. solani</i> mating population number.
                           </p>
                        </caption>
                     </caption>
                  </div>
               </p>
            </div>
            <div class="assemblyandannotationmetrics" title="sec">
               <div class="title" tagx="title" title="title">Assembly and annotation metrics</div>
               <p>Over 65% of the Illumina paired end reads resulting from sequencing the midgut contents
                  library and over 95% of the reads from the intact midgut library passed quality filtering.
                  Using Trinity, 161,177 transcript isoforms from 97,506 genes ranging in length from
                  200 nt to 31,393 nt (N50 contig length: 684 nt) were assembled from the midgut contents
                  library, while 61,812 transcript isoforms from 45,418 genes ranging in length from
                  200 nt to 26,118 nt (N50 contig length: 592) were assembled from the intact midgut
                  library (Table 
                  <a href="#Tab3">3</a>).
                  <table-wrap>
                     <span class="label" tagx="label" title="label">Table 3</span>
                     <caption>
                        <p>
                           <b>Trinity assembly metrics for</b>
                           <b>
                              <i>A. glabripennis</i>
                           </b>
                           <b>transcripts obtained separately from midgut contents and from intact midgut transcriptome
                              libraries (see
                           </b> Methods for more information)
                        </p>
                     </caption>
                     <table>
                        <thead>
                           <tr>
                              <th>Source</th>
                              <th>Number of transcripts</th>
                              <th>Minimum transcript length (nt)</th>
                              <th>N80 transcript length (nt)</th>
                              <th>N50 transcript length (nt)</th>
                              <th>N20 transcript length (nt)</th>
                              <th>Maximum transcript length (nt)</th>
                              <th>Total length of assembled transcripts (nt)</th>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td>
                                 <b>Midgut contents</b>
                              </td>
                              <td>161,117</td>
                              <td>200</td>
                              <td>323</td>
                              <td>684</td>
                              <td>1,945</td>
                              <td>31,383</td>
                              <td>90.09 Mb</td>
                           </tr>
                           <tr>
                              <td>
                                 <b>Midgut contents: Microbial</b>
                              </td>
                              <td>
                                 <b>7,952</b>
                              </td>
                              <td>
                                 <b>200</b>
                              </td>
                              <td>
                                 <b>245</b>
                              </td>
                              <td>
                                 <b>334</b>
                              </td>
                              <td>
                                 <b>546</b>
                              </td>
                              <td>
                                 <b>5,049</b>
                              </td>
                              <td>
                                 <b>2.69 Mb</b>
                              </td>
                           </tr>
                           <tr>
                              <td>
                                 <b>Midgut contents: Bacterial</b>
                              </td>
                              <td>3,084</td>
                              <td>200</td>
                              <td>247</td>
                              <td>363</td>
                              <td>653</td>
                              <td>5,049</td>
                              <td>1.12 Mb</td>
                           </tr>
                           <tr>
                              <td>
                                 <b>Midgut contents: Fungal</b>
                              </td>
                              <td>4,868</td>
                              <td>200</td>
                              <td>242</td>
                              <td>316</td>
                              <td>484</td>
                              <td>2,259</td>
                              <td>1.58 Mb</td>
                           </tr>
                           <tr>
                              <td>
                                 <b>Intact midgut</b>
                              </td>
                              <td>61,812</td>
                              <td>200</td>
                              <td>272</td>
                              <td>592</td>
                              <td>1,937</td>
                              <td>26,118</td>
                              <td>30.6 Mb</td>
                           </tr>
                           <tr>
                              <td>
                                 <b>Intact midgut: Microbial</b>
                              </td>
                              <td>
                                 <b>3,167</b>
                              </td>
                              <td>
                                 <b>200</b>
                              </td>
                              <td>
                                 <b>266</b>
                              </td>
                              <td>
                                 <b>492</b>
                              </td>
                              <td>
                                 <b>1,271</b>
                              </td>
                              <td>
                                 <b>14,152</b>
                              </td>
                              <td>
                                 <b>1.42 Mb</b>
                              </td>
                           </tr>
                           <tr>
                              <td>
                                 <b>Intact midgut: Bacterial</b>
                              </td>
                              <td>2,154</td>
                              <td>200</td>
                              <td>294</td>
                              <td>548</td>
                              <td>1,445</td>
                              <td>14,152</td>
                              <td>1.05 Mb</td>
                           </tr>
                           <tr>
                              <td>
                                 <b>Intact midgut: Fungal</b>
                              </td>
                              <td>1,013</td>
                              <td>200</td>
                              <td>234</td>
                              <td>353</td>
                              <td>799</td>
                              <td>3,643</td>
                              <td>0.36 Mb</td>
                           </tr>
                        </tbody>
                     </table>
                  </table-wrap>
               </p>
               <p>Of the assembled transcripts, 7,952 (2.7 Mb) and 3,167 (1.42 Mb) were identified as
                  microbial from the midgut contents and intact midgut libraries, respectively, and
                  had highest scoring BLASTX alignments to microbial protein coding genes present in
                  the NCBI non-redundant protein database at an e-value of 1e-5 or lower, a minimum
                  alignment length of 15 amino acids, and at least a 60% amino acid similarity (Table 
                  <a href="#Tab4">4</a>). Relative expression levels of microbial-derived functional gene categories that
                  are relevant for survival in woody tissue are presented in Table 
                  <a href="#Tab5">5</a>. Transcripts in the midgut library were assigned to six bacterial and eight fungal
                  orders, while transcripts in the midgut contents library were assigned to 14 bacterial
                  and five fungal orders (Figure 
                  <a href="#Fig6">6</a>). In both libraries, Saccharomycetales was the dominant fungal order, while the bacterial
                  orders Enterobacteriales and Lactobacillales were dominant in the midgut contents
                  and intact midgut libraries, respectively (Additional file 
                  <a href="#MOESM2">2</a>: Figure S1).
                  <table-wrap>
                     <span class="label" tagx="label" title="label">Table 4</span>
                     <caption>
                        <p>
                           <b>Annotation statistics for microbial transcripts detected in the</b>
                           <b>
                              <i>A. glabripennis</i>
                           </b>
                           <b>midgut contents and intact midgut libraries</b>
                        </p>
                     </caption>
                     <table>
                        <thead>
                           <tr>
                              <th></th>
                              <th>Midgut contents</th>
                              <th>Intact midgut</th>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td>Number of rRNAs</td>
                              <td>182</td>
                              <td>237</td>
                           </tr>
                           <tr>
                              <td>Number of Transcripts with BLASTX Alignments</td>
                              <td>7,952</td>
                              <td>3,167</td>
                           </tr>
                           <tr>
                              <td>Number of Transcripts with Gene Ontology Assignments</td>
                              <td>1,083</td>
                              <td>1,225</td>
                           </tr>
                           <tr>
                              <td>Number of Transcripts with KEGG Assignments</td>
                              <td>705</td>
                              <td>1,103</td>
                           </tr>
                           <tr>
                              <td>Number of Transcripts with Pfam Assignments</td>
                              <td>1,686</td>
                              <td>2,766</td>
                           </tr>
                           <tr>
                              <td>Number of Transcripts with KOG/COG assignments</td>
                              <td>1,554</td>
                              <td>2,328</td>
                           </tr>
                        </tbody>
                     </table>
                  </table-wrap>
                  <table-wrap>
                     <span class="label" tagx="label" title="label">Table 5</span>
                     <caption>
                        <p>
                           <b>Number of transcripts per million mapped reads (TPM) for bacterial and fungal transcripts
                              assembled from the midgut contents and intact midgut libraries
                           </b>
                        </p>
                     </caption>
                     <table>
                        <thead>
                           <tr>
                              <th>Class</th>
                              <th>COG assignment</th>
                              <th>TPM midgut contents</th>
                              <th>TPM intact midgut</th>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td>Actinobacteria</td>
                              <td>Amino acid transport and metabolism</td>
                              <td>0</td>
                              <td>0.72</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Carbohydrate transport and metabolism</td>
                              <td>7.43</td>
                              <td>2.04</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Cell motility and secretion</td>
                              <td>0</td>
                              <td>1.08</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Coenzyme metabolism</td>
                              <td>0</td>
                              <td>1.28</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Energy production and conversion</td>
                              <td>0</td>
                              <td>2.24</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Function unknown</td>
                              <td>0</td>
                              <td>2.67</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Lipid metabolism</td>
                              <td>0</td>
                              <td>1.08</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Nucleotide transport and metabolism</td>
                              <td>0</td>
                              <td>2.01</td>
                           </tr>
                           <tr>
                              <td>Alphaproteobacteria</td>
                              <td>Amino acid transport and metabolism</td>
                              <td>3.28</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Cell envelope biogenesis, outer membrane</td>
                              <td>6.47</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Coenzyme metabolism</td>
                              <td>3.03</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Energy production and conversion</td>
                              <td>5.04</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>General function prediction only</td>
                              <td>10.17</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td>Bacilli</td>
                              <td>Amino acid transport and metabolism</td>
                              <td>0</td>
                              <td>326.88</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Carbohydrate transport and metabolism</td>
                              <td>558.13</td>
                              <td>492.9</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Cell envelope biogenesis, outer membrane</td>
                              <td>1585.16</td>
                              <td>334.75</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Cell motility and secretion</td>
                              <td>1569.88</td>
                              <td>41.89</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Coenzyme metabolism</td>
                              <td>0</td>
                              <td>157</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Energy production and conversion</td>
                              <td>0</td>
                              <td>232.11</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>General function prediction only</td>
                              <td>30.88</td>
                              <td>568.52</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Inorganic ion transport and metabolism</td>
                              <td>0</td>
                              <td>257.6</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Lipid metabolism</td>
                              <td>0</td>
                              <td>95.11</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Nucleotide transport and metabolism</td>
                              <td>0</td>
                              <td>214.12</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Secondary metabolites biosynthesis, transport, and catabolism</td>
                              <td>0</td>
                              <td>86.14</td>
                           </tr>
                           <tr>
                              <td>Bacteroidetes</td>
                              <td>Energy production and conversion</td>
                              <td>1.51</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Function unknown</td>
                              <td>3.47</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td>Betaproteobacteria</td>
                              <td>Coenzyme metabolism</td>
                              <td>1.94</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Cell envelope biogenesis, outer membrane</td>
                              <td>21.71</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td>Clostridia</td>
                              <td>Function unknown</td>
                              <td>342.46</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td>Gammaproteobacteria</td>
                              <td>Amino acid transport and metabolism</td>
                              <td>105.84</td>
                              <td>1.11</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Carbohydrate transport and metabolism</td>
                              <td>147.84</td>
                              <td>0.33</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Cell envelope biogenesis, outer membrane</td>
                              <td>304.28</td>
                              <td>1.96</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Cell motility and secretion</td>
                              <td>111.13</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Coenzyme metabolism</td>
                              <td>49.54</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Defense mechanisms</td>
                              <td>20.49</td>
                              <td>1.36</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Energy production and conversion</td>
                              <td>168.14</td>
                              <td>1.17</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Function unknown</td>
                              <td>189.35</td>
                              <td>0.24</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>General function prediction only</td>
                              <td>187.36</td>
                              <td>1.94</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Inorganic ion transport and metabolism</td>
                              <td>185.12</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Lipid metabolism</td>
                              <td>15.69</td>
                              <td>0.23</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Nucleotide transport and metabolism</td>
                              <td>81.51</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td>Saccharomycetes</td>
                              <td>Amino acid transport and metabolism</td>
                              <td>172.97</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Carbohydrate transport and metabolism</td>
                              <td>106.88</td>
                              <td>0.33</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Cell motility</td>
                              <td>5.21</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Cell wall/membrane/envelope biogenesis</td>
                              <td>107.45</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Chromatin structure and dynamics</td>
                              <td>32.59</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Coenzyme transport and metabolism</td>
                              <td>17.03</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Defense mechanisms</td>
                              <td>98.79</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Energy production and conversion</td>
                              <td>64.95</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Extracellular structures</td>
                              <td>93.48</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Function unknown</td>
                              <td>156.44</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>General function prediction only</td>
                              <td>348.77</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Inorganic ion transport and metabolism</td>
                              <td>193.37</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Lipid transport and metabolism</td>
                              <td>85.25</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Nucleotide transport and metabolism</td>
                              <td>34.03</td>
                              <td>0.96</td>
                           </tr>
                           <tr>
                              <td>Sordariomycetes</td>
                              <td>Amino acid transport and metabolism</td>
                              <td>10.10</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Carbohydrate transport and metabolism</td>
                              <td>7.40</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Cell wall/membrane/envelope biogenesis</td>
                              <td>1.47</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Chromatin structure and dynamics</td>
                              <td>5.63</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Coenzyme transport and metabolism</td>
                              <td>1.02</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Energy production and conversion</td>
                              <td>8.14</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Function unknown</td>
                              <td>0.56</td>
                              <td>2.26</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>General function prediction only</td>
                              <td>14.59</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Inorganic ion transport and metabolism</td>
                              <td>8.62</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Lipid transport and metabolism</td>
                              <td>1.47</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Nucleotide transport and metabolism</td>
                              <td>3.35</td>
                              <td>0</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>Secondary metabolites biosynthesis, transport and catabolism</td>
                              <td>1.65</td>
                              <td>0</td>
                           </tr>
                        </tbody>
                     </table>
                     <table-wrap-foot>
                        <p>Class-level taxonomic assignments and COG (Clusters of Orthologous Genes) functional
                           assignments are indicated.
                        </p>
                     </table-wrap-foot>
                  </table-wrap>
                  <div class="fig" title="fig">
                     <div class="figure" title="figure">
                        <a href="http://doi.org/">
                           <span class="label" tagx="label" title="label">Figure 6</span>
                        </a>
                     </div>
                     <caption class="caption" title="caption">
                        <caption>
                           <p>
                              <b>Bacterial and fungal classes detected in</b>
                              <b>
                                 <i>A. glabripennis</i>
                              </b>
                              <b>larval intact midgut and midgut contents metatranscriptome assemblies through MEGAN
                                 analysis of putative protein coding genes.
                              </b>
                           </p>
                        </caption>
                     </caption>
                  </div>
               </p>
            </div>
            <div class="bacterial16sandfungalitsotusdetectedinmetatranscriptomedata" title="sec">
               <div class="title" tagx="title" title="title">Bacterial 16S and fungal ITS OTUs detected in metatranscriptome data</div>
               <p>Over 500 transcripts were classified as SSU (small ribosomal subunit) or LSU (large
                  ribosomal subunit) rRNAs (Additional file 
                  <a href="#MOESM3">3</a>: Table S2). The expression of several persistent OTUs in all beetle midgut communities
                  sampled for sequencing was confirmed. For example, rRNAs with highest scoring BLASTN
                  alignments to eight shared OTUs were detected and included OTUs classified to the
                  genera 
                  <i>Novosphingobium</i>, 
                  <i>Propionibacterium</i>, 
                  <i>Pseudomonas</i>, 
                  <i>Pediococcus</i>, and 
                  <i>Staphylococcus</i>; the families Burkholderiaceae and Enterobacteriaceae (2 OTUs); and the order Actinomycetales.
                  However, the majority of the identified rRNAs were from OTUs detected in association
                  with the midgut communities of just one or two of the larvae sampled (Table 
                  <a href="#Tab6">6</a>). Interestingly, the majority of the 16S rRNA transcripts constructed from the Illumina
                  reads did not have significant BLASTN matches to OTUs documented through community
                  profiling. In some cases, the transcripts originated from a different region of the
                  16S rRNA locus than the region targeted for amplicon sequencing and therefore would
                  not be expected to align with the OTUs detected in the amplicon analysis. However,
                  in many cases, the transcripts overlapped the region targeted for amplicon analysis,
                  but had low sequence identities to the 16S OTUs. This could be a reflection of the
                  dynamic nature of the midgut community, as has been observed for the gut communities
                  of other xylophagous insects [
                  <a href="#CR36">36</a>], or it could be an artifact of feeding on a non-sterile food source where DNA from
                  microbes ingested during feeding is more abundant than DNA from microbes that are
                  truly residents of the midgut. Consistent with the utility of targeting expressed
                  rRNAs for symbiont community analysis, the taxonomic classification of the rRNAs that
                  were assembled from the transcriptome libraries most strongly resembled the taxonomic
                  assignments obtained from MEGAN analysis of the microbial protein coding genes.
                  <table-wrap>
                     <span class="label" tagx="label" title="label">Table 6</span>
                     <caption>
                        <p>
                           <b>Taxonomic identity of 16S OTUs supported by rRNAs assembled from metatranscriptome
                              data for intact midgut and midgut contents
                           </b>
                        </p>
                     </caption>
                     <table>
                        <thead>
                           <tr>
                              <th>Classification to lowest possible taxonomic rank</th>
                              <th>Percent nucleotide identity</th>
                              <th>Number of assembled rRNAs</th>
                              <th>Persistant?</th>
                           </tr>
                        </thead>
                        <tbody>
                           <tr>
                              <td>Acinetobacter</td>
                              <td>97</td>
                              <td>1</td>
                              <td>No</td>
                           </tr>
                           <tr>
                              <td>Actinomyces</td>
                              <td>100</td>
                              <td>1</td>
                              <td>No</td>
                           </tr>
                           <tr>
                              <td>Actinomycetales</td>
                              <td>95</td>
                              <td>2</td>
                              <td>No</td>
                           </tr>
                           <tr>
                              <td>Actinomycetales</td>
                              <td>96</td>
                              <td>2</td>
                              <td>Yes</td>
                           </tr>
                           <tr>
                              <td>Actinomycetales</td>
                              <td>96</td>
                              <td>1</td>
                              <td>No</td>
                           </tr>
                           <tr>
                              <td>Bacteria</td>
                              <td>96</td>
                              <td>1</td>
                              <td>No</td>
                           </tr>
                           <tr>
                              <td>Burkholderiaceae</td>
                              <td>97</td>
                              <td>1</td>
                              <td>Yes</td>
                           </tr>
                           <tr>
                              <td>Caryophanon</td>
                              <td>100</td>
                              <td>1</td>
                              <td>No</td>
                           </tr>
                           <tr>
                              <td>Cellovibrio</td>
                              <td>98</td>
                              <td>2</td>
                              <td>No</td>
                           </tr>
                           <tr>
                              <td>Curtobacterium</td>
                              <td>97</td>
                              <td>1</td>
                              <td>No</td>
                           </tr>
                           <tr>
                              <td>Enterobacteriaceae</td>
                              <td>99</td>
                              <td>2</td>
                              <td>No</td>
                           </tr>
                           <tr>
                              <td>Enterobacteriaceae</td>
                              <td>97</td>
                              <td>3</td>
                              <td>No</td>
                           </tr>
                           <tr>
                              <td>Enterobacteriaceae</td>
                              <td>96</td>
                              <td>4</td>
                              <td>No</td>
                           </tr>
                           <tr>
                              <td>Enterobacteriaceae</td>
                              <td>98</td>
                              <td>1</td>
                              <td>Yes</td>
                           </tr>
                           <tr>
                              <td>Enterobacteriaceae</td>
                              <td>95</td>
                              <td>1</td>
                              <td>Yes</td>
                           </tr>
                           <tr>
                              <td>Novosphingobium</td>
                              <td>96</td>
                              <td>1</td>
                              <td>Yes</td>
                           </tr>
                           <tr>
                              <td>Pasteurellaceae</td>
                              <td>95</td>
                              <td>1</td>
                              <td>No</td>
                           </tr>
                           <tr>
                              <td>Pediococcus</td>
                              <td>95</td>
                              <td>1</td>
                              <td>Yes</td>
                           </tr>
                           <tr>
                              <td>Propionibacterium</td>
                              <td>99</td>
                              <td>1</td>
                              <td>Yes</td>
                           </tr>
                           <tr>
                              <td>Pseudomonadaceae</td>
                              <td>100</td>
                              <td>1</td>
                              <td>No</td>
                           </tr>
                           <tr>
                              <td>Pseudomonadaceae</td>
                              <td>97</td>
                              <td>1</td>
                              <td>No</td>
                           </tr>
                           <tr>
                              <td>Pseudomonas</td>
                              <td>97</td>
                              <td>1</td>
                              <td>Yes</td>
                           </tr>
                           <tr>
                              <td>Sphingobacterium</td>
                              <td>98</td>
                              <td>1</td>
                              <td>No</td>
                           </tr>
                           <tr>
                              <td>Staphylococcus</td>
                              <td>99</td>
                              <td>1</td>
                              <td>Yes</td>
                           </tr>
                           <tr>
                              <td>Streptococcus</td>
                              <td>98</td>
                              <td>1</td>
                              <td>No</td>
                           </tr>
                        </tbody>
                     </table>
                     <table-wrap-foot>
                        <p>Ribosomal RNAs assembled from the intact midgut and midgut contents metatranscriptome
                           data were compared to the OTUs detected in the 16S amplicon data using BLASTN to determine
                           which OTUs were transcriptionally active in the midgut. 16S OTUs were considered persistant
                           if they were detected in ≥ 3 
                           <i>A. glabripennis</i> midgut communities in the current study.
                        </p>
                     </table-wrap-foot>
                  </table-wrap>
               </p>
               <p>Although fungal ITS sequences were not numerous in the metatranscriptome data (the
                  ITS region is generally excised from mature rRNAs) they can be present in pre-ribosomal
                  RNAs. Five complete ITS transcripts were identified in both the intact midgut and
                  midgut contents. Two of these transcripts had 100% nucleotide identity to persistent
                  OTUs detected in all midgut communities and both were classified to the genus 
                  <i>Pichia</i>. While transcriptional activities of other fungal OTUs could not be confirmed, analysis
                  of SSUs and LSUs detected in the metatranscriptome data confirmed that many of the
                  persistent fungal OTUs identified through ITS community profiling were transcriptionally
                  active (e.g., 
                  <i>Fusarium</i> and 
                  <i>Candida</i>).
               </p>
            </div>
            <div class="metaboliccapacityofthea.glabripennismidgutmicrobiome" title="sec">
               <div class="title" tagx="title" title="title">Metabolic capacity of the 
                  <i>A. glabripennis</i>midgut microbiome
               </div>
               <p>The midgut microbiome comprised a much broader repertoire of genes with predicted
                  involvement in carbohydrate digestion and sugar assimilation relative to the 
                  <i>A. glabripennis</i> midgut transcriptome (Additional file 
                  <a href="#MOESM4">4</a>: Table S3). Many of these microbial genes could potentially complement 
                  <i>A. glabripennis’</i> existing repertiore of digestive and nutrient assimilating capacities. This capacity
                  was noted particularly with regard to utilization of hemicellulosic sugars. For example,
                  insect-derived xylanases, which are hypothesized to release xylose sugars from the
                  xylan chains that predominantly comprise hemicellulose found in angiospermous trees,
                  have been documented in the 
                  <i>A. glabripennis</i> midgut through both transcriptome and zymogram analyses [
                  <a href="#CR31">31</a>, 
                  <a href="#CR37">37</a>]. However, pathways for xylose sugar metabolism and utilization were not detected
                  in the 
                  <i>A. glabripennis</i> transcriptome [
                  <a href="#CR31">31</a>] and have not been identified in any other insect. The microbial community of 
                  <i>A. glabripennis</i> actively expressed genes with predicted roles in the assimilation and metabolism
                  of xylose sugars, which could be used for energy production or as substrates for various
                  biosynthetic pathways (i.e., carbon skeletons of amino acids or fatty acids). One
                  of the most striking discoveries was the expression of xylose sugar utilization pathways
                  by the fungal microbiota via the oxido-reductase pathway, which is a pathway used
                  to convert xylose to D-xylulose-5-phosphate that can then enter the pentose phosphate
                  pathway (Figure 
                  <a href="#Fig7">7</a>). In addition, we found lactic acid bacterial-derived transcripts (e.g. 
                  <i>Pediococcus</i>) from the xylose isomerase pathway that are capable of converting xylose directly
                  into D-xylulose. It has been previously hypothesized that xylose-fermenting yeasts
                  commonly found in the guts of cerambycid beetles play roles in processing xylose [
                  <a href="#CR38">38</a>, 
                  <a href="#CR39">39</a>], converting it into metabolites that could be used by the host or the microbiota
                  in various biosynthetic processes. The expression of yeast-derived oxido-reductase
                  and bacterial-derived xylose isomerase pathways in 
                  <i>A. glabripennis</i> supports a role for the gut microbiota in xylose sugar utilization. Whether these
                  microbes are simply exploiting xylose sugars released from hemicellulose by beetle-derived
                  xylanases or are actually working “in cooperation” with 
                  <i>A. glabripennis</i> to digest hemicellulose and utilize its metabolites is not known.
                  <div class="fig" title="fig">
                     <div class="figure" title="figure">
                        <a href="http://doi.org/">
                           <span class="label" tagx="label" title="label">Figure 7</span>
                        </a>
                     </div>
                     <caption class="caption" title="caption">
                        <caption>
                           <p>
                              <b>Putative pathways for xylose utilization based on BLASTX annotation of transcripts
                                 sampled from the midgut microbiota of larval
                              </b>
                              <b>
                                 <i>A. glabripennis</i>
                              </b>
                              <b>.</b> Xylose can be shuttled into the pentose phosphate pathway by two different routes
                              detected in the metatranscriptome: the oxoreductive pathway and the isomerase pathway.
                              Transcripts originating from the oxoreductive pathway were from yeasts while transcripts
                              originating from the isomerase pathway were from lactic acid bacteria. Both pathways
                              lead to the production of xylulose-5-phosphate, which is shuttled into the pentose
                              phosphate pathway and is used to produce pyruvate and shikimate. These compounds serve
                              as key intermediates in the synthesis of essential branched chain amino acids and
                              essential aromatic amino acids, demonstrating how these wood sugars can be used by
                              the gut microbiota to produce essential nutrients that are otherwise lacking from
                              the 
                              <i>A. glabripennis</i> diet.
                           </p>
                        </caption>
                     </caption>
                  </div>
               </p>
               <p>Fungal transcripts predicted to encode glycoside hydrolase family 28 (GH 28) polygalacturonases
                  and GH 5 cellulases were also expressed in the 
                  <i>A. glabripennis</i> midgut, indicating that the fungal community is capable of degrading cellulose and
                  other plant polysaccharides in woody tissue (e.g., pectin). Fungal and bacterial transcripts
                  predicted to encode ABC transporters, β-glucosidases, β-xylosidases, major facilitator
                  superfamily transporters, and phosphotransferase systems, were detected, suggesting
                  that microbes are also capable of assimilating and metabolizing sugars released from
                  secondary cell wall polysaccharides. Of particular interest are the transcripts predicted
                  to encode components of phosphotransferase systems since many of these can internalize
                  and metabolize cellobiose, xylose, mannose, galactose, and other sugars released from
                  woody tissue by beetle-derived cellulases and hemicellulases. While 
                  <i>A. glabripennis</i> can endogenously metabolize and use many of the hexose sugar substrates for energy
                  production, members of the microbiota expressed transcripts that could ferment both
                  5- and 6- carbon sugars, allowing hexoses and pentoses liberated from the plant cell
                  wall to be converted into substrates that can be used in various anabolic processes,
                  i.e., amino acid or fatty acid biosynthesis.
               </p>
               <p>Based on our data, the gut microbial community likely benefits 
                  <i>A. glabripennis</i> by 
                  <i>i)</i> increasing it’s capacity to utilize 5-carbon sugars and 
                  <i>ii)</i> by providing biosynthetic pathways for essential biosynthetic precursors (Figure 
                  <a href="#Fig8">8</a>). For example, while the beetle can metabolize glucose and fructose into pyruvate,
                  the gut microbiota can convert many other sugar substrates, including pentose sugars
                  found in hemicellulose from angiospermous trees into this key metabolic intermediate.
                  This is a potential mechanism for the conversion of glucose and xylose released from
                  plant cell walls by beetle digestive enzymes into essential nutrients by the gut community.
                  Microbial pathways for the conversion of pyruvate into essential branched chain amino
                  acids (valine/leucine/isoleucine) and homocitrate, which is a valuable substrate for
                  lysine biosynthesis (another essential amino acid), were expressed in the midgut (Figure 
                  <a href="#Fig9">9</a>). Although additional experimental evidence is needed to validate the microbial conversion
                  of wood sugars into essential nutrients, biochemical evidence using 
                  <sup>13</sup>C labeled carbon suggests a microbial origin of the carbon backbones of several essential
                  amino acids in the 
                  <i>A. glabripennis</i> midgut (Ayayee, in review).
                  <div class="fig" title="fig">
                     <div class="figure" title="figure">
                        <a href="http://doi.org/">
                           <span class="label" tagx="label" title="label">Figure 8</span>
                        </a>
                     </div>
                     <caption class="caption" title="caption">
                        <caption>
                           <p>
                              <b>Pathways for pyruvate utilization detected in the</b>
                              <b>
                                 <i>A. glabripennis</i>
                              </b>
                              <b>larval midgut microbiome.</b> The gut microbial community has an expanded capacity to synthesize pyruvate from
                              pentose sugars found in hemicellulose and convert pyruvate to essential branched-chain
                              amino acids and homocitrate, an essential component of the lysine biosynthetic pathway.
                           </p>
                        </caption>
                     </caption>
                  </div>
                  <div class="fig" title="fig">
                     <div class="figure" title="figure">
                        <a href="http://doi.org/">
                           <span class="label" tagx="label" title="label">Figure 9</span>
                        </a>
                     </div>
                     <caption class="caption" title="caption">
                        <caption>
                           <p>
                              <b>Partial pathways for aromatic amino acid biosynthesis detected in the</b>
                              <b>
                                 <i>A. glabripennis</i>
                              </b>
                              <b>larval midgut microbiome.</b> In some cases, pathways encoded by the gut microbiota can be complemented by transcripts
                              derived from 
                              <i>A. glabripennis</i>. For example, gut microbes encoded full pathways for the biosynthesis of the essential
                              aromatic amino acid phenylalanine, while 
                              <i>A. glabripennis</i> produces the enzymes necessary to convert phenylalanine derived from microbes into
                              tyrosine.
                           </p>
                        </caption>
                     </caption>
                  </div>
               </p>
               <p>Aside from its potential utility in the biosynthesis of essential and nonessential
                  amino acids, pyruvate can also be used to produce substrates for fatty acid biosynthesis.
                  For example, the gut microbiota can synthesize acetyl CoA and malonyl CoA from pyruvate,
                  which are required for biosynthesis of fatty acids and are scarce in woody tissue,
                  via a transcriptionally active pentose phosphate pathway capable of converting five-carbon
                  sugars (e.g. xylose and arabinose) into intermediates that can be used in biosynthesis
                  of essential aromatic amino acids by the shikimate pathway. Full pathways for phenylalanine
                  biosynthesis and partial pathways for histidine acid biosynthesis were detected in
                  the microbial-derived transcripts. Although we did not find pathways in the gut involved
                  in biosynthesis of tryptophan in either the metatranscriptome or the larval transcriptome,
                  
                  <i>A. glabripennis</i> produces transcripts predicted to encode enzymes necessary to convert phenylalanine
                  to tyrosine. The microbial origin several of essential aromatic and branched chain
                  amino acids in 
                  <i>A. glabripennis</i> was validated by 
                  <sup>15</sup> N enrichment experiments [
                  <a href="#CR12">12</a>]. Although the beetle is capable of synthesizing several non-essential amino acids
                  from pyruvate [
                  <a href="#CR31">31</a>], the microbiota also expressed components of pathways for the synthesis of several
                  non-essential amino acids, such as asparagine, glutamate, glutamine, and arginine.
                  The synthesis of non-essential amino acids by gut microbes may contribute to nitrogen
                  conservation in 
                  <i>A. glabripennis.</i>
               </p>
               <p>The gut microbiota can also make contributions to the nitrogen economy in the midgut
                  by recycling nitrogenous waste products produced either by the beetle or gut microbes,
                  which is consistent with previous work showing that nitrogen recycled from urea is
                  reincorporated into both essential and non-essential amino acids in larval 
                  <i>A. glabripennis</i>
                  [
                  <a href="#CR12">12</a>]. However, alternate pathways for recycling nitrogenous waste products were also
                  detected. For example, transcripts predicted to encode NAD-specific glutamate dehydrogenases,
                  taurine dehydrogenases, and glutamine synthetases were expressed by various members
                  of the gut community. Glutamate dehydrogenase typically converts glutamate to α-ketogluturate
                  for excretion; however, it can also catalyze the reverse reaction under ammonia-rich
                  conditions, recycling α-ketoglutarate waste products back into glutamate. Glutamine
                  synthetases can also have integral roles in nitrogen economy by incorporating ammonia
                  liberated from amino acid and nucleic acid deamination reactions back into glutamine.
                  Taurine dioxygenases may also contribute to nitrogen economy in the gut because these
                  enzymes can actively break down the organic acid taurine, which is one of the most
                  prominent free organic acids in both insect hemolymph and tissues [
                  <a href="#CR40">40</a>]. Taurine dioxygenases catalyze the conversion of taurine to sulfite, aminoacetaldehyde,
                  carbon dioxide, and succinate, allowing it to be used as a source of both nitrogen
                  and sulfur. Finally, transcripts containing xanthine uracil permease domains and phosphoribosyltransferase
                  domains were found, which participate in the purine salvage pathway and signify a
                  potential microbial role for the recycling of purine nucleotides.
               </p>
            </div>
            <div class="potentialopportunitiesfornutrientexchangebetweenmicrobesanda.glabripennis" title="sec">
               <div class="title" tagx="title" title="title">Potential opportunities for nutrient exchange between microbes and 
                  <i>A. glabripennis</i>
               </div>
               <p>We found numerous opportunities for nutrient and metabolite exchange between members
                  of the midgut microbiota and 
                  <i>A. glabripennis,</i> which could contribute to niche expansion by 
                  <i>A. glabripennis,</i> represented by its ability to colonize and thrive in an unusually broad range of
                  healthy host trees. For example, potential pathway complementarity was noted with
                  regard to carbohydrate digestion. 
                  <i>A. glabripennis</i> produces a large number of transcripts predicted to break down plant cell wall carbohydrates
                  [
                  <a href="#CR31">31</a>, 
                  <a href="#CR37">37</a>], while in contrast, the gut community expressed numerous transcripts predicted to
                  assimilate and ferment wood sugars (i.e., glucose and xylose). 
                  <i>A. glabripennis</i> also has the capacity to assimilate and use fermentation products via a diverse repertoire
                  of insect-derived alcohol dehydrogenases [
                  <a href="#CR31">31</a>].
               </p>
               <p>Complementarity was also noted with regard to amino acid and vitamin metabolism. The
                  microbiota expressed full pathways for the biosynthesis of branched chain and aromatic
                  amino acids, while the beetle expressed full pathways for the decomposition of branched-chain
                  amino acids, aromatic amino acids, and lysine [
                  <a href="#CR31">31</a>], thus providing a direct example of how 
                  <i>A. glabripennnis</i> can directly catabolize nutrients synthesized by the gut microbial community. While
                  the gut microbiota express transcripts involved in biosynthesis of methionine and
                  vitamins, 
                  <i>A. glabripennis</i> itself expressed transcripts with predicted roles in recycling these essential nutrients.
                  Further, the microbiome may augment production of non-essential amino acids, which
                  may be important for nitrogen conservation in the nutrient poor environment of wood.
                  For example, although 
                  <i>A. glabripennis</i> can synthesize cysteine, [
                  <a href="#CR31">31</a>] the low abundance of sulfur in woody tissues may make cysteine a conditionally essential
                  amino acid in this system. Therefore, the role of recycled sulfur obtained from taurine
                  or other insect-derived compounds could also be critical for the synthesis of cysteine,
                  methionine, and other sulfur-containing essential nutrients. The 
                  <i>A. glabripennis</i> transcriptome contains a larger repertoire of expressed peptidases and amino acid
                  ligases relative to transcriptomes compiled from the guts of other phytophagous insects
                  [
                  <a href="#CR31">31</a>]. While the source of nitrogen used by larvae as they feed in the high C:N heartwood
                  of their host trees is uncertain, this analysis suggests that the microbiota can provide
                  a suitable store of amino acids and proteins that could be assimilated by 
                  <i>A. glabripennis</i> and its gut microbes.
               </p>
            </div>
            <div class="potentialcontributionsofmicrobialotustoa.glabripennisdigestivephysiology" title="sec">
               <div class="title" tagx="title" title="title">Potential contributions of microbial OTUs to 
                  <i>A. glabripennis</i>digestive physiology
               </div>
               <p>In all beetle gut communities in this study we found rRNAs from several bacterial
                  and fungal taxa that were taxonomically classified to the genera Fusarium, Pichia,
                  and Pediococcus. Although these taxa all expressed transcripts that could serve key
                  roles in the digestive and nutritional physiology of A. glabripennis, some of the
                  potential contributions to metabolism were partitioned differently among taxa. For
                  example, all three taxa expressed genes with predicted involvement in plant cell wall
                  digestion (i.e. cellulases, β-glucosidases, β-xylosidases), nitrogen uptake and recycling,
                  and detoxification. Transcripts with predicted involvement in xylose utilization were
                  taxonomically assigned to Pediococcus and Pichia. However, several functions were
                  differentially partitioned among these taxa. For example, the lactic acid bacterium
                  Pediococcus had predicted roles in xylose utilization, sugar fermentation, aromatic
                  amino acid biosynthesis, and vitamin biosynthesis, while genes involved in sterol
                  biosynthesis were exclusively assigned to the fungal taxa.
               </p>
               <div class="potentialnutritionprovisioningrolesoffusariumsolani" title="sec">
                  <div class="title" tagx="title" title="title">Potential nutrition provisioning roles of Fusarium solani</div>
                  <p>OTUs taxonomically classified to the genus 
                     <i>Fusarium</i> are consistently found in the 
                     <i>A. glabripennis</i> larval midgut [
                     <a href="#CR20">20</a>, 
                     <a href="#CR33">33</a>] and are capable of secreting numerous plant cell wall degrading enzymes, detoxification
                     enzymes, and laccases with putative involvement in lignin degradation [
                     <a href="#CR32">32</a>]. Over 200 transcripts assigned to the genera 
                     <i>Fusarium</i> or 
                     <i>Nectria</i> (Family Nectriaceae) were detected in the gut contents library and many of these
                     transcripts were classified as glycoside hydrolases (GH) based on Pfam domain annotations.
                     One of the most striking findings was the presence of over 30 transcripts predicted
                     to originate from GH 28 polygalacturonases, which have predicted involvement in degradation
                     of pectin. While pectin is not prevalent in woody tissue, it is periodically deposited
                     in secondary growth and represents stores of galactose sugars and essential minerals
                     (e.g., calcium) [
                     <a href="#CR41">41</a>] that could be assimilated by the beetle or gut fungi. A single transcript predicted
                     to encode a GH 5 cellulase was also detected. While its ability to catalyze endo-
                     or exo- type glucanase reactions could not be predicted based on 
                     <i>in silico</i> annotations alone, it could work in collaboration with beetle-derived cellulases
                     to facilitate cell wall digestion [
                     <a href="#CR20">20</a>, 
                     <a href="#CR27">27</a>]. In addition, it was previously hypothesized that 
                     <i>F. solani</i> could play a key role in lignin degradation in the midgut [
                     <a href="#CR42">42</a>]; several secreted laccases were detected previously through MudPIT shotgun proteomic
                     analysis [
                     <a href="#CR32">32</a>]. Although no known lignin peroxidase transcripts were detected in this analysis,
                     
                     <i>Fusarium</i>-derived transcripts with predicted involvement in metabolizing aromatic compounds
                     were detected that could play accessory roles in processing the lignin biopolymer.
                     For example, several transcripts were detected that are predicted to encode phenolic
                     acid decarboxylases, aromatic ring hydroxylases, cytochrome P450s, and alcohol and
                     aldehyde dehydrogenases, which can enhance oxidation of inter-phenylpropanoid linkages
                     in lignin in the presence of other lignin degrading enzymes [
                     <a href="#CR43">43</a>] or have roles in detoxification.
                  </p>
                  <p>Non-entomopathogenic fungi associated with insects are also hypothesized to serve
                     key roles in nutrient provisioning, including (but not limited to) sterol synthesis
                     and nitrogen concentration [
                     <a href="#CR44">44</a>]. Several transcripts derived from 
                     <i>Fusarium spp.</i> with predicted involvement in amino acid salvage and recycling of nitrogenous waste
                     products were detected. These included nitrate transporters and xanthine/uracil family
                     permeases, which can assimilate uric acid, xanthine, cysteine and other nitrogenous
                     waste products. Additionally, transcripts linked to sterol biosynthetic pathways were
                     detected and included transcripts with highest scoring BLASTX alignments to 3-keto
                     sterol reductases, sulfatases, sterol regulatory element-binding protein (SREBP) cleavage-activating
                     proteins, and di-trans-poly-cis-decaprenylcistransferases, which produce key intermediates
                     in the biosynthesis of terpenoids.
                  </p>
               </div>
               <div class="potentialnutritionprovisioningrolesofyeastsinthea.glabripennismidgut" title="sec">
                  <div class="title" tagx="title" title="title">Potential nutrition provisioning roles of yeasts in the A. glabripennis midgut</div>
                  <p>OTUs derived from yeasts are consistently detected in the midgut of 
                     <i>A. glabripennis</i> from field and laboratory reared larvae (unpublished data). The association of yeasts
                     with beetle larvae has been extensively studied in a variety of cerambycids [
                     <a href="#CR7">7</a>, 
                     <a href="#CR45">45</a>, 
                     <a href="#CR46">46</a>] and the importance of fungal enzymes to plant cell wall degradation in several cerambycid
                     species has been demonstrated [
                     <a href="#CR47">47</a>]. Previously, it was hypothesized that yeasts associated with the guts of multiple
                     species of cerambycids ferment xylose [
                     <a href="#CR38">38</a>] and the detection of transcripts derived from the xylose isomerase and various fermentation
                     pathways provide support to this hypothesis. However, additional yeast-derived transcripts
                     detected in this study suggest that these microbes could have roles in digestion and
                     nutrient biosynthesis beyond xylose fermentation. Several yeast-derived transcripts
                     annotated as multicopper oxidases, laccases, and C
                     <sub>α</sub> –dehydrogenases were detected. Although yeasts are not known to degrade lignin and
                     laccases and C
                     <sub>α</sub> dehydrogenases are often involved in detoxification processes in yeasts, similar
                     genes in other bacterial and fungal taxa can expedite oxidation of linkages in lignin,
                     enhancing the degradation of lignin in the presence of other lignin degrading enzymes
                     [
                     <a href="#CR48">48</a>, 
                     <a href="#CR49">49</a>]. To our knowledge, the ability of yeast laccases and C
                     <sub>α</sub> dehydrogenases to participate in lignin degradation has not been directly tested.
                  </p>
                  <p>Yeasts associated with the beetle midgut also have the potential to produce enzymes
                     with pivotal roles in nitrogen recycling. In addition to the glutamate dehydrogenase
                     and taurine dioxygenase transcripts, which were derived predominately from yeasts,
                     several transcripts annotated as putative uricases and ureases were detected, suggesting
                     that yeasts associated with the midgut can also decompose and recycle nitrogenous
                     waste products. Transcripts that encode essential nutrient biosynthetic pathways were
                     also associated with yeasts, such as those involved in the synthesis of methionine,
                     branched chain amino acids, aromatic amino acids, and α-ketoglutarate, suggesting
                     that yeasts could also play roles in essential amino acid biosynthesis.
                  </p>
                  <p>Several enzymes with predicted involvement in ergosterol biosynthesis were detected;
                     it has been hypothesized that insects can utilize ergosterols produced by fungal symbionts
                     for production of pheromones and cholesterol and, in some cases, biochemical evidence
                     supports the utilization of fungal ergosterols [
                     <a href="#CR50">50</a>]. Fungi associated with the midgut also produced transcripts that are predicted to
                     encode enzymes involved in the biosynthesis of several vitamins, including riboflavin,
                     thiamine, and thiazole, which are all deficient in woody tissue.
                  </p>
               </div>
            </div>
            <div class="potentialnutrientprovisioningrolesoflacticacidbacteria" title="sec">
               <div class="title" tagx="title" title="title">Potential nutrient provisioning roles of lactic acid bacteria</div>
               <p>By far, the most abundant microbial transcripts detected in the 
                  <i>A. glabripennis</i> midgut were taxonomically classified to the genus 
                  <i>Pediococcus</i> (Family Lactobacillaceae). This taxon has been previously detected in association
                  with the 
                  <i>A. glabripennis</i> midgut [
                  <a href="#CR27">27</a>] and lactic acid bacterial reads taxonomically classified to the genus 
                  <i>Leuconostoc</i> were previously detected through shotgun metagenomic and 16S rDNA amplicon analyses
                  of beetles collected from another population of 
                  <i>A. glabripennis</i>
                  [
                  <a href="#CR20">20</a>]. Although no major pathways for carbohydrate digestion were associated with this
                  taxon, pathways for the assimilation and utilization of β-1,4 linked di- and oligo-saccharides
                  (e.g. cellobiose, β-1,4-linked xylose, arabinose, galactan, and rhamnose), N-acetylglucosamines
                  (e.g., chitin and other aminosugars), and α-1,3 and α-1,6 linked mannose oligomers
                  were highly expressed. This suggests that these bacteria metabolize and utilize the
                  products of larger scale degradative processes catalyzed by either the beetle or other
                  members of the gut community. The overabundance of β-glucosidases and cellobiose phosphotransferase
                  systems associated with this genus and with the gut community in general can partially
                  explain the enhanced cellulase complex activity in the presence of diverse gut microbial
                  communities previously observed in the 
                  <i>A. glabripennis</i> midgut [
                  <a href="#CR27">27</a>]. Heightened β-glucosidase activities could reduce the impact of end product inhibition
                  of cellulases. Aldose epimerases were also detected in association with this taxon,
                  which can catalyze the inter-conversion of α-D-glucose and β-D-glucose. This may explain
                  the overabundance of α-glucoside transporters in the 
                  <i>A. glabripennis</i> midgut transcriptome [
                  <a href="#CR31">31</a>] relative to the microbial community since the beetle may acquire some of its carbon
                  sources from α-D-glucose synthesized by various members of the midgut community. Aldo-keto
                  reductases (AKR) and laccases associated with the genus 
                  <i>Pediococcus</i> were also highly abundant. AKRs were highly abundant in both the 
                  <i>A. glabripennis</i> larval midgut transcriptome [
                  <a href="#CR31">31</a>] and larval midgut metagenome [
                  <a href="#CR20">20</a>] where they were hypothesized to serve key roles in lignin degradation because they
                  can enhance the cleavage of β-aryl ether linkages in the presence of other lignin
                  degrading enzymes. Aside from lignin degradation, aldo-keto reductases can be integrally
                  involved in detoxification, metabolism of monosaccharides, and a variety of other
                  diverse oxidoreductive processes; thus, the roles of these transcripts in the midgut
                  could be diverse. Finally, many transcripts were detected with predicted roles in
                  the biosynthesis of vitamins, including folate, coenzyme A, and thiamine in the biosynthesis
                  of branched chain amino acids, lysine, asparagine, arginine, aspartate, and aromatic
                  amino acids.
               </p>
               <p>Previous analysis of metagenomic DNA predicted to originate from bacteria assigned
                  to the genus 
                  <i>Leuconostoc</i> detected in 
                  <i>A. glabripennis</i> midguts collected from an established population in Worcester, MA indicated that
                  these lactic acid bacteria have a similar metabolic potential as the 
                  <i>Pediococcus</i>-derived transcripts detected in the current study [
                  <a href="#CR20">20</a>]. Currently, the relationship between 
                  <i>A. glabripennis</i> and these lactic acid bacteria is uncharacterized; however, the conversion of xylose
                  sugars released by insect-derived xylanase and the synthesis of essential amino acids
                  and other nutrients that are generally deficient from woody tissue suggests that these
                  bacteria could be beneficial.
               </p>
            </div>
         </div>
         <div class="conclusions" title="conclusions">
            <div class="title" tagx="title" title="title">Conclusions</div>
            <p>The midgut microbiome of 
               <i>A. glabripennis</i> is enriched in biosynthetic pathways for the synthesis of essential amino acids,
               vitamins, and sterols. Consequently, microbes can provide 
               <i>A. glabripennis</i> with essential nutrients that the beetle cannot obtain directly from its woody diet,
               or synthesize itself. The gut microbiota also produces its own suite of transcripts
               that can enhance lignin degradation, degrade hemicellulose, and ferment xylose and
               wood sugars. An abundance of cellulases (from several glycoside hydrolase families)
               plus transcripts that allow the beetle to convert microbe-synthesized essential amino
               acids into non-essential amino acids are expressed by the beetle. The beetle and its
               gut microbiota likely collaborate to digest carbohydrates and convert released sugars
               and amino acid intermediates into essential nutrients otherwise lacking from woody
               plants. Further, the nutritional provisioning capabilities of the 
               <i>A. glabripennis</i> gut microbiome may contribute to the unusually broad host range of this invasive
               beetle species.
            </p>
            <p>Many studies have sought to characterize the microbiota associated with xylophagous
               insects to determine how they are able to thrive in woody tissue and to help develop
               targets for biological control. The challenge is that these communities are taxonomically
               diverse, temporally dynamic and composed of large numbers of facultative microbial
               taxa, fueling debate about whether such microbes enhance fitness or are transient
               commensals [
               <a href="#CR7">7</a>, 
               <a href="#CR8">8</a>, 
               <a href="#CR51">51</a>]. The presence of some of the same microbial taxa in the midguts of 
               <i>A. glabripennis</i> and exemplars of other beetle lineages that contain large numbers of tree-feeding
               species (e.g., the Buprestidae [
               <a href="#CR8">8</a>] and Scolytinae), in addition to the apparent importance of certain OTUs (e.g., 
               <i>Fusarium solani</i> and yeasts) to the nutritional ecology of 
               <i>A. glabripennis</i> and other wood-feeding beetles (e.g., (Scarabaeidae) [
               <a href="#CR52">52</a>]; (Passalidae) [
               <a href="#CR38">38</a>]; (Curculionidae) [
               <a href="#CR53">53</a>]), suggests that indepedently-derived partnerships with microbes may be common facilitators
               of taxonomic and ecological diversification in beetles, as they are in certain other
               groups of insects (e.g., aphids [
               <a href="#CR11">11</a>]). These partnerships likely facilitate access to novel food sources through enhanced
               nutritional provisioning capabilities. Interestingly, several expressed rRNAs and
               mRNAs predicted to originate from rare OTUs were also detected in this study, indicating
               that persistence and abundance are not necessarily correlated with transcriptional
               activity and that even rare microbial taxa can express genes integral to nutritional
               ecology.
            </p>
            <p>Microbes associated with xylophagous insects have been hypothesized to have primary
               involvement in the degradation of woody tissue; however, our analyses demonstrate
               that pathways for the synthesis of essential nutrients, including aromatic and branched
               chain amino acids, sterols, and vitamins, and for recycling of nitrogenous waste products,
               are relatively more abundant, suggesting that the majority of the microbial community
               may be more important for nutrient provisioning. In addition, the 
               <i>A. glabripennis</i> microbiota produce transcripts with predicted roles in pentose sugar fermentation
               and utilization, which could allow xylose to be used as a substrate for energy production
               and essential and non-essential amino acid biosynthesis.
            </p>
            <p>Although no canonical lignin degrading genes were detected, several enzymes with axillary
               roles in this process were identified, including C
               <sub>α</sub> dehydrogenases, multicopper oxidases, laccases, and aldo-keto reductases, suggesting
               that the gut microbiota can facilitate the degradation of lignin [
               <a href="#CR42">42</a>]. In contrast, 
               <i>A. glabripennis</i> produces several endogenous cellulases, xylanases, β-glucosidases, and other cell
               wall degrading enzymes, suggesting that the beetle has a primary role in digesting
               plant polysaccharides and its interactions with microbes could augment these processes.
               Therefore, disrupting microbes with predicted involvement in essential nutrient biosynthesis,
               such as lactic acid bacteria and yeasts, could have deleterious phenotypic impacts
               on 
               <i>A. glabripennis</i> and serve as a viable approach for controlling populations of 
               <i>A. glabripennis</i> and other wood-feeding beetles.
            </p>
            <p>Many of the same bacterial genera detected in the 
               <i>A. glabripennis</i> midgut are shared among all life stages of the phloephagous emerald ash borer (Buprestidae:
               
               <i>Agrilus planipennis</i>) [
               <a href="#CR8">8</a>]. 
               <i>Agrilus planipennis</i> and 
               <i>A. glabripennis</i> have different biologies (feeding primarily in phloem and xylem, respectively) and
               are very distantly related (last common ancestor ~200 MYA) [
               <a href="#CR54">54</a>], suggesting that their wood-feeding habits evolved independently. Similarly, the
               bark beetle 
               <i>Dendroctonus ponderosae,</i> which last shared a common ancestor with 
               <i>A. glabripennis</i> ~170 MYA [
               <a href="#CR55">55</a>], produces different types of cell wall degrading enzymes than 
               <i>A. glabripennis</i>, suggesting that these beetles, while members of sister superfamilies, also likely
               have independently evolved to feed in trees
               <i>.</i> No well-resolved and/or comprehensive higher-level molecular phylogenies are available
               for Cerambycidae or their closest relatives and few data are available about their
               gut microbiota, making it difficult to speculate on the timing and number of phylogenetic
               origins of xylophagy in longhorned beetles or in Chrysomeloidea as a whole. Nonetheless,
               through multiple comparisons of microbial communities associated with xylophagous
               beetles, it should be possible to characterize similarities and differences in microbial
               community composition and function and genetic/physiological adaptations associated
               with xylophagy within the species rich family Cerambycidae, and also within other
               families of Coleoptera with xylophagous members.
            </p>
            <p>While this study demonstrates that various members of the gut microbiome express genes
               that can complement the beetle’s existing repertoire of digestive and nutrient scavenging
               capabilities, more research is needed to determine which of these symbionts are required
               for survival in woody tissue. However, experiments to manipulate the composition of
               the gut community in 
               <i>A. glabripennis</i> and other phloephagous and xylophagous beetles to demonstrate the impact of disruption
               of select microbes on fitness and digestive physiology have been impeded by the difficulty
               of rearing these insects on anti-microbial free artificial diet. Also, the composition
               of the gut microbial community and the genes expressed by its members were sampled
               only at a single life stage in this study (third instar) and therefore, this analysis
               cannot account for shifts in community composition and its metabolic potential that
               may occur as the larvae feed in different tissues of the tree throughout their life
               cycle.
            </p>
            <p>Rarefaction curves computed for the 16S bacterial communities were not completely
               saturated, indicating that OTUs present in lower relative abundances in the midgut
               may not have been sampled for sequencing. Although deeper sequencing and a more comprehensive
               survey of the microbial genes expressed in the gut would allow us to identify additional
               microbial contributions to digestive physiology and more opportunities for nutrient
               exchange, it is often difficult to achieve complete saturation in complex communities
               with large numbers of rare OTUs as has been previously observed in 
               <i>A. glabripennis</i>
               [
               <a href="#CR17">17</a>, 
               <a href="#CR27">27</a>, 
               <a href="#CR31">31</a>] and other wood-feeding insects [
               <a href="#CR56">56</a>]. Both the community and the expression of various digestive and nutrient scavenging
               genes are dynamic; therefore, this survey is not exhaustive and additional bacterial
               and fungal OTUs and microbial contributions to digestive physiology will likely be
               identified with further study.
            </p>
         </div>
         <div class="methods" title="methods">
            <div class="title" tagx="title" title="title">Methods</div>
            <div class="fungalandbacterialcommunitiesofthea.glabripennismidgut" title="sec">
               <div class="title" tagx="title" title="title">Fungal and bacterial communities of the 
                  <i>A. glabripennis</i>midgut
               </div>
               <p>We characterized the bacterial and fungal communities associated with the 
                  <i>A. glabripennis</i> midgut by high-throughput sequencing of 16S bacterial rRNA and fungal ITS amplicon
                  libraries. Larval 
                  <i>A. glabripennis</i> were reared on living, potted 
                  <i>Acer saccharum</i> (sugar maple) in a USDA-approved quarantine greenhouse as described previously [
                  <a href="#CR27">27</a>, 
                  <a href="#CR31">31</a>]. 
                  <i>A. glabripennis</i> adults and larvae used in these experiments were reared in colony at The Pennsylvania
                  State University. This colony is of mixed ancestry from several invasive populations
                  of 
                  <i>A. glabripennis</i> obtained within the U.S. and has been in culture for over 10 years, but field-collected
                  insects from regulated areas in the U.S. are routinely added to the colony to maintain
                  genetic diversity. In brief, potted 
                  <i>A. saccharum</i> trees were grown at an outdoor nursery until they were 3–4 years old. Several weeks
                  before use, trees were moved into the quarantine greenhouse to allow for acclimation
                  to greenhouse conditions. Three trees were placed in a walk-in insect cage (~3 m high,
                  3 m long, and 2 m wide) and five mating pairs of 
                  <i>A. glabripennis</i> adults were placed in the cage and allowed to mate and lay eggs.
               </p>
               <p>After 8 weeks of feeding in trees, larvae were removed and surface sterilized with
                  two washes of 70% ethanol followed by a single wash in sterile distilled water to
                  remove residual ethanol. Larvae were dissected and their midguts removed for DNA extraction
                  using the PowerSoil DNA Isolation Kit (MoBio, Carlsbad, CA). DNA integrity and concentration
                  were verified using a NanoDrop spectrophotometer (Thermo-Fisher, Pittsburgh, PA) and
                  the Quant-It dsDNA assay (Life Technologies, Carlsbad, CA), which was analyzed on
                  a Qubit fluorometer (Life Technologies). DNA collected from each midgut was used to
                  construct partial 16S bacterial amplicon libraries, ranging from position 27 F to
                  907R, and full ITS amplicon libraries, ranging from ITS5 to ITS4. 454 multiplex identifiers
                  (MIDs) and 454 Titanium library adapters were directly incorporated into the primer
                  sequences as described previously [
                  <a href="#CR20">20</a>]. In brief, 100 ng of DNA were added to a PCR reaction mixture containing 1.0 μL
                  10X Buffer Mix (Roche, Branford, CT), 2 mM dNTPs (Roche), 0.5 U Taq polymerase (Roche),
                  5 μM forward primer (27 F: 5’-AGAGTTTGATCMTGGCTCAG-3’) and 5 μM reverse primer (907R:
                  5’-CCCCGTCAATTCMTTTGAGTTT-3’) [
                  <a href="#CR57">57</a>]. PCR cycling conditions were as follows: initial denaturation for 3 minutes at 94°C,
                  30 cycles of 94°C for 15 seconds, 55°C for 45 seconds, and 72°C for 1 minute, and
                  a final extension at 72°C for 8 minutes. The PCR reaction mixture and thermal cycling
                  conditions for the ITS amplicon library were identical to those used to generate the
                  bacterial library except that the 5 μM forward primer (ITS5: 5’-GGAAGTAAAAGTCGTAACAAGG-3’)
                  and the 5 μM reverse primer (ITS4: 5’- TCCTCCGCTTATTGATATGC-3’) [
                  <a href="#CR58">58</a>] were substituted and 35 PCR cycles were used. PCR products were evaluated using
                  agarose gel electrophoresis and bands corresponding to the sizes of the desired products
                  were eluted from the gel using the Agarose Gel Extraction Kit (Roche). For the 16S
                  amplicon libraries, products of approximately 900 bp in size were eluted from the
                  gel. Since the length of the ITS region varies in different fungal taxa, PCR products
                  ranging in size from 500 bp to 1800 bp were eluted from the gel and were used for
                  library preparation. Libraries were quantified using the Qubit dsDNA assay (Invitrogen),
                  samples were multiplexed, and library titers were calculated using quantitative PCR
                  against a library standard (Kapa Biosystems, Woburn, MA). Approximately 5,000 reads
                  were sequenced from each 16S bacterial library and 1,000 reads were sequenced from
                  each ITS fungal library using 454 Titanium XLR chemistry. Raw reads from each 16S
                  and ITS library were deposited in the NCBI Sequence Read Archive (SRA) under the accession
                  numbers SRX367813 and SRX369139, respectively. These reads are associated with BioProjects
                  PRJNA222386 and PRJNA222384, respectively.
               </p>
            </div>
            <div class="operationaltaxonomicunit-basedanalysisofmicrobial16sanditsampliconsfromthea.glabripennismidgut" title="sec">
               <div class="title" tagx="title" title="title">Operational taxonomic unit-based analysis of microbial 16S and ITS amplicons from
                  the 
                  <i>A. glabripennis</i>midgut
               </div>
               <p>16S amplicons were assigned to OTUs using the program Mothur (version 1.32.0). Pyrosequencing
                  flowgrams were denoised to reduce the impact of 454 homopolymer errors on OTU categorization
                  [
                  <a href="#CR59">59</a>, 
                  <a href="#CR60">60</a>] and low quality amplicons with ≥ 80% of the bases containing a quality score of
                  less than 25 were removed from the dataset. Chimeras were detected and removed with
                  the program UCHIME [
                  <a href="#CR61">61</a>], high quality reads greater than 700 bp in length were clustered into OTUs at 97%
                  similarity; rarefaction curves, richness estimates, and other indices of ecological
                  diversity were computed using Mothur [
                  <a href="#CR62">62</a>]. Bacterial OTUs were classified to higher taxa using the Ribosomal Database Project
                  (RDP) Classifier [
                  <a href="#CR63">63</a>], with an 80% confidence threshold for taxonomic classifications [
                  <a href="#CR60">60</a>]; sequences classified as mitochondrial, chloroplast, or eukaryotic in origin were
                  omitted from the analysis. Before calculating richness and diversity metrics, the
                  16S and ITS amplicon libraries were normalized by randomly subsampling (without replacment)
                  the same number of reads from each library to equalize sampling depth across communities.
               </p>
               <p>For analysis of ITS amplicons, flowgrams were denoised and low quality reads discarded
                  as described above. High quality reads ranging from 450 bp to 850 bp in length were
                  clustered into OTUs at 97% sequence similarity. However, ITS is highly prone to indel
                  events, making it difficult to accurately align across distantly related taxa [
                  <a href="#CR64">64</a>], leading to artificial inflations in richness calculations and estimates when using
                  alignment-based methods for OTU classification [
                  <a href="#CR65">65</a>]. Therefore, instead of implementing Mothur’s alignment based approach for OTU assignment,
                  amplicons were clustered into OTUs at 97% sequence similarity using the program CD-HIT-EST
                  [
                  <a href="#CR66">66</a>]. The program UCHIME [
                  <a href="#CR61">61</a>] was used to detect putative chimeras using the more highly abundant OTUs as a reference,
                  since templates for chimera detection were not available for the ITS region. Taxonomic
                  classification of ITS OTUs was conducted by first comparing representative sequences
                  from each OTU to NCBI’s non-redundant nucleotide database using BLASTN [
                  <a href="#CR67">67</a>] to identify plant or insect derived ITS amplicons. After excluding these, fungal
                  OTUs were classified to higher taxa using the UNITE database [
                  <a href="#CR68">68</a>] at a 90% confidence level and diversity and richness indices were computed using
                  Mothur. A higher confidence threshold was used for fungal classification to ensure
                  accurate taxonomic assignment despite the relative underrepresentation of fungal taxa
                  in NCBI databases. As in the 16S analysis, the ITS amplicon libraries were normalized
                  by randomly subsampling the same number of reads from each library before computing
                  and comparing diversity indices. To determine which OTUs were transcriptionally active
                  in the midgut, rRNAs and fungal ITS sequences assembled from the Illumina data were
                  also compared to representative sequences derived from each 16S or ITS OTU detected
                  through amplicon sequencing via BLASTN. Due to potential homopolymer errors in OTU
                  sequences derived from 454 amplicons, rRNA and ITS sequences from the transcriptome
                  data were considered to have a significant match if they were ≥ 95% identical at the
                  nucleotide level to an OTU detected through 454 amplicon sequencing.
               </p>
            </div>
            <div class="metatranscriptomesequencingofthea.glabripennismidgut" title="sec">
               <div class="title" tagx="title" title="title">Metatranscriptome sequencing of the 
                  <i>A. glabripennis</i>midgut
               </div>
               <p>Larvae feeding in 
                  <i>A. saccharum</i> for 8 weeks were removed from the tree and midguts collected from 10 instars. RNA
                  was sampled from two gut locations. First, total RNA was sampled from the midgut contents
                  collected from five larvae to identify genes from transcriptionally active microbes
                  associated with the food bolus. Second, total RNA was sampled from intact midguts
                  removed from four larvae to capture RNAs expressed by microbes that may be associated
                  with the gut tissue. This is of significant interest because microbes are often observed
                  in the epithelium of the 
                  <i>A. glabripennis</i> midgut and are poised to directly interact with host cells. One shortfall of the
                  previous metagenomic survey of the 
                  <i>A. glabripennis</i> midgut microbiota [
                  <a href="#CR20">20</a>] was that only microbes associated with the midgut contents were sampled and sequenced,
                  and microbes associated with the midgut epithelial cells could have been missed.
               </p>
            </div>
            <div class="shotgunsequencingofmrnafroma.glabripennismidgutcontents" title="sec">
               <div class="title" tagx="title" title="title">Shotgun sequencing of mRNA from 
                  <i>A. glabripennis</i>midgut contents
               </div>
               <p>Third instars of 
                  <i>A. glabripennis</i> were reared in potted 
                  <i>A. saccharum</i>, then collected and dissected under sterile conditions as described above, except
                  in this case the peritrophic matrix was removed to enrich the sample for microbial
                  cells associated with the food bolus. Midgut contents were flash frozen in liquid
                  nitrogen and total RNA was immediately extracted using the FastRNA Spin Kit for Soil
                  (MP Biomedicals, Solon, OH). A post-RNA extraction clean-up was performed using RNA
                  Clean &amp; Concentrator (Zymo Research, Irvine, CA) to remove residual salts and phenolics;
                  the sample was also treated with DNase I (Zymo Research, Irvine, CA). Sample integrity
                  was verified using the RNA Pico Assay (Life Technologies, Carlsbad, CA) and Nano Drop
                  (Thermo-Fisher), while the sample concentration was determined with the Quant-It RNA
                  Assay (Life Technologies). Removal of DNA was confirmed using the Quant-iT High-Sensitivity
                  DNA Assay (Life Technologies, Carlsbad, CA). Insect- and microbe-derived rRNAs were
                  depleted from the sample as described previously [
                  <a href="#CR31">31</a>].
               </p>
               <p>The total RNA recovered from the midgut contents was of relatively low quality (RNA
                  integrity score of 5, on a scale ranging from 1 to 10) and the amount of recovered
                  RNA was low due to the presence of nucleases and harsh conditions within the midgut
                  lumen. To obtain sufficient RNA for sequencing, 20 ng of enriched mRNA was amplified
                  using Ovation RNA Seq (NuGEN, San Carlos, CA) to produce 2 μg of double-stranded cDNA.
                  The library was sheared using a Covaris Focused-ultrasonicator (Covaris, Woburn, MA),
                  enriched for 175 nt fragments, and prepared using TruSeq Genomic DNA library adapters
                  (Illumina, San Diego, CA). Approximately 40 million 130 nt paired end reads (a total
                  of 10.6 Gb sequence data) were obtained using the Illumina GAIIx platform. An overlapping
                  paired end library enriched for 175 nt fragments was constructed so that a longer
                  contiguous sequence of this length could be constructed by merging a single read pair,
                  thus reducing the likelihood of cross-assembling reads derived from orthologous genes
                  from different microbial taxa [
                  <a href="#CR69">69</a>]. The raw Illumina reads were deposited in NCBI’s SRA under the accession number
                  SRX352195 and are associated with Bioproject PRJNA219402.
               </p>
            </div>
            <div class="shotgunsequencingofmrnacollectedfroma.glabripennismidguttissue" title="sec">
               <div class="title" tagx="title" title="title">Shotgun sequencing of mRNA collected from 
                  <i>A. glabripennis</i>midgut tissue
               </div>
               <p>Four individual third instars of 
                  <i>A. glabripennis</i> were dissected and their midguts removed, pooled, total RNA extracted, and ribosomal
                  RNA depleted, as described above. The quality and concentration of RNA recovered from
                  the intact midguts were high relative to the above-mentioned gut contents library,
                  so amplification prior to library construction was not necessary. Approximately 200 ng
                  of enriched RNA was used for library preparation with the TruSeq RNA Library Prep
                  Kit (Illumina), omitting the polyA enrichment step to enhance recovery of non-polyA
                  bacterial mRNA. The library was enriched for 175 nt fragments so that paired end reads
                  overlapped by ~30 nt. Approximately 130 million 101 nt paired end reads (36 Gb) were
                  generated using the Illumina HiSeq 2000 platform. Raw reads were deposited in NCBI’s
                  SRA under the accession number SRX265389 and are associated with Bioproject PRJNA193436.
               </p>
            </div>
            <div class="assemblyofthea.glabripennislarvalmidgutmetatranscriptome" title="sec">
               <div class="title" tagx="title" title="title">Assembly of the 
                  <i>A. glabripennis</i>larval midgut metatranscriptome
               </div>
               <p>Due to differences in library construction between the whole gut and gut contents
                  libraries, the two libraries were quality filtered and assembled separately. Low quality
                  reads or reads with ≥ 20% of the bases possessing quality scores less than 20 were
                  filtered from the dataset using the FastX Toolkit and residual library adapters were
                  removed using Cutadapt [
                  <a href="#CR70">70</a>]. Remaining read pairs and orphans were assembled with the Trinity 
                  <i>de novo</i> assembler [
                  <a href="#CR71">71</a>] in paired-end mode. To reduce the coverage of highly expressed genes and to improve
                  the ability to assemble transcripts and transcript isoforms originating from lowly
                  expressed genes, k-mers (k = 25) from quality filtered Illumina paired end reads were
                  reduced to ≤ 30X coverage using digital normalization [
                  <a href="#CR72">72</a>]. Normalized reads were assembled with Trinity (version r2012-10-05) [
                  <a href="#CR71">71</a>]. We used Trinity for assembly due to its ability to discriminate and assemble gene
                  isoforms and splice variants in data from Eukaryotes [
                  <a href="#CR71">71</a>]. We anticipated that Trinity’s ability to distinguish between isoforms from the
                  same gene would prevent cross-assembling reads originating from orthologous genes
                  from different microbial taxa and/or strains.
               </p>
            </div>
            <div class="annotationofmetatranscriptomegenesandisoforms" title="sec">
               <div class="title" tagx="title" title="title">Annotation of metatranscriptome genes and isoforms</div>
               <p>Transcripts assembled by Trinity from both the midgut tissue and midgut contents libraries
                  were used in downstream annotations and analyses. Since transcript isoforms could
                  be derived from orthologous genes from closely related bacterial species or strains,
                  they were not collapsed to the gene level prior to 
                  <i>in silico</i> functional annotation. SSU (small subunit) and LSU (large submit) rRNAs were detected
                  and removed with HMMer [
                  <a href="#CR73">73</a>] using profiles for prokaryotic, eukaryotic, and archeael SSU, LSU, and 5.8S/8S rRNAs
                  [
                  <a href="#CR74">74</a>]. Bacterial, fungal, insect, and tree SSU and LSU rRNAs detected in the transcriptome
                  were taxonomically classified by comparison to the Silva database, a manually curated
                  collection of high-quality, full length SSU and LSU ribosomal RNAs from all domains
                  of life [
                  <a href="#CR75">75</a>]. The remainder of the isoforms were annotated by a BLASTX comparison [
                  <a href="#CR67">67</a>] to the NCBI non-redundant protein database and were taxonomically classified using
                  MEGAN metagenomic analyzer [
                  <a href="#CR76">76</a>] to identify transcripts that were likely microbial in origin. Microbial transcripts
                  were functionally grouped into Gene Ontology terms [
                  <a href="#CR77">77</a>] and mapped onto KEGG pathways [
                  <a href="#CR78">78</a>] using the Trinotate pipeline and the KAAS server [
                  <a href="#CR79">79</a>], respectively. BLASTX results were corroborated and glycoside hydrolase (GH) family
                  assignments were computed by scanning for Pfam A domains [
                  <a href="#CR80">80</a>] using HmmSearch [
                  <a href="#CR81">81</a>]. Transcripts were also aligned to the 
                  <i>A. glabripennis</i> genome using BLASTN to ensure that they did not arise from genes that were laterally
                  transferred from microbes to 
                  <i>A. glabripennis</i>.
               </p>
            </div>
            <div class="comparisonofmicrobialandinsectannotations" title="sec">
               <div class="title" tagx="title" title="title">Comparison of microbial and insect annotations</div>
               <p>To determine how the gut microbiota can augment and complement 
                  <i>A. glabripennis’</i> endogenous digestive repertoire, annotations of microbial genes assembled from the
                  intact midgut and midgut contents libraries were compared to annotations of insect-derived
                  genes that were previously assembled and annotated from the 
                  <i>A. glabripennis</i> midgut [
                  <a href="#CR33">33</a>]. The full genome sequence of 
                  <i>A. glabripennis</i> is available pre-release and was used to confirm the absence of various host digestive
                  genes and metabolic pathways.
               </p>
            </div>
            <div class="availabilityofsupportingdata" title="sec">
               <div class="title" tagx="title" title="title">Availability of supporting data</div>
               <p>Raw reads from each 16S and ITS library were deposited in the NCBI Sequence Read Archive
                  (SRA) under the accession numbers SRX367813 and SRX369139, respectively. These reads
                  are associated with BioProjects PRJNA222386 and PRJNA222384, respectively. Barcoding
                  information is presented in Additional file 
                  <a href="#MOESM5">5</a>: Table S4. Raw Illumina reads from the whole midgut library were deposited in NCBI’s
                  SRA under the accession number SRX265389 and are associated with Bioproject PRJNA193436.
                  The raw Illumina reads for the midgut contents library were deposited in NCBI’s SRA
                  under the accession number SRX352195 and are associated with Bioproject PRJNA219402.
                  The assemblies and annotations for the 
                  <i>A. glabripennis</i> midgut transcriptome used for comparisons in this manuscript can be found in in NCBI’s
                  Transcript Shotgun Assembly database under the accession number [GALX00000000] and
                  was previously published [
                  <a href="#CR28">28</a>]. Assembled microbial transcripts from the intact midgut and midgut contents and
                  their corresponding annotations are included as Additional file 
                  <a href="#MOESM6">6</a>: Supplemental file 1 and Additional file 
                  <a href="#MOESM7">7</a>: Supplemental file 2. Highest scoring BLASTX results for the intact midgut and midgut
                  contents are presented in Additional file 
                  <a href="#MOESM8">8</a>: Supplemental file 3 and Additional file 
                  <a href="#MOESM9">9</a>: Supplemental file 4.
               </p>
            </div>
         </div>
         <div class="supplementary-material" title="supplementary-material">
            <div class="title" tagx="title" title="title">Electronic supplementary material</div>
            <div class="sec" title="sec">
               <div class="local-data" title="local-data">
                  <div class="media" title="media">
                     <a href="12864_2014_6803_MOESM1_ESM.docx">LINK</a>
                     <caption>
                        <p>Additional file 1: Table S1: Taxonomic classifications of 16S bacterial OTUs detected
                           in all A. glabripennis larval midguts sampled. (DOCX 18 KB)
                        </p>
                     </caption>
                  </div>
               </div>
               <div class="local-data" title="local-data">
                  <div class="media" title="media">
                     <a href="12864_2014_6803_MOESM2_ESM.tif">LINK</a>
                     <caption>
                        <p>Additional file 2: Figure S1: Taxonomic composition of midgut contents (top) and intact
                           midgut (bottom) metatranscriptome libraries. (TIF 1 MB)
                        </p>
                     </caption>
                  </div>
               </div>
               <div class="local-data" title="local-data">
                  <div class="media" title="media">
                     <a href="12864_2014_6803_MOESM3_ESM.docx">LINK</a>
                     <caption>
                        <p>Additional file 3: Table S2: Taxonomic classification of microbial rRNAs detected
                           in the midgut contents and intact midgut libraries. (DOCX 19 KB)
                        </p>
                     </caption>
                  </div>
               </div>
               <div class="local-data" title="local-data">
                  <div class="media" title="media">
                     <a href="12864_2014_6803_MOESM4_ESM.docx">LINK</a>
                     <caption>
                        <p>Additional file 4: Table S3: Number of unique KO terms found in KEGG pathways associated
                           with carbon metabolism, nitrogen acquisition and amino acid biosynthesis, nutrient
                           acquisition, and detoxification. (DOCX 21 KB)
                        </p>
                     </caption>
                  </div>
               </div>
               <div class="local-data" title="local-data">
                  <div class="media" title="media">
                     <a href="12864_2014_6803_MOESM5_ESM.docx">LINK</a>
                     <caption>
                        <p>Additional file 5: Table S4: 454 Barcodes used for 16S and ITS amplicon studies. (DOCX
                           11 KB)
                        </p>
                     </caption>
                  </div>
               </div>
               <div class="local-data" title="local-data">
                  <div class="media" title="media">
                     <a href="12864_2014_6803_MOESM6_ESM.zip">LINK</a>
                     <caption>
                        <p>Additional file 6:
                           
                           <b>Supplemental file 1.</b> Fasta file containing microbial transcripts from the A. glabripennis intact midgut.
                           (ZIP 823 KB)
                        </p>
                     </caption>
                  </div>
               </div>
               <div class="local-data" title="local-data">
                  <div class="media" title="media">
                     <a href="12864_2014_6803_MOESM7_ESM.zip">LINK</a>
                     <caption>
                        <p>Additional file 7:
                           
                           <b>Supplemental file 2.</b> Fasta file containing microbial transcripts assembled from the A. glabripennis intact
                           midgut. (ZIP 419 KB)
                        </p>
                     </caption>
                  </div>
               </div>
               <div class="local-data" title="local-data">
                  <div class="media" title="media">
                     <a href="12864_2014_6803_MOESM8_ESM.xlsx">LINK</a>
                     <caption>
                        <p>Additional file 8:
                           
                           <b>Supplemental file 3.</b> Highest scoring BLASTX alignments of protein coding genes assembled from the A. glabripennis
                           midgut contents. (XLSX 251 KB)
                        </p>
                     </caption>
                  </div>
               </div>
               <div class="local-data" title="local-data">
                  <div class="media" title="media">
                     <a href="12864_2014_6803_MOESM9_ESM.xlsx">LINK</a>
                     <caption>
                        <p>Additional file 9:
                           
                           <b>Supplemental file 4.</b> Highest scoring BLASTX alignments of protein coding genes assembled from the A. glabripennis
                           intact midgut. (XLSX 89 KB)
                        </p>
                     </caption>
                  </div>
               </div>
            </div>
         </div>
      </div>
      <div class="back" title="back">
         <div class="fn-group" title="fn-group">
            <div class="fn-type-" title="">
               <p>
                  <b>Competing interests</b>
               </p>
               <p>The author(s) declare that they have no competing interests.</p>
            </div>
            <div class="fn-type-" title="">
               <p>
                  <b>Authors’ contributions</b>
               </p>
               <p>Conceived and designed the experiments: EDS, SMG, JEC, MT and KH. Performed the experiments:
                  EDS and SMG. Analyzed the data: EDS and SMG. Evaluated conclusions: EDS, SMG, JEC,
                  MT, DM, and KH. Contributed reagents/materials/analysis tools: EDS, SMG, JEC, MT,
                  and KH. Wrote the paper: EDS, SMG, JEC, MT, DM, and KH. All authors read and approved
                  the final manuscript.
               </p>
            </div>
         </div>
         <div class="ack" title="ack">
            <div class="title" tagx="title" title="title">Acknowledgements</div>
            <p>Illumina GAIIx and HiSeq 2000 sequencing were performed at the University of Delaware
               Biotechnology Institute. 16S and ITS amplicon sequencing were performed at the Pennsylvania
               State University Genomics Core Facility-University Park, PA. Trinity assembly and
               BLAST and Pfam searches were performed using computing resources at the Hawaii Open
               Supercomputing Center at University of Hawaii (Jaws cluster; Maui, HI), the Research
               Computing and Cyberinfrastructure Group at The Pennsylvania State University (LionX
               clusters; University Park, PA), the Extreme Science and Engineering Discovery Environment
               (XSEDE), which is supported by National Science Foundation grant number OCI-1053575XSEDE
               under allocation TG-MCB140032 to SMG, and USDA-ARS Pacific Basin Agricultural Research
               Center (Moana cluster; Hilo, HI). We thank David Long, Katie Cassidy, and Karen Ferlez
               for assistance with insect rearing, Drs. Deb Gove and Greg Grove for assistance with
               16S and ITS amplicon library construction and sequencing, Dr. Bruce Kingham for assistance
               with Illumina library construction and sequencing, and Dr. Stephen Richards and the
               Baylor College of Medicine Human Genome Sequencing Center i5K Pilot project staff
               for access to pre-publication data from the 
               <i>A. glabripennis</i> genome. The funders had no role in study design, data collection and analysis, decision
               to publish, or preparation of the manuscript.
            </p>
            <p>
               <b>Funding</b>
            </p>
            <p>Funding for this project was provided by USDA-NRI-CRSEES grant 2008-35504-04464, USDA-NRI-CREES
               grant 2009-35302-05286, the Alphawood Foundation, Chicago, Illinois, a Seed Grant
               to Dr. Hoover from the Pennsylvania State University College of Agricultural Sciences,
               and a Microbial Genomics Fellowship from USDA-AFRI to EDS. Dr. Carlson was partially
               supported by World Class University Project R31-2009-000-20025-0 grant from the Ministry
               of Education, Science and Technology of South Korea. Opinions, findings, conclusions
               or recommendations expressed in this publication are those of the authors and do not
               necessarily reflect the views of the USDA. USDA is an equal opportunity provider and
               employer.
            </p>
         </div>
         <div class="references">References</div>
         <div tag="ref-list">
            <ul>
               <div class="title" tagx="title" title="title">References</div>
               <li tag="ref">
                  <a name="CR1"></a>
                  <span class="label" tagx="label" title="label">1.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Dillon</span>
                           <span class="given-names" tagx="given-names" title="given-names">RJ</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Dillon</span>
                           <span class="given-names" tagx="given-names" title="given-names">VM</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">The gut bacteria of insects: nonpathogenic interactions</span>
                     <span class="source" tagx="source" title="source">Annu Rev Entomol</span>
                     <span class="year" tagx="year" title="year">2004</span>
                     <span class="volume" tagx="volume" title="volume">49</span>
                     <span class="fpage" tagx="fpage" title="fpage">71</span>
                     <span class="lpage" tagx="lpage" title="lpage">92</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1146/annurev.ento.49.061802.123416">10.1146/annurev.ento.49.061802.123416</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/14651457">14651457</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR2"></a>
                  <span class="label" tagx="label" title="label">2.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Lee</span>
                           <span class="given-names" tagx="given-names" title="given-names">RE</span>
                           <span class="suffix" tagx="suffix" title="suffix">Jr</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Lee</span>
                           <span class="given-names" tagx="given-names" title="given-names">MR</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Strong-Gunderson</span>
                           <span class="given-names" tagx="given-names" title="given-names">JM</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Insect cold-hardiness and ice nucleating active microorganisms including their potential
                        use for biological control
                     </span>
                     <span class="source" tagx="source" title="source">J Insect Physiol</span>
                     <span class="year" tagx="year" title="year">1993</span>
                     <span class="volume" tagx="volume" title="volume">39</span>
                     <span class="issue" tagx="issue" title="issue">1</span>
                     <span class="fpage" tagx="fpage" title="fpage">1</span>
                     <span class="lpage" tagx="lpage" title="lpage">12</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1016/0022-1910(93)90011-F">10.1016/0022-1910(93)90011-F</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR3"></a>
                  <span class="label" tagx="label" title="label">3.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Janson</span>
                           <span class="given-names" tagx="given-names" title="given-names">EM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Stireman</span>
                           <span class="given-names" tagx="given-names" title="given-names">JO</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Singer</span>
                           <span class="given-names" tagx="given-names" title="given-names">MS</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Abbot</span>
                           <span class="given-names" tagx="given-names" title="given-names">P</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Phytophagous insect-microbe mutualisms and adaptive evolutionary diversification</span>
                     <span class="source" tagx="source" title="source">Evolution</span>
                     <span class="year" tagx="year" title="year">2008</span>
                     <span class="volume" tagx="volume" title="volume">62</span>
                     <span class="issue" tagx="issue" title="issue">5</span>
                     <span class="fpage" tagx="fpage" title="fpage">997</span>
                     <span class="lpage" tagx="lpage" title="lpage">1012</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1111/j.1558-5646.2008.00348.x">10.1111/j.1558-5646.2008.00348.x</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/18298649">18298649</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR4"></a>
                  <span class="label" tagx="label" title="label">4.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Feldhaar</span>
                           <span class="given-names" tagx="given-names" title="given-names">H</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Bacterial symbionts as mediators of ecologically important traits of insect hosts</span>
                     <span class="source" tagx="source" title="source">Ecol Entomol</span>
                     <span class="year" tagx="year" title="year">2011</span>
                     <span class="volume" tagx="volume" title="volume">36</span>
                     <span class="issue" tagx="issue" title="issue">5</span>
                     <span class="fpage" tagx="fpage" title="fpage">533</span>
                     <span class="lpage" tagx="lpage" title="lpage">543</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1111/j.1365-2311.2011.01318.x">10.1111/j.1365-2311.2011.01318.x</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR5"></a>
                  <span class="label" tagx="label" title="label">5.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Beaver</span>
                           <span class="given-names" tagx="given-names" title="given-names">R</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Wilding</span>
                           <span class="given-names" tagx="given-names" title="given-names">N</span>
                        </span>
                     </span>
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Wilding</span>
                           <span class="given-names" tagx="given-names" title="given-names">N</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Collins</span>
                           <span class="given-names" tagx="given-names" title="given-names">NM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hammond</span>
                           <span class="given-names" tagx="given-names" title="given-names">PM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Webber</span>
                           <span class="given-names" tagx="given-names" title="given-names">JF</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Insect-fungus relationships in the bark and ambrosia beetles</span>
                     <span class="source" tagx="source" title="source">Insect-fungus interactions</span>
                     <span class="year" tagx="year" title="year">1989</span>
                     <span class="fpage" tagx="fpage" title="fpage">121</span>
                     <span class="lpage" tagx="lpage" title="lpage">143</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR6"></a>
                  <span class="label" tagx="label" title="label">6.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Warnecke</span>
                           <span class="given-names" tagx="given-names" title="given-names">F</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Luginbuhl</span>
                           <span class="given-names" tagx="given-names" title="given-names">P</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Ivanova</span>
                           <span class="given-names" tagx="given-names" title="given-names">N</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Ghassemian</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Richardson</span>
                           <span class="given-names" tagx="given-names" title="given-names">TH</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Stege</span>
                           <span class="given-names" tagx="given-names" title="given-names">JT</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Cayouette</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">McHardy</span>
                           <span class="given-names" tagx="given-names" title="given-names">AC</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Djordjevic</span>
                           <span class="given-names" tagx="given-names" title="given-names">G</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Aboushadi</span>
                           <span class="given-names" tagx="given-names" title="given-names">N</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Sorek</span>
                           <span class="given-names" tagx="given-names" title="given-names">R</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Tringe</span>
                           <span class="given-names" tagx="given-names" title="given-names">SG</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Podar</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Martin</span>
                           <span class="given-names" tagx="given-names" title="given-names">HG</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kunin</span>
                           <span class="given-names" tagx="given-names" title="given-names">V</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Delevi</span>
                           <span class="given-names" tagx="given-names" title="given-names">D</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Madejska</span>
                           <span class="given-names" tagx="given-names" title="given-names">J</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kirton</span>
                           <span class="given-names" tagx="given-names" title="given-names">E</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Platt</span>
                           <span class="given-names" tagx="given-names" title="given-names">D</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Szeto</span>
                           <span class="given-names" tagx="given-names" title="given-names">E</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Salamov</span>
                           <span class="given-names" tagx="given-names" title="given-names">A</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Barry</span>
                           <span class="given-names" tagx="given-names" title="given-names">K</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Mikhailova</span>
                           <span class="given-names" tagx="given-names" title="given-names">N</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kyrpides</span>
                           <span class="given-names" tagx="given-names" title="given-names">NC</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Matson</span>
                           <span class="given-names" tagx="given-names" title="given-names">EG</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Ottsen</span>
                           <span class="given-names" tagx="given-names" title="given-names">EA</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Zhang</span>
                           <span class="given-names" tagx="given-names" title="given-names">X</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Murillo</span>
                           <span class="given-names" tagx="given-names" title="given-names">C</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Metagenomic and functional analysis of hindgut microbiota of a wood-feeding higher
                        termite
                     </span>
                     <span class="source" tagx="source" title="source">Nature</span>
                     <span class="year" tagx="year" title="year">2007</span>
                     <span class="volume" tagx="volume" title="volume">450</span>
                     <span class="issue" tagx="issue" title="issue">7169</span>
                     <span class="fpage" tagx="fpage" title="fpage">560</span>
                     <span class="lpage" tagx="lpage" title="lpage">565</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1038/nature06269">10.1038/nature06269</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/18033299">18033299</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR7"></a>
                  <span class="label" tagx="label" title="label">7.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Grunwald</span>
                           <span class="given-names" tagx="given-names" title="given-names">S</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Pilhofer</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Holl</span>
                           <span class="given-names" tagx="given-names" title="given-names">W</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Microbial associations in gut systems of wood- and bark-inhabiting longhorned beetles
                        [Coleoptera: Cerambycidae]
                     </span>
                     <span class="source" tagx="source" title="source">Syst Appl Microbiol</span>
                     <span class="year" tagx="year" title="year">2010</span>
                     <span class="volume" tagx="volume" title="volume">33</span>
                     <span class="issue" tagx="issue" title="issue">1</span>
                     <span class="fpage" tagx="fpage" title="fpage">25</span>
                     <span class="lpage" tagx="lpage" title="lpage">34</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1016/j.syapm.2009.10.002">10.1016/j.syapm.2009.10.002</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/19962263">19962263</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR8"></a>
                  <span class="label" tagx="label" title="label">8.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Vasanthakumar</span>
                           <span class="given-names" tagx="given-names" title="given-names">A</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Handelsman</span>
                           <span class="given-names" tagx="given-names" title="given-names">J</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Schloss</span>
                           <span class="given-names" tagx="given-names" title="given-names">PD</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Bauer</span>
                           <span class="given-names" tagx="given-names" title="given-names">LS</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Raffa</span>
                           <span class="given-names" tagx="given-names" title="given-names">KF</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Gut microbiota of an invasive subcortical beetle, 
                        <i>Agrilus planipennis</i> Fairmarine, across various life stages
                     </span>
                     <span class="source" tagx="source" title="source">Environ Entomol</span>
                     <span class="year" tagx="year" title="year">2008</span>
                     <span class="volume" tagx="volume" title="volume">37</span>
                     <span class="issue" tagx="issue" title="issue">5</span>
                     <span class="fpage" tagx="fpage" title="fpage">1344</span>
                     <span class="lpage" tagx="lpage" title="lpage">1353</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1093/ee/37.5.1344">10.1093/ee/37.5.1344</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/19036215">19036215</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR9"></a>
                  <span class="label" tagx="label" title="label">9.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Barr</span>
                           <span class="given-names" tagx="given-names" title="given-names">KL</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hearne</span>
                           <span class="given-names" tagx="given-names" title="given-names">LB</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Briesacher</span>
                           <span class="given-names" tagx="given-names" title="given-names">S</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Clark</span>
                           <span class="given-names" tagx="given-names" title="given-names">TL</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Davis</span>
                           <span class="given-names" tagx="given-names" title="given-names">GE</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Microbial symbionts in insects influence down-regulation of defense genes in maize</span>
                     <span class="source" tagx="source" title="source">PLoS One</span>
                     <span class="year" tagx="year" title="year">2010</span>
                     <span class="volume" tagx="volume" title="volume">5</span>
                     <span class="issue" tagx="issue" title="issue">6</span>
                     <span class="fpage" tagx="fpage" title="fpage">e11339</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1371/journal.pone.0011339">10.1371/journal.pone.0011339</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/20596533">20596533</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR10"></a>
                  <span class="label" tagx="label" title="label">10.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Chung</span>
                           <span class="given-names" tagx="given-names" title="given-names">SH</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Rosa</span>
                           <span class="given-names" tagx="given-names" title="given-names">C</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Scully</span>
                           <span class="given-names" tagx="given-names" title="given-names">ED</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Peiffer</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Tooker</span>
                           <span class="given-names" tagx="given-names" title="given-names">JF</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hoover</span>
                           <span class="given-names" tagx="given-names" title="given-names">K</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Luthe</span>
                           <span class="given-names" tagx="given-names" title="given-names">DS</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Felton</span>
                           <span class="given-names" tagx="given-names" title="given-names">GW</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Herbivore exploits orally secreted bacteria to suppress plant defenses</span>
                     <span class="source" tagx="source" title="source">Proc Natl Acad Sci</span>
                     <span class="year" tagx="year" title="year">2013</span>
                     <span class="volume" tagx="volume" title="volume">10</span>
                     <span class="issue" tagx="issue" title="issue">39</span>
                     <span class="fpage" tagx="fpage" title="fpage">15728</span>
                     <span class="lpage" tagx="lpage" title="lpage">15733</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1073/pnas.1308867110">10.1073/pnas.1308867110</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/24019469">24019469</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR11"></a>
                  <span class="label" tagx="label" title="label">11.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Douglas</span>
                           <span class="given-names" tagx="given-names" title="given-names">AE</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Nutritional interactions in insect-microbial symbioses: aphids and their symbiotic
                        bacteria Buchnera
                     </span>
                     <span class="source" tagx="source" title="source">Annu Rev Entomol</span>
                     <span class="year" tagx="year" title="year">1998</span>
                     <span class="volume" tagx="volume" title="volume">43</span>
                     <span class="fpage" tagx="fpage" title="fpage">17</span>
                     <span class="lpage" tagx="lpage" title="lpage">37</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1146/annurev.ento.43.1.17">10.1146/annurev.ento.43.1.17</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/15012383">15012383</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR12"></a>
                  <span class="label" tagx="label" title="label">12.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Ayayee</span>
                           <span class="given-names" tagx="given-names" title="given-names">P</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Rosa</span>
                           <span class="given-names" tagx="given-names" title="given-names">C</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Ferry</span>
                           <span class="given-names" tagx="given-names" title="given-names">JG</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Felton</span>
                           <span class="given-names" tagx="given-names" title="given-names">GW</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Saunders</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hoover</span>
                           <span class="given-names" tagx="given-names" title="given-names">K</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Gut microbes contribute to nitrogen provisioning in a wood-feeding cerambycid</span>
                     <span class="source" tagx="source" title="source">Environ Entomol</span>
                     <span class="year" tagx="year" title="year">2014</span>
                     <span class="volume" tagx="volume" title="volume">43</span>
                     <span class="issue" tagx="issue" title="issue">4</span>
                     <span class="fpage" tagx="fpage" title="fpage">903</span>
                     <span class="lpage" tagx="lpage" title="lpage">912</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1603/EN14045">10.1603/EN14045</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/24937261">24937261</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR13"></a>
                  <span class="label" tagx="label" title="label">13.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Brownlie</span>
                           <span class="given-names" tagx="given-names" title="given-names">JC</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Johnson</span>
                           <span class="given-names" tagx="given-names" title="given-names">KN</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Symbiont-mediated protection in insect hosts</span>
                     <span class="source" tagx="source" title="source">Trends Microbiol</span>
                     <span class="year" tagx="year" title="year">2009</span>
                     <span class="volume" tagx="volume" title="volume">17</span>
                     <span class="issue" tagx="issue" title="issue">8</span>
                     <span class="fpage" tagx="fpage" title="fpage">348</span>
                     <span class="lpage" tagx="lpage" title="lpage">354</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1016/j.tim.2009.05.005">10.1016/j.tim.2009.05.005</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/19660955">19660955</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR14"></a>
                  <span class="label" tagx="label" title="label">14.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Engel</span>
                           <span class="given-names" tagx="given-names" title="given-names">P</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Martinson</span>
                           <span class="given-names" tagx="given-names" title="given-names">VG</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Moran</span>
                           <span class="given-names" tagx="given-names" title="given-names">NA</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Functional diversity within the simple gut microbiota of the honey bee</span>
                     <span class="source" tagx="source" title="source">Proc Natl Acad Sci</span>
                     <span class="year" tagx="year" title="year">2012</span>
                     <span class="volume" tagx="volume" title="volume">109</span>
                     <span class="issue" tagx="issue" title="issue">27</span>
                     <span class="fpage" tagx="fpage" title="fpage">11002</span>
                     <span class="lpage" tagx="lpage" title="lpage">11007</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1073/pnas.1202970109">10.1073/pnas.1202970109</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/22711827">22711827</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR15"></a>
                  <span class="label" tagx="label" title="label">15.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Delalibera</span>
                           <span class="given-names" tagx="given-names" title="given-names">I</span>
                           <span class="suffix" tagx="suffix" title="suffix">Jr</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Vasanthakumar</span>
                           <span class="given-names" tagx="given-names" title="given-names">A</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Burwitz</span>
                           <span class="given-names" tagx="given-names" title="given-names">BJ</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Schloss</span>
                           <span class="given-names" tagx="given-names" title="given-names">PD</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Klepzig</span>
                           <span class="given-names" tagx="given-names" title="given-names">KD</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Handelsman</span>
                           <span class="given-names" tagx="given-names" title="given-names">J</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Raffa</span>
                           <span class="given-names" tagx="given-names" title="given-names">KF</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Composition of the bacterial community in the gut of the pine engraver, 
                        <i>Ips pini</i> (Say)(Coleoptera) colonizing red pine
                     </span>
                     <span class="source" tagx="source" title="source">Symbiosis (Rehovot)</span>
                     <span class="year" tagx="year" title="year">2007</span>
                     <span class="volume" tagx="volume" title="volume">43</span>
                     <span class="issue" tagx="issue" title="issue">2</span>
                     <span class="fpage" tagx="fpage" title="fpage">97</span>
                     <span class="lpage" tagx="lpage" title="lpage">104</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR16"></a>
                  <span class="label" tagx="label" title="label">16.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Coleman</span>
                           <span class="given-names" tagx="given-names" title="given-names">JJ</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Rounsley</span>
                           <span class="given-names" tagx="given-names" title="given-names">SD</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Rodriguez-Carres</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kuo</span>
                           <span class="given-names" tagx="given-names" title="given-names">A</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Wasmann</span>
                           <span class="given-names" tagx="given-names" title="given-names">CC</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Grimwood</span>
                           <span class="given-names" tagx="given-names" title="given-names">J</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Schmutz</span>
                           <span class="given-names" tagx="given-names" title="given-names">J</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Taga</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">White</span>
                           <span class="given-names" tagx="given-names" title="given-names">GJ</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Zhou</span>
                           <span class="given-names" tagx="given-names" title="given-names">SG</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Schwartz</span>
                           <span class="given-names" tagx="given-names" title="given-names">DC</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Freitag</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Ma</span>
                           <span class="given-names" tagx="given-names" title="given-names">L</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Danchin</span>
                           <span class="given-names" tagx="given-names" title="given-names">EGJ</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Henrissat</span>
                           <span class="given-names" tagx="given-names" title="given-names">B</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Coutinho</span>
                           <span class="given-names" tagx="given-names" title="given-names">PM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Nelson</span>
                           <span class="given-names" tagx="given-names" title="given-names">DR</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Straney</span>
                           <span class="given-names" tagx="given-names" title="given-names">D</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Napoli</span>
                           <span class="given-names" tagx="given-names" title="given-names">CA</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Barker</span>
                           <span class="given-names" tagx="given-names" title="given-names">BM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Gribskov</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Rep</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kroken</span>
                           <span class="given-names" tagx="given-names" title="given-names">S</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Molnar</span>
                           <span class="given-names" tagx="given-names" title="given-names">I</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Rensing</span>
                           <span class="given-names" tagx="given-names" title="given-names">C</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kennell</span>
                           <span class="given-names" tagx="given-names" title="given-names">JC</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Zamora</span>
                           <span class="given-names" tagx="given-names" title="given-names">J</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Farman</span>
                           <span class="given-names" tagx="given-names" title="given-names">ML</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Selker</span>
                           <span class="given-names" tagx="given-names" title="given-names">EU</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Salamov</span>
                           <span class="given-names" tagx="given-names" title="given-names">S</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">The genome of 
                        <i>Nectria haematococca</i>: contribution of supernumerary chromosomes to gene expansion
                     </span>
                     <span class="source" tagx="source" title="source">PLoS Genet</span>
                     <span class="year" tagx="year" title="year">2009</span>
                     <span class="volume" tagx="volume" title="volume">5</span>
                     <span class="issue" tagx="issue" title="issue">8</span>
                     <span class="fpage" tagx="fpage" title="fpage">e1000618</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1371/journal.pgen.1000618">10.1371/journal.pgen.1000618</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/19714214">19714214</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR17"></a>
                  <span class="label" tagx="label" title="label">17.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Schloss</span>
                           <span class="given-names" tagx="given-names" title="given-names">PD</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Delalibera</span>
                           <span class="given-names" tagx="given-names" title="given-names">I</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Handelsman</span>
                           <span class="given-names" tagx="given-names" title="given-names">J</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Raffa</span>
                           <span class="given-names" tagx="given-names" title="given-names">KF</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Bacteria associated with the guts of two wood-boring beetles: 
                        <i>Anoplophora glabripennis</i> and 
                        <i>Saperda vestita</i> (Cerambycidae)
                     </span>
                     <span class="source" tagx="source" title="source">Environ Entomol</span>
                     <span class="year" tagx="year" title="year">2006</span>
                     <span class="volume" tagx="volume" title="volume">35</span>
                     <span class="issue" tagx="issue" title="issue">3</span>
                     <span class="fpage" tagx="fpage" title="fpage">625</span>
                     <span class="lpage" tagx="lpage" title="lpage">629</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1603/0046-225X-35.3.625">10.1603/0046-225X-35.3.625</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR18"></a>
                  <span class="label" tagx="label" title="label">18.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Ehinger</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Mohr</span>
                           <span class="given-names" tagx="given-names" title="given-names">TJ</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Starcevich</span>
                           <span class="given-names" tagx="given-names" title="given-names">JB</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Sachs</span>
                           <span class="given-names" tagx="given-names" title="given-names">JL</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Porter</span>
                           <span class="given-names" tagx="given-names" title="given-names">SS</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Simms</span>
                           <span class="given-names" tagx="given-names" title="given-names">EL</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Specialization-generalization trade-off in a 
                        <i>Bradyrhizobium</i> symbiosis with wild legume hosts
                     </span>
                     <span class="source" tagx="source" title="source">BMC Ecol</span>
                     <span class="year" tagx="year" title="year">2014</span>
                     <span class="volume" tagx="volume" title="volume">14</span>
                     <span class="issue" tagx="issue" title="issue">1</span>
                     <span class="fpage" tagx="fpage" title="fpage">8</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1186/1472-6785-14-8">10.1186/1472-6785-14-8</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/24641813">24641813</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR19"></a>
                  <span class="label" tagx="label" title="label">19.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Adams</span>
                           <span class="given-names" tagx="given-names" title="given-names">AS</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Aylward</span>
                           <span class="given-names" tagx="given-names" title="given-names">FO</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Adams</span>
                           <span class="given-names" tagx="given-names" title="given-names">SM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Erbilgin</span>
                           <span class="given-names" tagx="given-names" title="given-names">N</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Aukema</span>
                           <span class="given-names" tagx="given-names" title="given-names">BH</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Currie</span>
                           <span class="given-names" tagx="given-names" title="given-names">CR</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Suen</span>
                           <span class="given-names" tagx="given-names" title="given-names">G</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Raffa</span>
                           <span class="given-names" tagx="given-names" title="given-names">KF</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Mountain pine beetles colonizing historical and naïve host trees are associated with
                        a bacterial community highly enriched in genes contributing to terpene metabolism
                     </span>
                     <span class="source" tagx="source" title="source">Appl Environ Microbiol</span>
                     <span class="year" tagx="year" title="year">2013</span>
                     <span class="volume" tagx="volume" title="volume">79</span>
                     <span class="issue" tagx="issue" title="issue">11</span>
                     <span class="fpage" tagx="fpage" title="fpage">3468</span>
                     <span class="lpage" tagx="lpage" title="lpage">3475</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1128/AEM.00068-13">10.1128/AEM.00068-13</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/23542624">23542624</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR20"></a>
                  <span class="label" tagx="label" title="label">20.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Scully</span>
                           <span class="given-names" tagx="given-names" title="given-names">ED</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Geib</span>
                           <span class="given-names" tagx="given-names" title="given-names">SM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hoover</span>
                           <span class="given-names" tagx="given-names" title="given-names">K</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Tien</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Tringe</span>
                           <span class="given-names" tagx="given-names" title="given-names">SG</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Barry</span>
                           <span class="given-names" tagx="given-names" title="given-names">KW</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">del Rio</span>
                           <span class="given-names" tagx="given-names" title="given-names">TG</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Chovatia</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Herr</span>
                           <span class="given-names" tagx="given-names" title="given-names">JR</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Carlson</span>
                           <span class="given-names" tagx="given-names" title="given-names">JE</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Metagenomic profiling reveals lignocellulose degrading system in a microbial community
                        associated with a wood-feeding beetle
                     </span>
                     <span class="source" tagx="source" title="source">PLoS One</span>
                     <span class="year" tagx="year" title="year">2013</span>
                     <span class="volume" tagx="volume" title="volume">8</span>
                     <span class="issue" tagx="issue" title="issue">9</span>
                     <span class="fpage" tagx="fpage" title="fpage">e73827</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1371/journal.pone.0073827">10.1371/journal.pone.0073827</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/24023907">24023907</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR21"></a>
                  <span class="label" tagx="label" title="label">21.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Shi</span>
                           <span class="given-names" tagx="given-names" title="given-names">W</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Xie</span>
                           <span class="given-names" tagx="given-names" title="given-names">S</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Chen</span>
                           <span class="given-names" tagx="given-names" title="given-names">X</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Sun</span>
                           <span class="given-names" tagx="given-names" title="given-names">S</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Zhou</span>
                           <span class="given-names" tagx="given-names" title="given-names">X</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Liu</span>
                           <span class="given-names" tagx="given-names" title="given-names">L</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Gao</span>
                           <span class="given-names" tagx="given-names" title="given-names">P</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kyrpides</span>
                           <span class="given-names" tagx="given-names" title="given-names">NC</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">No</span>
                           <span class="given-names" tagx="given-names" title="given-names">E-G</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Yuan</span>
                           <span class="given-names" tagx="given-names" title="given-names">JS</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Comparative genomic analysis of the endosymbionts of herbivorous insects reveals eco-environmental
                        adaptations: biotechnology applications
                     </span>
                     <span class="source" tagx="source" title="source">PLoS Genet</span>
                     <span class="year" tagx="year" title="year">2013</span>
                     <span class="volume" tagx="volume" title="volume">9</span>
                     <span class="issue" tagx="issue" title="issue">1</span>
                     <span class="fpage" tagx="fpage" title="fpage">e1003131</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1371/journal.pgen.1003131">10.1371/journal.pgen.1003131</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/23326236">23326236</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR22"></a>
                  <span class="label" tagx="label" title="label">22.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Helbling</span>
                           <span class="given-names" tagx="given-names" title="given-names">DE</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Ackermann</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Fenner</span>
                           <span class="given-names" tagx="given-names" title="given-names">K</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kohler</span>
                           <span class="given-names" tagx="given-names" title="given-names">H-PE</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Johnson</span>
                           <span class="given-names" tagx="given-names" title="given-names">DR</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">The activity level of a microbial community function can be predicted from its metatranscriptome</span>
                     <span class="source" tagx="source" title="source">ISME J</span>
                     <span class="year" tagx="year" title="year">2012</span>
                     <span class="volume" tagx="volume" title="volume">6</span>
                     <span class="issue" tagx="issue" title="issue">4</span>
                     <span class="fpage" tagx="fpage" title="fpage">902</span>
                     <span class="lpage" tagx="lpage" title="lpage">904</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1038/ismej.2011.158">10.1038/ismej.2011.158</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/22094344">22094344</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR23"></a>
                  <span class="label" tagx="label" title="label">23.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Lawrence</span>
                           <span class="given-names" tagx="given-names" title="given-names">JF</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Newton</span>
                           <span class="given-names" tagx="given-names" title="given-names">AF</span>
                           <span class="suffix" tagx="suffix" title="suffix">Jr</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Evolution and classification of beetles</span>
                     <span class="source" tagx="source" title="source">Annu Rev Ecol Syst</span>
                     <span class="year" tagx="year" title="year">1982</span>
                     <span class="volume" tagx="volume" title="volume">13</span>
                     <span class="fpage" tagx="fpage" title="fpage">261</span>
                     <span class="lpage" tagx="lpage" title="lpage">290</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1146/annurev.es.13.110182.001401">10.1146/annurev.es.13.110182.001401</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR24"></a>
                  <span class="label" tagx="label" title="label">24.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Haack</span>
                           <span class="given-names" tagx="given-names" title="given-names">RA</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Herard</span>
                           <span class="given-names" tagx="given-names" title="given-names">F</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Sun</span>
                           <span class="given-names" tagx="given-names" title="given-names">JH</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Turgeon</span>
                           <span class="given-names" tagx="given-names" title="given-names">JJ</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Managing invasive populations of Asian longhorned beetle and citrus longhorned beetle:
                        a worldwide perspective
                     </span>
                     <span class="source" tagx="source" title="source">Annu Rev Entomol</span>
                     <span class="year" tagx="year" title="year">2010</span>
                     <span class="volume" tagx="volume" title="volume">55</span>
                     <span class="fpage" tagx="fpage" title="fpage">521</span>
                     <span class="lpage" tagx="lpage" title="lpage">546</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1146/annurev-ento-112408-085427">10.1146/annurev-ento-112408-085427</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/19743916">19743916</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR25"></a>
                  <span class="label" tagx="label" title="label">25.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hu</span>
                           <span class="given-names" tagx="given-names" title="given-names">JF</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Angeli</span>
                           <span class="given-names" tagx="given-names" title="given-names">S</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Schuetz</span>
                           <span class="given-names" tagx="given-names" title="given-names">S</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Luo</span>
                           <span class="given-names" tagx="given-names" title="given-names">YQ</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hajek</span>
                           <span class="given-names" tagx="given-names" title="given-names">AE</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Ecology and management of exotic and endemic Asian longhorned beetle 
                        <i>Anoplophora glabripennis</i>
                     </span>
                     <span class="source" tagx="source" title="source">Agric Forest Entomol</span>
                     <span class="year" tagx="year" title="year">2009</span>
                     <span class="volume" tagx="volume" title="volume">11</span>
                     <span class="issue" tagx="issue" title="issue">4</span>
                     <span class="fpage" tagx="fpage" title="fpage">359</span>
                     <span class="lpage" tagx="lpage" title="lpage">375</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1111/j.1461-9563.2009.00443.x">10.1111/j.1461-9563.2009.00443.x</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR26"></a>
                  <span class="label" tagx="label" title="label">26.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">MacLeod</span>
                           <span class="given-names" tagx="given-names" title="given-names">A</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Evans</span>
                           <span class="given-names" tagx="given-names" title="given-names">HF</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Baker</span>
                           <span class="given-names" tagx="given-names" title="given-names">RHA</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">An analysis of pest risk from an Asian longhorn beetle (
                        <i>Anoplophora glabripennis)</i> to hardwood trees in the European community
                     </span>
                     <span class="source" tagx="source" title="source">Crop Prot</span>
                     <span class="year" tagx="year" title="year">2002</span>
                     <span class="volume" tagx="volume" title="volume">21</span>
                     <span class="issue" tagx="issue" title="issue">8</span>
                     <span class="fpage" tagx="fpage" title="fpage">635</span>
                     <span class="lpage" tagx="lpage" title="lpage">645</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1016/S0261-2194(02)00016-9">10.1016/S0261-2194(02)00016-9</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR27"></a>
                  <span class="label" tagx="label" title="label">27.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Geib</span>
                           <span class="given-names" tagx="given-names" title="given-names">SM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Jimenez-Gasco Mdel</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Carlson</span>
                           <span class="given-names" tagx="given-names" title="given-names">JE</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Tien</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hoover</span>
                           <span class="given-names" tagx="given-names" title="given-names">K</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Effect of host tree species on cellulase activity and bacterial community composition
                        in the gut of larval Asian longhorned beetle
                     </span>
                     <span class="source" tagx="source" title="source">Environ Entomol</span>
                     <span class="year" tagx="year" title="year">2009</span>
                     <span class="volume" tagx="volume" title="volume">38</span>
                     <span class="issue" tagx="issue" title="issue">3</span>
                     <span class="fpage" tagx="fpage" title="fpage">686</span>
                     <span class="lpage" tagx="lpage" title="lpage">699</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1603/022.038.0320">10.1603/022.038.0320</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/19508777">19508777</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR28"></a>
                  <span class="label" tagx="label" title="label">28.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Ugine</span>
                           <span class="given-names" tagx="given-names" title="given-names">TA</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Gardescu</span>
                           <span class="given-names" tagx="given-names" title="given-names">S</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Lewis</span>
                           <span class="given-names" tagx="given-names" title="given-names">PA</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hajek</span>
                           <span class="given-names" tagx="given-names" title="given-names">AE</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Efficacy of imidacloprid, trunk-injected into 
                        <i>Acer platanoides</i>, for control of adult Asian longhorned beetles (Coleoptera: Cerambycidae)
                     </span>
                     <span class="source" tagx="source" title="source">J Econ Entomol</span>
                     <span class="year" tagx="year" title="year">2012</span>
                     <span class="volume" tagx="volume" title="volume">105</span>
                     <span class="issue" tagx="issue" title="issue">6</span>
                     <span class="fpage" tagx="fpage" title="fpage">2015</span>
                     <span class="lpage" tagx="lpage" title="lpage">2028</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1603/EC12188">10.1603/EC12188</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/23356066">23356066</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR29"></a>
                  <span class="label" tagx="label" title="label">29.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Rosengaus</span>
                           <span class="given-names" tagx="given-names" title="given-names">RB</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Zecher</span>
                           <span class="given-names" tagx="given-names" title="given-names">CN</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Schultheis</span>
                           <span class="given-names" tagx="given-names" title="given-names">KF</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Brucker</span>
                           <span class="given-names" tagx="given-names" title="given-names">RM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Bordenstein</span>
                           <span class="given-names" tagx="given-names" title="given-names">SR</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Disruption of the termite gut microbiota and its prolonged consequences for fitness</span>
                     <span class="source" tagx="source" title="source">Appl Environ Microbiol</span>
                     <span class="year" tagx="year" title="year">2011</span>
                     <span class="volume" tagx="volume" title="volume">77</span>
                     <span class="issue" tagx="issue" title="issue">13</span>
                     <span class="fpage" tagx="fpage" title="fpage">4303</span>
                     <span class="lpage" tagx="lpage" title="lpage">4312</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1128/AEM.01886-10">10.1128/AEM.01886-10</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/21571887">21571887</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR30"></a>
                  <span class="label" tagx="label" title="label">30.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Pauchet</span>
                           <span class="given-names" tagx="given-names" title="given-names">Y</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kirsch</span>
                           <span class="given-names" tagx="given-names" title="given-names">R</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Giraud</span>
                           <span class="given-names" tagx="given-names" title="given-names">S</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Vogel</span>
                           <span class="given-names" tagx="given-names" title="given-names">H</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Heckel</span>
                           <span class="given-names" tagx="given-names" title="given-names">DG</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Identification and characterization of plant cell wall degrading enzymes from three
                        glycoside hydrolase families in the cerambycid beetle 
                        <i>Apriona japonica</i>
                     </span>
                     <span class="source" tagx="source" title="source">Insect Biochem Mol Biol</span>
                     <span class="year" tagx="year" title="year">2014</span>
                     <span class="volume" tagx="volume" title="volume">49</span>
                     <span class="fpage" tagx="fpage" title="fpage">1</span>
                     <span class="lpage" tagx="lpage" title="lpage">13</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1016/j.ibmb.2014.03.004">10.1016/j.ibmb.2014.03.004</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/24657889">24657889</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR31"></a>
                  <span class="label" tagx="label" title="label">31.</span>
                  <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Scully ED, Hoover K, Carlson JE, Tien M, Geib SM: 
                     <b>Midgut transcriptome profiling of</b>
                     <b>
                        <i>Anoplophora glabripennis</i>
                     </b>
                     <b>, a lignocellulose degrading cerambycid beetle.</b>
                     <i>BMC Genomics</i> 2013.,
                     <b>14</b>(850)
                     <b>:</b> doi:10.1186/1471–2164–14–850
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR32"></a>
                  <span class="label" tagx="label" title="label">32.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Scully</span>
                           <span class="given-names" tagx="given-names" title="given-names">ED</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hoover</span>
                           <span class="given-names" tagx="given-names" title="given-names">K</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Carlson</span>
                           <span class="given-names" tagx="given-names" title="given-names">J</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Tien</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Geib</span>
                           <span class="given-names" tagx="given-names" title="given-names">SM</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Proteomic analysis of 
                        <i>Fusarium solani</i> isolated from the Asian longhorned beetle, 
                        <i>Anoplophora glabripennis</i>
                     </span>
                     <span class="source" tagx="source" title="source">PLoS One</span>
                     <span class="year" tagx="year" title="year">2012</span>
                     <span class="volume" tagx="volume" title="volume">7</span>
                     <span class="issue" tagx="issue" title="issue">4</span>
                     <span class="fpage" tagx="fpage" title="fpage">e32990</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1371/journal.pone.0032990">10.1371/journal.pone.0032990</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/22496740">22496740</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR33"></a>
                  <span class="label" tagx="label" title="label">33.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Geib</span>
                           <span class="given-names" tagx="given-names" title="given-names">SM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Scully</span>
                           <span class="given-names" tagx="given-names" title="given-names">ED</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Jimenez-Gasco</span>
                           <span class="given-names" tagx="given-names" title="given-names">MM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Carlson</span>
                           <span class="given-names" tagx="given-names" title="given-names">JE</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Tien</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hoover</span>
                           <span class="given-names" tagx="given-names" title="given-names">K</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Phylogenetic analysis of 
                        <i>Fusarium solani</i> associated with the Asian longhorned beetle, 
                        <i>Anoplophora glabripennis</i>
                     </span>
                     <span class="source" tagx="source" title="source">Insects</span>
                     <span class="year" tagx="year" title="year">2012</span>
                     <span class="volume" tagx="volume" title="volume">3</span>
                     <span class="issue" tagx="issue" title="issue">1</span>
                     <span class="fpage" tagx="fpage" title="fpage">141</span>
                     <span class="lpage" tagx="lpage" title="lpage">160</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.3390/insects3010141">10.3390/insects3010141</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR34"></a>
                  <span class="label" tagx="label" title="label">34.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Geib</span>
                           <span class="given-names" tagx="given-names" title="given-names">SM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Jimenez-Gasco</span>
                           <span class="given-names" tagx="given-names" title="given-names">MM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Carlson</span>
                           <span class="given-names" tagx="given-names" title="given-names">JE</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Tien</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Jabbour</span>
                           <span class="given-names" tagx="given-names" title="given-names">R</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hoover</span>
                           <span class="given-names" tagx="given-names" title="given-names">K</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Microbial community profiling to investigate transmission of bacteria between life
                        stages of the wood-boring beetle, 
                        <i>Anoplophora glabripennis</i>
                     </span>
                     <span class="source" tagx="source" title="source">Microb Ecol</span>
                     <span class="year" tagx="year" title="year">2009</span>
                     <span class="volume" tagx="volume" title="volume">58</span>
                     <span class="issue" tagx="issue" title="issue">1</span>
                     <span class="fpage" tagx="fpage" title="fpage">199</span>
                     <span class="lpage" tagx="lpage" title="lpage">211</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1007/s00248-009-9501-4">10.1007/s00248-009-9501-4</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/19277770">19277770</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR35"></a>
                  <span class="label" tagx="label" title="label">35.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Carter</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Smith</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Harrison</span>
                           <span class="given-names" tagx="given-names" title="given-names">R</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Genetic analyses of the Asian longhorned beetle (Coleoptera, Cerambycidae, 
                        <i>Anoplophora glabripennis</i>), in North America, Europe and Asia
                     </span>
                     <span class="source" tagx="source" title="source">Biol Invasions</span>
                     <span class="year" tagx="year" title="year">2010</span>
                     <span class="volume" tagx="volume" title="volume">12</span>
                     <span class="issue" tagx="issue" title="issue">5</span>
                     <span class="fpage" tagx="fpage" title="fpage">1165</span>
                     <span class="lpage" tagx="lpage" title="lpage">1182</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1007/s10530-009-9538-9">10.1007/s10530-009-9538-9</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR36"></a>
                  <span class="label" tagx="label" title="label">36.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Nardi</span>
                           <span class="given-names" tagx="given-names" title="given-names">JB</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Bee</span>
                           <span class="given-names" tagx="given-names" title="given-names">CM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Miller</span>
                           <span class="given-names" tagx="given-names" title="given-names">LA</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Nguyen</span>
                           <span class="given-names" tagx="given-names" title="given-names">NH</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Suh</span>
                           <span class="given-names" tagx="given-names" title="given-names">S-O</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Blackwell</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Communities of microbes that inhabit the changing hindgut landscape of a subsocial
                        beetle
                     </span>
                     <span class="source" tagx="source" title="source">Arthropod Struct Dev</span>
                     <span class="year" tagx="year" title="year">2006</span>
                     <span class="volume" tagx="volume" title="volume">35</span>
                     <span class="issue" tagx="issue" title="issue">1</span>
                     <span class="fpage" tagx="fpage" title="fpage">57</span>
                     <span class="lpage" tagx="lpage" title="lpage">68</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1016/j.asd.2005.06.003">10.1016/j.asd.2005.06.003</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/18089058">18089058</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR37"></a>
                  <span class="label" tagx="label" title="label">37.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Geib</span>
                           <span class="given-names" tagx="given-names" title="given-names">SM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Tien</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hoover</span>
                           <span class="given-names" tagx="given-names" title="given-names">K</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Identification of proteins involved in lignocellulose degradation using in gel zymogram
                        analysis combined with mass spectroscopy-based peptide analysis of gut proteins from
                        larval Asian longhorned beetles, 
                        <i>Anoplophora glabripennis</i>
                     </span>
                     <span class="source" tagx="source" title="source">Insect Science</span>
                     <span class="year" tagx="year" title="year">2010</span>
                     <span class="volume" tagx="volume" title="volume">17</span>
                     <span class="issue" tagx="issue" title="issue">3</span>
                     <span class="fpage" tagx="fpage" title="fpage">253</span>
                     <span class="lpage" tagx="lpage" title="lpage">264</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1111/j.1744-7917.2010.01323.x">10.1111/j.1744-7917.2010.01323.x</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR38"></a>
                  <span class="label" tagx="label" title="label">38.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Suh</span>
                           <span class="given-names" tagx="given-names" title="given-names">SO</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Marshall</span>
                           <span class="given-names" tagx="given-names" title="given-names">CJ</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">McHugh</span>
                           <span class="given-names" tagx="given-names" title="given-names">JV</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Blackwell</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Wood ingestion by passalid beetles in the presence of xylose-fermenting gut yeasts</span>
                     <span class="source" tagx="source" title="source">Mol Ecol</span>
                     <span class="year" tagx="year" title="year">2003</span>
                     <span class="volume" tagx="volume" title="volume">12</span>
                     <span class="issue" tagx="issue" title="issue">11</span>
                     <span class="fpage" tagx="fpage" title="fpage">3137</span>
                     <span class="lpage" tagx="lpage" title="lpage">3145</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1046/j.1365-294X.2003.01973.x">10.1046/j.1365-294X.2003.01973.x</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/14629392">14629392</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR39"></a>
                  <span class="label" tagx="label" title="label">39.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Urbina</span>
                           <span class="given-names" tagx="given-names" title="given-names">H</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Blackwell</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Multilocus phylogenetic study of the scheffersomyces yeast clade and characterization
                        of the N-terminal region of xylose reductase gene
                     </span>
                     <span class="source" tagx="source" title="source">PLoS One</span>
                     <span class="year" tagx="year" title="year">2012</span>
                     <span class="volume" tagx="volume" title="volume">7</span>
                     <span class="issue" tagx="issue" title="issue">6</span>
                     <span class="fpage" tagx="fpage" title="fpage">e39128</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1371/journal.pone.0039128">10.1371/journal.pone.0039128</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/22720049">22720049</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR40"></a>
                  <span class="label" tagx="label" title="label">40.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Wyatt</span>
                           <span class="given-names" tagx="given-names" title="given-names">GR</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">The biochemistry of insect hemolymph</span>
                     <span class="source" tagx="source" title="source">Annu Rev Entomol</span>
                     <span class="year" tagx="year" title="year">1961</span>
                     <span class="volume" tagx="volume" title="volume">6</span>
                     <span class="issue" tagx="issue" title="issue">1</span>
                     <span class="fpage" tagx="fpage" title="fpage">75</span>
                     <span class="lpage" tagx="lpage" title="lpage">102</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1146/annurev.en.06.010161.000451">10.1146/annurev.en.06.010161.000451</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR41"></a>
                  <span class="label" tagx="label" title="label">41.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kohn RaL</span>
                           <span class="given-names" tagx="given-names" title="given-names">O</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Intermolecular calcium ion binding on polyuronate-polygalacturonate and polyguluronate</span>
                     <span class="source" tagx="source" title="source">Cxollect Czech Chem Comun</span>
                     <span class="year" tagx="year" title="year">1977</span>
                     <span class="volume" tagx="volume" title="volume">42</span>
                     <span class="fpage" tagx="fpage" title="fpage">731</span>
                     <span class="lpage" tagx="lpage" title="lpage">744</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1135/cccc19770731">10.1135/cccc19770731</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR42"></a>
                  <span class="label" tagx="label" title="label">42.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Geib</span>
                           <span class="given-names" tagx="given-names" title="given-names">SM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Filley</span>
                           <span class="given-names" tagx="given-names" title="given-names">TR</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hatcher</span>
                           <span class="given-names" tagx="given-names" title="given-names">PG</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hoover</span>
                           <span class="given-names" tagx="given-names" title="given-names">K</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Carlson</span>
                           <span class="given-names" tagx="given-names" title="given-names">JE</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Jimenez-Gasco</span>
                           <span class="given-names" tagx="given-names" title="given-names">MM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Nakagawa-Izumi</span>
                           <span class="given-names" tagx="given-names" title="given-names">A</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Sleighter</span>
                           <span class="given-names" tagx="given-names" title="given-names">RL</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Tien</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Lignin degradation in wood-feeding insects</span>
                     <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span>
                     <span class="year" tagx="year" title="year">2008</span>
                     <span class="volume" tagx="volume" title="volume">105</span>
                     <span class="issue" tagx="issue" title="issue">35</span>
                     <span class="fpage" tagx="fpage" title="fpage">12932</span>
                     <span class="lpage" tagx="lpage" title="lpage">12937</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1073/pnas.0805257105">10.1073/pnas.0805257105</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/18725643">18725643</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR43"></a>
                  <span class="label" tagx="label" title="label">43.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Zimmermann</span>
                           <span class="given-names" tagx="given-names" title="given-names">W</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Degradation of lignin by bacteria</span>
                     <span class="source" tagx="source" title="source">J Biotechnol</span>
                     <span class="year" tagx="year" title="year">1990</span>
                     <span class="volume" tagx="volume" title="volume">13</span>
                     <span class="issue" tagx="issue" title="issue">2</span>
                     <span class="fpage" tagx="fpage" title="fpage">119</span>
                     <span class="lpage" tagx="lpage" title="lpage">130</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1016/0168-1656(90)90098-V">10.1016/0168-1656(90)90098-V</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR44"></a>
                  <span class="label" tagx="label" title="label">44.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Martin</span>
                           <span class="given-names" tagx="given-names" title="given-names">MMC</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">The evolution of insect-fungus associations: from contact to stable symbiosis</span>
                     <span class="source" tagx="source" title="source">Am Zool</span>
                     <span class="year" tagx="year" title="year">1992</span>
                     <span class="volume" tagx="volume" title="volume">32</span>
                     <span class="issue" tagx="issue" title="issue">4</span>
                     <span class="fpage" tagx="fpage" title="fpage">593</span>
                     <span class="lpage" tagx="lpage" title="lpage">605</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR45"></a>
                  <span class="label" tagx="label" title="label">45.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Jones</span>
                           <span class="given-names" tagx="given-names" title="given-names">KG</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Dowd</span>
                           <span class="given-names" tagx="given-names" title="given-names">PF</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Blackwell</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Polyphyletic origins of yeast-like endocytobionts from anobiid and cerambycid beetles</span>
                     <span class="source" tagx="source" title="source">Mycol Res</span>
                     <span class="year" tagx="year" title="year">1999</span>
                     <span class="volume" tagx="volume" title="volume">103</span>
                     <span class="fpage" tagx="fpage" title="fpage">542</span>
                     <span class="lpage" tagx="lpage" title="lpage">546</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1017/S0953756298007308">10.1017/S0953756298007308</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR46"></a>
                  <span class="label" tagx="label" title="label">46.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Suh</span>
                           <span class="given-names" tagx="given-names" title="given-names">SO</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">McHugh</span>
                           <span class="given-names" tagx="given-names" title="given-names">JV</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Pollock</span>
                           <span class="given-names" tagx="given-names" title="given-names">DD</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Blackwell</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">The beetle gut: a hyperdiverse source of novel yeasts</span>
                     <span class="source" tagx="source" title="source">Mycol Res</span>
                     <span class="year" tagx="year" title="year">2005</span>
                     <span class="volume" tagx="volume" title="volume">109</span>
                     <span class="issue" tagx="issue" title="issue">Pt 3</span>
                     <span class="fpage" tagx="fpage" title="fpage">261</span>
                     <span class="lpage" tagx="lpage" title="lpage">265</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1017/S0953756205002388">10.1017/S0953756205002388</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/15912941">15912941</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR47"></a>
                  <span class="label" tagx="label" title="label">47.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kukor</span>
                           <span class="given-names" tagx="given-names" title="given-names">JJ</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Cowan</span>
                           <span class="given-names" tagx="given-names" title="given-names">DP</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Martin</span>
                           <span class="given-names" tagx="given-names" title="given-names">MM</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">The role of ingested fungal enzymes in cellulose digestion in the larvae of cerambycid
                        beetles
                     </span>
                     <span class="source" tagx="source" title="source">Physiol Zool</span>
                     <span class="year" tagx="year" title="year">1988</span>
                     <span class="volume" tagx="volume" title="volume">61</span>
                     <span class="issue" tagx="issue" title="issue">4</span>
                     <span class="fpage" tagx="fpage" title="fpage">364</span>
                     <span class="lpage" tagx="lpage" title="lpage">371</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR48"></a>
                  <span class="label" tagx="label" title="label">48.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Johannes</span>
                           <span class="given-names" tagx="given-names" title="given-names">C</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Majcherczyk</span>
                           <span class="given-names" tagx="given-names" title="given-names">A</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Natural mediators in the oxidation of polycyclic aromatic hydrocarbons by laccase
                        mediator systems
                     </span>
                     <span class="source" tagx="source" title="source">Appl Environ Microbiol</span>
                     <span class="year" tagx="year" title="year">2000</span>
                     <span class="volume" tagx="volume" title="volume">66</span>
                     <span class="issue" tagx="issue" title="issue">2</span>
                     <span class="fpage" tagx="fpage" title="fpage">524</span>
                     <span class="lpage" tagx="lpage" title="lpage">528</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1128/AEM.66.2.524-528.2000">10.1128/AEM.66.2.524-528.2000</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/10653713">10653713</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR49"></a>
                  <span class="label" tagx="label" title="label">49.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Masai</span>
                           <span class="given-names" tagx="given-names" title="given-names">E</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kubota</span>
                           <span class="given-names" tagx="given-names" title="given-names">S</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Katayama</span>
                           <span class="given-names" tagx="given-names" title="given-names">Y</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kawai</span>
                           <span class="given-names" tagx="given-names" title="given-names">S</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Yamasaki</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Morohoshi</span>
                           <span class="given-names" tagx="given-names" title="given-names">N</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Characterization of the C alpha-dehydrogenase gene involved in the cleavage of beta-aryl
                        ether by 
                        <i>Pseudomonas paucimobilis</i>
                     </span>
                     <span class="source" tagx="source" title="source">Biosci Biotechnol Biochem</span>
                     <span class="year" tagx="year" title="year">1993</span>
                     <span class="volume" tagx="volume" title="volume">57</span>
                     <span class="issue" tagx="issue" title="issue">10</span>
                     <span class="fpage" tagx="fpage" title="fpage">1655</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1271/bbb.57.1655">10.1271/bbb.57.1655</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/7764263">7764263</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR50"></a>
                  <span class="label" tagx="label" title="label">50.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Nasir</span>
                           <span class="given-names" tagx="given-names" title="given-names">H</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Noda</span>
                           <span class="given-names" tagx="given-names" title="given-names">H</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Yeast‒like symbiotes as a sterol source in anobiid beetles (Coleoptera, Anobiidae):
                        possible metabolic pathways from fungal sterols to 7‒dehydrocholesterol
                     </span>
                     <span class="source" tagx="source" title="source">Arch Insect Biochem Physiol</span>
                     <span class="year" tagx="year" title="year">2003</span>
                     <span class="volume" tagx="volume" title="volume">52</span>
                     <span class="issue" tagx="issue" title="issue">4</span>
                     <span class="fpage" tagx="fpage" title="fpage">175</span>
                     <span class="lpage" tagx="lpage" title="lpage">182</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1002/arch.10079">10.1002/arch.10079</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/12655605">12655605</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR51"></a>
                  <span class="label" tagx="label" title="label">51.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Colman</span>
                           <span class="given-names" tagx="given-names" title="given-names">DR</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Toolson</span>
                           <span class="given-names" tagx="given-names" title="given-names">EC</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Takacs-Vesbach</span>
                           <span class="given-names" tagx="given-names" title="given-names">CD</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Do diet and taxonomy influence insect gut bacterial communities?</span>
                     <span class="source" tagx="source" title="source">Mol Ecol</span>
                     <span class="year" tagx="year" title="year">2012</span>
                     <span class="volume" tagx="volume" title="volume">21</span>
                     <span class="issue" tagx="issue" title="issue">20</span>
                     <span class="fpage" tagx="fpage" title="fpage">5124</span>
                     <span class="lpage" tagx="lpage" title="lpage">5137</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1111/j.1365-294X.2012.05752.x">10.1111/j.1365-294X.2012.05752.x</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/22978555">22978555</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR52"></a>
                  <span class="label" tagx="label" title="label">52.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Vishniac</span>
                           <span class="given-names" tagx="given-names" title="given-names">H</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Johnson</span>
                           <span class="given-names" tagx="given-names" title="given-names">D</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Development of a yeast flora in the adult green June beetle (
                        <i>Cotinis nitida</i>, Scarabaeidae)
                     </span>
                     <span class="source" tagx="source" title="source">Mycologia</span>
                     <span class="year" tagx="year" title="year">1990</span>
                     <span class="volume" tagx="volume" title="volume">82</span>
                     <span class="issue" tagx="issue" title="issue">4</span>
                     <span class="fpage" tagx="fpage" title="fpage">471</span>
                     <span class="lpage" tagx="lpage" title="lpage">479</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.2307/3760018">10.2307/3760018</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR53"></a>
                  <span class="label" tagx="label" title="label">53.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Keeling</span>
                           <span class="given-names" tagx="given-names" title="given-names">CI</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Yuen</span>
                           <span class="given-names" tagx="given-names" title="given-names">MM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Liao</span>
                           <span class="given-names" tagx="given-names" title="given-names">NY</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Docking</span>
                           <span class="given-names" tagx="given-names" title="given-names">TR</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Chan</span>
                           <span class="given-names" tagx="given-names" title="given-names">SK</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Taylor</span>
                           <span class="given-names" tagx="given-names" title="given-names">GA</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Palmquist</span>
                           <span class="given-names" tagx="given-names" title="given-names">DL</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Jackman</span>
                           <span class="given-names" tagx="given-names" title="given-names">SD</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Nguyen</span>
                           <span class="given-names" tagx="given-names" title="given-names">A</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Li</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Draft genome of the mountain pine beetle, 
                        <i>Dendroctonus ponderosae</i> Hopkins, a major forest pest
                     </span>
                     <span class="source" tagx="source" title="source">Genome Biol</span>
                     <span class="year" tagx="year" title="year">2013</span>
                     <span class="volume" tagx="volume" title="volume">14</span>
                     <span class="issue" tagx="issue" title="issue">3</span>
                     <span class="fpage" tagx="fpage" title="fpage">R27</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1186/gb-2013-14-3-r27">10.1186/gb-2013-14-3-r27</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/23537049">23537049</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR54"></a>
                  <span class="label" tagx="label" title="label">54.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">McKenna</span>
                           <span class="given-names" tagx="given-names" title="given-names">DD</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Farrell</span>
                           <span class="given-names" tagx="given-names" title="given-names">BD</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Beetles (Coleoptera)</span>
                     <span class="source" tagx="source" title="source">The Timetree of Life</span>
                     <span class="year" tagx="year" title="year">2009</span>
                     <span class="fpage" tagx="fpage" title="fpage">278</span>
                     <span class="lpage" tagx="lpage" title="lpage">289</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR55"></a>
                  <span class="label" tagx="label" title="label">55.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">McKenna</span>
                           <span class="given-names" tagx="given-names" title="given-names">DD</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Sequeira</span>
                           <span class="given-names" tagx="given-names" title="given-names">AS</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Marvaldi</span>
                           <span class="given-names" tagx="given-names" title="given-names">AE</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Farrell</span>
                           <span class="given-names" tagx="given-names" title="given-names">BD</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Temporal lags and overlap in the diversification of weevils and flowering plants</span>
                     <span class="source" tagx="source" title="source">Proc Natl Acad Sci</span>
                     <span class="year" tagx="year" title="year">2009</span>
                     <span class="volume" tagx="volume" title="volume">106</span>
                     <span class="issue" tagx="issue" title="issue">17</span>
                     <span class="fpage" tagx="fpage" title="fpage">7083</span>
                     <span class="lpage" tagx="lpage" title="lpage">7088</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1073/pnas.0810618106">10.1073/pnas.0810618106</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/19365072">19365072</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR56"></a>
                  <span class="label" tagx="label" title="label">56.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Reid</span>
                           <span class="given-names" tagx="given-names" title="given-names">NM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Addison</span>
                           <span class="given-names" tagx="given-names" title="given-names">SL</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Macdonald</span>
                           <span class="given-names" tagx="given-names" title="given-names">LJ</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Lloyd-Jones</span>
                           <span class="given-names" tagx="given-names" title="given-names">G</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Biodiversity of active and inactive bacteria in the gut flora of wood-feeding huhu
                        beetle larvae (
                        <i>Prionoplus reticularis</i>)
                     </span>
                     <span class="source" tagx="source" title="source">Appl Environ Microbiol</span>
                     <span class="year" tagx="year" title="year">2011</span>
                     <span class="volume" tagx="volume" title="volume">77</span>
                     <span class="issue" tagx="issue" title="issue">19</span>
                     <span class="fpage" tagx="fpage" title="fpage">7000</span>
                     <span class="lpage" tagx="lpage" title="lpage">7006</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1128/AEM.05609-11">10.1128/AEM.05609-11</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/21841025">21841025</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR57"></a>
                  <span class="label" tagx="label" title="label">57.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Weisburg</span>
                           <span class="given-names" tagx="given-names" title="given-names">WG</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Barns</span>
                           <span class="given-names" tagx="given-names" title="given-names">SM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Pelletier</span>
                           <span class="given-names" tagx="given-names" title="given-names">DA</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Lane</span>
                           <span class="given-names" tagx="given-names" title="given-names">DJ</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">16S ribosomal DNA amplification for phylogenetic study</span>
                     <span class="source" tagx="source" title="source">J Bacteriol</span>
                     <span class="year" tagx="year" title="year">1991</span>
                     <span class="volume" tagx="volume" title="volume">173</span>
                     <span class="issue" tagx="issue" title="issue">2</span>
                     <span class="fpage" tagx="fpage" title="fpage">697</span>
                     <span class="lpage" tagx="lpage" title="lpage">703</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/1987160">1987160</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR58"></a>
                  <span class="label" tagx="label" title="label">58.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Lord</span>
                           <span class="given-names" tagx="given-names" title="given-names">NS</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kaplan</span>
                           <span class="given-names" tagx="given-names" title="given-names">CW</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Shank</span>
                           <span class="given-names" tagx="given-names" title="given-names">P</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kitts</span>
                           <span class="given-names" tagx="given-names" title="given-names">CL</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Elrod</span>
                           <span class="given-names" tagx="given-names" title="given-names">SL</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Assessment of fungal diversity using terminal restriction fragment (TRF) pattern analysis:
                        comparison of 18S and ITS ribosomal regions
                     </span>
                     <span class="source" tagx="source" title="source">Fems Microbiology Ecology</span>
                     <span class="year" tagx="year" title="year">2002</span>
                     <span class="volume" tagx="volume" title="volume">42</span>
                     <span class="issue" tagx="issue" title="issue">3</span>
                     <span class="fpage" tagx="fpage" title="fpage">327</span>
                     <span class="lpage" tagx="lpage" title="lpage">337</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1111/j.1574-6941.2002.tb01022.x">10.1111/j.1574-6941.2002.tb01022.x</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/19709292">19709292</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR59"></a>
                  <span class="label" tagx="label" title="label">59.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Quince</span>
                           <span class="given-names" tagx="given-names" title="given-names">C</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Lanzén</span>
                           <span class="given-names" tagx="given-names" title="given-names">A</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Curtis</span>
                           <span class="given-names" tagx="given-names" title="given-names">TP</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Davenport</span>
                           <span class="given-names" tagx="given-names" title="given-names">RJ</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hall</span>
                           <span class="given-names" tagx="given-names" title="given-names">N</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Head</span>
                           <span class="given-names" tagx="given-names" title="given-names">IM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Read</span>
                           <span class="given-names" tagx="given-names" title="given-names">LF</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Sloan</span>
                           <span class="given-names" tagx="given-names" title="given-names">WT</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Accurate determination of microbial diversity from 454 pyrosequencing data</span>
                     <span class="source" tagx="source" title="source">Nat Methods</span>
                     <span class="year" tagx="year" title="year">2009</span>
                     <span class="volume" tagx="volume" title="volume">6</span>
                     <span class="issue" tagx="issue" title="issue">9</span>
                     <span class="fpage" tagx="fpage" title="fpage">639</span>
                     <span class="lpage" tagx="lpage" title="lpage">641</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1038/nmeth.1361">10.1038/nmeth.1361</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/19668203">19668203</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR60"></a>
                  <span class="label" tagx="label" title="label">60.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Schloss</span>
                           <span class="given-names" tagx="given-names" title="given-names">PD</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Gevers</span>
                           <span class="given-names" tagx="given-names" title="given-names">D</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Westcott</span>
                           <span class="given-names" tagx="given-names" title="given-names">SL</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Reducing the effects of PCR amplification and sequencing artifacts on 16S rRNA-based
                        studies
                     </span>
                     <span class="source" tagx="source" title="source">PLoS One</span>
                     <span class="year" tagx="year" title="year">2011</span>
                     <span class="volume" tagx="volume" title="volume">6</span>
                     <span class="issue" tagx="issue" title="issue">12</span>
                     <span class="fpage" tagx="fpage" title="fpage">e27310</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1371/journal.pone.0027310">10.1371/journal.pone.0027310</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/22194782">22194782</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR61"></a>
                  <span class="label" tagx="label" title="label">61.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Edgar</span>
                           <span class="given-names" tagx="given-names" title="given-names">RC</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Haas</span>
                           <span class="given-names" tagx="given-names" title="given-names">BJ</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Clemente</span>
                           <span class="given-names" tagx="given-names" title="given-names">JC</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Quince</span>
                           <span class="given-names" tagx="given-names" title="given-names">C</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Knight</span>
                           <span class="given-names" tagx="given-names" title="given-names">R</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">UCHIME improves sensitivity and speed of chimera detection</span>
                     <span class="source" tagx="source" title="source">Bioinformatics</span>
                     <span class="year" tagx="year" title="year">2011</span>
                     <span class="volume" tagx="volume" title="volume">27</span>
                     <span class="issue" tagx="issue" title="issue">16</span>
                     <span class="fpage" tagx="fpage" title="fpage">2194</span>
                     <span class="lpage" tagx="lpage" title="lpage">2200</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1093/bioinformatics/btr381">10.1093/bioinformatics/btr381</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/21700674">21700674</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR62"></a>
                  <span class="label" tagx="label" title="label">62.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Schloss</span>
                           <span class="given-names" tagx="given-names" title="given-names">PD</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Westcott</span>
                           <span class="given-names" tagx="given-names" title="given-names">SL</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Ryabin</span>
                           <span class="given-names" tagx="given-names" title="given-names">T</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hall</span>
                           <span class="given-names" tagx="given-names" title="given-names">JR</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hartmann</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hollister</span>
                           <span class="given-names" tagx="given-names" title="given-names">EB</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Lesniewski</span>
                           <span class="given-names" tagx="given-names" title="given-names">RA</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Oakley</span>
                           <span class="given-names" tagx="given-names" title="given-names">BB</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Parks</span>
                           <span class="given-names" tagx="given-names" title="given-names">DH</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Robinson</span>
                           <span class="given-names" tagx="given-names" title="given-names">CJ</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Sahl</span>
                           <span class="given-names" tagx="given-names" title="given-names">JW</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Stres</span>
                           <span class="given-names" tagx="given-names" title="given-names">B</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Thallinger</span>
                           <span class="given-names" tagx="given-names" title="given-names">GG</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Van Horn</span>
                           <span class="given-names" tagx="given-names" title="given-names">DJ</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Weber</span>
                           <span class="given-names" tagx="given-names" title="given-names">CF</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Introducing mothur: open-source, platform-independent, community-supported software
                        for describing and comparing microbial communities
                     </span>
                     <span class="source" tagx="source" title="source">Appl Environ Microbiol</span>
                     <span class="year" tagx="year" title="year">2009</span>
                     <span class="volume" tagx="volume" title="volume">75</span>
                     <span class="issue" tagx="issue" title="issue">23</span>
                     <span class="fpage" tagx="fpage" title="fpage">7537</span>
                     <span class="lpage" tagx="lpage" title="lpage">7541</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1128/AEM.01541-09">10.1128/AEM.01541-09</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/19801464">19801464</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR63"></a>
                  <span class="label" tagx="label" title="label">63.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Wang</span>
                           <span class="given-names" tagx="given-names" title="given-names">Q</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Garrity</span>
                           <span class="given-names" tagx="given-names" title="given-names">GM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Tiedje</span>
                           <span class="given-names" tagx="given-names" title="given-names">JM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Cole</span>
                           <span class="given-names" tagx="given-names" title="given-names">JR</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Naive Bayesian classifier for rapid assignment of rRNA sequences into the new bacterial
                        taxonomy
                     </span>
                     <span class="source" tagx="source" title="source">Appl Environ Microbiol</span>
                     <span class="year" tagx="year" title="year">2007</span>
                     <span class="volume" tagx="volume" title="volume">73</span>
                     <span class="issue" tagx="issue" title="issue">16</span>
                     <span class="fpage" tagx="fpage" title="fpage">5261</span>
                     <span class="lpage" tagx="lpage" title="lpage">5267</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1128/AEM.00062-07">10.1128/AEM.00062-07</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/17586664">17586664</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR64"></a>
                  <span class="label" tagx="label" title="label">64.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Ryberg</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kristiansson</span>
                           <span class="given-names" tagx="given-names" title="given-names">E</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Sjökvist</span>
                           <span class="given-names" tagx="given-names" title="given-names">E</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Nilsson</span>
                           <span class="given-names" tagx="given-names" title="given-names">RH</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">An outlook on the fungal internal transcribed spacer sequences in GenBank and the
                        introduction of a web-based tool for the exploration of fungal diversity
                     </span>
                     <span class="source" tagx="source" title="source">New Phytologist</span>
                     <span class="year" tagx="year" title="year">2009</span>
                     <span class="volume" tagx="volume" title="volume">181</span>
                     <span class="issue" tagx="issue" title="issue">2</span>
                     <span class="fpage" tagx="fpage" title="fpage">471</span>
                     <span class="lpage" tagx="lpage" title="lpage">477</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1111/j.1469-8137.2008.02667.x">10.1111/j.1469-8137.2008.02667.x</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/19121041">19121041</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR65"></a>
                  <span class="label" tagx="label" title="label">65.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Nilsson</span>
                           <span class="given-names" tagx="given-names" title="given-names">RH</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kristiansson</span>
                           <span class="given-names" tagx="given-names" title="given-names">E</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Ryberg</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hallenberg</span>
                           <span class="given-names" tagx="given-names" title="given-names">N</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Larsson</span>
                           <span class="given-names" tagx="given-names" title="given-names">KH</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Intraspecific ITS variability in the kingdom fungi as expressed in the international
                        sequence databases and its implications for molecular species identification
                     </span>
                     <span class="source" tagx="source" title="source">Evol Bioinform Online</span>
                     <span class="year" tagx="year" title="year">2008</span>
                     <span class="volume" tagx="volume" title="volume">4</span>
                     <span class="fpage" tagx="fpage" title="fpage">193</span>
                     <span class="lpage" tagx="lpage" title="lpage">201</span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/19204817">19204817</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR66"></a>
                  <span class="label" tagx="label" title="label">66.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Li</span>
                           <span class="given-names" tagx="given-names" title="given-names">W</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Godzik</span>
                           <span class="given-names" tagx="given-names" title="given-names">A</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide
                        sequences
                     </span>
                     <span class="source" tagx="source" title="source">Bioinformatics</span>
                     <span class="year" tagx="year" title="year">2006</span>
                     <span class="volume" tagx="volume" title="volume">22</span>
                     <span class="issue" tagx="issue" title="issue">13</span>
                     <span class="fpage" tagx="fpage" title="fpage">1658</span>
                     <span class="lpage" tagx="lpage" title="lpage">1659</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1093/bioinformatics/btl158">10.1093/bioinformatics/btl158</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/16731699">16731699</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR67"></a>
                  <span class="label" tagx="label" title="label">67.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Altschul</span>
                           <span class="given-names" tagx="given-names" title="given-names">SF</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Madden</span>
                           <span class="given-names" tagx="given-names" title="given-names">TL</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Schaffer</span>
                           <span class="given-names" tagx="given-names" title="given-names">AA</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Zhang</span>
                           <span class="given-names" tagx="given-names" title="given-names">JH</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Zhang</span>
                           <span class="given-names" tagx="given-names" title="given-names">Z</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Miller</span>
                           <span class="given-names" tagx="given-names" title="given-names">W</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Lipman</span>
                           <span class="given-names" tagx="given-names" title="given-names">DJ</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</span>
                     <span class="source" tagx="source" title="source">Nucleic Acids Res</span>
                     <span class="year" tagx="year" title="year">1997</span>
                     <span class="volume" tagx="volume" title="volume">25</span>
                     <span class="issue" tagx="issue" title="issue">17</span>
                     <span class="fpage" tagx="fpage" title="fpage">3389</span>
                     <span class="lpage" tagx="lpage" title="lpage">3402</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1093/nar/25.17.3389">10.1093/nar/25.17.3389</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694">9254694</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR68"></a>
                  <span class="label" tagx="label" title="label">68.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kõljalg</span>
                           <span class="given-names" tagx="given-names" title="given-names">U</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Larsson</span>
                           <span class="given-names" tagx="given-names" title="given-names">KH</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Abarenkov</span>
                           <span class="given-names" tagx="given-names" title="given-names">K</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Nilsson</span>
                           <span class="given-names" tagx="given-names" title="given-names">RH</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Alexander</span>
                           <span class="given-names" tagx="given-names" title="given-names">IJ</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Eberhardt</span>
                           <span class="given-names" tagx="given-names" title="given-names">U</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Erland</span>
                           <span class="given-names" tagx="given-names" title="given-names">S</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Høiland</span>
                           <span class="given-names" tagx="given-names" title="given-names">K</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kjøller</span>
                           <span class="given-names" tagx="given-names" title="given-names">R</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Larsson</span>
                           <span class="given-names" tagx="given-names" title="given-names">E</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">UNITE: a database providing web‒based methods for the molecular identification of
                        ectomycorrhizal fungi
                     </span>
                     <span class="source" tagx="source" title="source">New Phytologist</span>
                     <span class="year" tagx="year" title="year">2005</span>
                     <span class="volume" tagx="volume" title="volume">166</span>
                     <span class="issue" tagx="issue" title="issue">3</span>
                     <span class="fpage" tagx="fpage" title="fpage">1063</span>
                     <span class="lpage" tagx="lpage" title="lpage">1068</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1111/j.1469-8137.2005.01376.x">10.1111/j.1469-8137.2005.01376.x</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/15869663">15869663</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR69"></a>
                  <span class="label" tagx="label" title="label">69.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Rodrigue</span>
                           <span class="given-names" tagx="given-names" title="given-names">S</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Materna</span>
                           <span class="given-names" tagx="given-names" title="given-names">AC</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Timberlake</span>
                           <span class="given-names" tagx="given-names" title="given-names">SC</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Blackburn</span>
                           <span class="given-names" tagx="given-names" title="given-names">MC</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Malmstrom</span>
                           <span class="given-names" tagx="given-names" title="given-names">RR</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Alm</span>
                           <span class="given-names" tagx="given-names" title="given-names">EJ</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Chisholm</span>
                           <span class="given-names" tagx="given-names" title="given-names">SW</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Unlocking short read sequencing for metagenomics</span>
                     <span class="source" tagx="source" title="source">PLoS One</span>
                     <span class="year" tagx="year" title="year">2010</span>
                     <span class="volume" tagx="volume" title="volume">5</span>
                     <span class="issue" tagx="issue" title="issue">7</span>
                     <span class="fpage" tagx="fpage" title="fpage">e11840</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1371/journal.pone.0011840">10.1371/journal.pone.0011840</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/20676378">20676378</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR70"></a>
                  <span class="label" tagx="label" title="label">70.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Martin</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Cutadapt removes adapter sequences from high-throughput sequencing reads</span>
                     <span class="source" tagx="source" title="source">EMBNet.journal</span>
                     <span class="year" tagx="year" title="year">2011</span>
                     <span class="volume" tagx="volume" title="volume">17</span>
                     <span class="issue" tagx="issue" title="issue">1</span>
                     <span class="fpage" tagx="fpage" title="fpage">10</span>
                     <span class="lpage" tagx="lpage" title="lpage">12</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.14806/ej.17.1.200">10.14806/ej.17.1.200</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR71"></a>
                  <span class="label" tagx="label" title="label">71.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Grabherr</span>
                           <span class="given-names" tagx="given-names" title="given-names">MG</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Haas</span>
                           <span class="given-names" tagx="given-names" title="given-names">BJ</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Yassour</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Levin</span>
                           <span class="given-names" tagx="given-names" title="given-names">JZ</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Thompson</span>
                           <span class="given-names" tagx="given-names" title="given-names">DA</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Amit</span>
                           <span class="given-names" tagx="given-names" title="given-names">I</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Adiconis</span>
                           <span class="given-names" tagx="given-names" title="given-names">X</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Fan</span>
                           <span class="given-names" tagx="given-names" title="given-names">L</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Raychowdhury</span>
                           <span class="given-names" tagx="given-names" title="given-names">R</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Zeng</span>
                           <span class="given-names" tagx="given-names" title="given-names">QD</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Chen</span>
                           <span class="given-names" tagx="given-names" title="given-names">Z</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Mauceli</span>
                           <span class="given-names" tagx="given-names" title="given-names">E</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hacohen</span>
                           <span class="given-names" tagx="given-names" title="given-names">N</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Gnirke</span>
                           <span class="given-names" tagx="given-names" title="given-names">A</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Rhind</span>
                           <span class="given-names" tagx="given-names" title="given-names">N</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">di Palma</span>
                           <span class="given-names" tagx="given-names" title="given-names">F</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Birren</span>
                           <span class="given-names" tagx="given-names" title="given-names">BW</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Nusbaum</span>
                           <span class="given-names" tagx="given-names" title="given-names">C</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Lindblad-Toh</span>
                           <span class="given-names" tagx="given-names" title="given-names">K</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Friedman</span>
                           <span class="given-names" tagx="given-names" title="given-names">N</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Regev</span>
                           <span class="given-names" tagx="given-names" title="given-names">A</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome</span>
                     <span class="source" tagx="source" title="source">Nat Biotechnol</span>
                     <span class="year" tagx="year" title="year">2011</span>
                     <span class="volume" tagx="volume" title="volume">29</span>
                     <span class="issue" tagx="issue" title="issue">7</span>
                     <span class="fpage" tagx="fpage" title="fpage">644</span>
                     <span class="lpage" tagx="lpage" title="lpage">652</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1038/nbt.1883">10.1038/nbt.1883</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/21572440">21572440</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR72"></a>
                  <span class="label" tagx="label" title="label">72.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Brown</span>
                           <span class="given-names" tagx="given-names" title="given-names">CT</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Howe</span>
                           <span class="given-names" tagx="given-names" title="given-names">A</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Zhang</span>
                           <span class="given-names" tagx="given-names" title="given-names">Q</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Pyrkosz</span>
                           <span class="given-names" tagx="given-names" title="given-names">AB</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Brom</span>
                           <span class="given-names" tagx="given-names" title="given-names">TH</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">arXiv preprint arXiv:12034802</span>
                     <span class="source" tagx="source" title="source">A reference-free algorithm for computational normalization of shotgun sequencing data</span>
                     <span class="year" tagx="year" title="year">2012</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR73"></a>
                  <span class="label" tagx="label" title="label">73.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Eddy</span>
                           <span class="given-names" tagx="given-names" title="given-names">SR</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">HMMER: Profile hidden Markov models for biological sequence analysis</span>
                     <span class="source" tagx="source" title="source">Bioinformatics</span>
                     <span class="year" tagx="year" title="year">1998</span>
                     <span class="volume" tagx="volume" title="volume">14</span>
                     <span class="fpage" tagx="fpage" title="fpage">755</span>
                     <span class="lpage" tagx="lpage" title="lpage">763</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1093/bioinformatics/14.9.755">10.1093/bioinformatics/14.9.755</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/9918945">9918945</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR74"></a>
                  <span class="label" tagx="label" title="label">74.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Huang</span>
                           <span class="given-names" tagx="given-names" title="given-names">Y</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Gilna</span>
                           <span class="given-names" tagx="given-names" title="given-names">P</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Li</span>
                           <span class="given-names" tagx="given-names" title="given-names">W</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Identification of ribosomal RNA genes in metagenomic fragments</span>
                     <span class="source" tagx="source" title="source">Bioinformatics</span>
                     <span class="year" tagx="year" title="year">2009</span>
                     <span class="volume" tagx="volume" title="volume">25</span>
                     <span class="issue" tagx="issue" title="issue">10</span>
                     <span class="fpage" tagx="fpage" title="fpage">1338</span>
                     <span class="lpage" tagx="lpage" title="lpage">1340</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1093/bioinformatics/btp161">10.1093/bioinformatics/btp161</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/19346323">19346323</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR75"></a>
                  <span class="label" tagx="label" title="label">75.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Pruesse</span>
                           <span class="given-names" tagx="given-names" title="given-names">E</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Quast</span>
                           <span class="given-names" tagx="given-names" title="given-names">C</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Knittel</span>
                           <span class="given-names" tagx="given-names" title="given-names">K</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Fuchs</span>
                           <span class="given-names" tagx="given-names" title="given-names">BM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Ludwig</span>
                           <span class="given-names" tagx="given-names" title="given-names">W</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Peplies</span>
                           <span class="given-names" tagx="given-names" title="given-names">J</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Glöckner</span>
                           <span class="given-names" tagx="given-names" title="given-names">FO</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">SILVA: a comprehensive online resource for quality checked and aligned ribosomal RNA
                        sequence data compatible with ARB
                     </span>
                     <span class="source" tagx="source" title="source">Nucleic Acids Res</span>
                     <span class="year" tagx="year" title="year">2007</span>
                     <span class="volume" tagx="volume" title="volume">35</span>
                     <span class="issue" tagx="issue" title="issue">21</span>
                     <span class="fpage" tagx="fpage" title="fpage">7188</span>
                     <span class="lpage" tagx="lpage" title="lpage">7196</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1093/nar/gkm864">10.1093/nar/gkm864</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/17947321">17947321</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR76"></a>
                  <span class="label" tagx="label" title="label">76.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Huson</span>
                           <span class="given-names" tagx="given-names" title="given-names">DH</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Auch</span>
                           <span class="given-names" tagx="given-names" title="given-names">AF</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Qi</span>
                           <span class="given-names" tagx="given-names" title="given-names">J</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Schuster</span>
                           <span class="given-names" tagx="given-names" title="given-names">SC</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">MEGAN analysis of metagenomic data</span>
                     <span class="source" tagx="source" title="source">Genome Res</span>
                     <span class="year" tagx="year" title="year">2007</span>
                     <span class="volume" tagx="volume" title="volume">17</span>
                     <span class="issue" tagx="issue" title="issue">3</span>
                     <span class="fpage" tagx="fpage" title="fpage">377</span>
                     <span class="lpage" tagx="lpage" title="lpage">386</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1101/gr.5969107">10.1101/gr.5969107</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/17255551">17255551</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR77"></a>
                  <span class="label" tagx="label" title="label">77.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Ashburner</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Ball</span>
                           <span class="given-names" tagx="given-names" title="given-names">CA</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Blake</span>
                           <span class="given-names" tagx="given-names" title="given-names">JA</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Botstein</span>
                           <span class="given-names" tagx="given-names" title="given-names">D</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Butler</span>
                           <span class="given-names" tagx="given-names" title="given-names">H</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Cherry</span>
                           <span class="given-names" tagx="given-names" title="given-names">JM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Davis</span>
                           <span class="given-names" tagx="given-names" title="given-names">AP</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Dolinski</span>
                           <span class="given-names" tagx="given-names" title="given-names">K</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Dwight</span>
                           <span class="given-names" tagx="given-names" title="given-names">SS</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Eppig</span>
                           <span class="given-names" tagx="given-names" title="given-names">JT</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Harris</span>
                           <span class="given-names" tagx="given-names" title="given-names">MA</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hill</span>
                           <span class="given-names" tagx="given-names" title="given-names">DP</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Issel-Traver</span>
                           <span class="given-names" tagx="given-names" title="given-names">L</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kasarskis</span>
                           <span class="given-names" tagx="given-names" title="given-names">A</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Lewis</span>
                           <span class="given-names" tagx="given-names" title="given-names">S</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Matese</span>
                           <span class="given-names" tagx="given-names" title="given-names">JC</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Richardson</span>
                           <span class="given-names" tagx="given-names" title="given-names">JE</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Ringwald</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Rubin</span>
                           <span class="given-names" tagx="given-names" title="given-names">GM</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Sherlock</span>
                           <span class="given-names" tagx="given-names" title="given-names">G</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Gene Ontology: tool for the unification of biology</span>
                     <span class="source" tagx="source" title="source">Nat Genet</span>
                     <span class="year" tagx="year" title="year">2000</span>
                     <span class="volume" tagx="volume" title="volume">25</span>
                     <span class="issue" tagx="issue" title="issue">1</span>
                     <span class="fpage" tagx="fpage" title="fpage">25</span>
                     <span class="lpage" tagx="lpage" title="lpage">29</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1038/75556">10.1038/75556</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/10802651">10802651</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR78"></a>
                  <span class="label" tagx="label" title="label">78.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Kanehisa</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">The KEGG Database</span>
                     <span class="source" tagx="source" title="source">‘In silico’ simulation of biological processes</span>
                     <span class="year" tagx="year" title="year">2008</span>
                     <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Manhattan, NY, USA</span>
                     <span class="publisher-name" tagx="publisher-name" title="publisher-name">John Wiley &amp; Sons, Ltd</span>
                     <span class="fpage" tagx="fpage" title="fpage">91</span>
                     <span class="lpage" tagx="lpage" title="lpage">103</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR79"></a>
                  <span class="label" tagx="label" title="label">79.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Ye</span>
                           <span class="given-names" tagx="given-names" title="given-names">Y</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Doak</span>
                           <span class="given-names" tagx="given-names" title="given-names">TG</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">A parsimony approach to biological pathway reconstruction/inference for genomes and
                        metagenomes
                     </span>
                     <span class="source" tagx="source" title="source">PLoS Comput Biol</span>
                     <span class="year" tagx="year" title="year">2009</span>
                     <span class="volume" tagx="volume" title="volume">5</span>
                     <span class="issue" tagx="issue" title="issue">8</span>
                     <span class="fpage" tagx="fpage" title="fpage">e1000465</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1371/journal.pcbi.1000465">10.1371/journal.pcbi.1000465</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/19680427">19680427</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR80"></a>
                  <span class="label" tagx="label" title="label">80.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Bateman</span>
                           <span class="given-names" tagx="given-names" title="given-names">A</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Coin</span>
                           <span class="given-names" tagx="given-names" title="given-names">L</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Durbin</span>
                           <span class="given-names" tagx="given-names" title="given-names">R</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Finn</span>
                           <span class="given-names" tagx="given-names" title="given-names">RD</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Hollich</span>
                           <span class="given-names" tagx="given-names" title="given-names">V</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Griffiths Jones</span>
                           <span class="given-names" tagx="given-names" title="given-names">S</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Khanna</span>
                           <span class="given-names" tagx="given-names" title="given-names">A</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Marshall</span>
                           <span class="given-names" tagx="given-names" title="given-names">M</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Moxon</span>
                           <span class="given-names" tagx="given-names" title="given-names">S</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Sonnhammer</span>
                           <span class="given-names" tagx="given-names" title="given-names">ELL</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Studholme</span>
                           <span class="given-names" tagx="given-names" title="given-names">DJ</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Yeats</span>
                           <span class="given-names" tagx="given-names" title="given-names">C</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Eddy</span>
                           <span class="given-names" tagx="given-names" title="given-names">SR</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">The Pfam protein families database</span>
                     <span class="source" tagx="source" title="source">Nucleic Acids Res</span>
                     <span class="year" tagx="year" title="year">2004</span>
                     <span class="volume" tagx="volume" title="volume">32</span>
                     <span class="issue" tagx="issue" title="issue">suppl 1</span>
                     <span class="fpage" tagx="fpage" title="fpage">D138</span>
                     <span class="lpage" tagx="lpage" title="lpage">D141</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1093/nar/gkh121">10.1093/nar/gkh121</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/14681378">14681378</a>
                     </span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR81"></a>
                  <span class="label" tagx="label" title="label">81.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Sun</span>
                           <span class="given-names" tagx="given-names" title="given-names">Y</span>
                        </span>
                        <span class="name" tagx="name" title="name">
                           <span class="surname" tagx="surname" title="surname">Buhler</span>
                           <span class="given-names" tagx="given-names" title="given-names">J</span>
                        </span>
                     </span>
                     <span class="mixed-article-title" title="mixed-article-title">Designing patterns for profile HMM search</span>
                     <span class="source" tagx="source" title="source">Bioinformatics</span>
                     <span class="year" tagx="year" title="year">2007</span>
                     <span class="volume" tagx="volume" title="volume">23</span>
                     <span class="issue" tagx="issue" title="issue">2</span>
                     <span class="fpage" tagx="fpage" title="fpage">e36</span>
                     <span class="lpage" tagx="lpage" title="lpage">e43</span>
                     <span class="pub-id">
                        <a href="https://dx.doi.org/10.1093/bioinformatics/btl323">10.1093/bioinformatics/btl323</a>
                     </span>
                     <span class="pub-id">
                        <a href="http://www.ncbi.nlm.nih.gov/pubmed/17237102">17237102</a>
                     </span>
                  </span>
               </li>
            </ul>
         </div>
      </div>
   </body>
</html>